<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.4.549">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Patrick Niekamp">
<meta name="author" content="Gaelen Guzman">
<meta name="author" content="Hans C. Leier">
<meta name="author" content="Ali Rashidforokhi">
<meta name="author" content="Veronica Richina">
<meta name="author" content="Fabian Pott">
<meta name="author" content="Caroline Barisch">
<meta name="author" content="Joost C.M. Holthuis">
<meta name="author" content="Fikadu G. Tafesse">

<title>TB &amp; The Sphinx - 3&nbsp; Breaking and Entering</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../">
<link href="../../parts/results/results_Mtb_Membrane_Damage_V2.html" rel="next">
<link href="../../parts/results.html" rel="prev">
<script src="../../site_libs/quarto-html/quarto.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../../site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script src="../../site_libs/quarto-contrib/glightbox/glightbox.min.js"></script>
<link href="../../site_libs/quarto-contrib/glightbox/glightbox.min.css" rel="stylesheet">
<link href="../../site_libs/quarto-contrib/glightbox/lightbox.css" rel="stylesheet">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="../../styles.css">
</head>

<body class="nav-sidebar floating nav-fixed slimcontent">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="../../index.html">
    <span class="navbar-title">TB and the Sphinx</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="https://drive.google.com/file/d/1Cdb-DDU9dsD4KLxv5WlcxFEM6ViPRuXl/view?usp=share_link" target="_blank"> <i class="bi bi-file-pdf" role="img" aria-label="Download .pdf">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://docs.google.com/document/d/1L0fe7AC-k4hhSJYj_NG2i_aBLd4QHr9T/edit?usp=share_link&amp;ouid=107014449567027741510&amp;rtpof=true&amp;sd=true" target="_blank"> <i class="bi bi-file-word" role="img" aria-label="Download .docx">
</i> 
<span class="menu-text"></span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
          <div class="quarto-navbar-tools">
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../../parts/results.html">Main Results</a></li><li class="breadcrumb-item"><a href="../../parts/results/results_Mtb_phagocytosis.html"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Breaking and Entering</span></a></li></ol></nav>
        <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Certificate of Approval</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Acknowledgements.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Acknowledgements</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Glossary.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Glossary</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../parts/introduction.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Introduction: TB and the Sphinx</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/introduction/intro_Layperson.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Introduction Part 1</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/introduction/intro_Scientific.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Introduction Part 2</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../parts/results.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Main Results</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/results/results_Mtb_phagocytosis.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Breaking and Entering</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/results/results_Mtb_Membrane_Damage_V2.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Sealing the Gaps</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/results/results_Mtb_lipidomics_v2.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Living Off the Fat of the Cell</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/results/results_trifunctionalProteomics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">An Affinity for Sphingolipids</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../parts/addenda.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Additional work</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Addenda/addenda_SLs_GeneticTools.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Genetic tools to study sphingolipids in disease</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Addenda/addenda_SLs_Fungus.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">The Squeaky Yeast Gets Greased</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Addenda/addenda_MethodsForVisualizingPhagocytosis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Rapid visualization and quantification of phagocytosis in neutrophils</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Addenda/addenda_Celegans_SMS_family.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Systematic analysis of the sphingomyelin synthase family in <em>C. elegans</em></span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../parts/discussionInterstitial.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Final thoughts &amp; future directions</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-4" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Discussion/discussion.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Discussion</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/references.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">References</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/about.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">About</span></a>
  </div>
</li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#abstract" id="toc-abstract" class="nav-link active" data-scroll-target="#abstract"><span class="header-section-number">3.1</span> Abstract</a></li>
  <li><a href="#importance" id="toc-importance" class="nav-link" data-scroll-target="#importance"><span class="header-section-number">3.2</span> Importance</a></li>
  <li><a href="#introduction" id="toc-introduction" class="nav-link" data-scroll-target="#introduction"><span class="header-section-number">3.3</span> Introduction</a></li>
  <li><a href="#results" id="toc-results" class="nav-link" data-scroll-target="#results"><span class="header-section-number">3.4</span> Results</a>
  <ul class="collapse">
  <li><a href="#manipulation-of-sphingolipid-levels-in-mammalian-phagocytes" id="toc-manipulation-of-sphingolipid-levels-in-mammalian-phagocytes" class="nav-link" data-scroll-target="#manipulation-of-sphingolipid-levels-in-mammalian-phagocytes"><span class="header-section-number">3.4.1</span> Manipulation of sphingolipid levels in mammalian phagocytes</a></li>
  <li><a href="#sphingolipid-deficient-cells-display-a-reduced-capacity-to-phagocytose-mtb" id="toc-sphingolipid-deficient-cells-display-a-reduced-capacity-to-phagocytose-mtb" class="nav-link" data-scroll-target="#sphingolipid-deficient-cells-display-a-reduced-capacity-to-phagocytose-mtb"><span class="header-section-number">3.4.2</span> Sphingolipid-deficient cells display a reduced capacity to phagocytose <em>Mtb</em></a></li>
  <li><a href="#sptlc2---cells-are-permissive-to-herpes-simplex-virus-entry-and-clathrin-mediated-endocytosis." id="toc-sptlc2---cells-are-permissive-to-herpes-simplex-virus-entry-and-clathrin-mediated-endocytosis." class="nav-link" data-scroll-target="#sptlc2---cells-are-permissive-to-herpes-simplex-virus-entry-and-clathrin-mediated-endocytosis."><span class="header-section-number">3.4.3</span> Sptlc2<sup>-/-</sup> cells are permissive to herpes simplex virus entry and clathrin-mediated endocytosis.</a></li>
  <li><a href="#sptlc2-cells-display-defects-in-filamentous-actin-dynamics-during-phagocytic-uptake." id="toc-sptlc2-cells-display-defects-in-filamentous-actin-dynamics-during-phagocytic-uptake." class="nav-link" data-scroll-target="#sptlc2-cells-display-defects-in-filamentous-actin-dynamics-during-phagocytic-uptake."><span class="header-section-number">3.4.4</span> Sptlc2<sup>−/−</sup> cells display defects in filamentous actin dynamics during phagocytic uptake.</a></li>
  <li><a href="#sptlc2-cells-fail-to-exclude-cd45-from-the-pathogen-contact-site." id="toc-sptlc2-cells-fail-to-exclude-cd45-from-the-pathogen-contact-site." class="nav-link" data-scroll-target="#sptlc2-cells-fail-to-exclude-cd45-from-the-pathogen-contact-site."><span class="header-section-number">3.4.5</span> Sptlc2<sup>−/−</sup> cells fail to exclude CD45 from the pathogen contact site.</a></li>
  <li><a href="#activation-of-rho-gtpases-rac1-and-cdc42-is-impaired-in-sptlc2-cells." id="toc-activation-of-rho-gtpases-rac1-and-cdc42-is-impaired-in-sptlc2-cells." class="nav-link" data-scroll-target="#activation-of-rho-gtpases-rac1-and-cdc42-is-impaired-in-sptlc2-cells."><span class="header-section-number">3.4.6</span> Activation of Rho GTPases Rac1 and Cdc42 is impaired in Sptlc2<sup>−/−</sup> cells.</a></li>
  <li><a href="#phosphoinositide-turnover-is-impaired-in-sptlc2-cells." id="toc-phosphoinositide-turnover-is-impaired-in-sptlc2-cells." class="nav-link" data-scroll-target="#phosphoinositide-turnover-is-impaired-in-sptlc2-cells."><span class="header-section-number">3.4.7</span> Phosphoinositide turnover is impaired in Sptlc2<sup>−/−</sup> cells.</a></li>
  <li><a href="#production-of-sphingomyelin-not-glycosphingolipids-is-critical-for-efficient-mtb-uptake." id="toc-production-of-sphingomyelin-not-glycosphingolipids-is-critical-for-efficient-mtb-uptake." class="nav-link" data-scroll-target="#production-of-sphingomyelin-not-glycosphingolipids-is-critical-for-efficient-mtb-uptake."><span class="header-section-number">3.4.8</span> Production of sphingomyelin, not glycosphingolipids, is critical for efficient <em>Mtb</em> uptake.</a></li>
  </ul></li>
  <li><a href="#discussion" id="toc-discussion" class="nav-link" data-scroll-target="#discussion"><span class="header-section-number">3.5</span> Discussion</a></li>
  <li><a href="#materials-and-methods" id="toc-materials-and-methods" class="nav-link" data-scroll-target="#materials-and-methods"><span class="header-section-number">3.6</span> Materials and Methods</a>
  <ul class="collapse">
  <li><a href="#reagents" id="toc-reagents" class="nav-link" data-scroll-target="#reagents"><span class="header-section-number">3.6.1</span> Reagents</a></li>
  <li><a href="#plasmids" id="toc-plasmids" class="nav-link" data-scroll-target="#plasmids"><span class="header-section-number">3.6.2</span> Plasmids</a></li>
  <li><a href="#cell-culture-and-small-molecule-inhibitor-treatment" id="toc-cell-culture-and-small-molecule-inhibitor-treatment" class="nav-link" data-scroll-target="#cell-culture-and-small-molecule-inhibitor-treatment"><span class="header-section-number">3.6.3</span> Cell culture and small-molecule-inhibitor treatment</a></li>
  <li><a href="#metabolic-labeling-and-thin-layer-chromatography" id="toc-metabolic-labeling-and-thin-layer-chromatography" class="nav-link" data-scroll-target="#metabolic-labeling-and-thin-layer-chromatography"><span class="header-section-number">3.6.4</span> Metabolic labeling and thin-layer chromatography</a></li>
  <li><a href="#lipodome-analysis" id="toc-lipodome-analysis" class="nav-link" data-scroll-target="#lipodome-analysis"><span class="header-section-number">3.6.5</span> Lipodome analysis</a></li>
  <li><a href="#generation-of-lentiviruses" id="toc-generation-of-lentiviruses" class="nav-link" data-scroll-target="#generation-of-lentiviruses"><span class="header-section-number">3.6.6</span> Generation of lentiviruses</a></li>
  <li><a href="#generation-of-crisprcas9-mediated-knockout-cell-lines" id="toc-generation-of-crisprcas9-mediated-knockout-cell-lines" class="nav-link" data-scroll-target="#generation-of-crisprcas9-mediated-knockout-cell-lines"><span class="header-section-number">3.6.7</span> Generation of CRISPR/Cas9-mediated knockout cell lines</a></li>
  <li><a href="#culturing-of-mtb-and-m.-marinum" id="toc-culturing-of-mtb-and-m.-marinum" class="nav-link" data-scroll-target="#culturing-of-mtb-and-m.-marinum"><span class="header-section-number">3.6.8</span> Culturing of <em>Mtb</em> and <em>M. marinum</em></a></li>
  <li><a href="#mtb-and-m.-marinum-infection" id="toc-mtb-and-m.-marinum-infection" class="nav-link" data-scroll-target="#mtb-and-m.-marinum-infection"><span class="header-section-number">3.6.9</span> <em>Mtb</em> and <em>M. marinum</em> infection</a></li>
  <li><a href="#zymosan-a-uptake-assay" id="toc-zymosan-a-uptake-assay" class="nav-link" data-scroll-target="#zymosan-a-uptake-assay"><span class="header-section-number">3.6.10</span> Zymosan A uptake assay</a></li>
  <li><a href="#phagocytosis-assay" id="toc-phagocytosis-assay" class="nav-link" data-scroll-target="#phagocytosis-assay"><span class="header-section-number">3.6.11</span> Phagocytosis assay</a></li>
  <li><a href="#herpes-simplex-virus-1-infection" id="toc-herpes-simplex-virus-1-infection" class="nav-link" data-scroll-target="#herpes-simplex-virus-1-infection"><span class="header-section-number">3.6.12</span> Herpes simplex virus 1 infection</a></li>
  <li><a href="#binding-assay-zymosan-a-particles" id="toc-binding-assay-zymosan-a-particles" class="nav-link" data-scroll-target="#binding-assay-zymosan-a-particles"><span class="header-section-number">3.6.13</span> Binding assay zymosan A particles</a></li>
  <li><a href="#real-time-imaging-of-phagocytosis" id="toc-real-time-imaging-of-phagocytosis" class="nav-link" data-scroll-target="#real-time-imaging-of-phagocytosis"><span class="header-section-number">3.6.14</span> Real time imaging of phagocytosis</a></li>
  <li><a href="#live-imaging-of-dectin-1gfp" id="toc-live-imaging-of-dectin-1gfp" class="nav-link" data-scroll-target="#live-imaging-of-dectin-1gfp"><span class="header-section-number">3.6.15</span> Live imaging of dectin-1–GFP</a></li>
  <li><a href="#live-image-analysis-quantification" id="toc-live-image-analysis-quantification" class="nav-link" data-scroll-target="#live-image-analysis-quantification"><span class="header-section-number">3.6.16</span> Live image analysis (quantification)</a></li>
  <li><a href="#dectin-1-and-cd45-displacement-assay" id="toc-dectin-1-and-cd45-displacement-assay" class="nav-link" data-scroll-target="#dectin-1-and-cd45-displacement-assay"><span class="header-section-number">3.6.17</span> Dectin-1 and CD45 displacement assay</a></li>
  <li><a href="#immunoblot-analysis" id="toc-immunoblot-analysis" class="nav-link" data-scroll-target="#immunoblot-analysis"><span class="header-section-number">3.6.18</span> Immunoblot analysis</a></li>
  <li><a href="#flow-cytometry" id="toc-flow-cytometry" class="nav-link" data-scroll-target="#flow-cytometry"><span class="header-section-number">3.6.19</span> Flow cytometry</a></li>
  <li><a href="#transferrin-uptake-analysis" id="toc-transferrin-uptake-analysis" class="nav-link" data-scroll-target="#transferrin-uptake-analysis"><span class="header-section-number">3.6.20</span> Transferrin uptake analysis</a></li>
  </ul></li>
  <li><a href="#acknowledgements" id="toc-acknowledgements" class="nav-link" data-scroll-target="#acknowledgements"><span class="header-section-number">3.7</span> Acknowledgements</a></li>
  <li><a href="#supplementary-materials" id="toc-supplementary-materials" class="nav-link" data-scroll-target="#supplementary-materials"><span class="header-section-number">3.8</span> Supplementary materials</a>
  <ul class="collapse">
  <li><a href="#supplementary-movies" id="toc-supplementary-movies" class="nav-link" data-scroll-target="#supplementary-movies"><span class="header-section-number">3.8.1</span> Supplementary Movies</a></li>
  </ul></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content page-columns page-full" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../../parts/results.html">Main Results</a></li><li class="breadcrumb-item"><a href="../../parts/results/results_Mtb_phagocytosis.html"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Breaking and Entering</span></a></li></ol></nav>
<div class="quarto-title">
<h1 class="title"><span id="sec-Mtb-Phagocytosis" class="quarto-section-identifier"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Breaking and Entering</span></span></h1>
<p class="subtitle lead">Sphingomyelin Biosynthesis Is Essential for Phagocytic Signaling during <em>Mycobacterium tuberculosis</em> Host Cell Entry</p>
</div>


<div class="quarto-title-meta-author">
  <div class="quarto-title-meta-heading">Authors</div>
  <div class="quarto-title-meta-heading">Affiliations</div>
  
    <div class="quarto-title-meta-contents">
    <p class="author">Patrick Niekamp </p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Molecular Cell Biology Division, Department of Biology and Chemistry, University of Osnabrück, Osnabrück, Germany
          </p>
      </div>
    <div class="quarto-title-meta-contents">
    <p class="author">Gaelen Guzman </p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Department of Molecular Microbiology and Immunology, Oregon Health &amp; Science University, Portland, Oregon, USA
          </p>
      </div>
    <div class="quarto-title-meta-contents">
    <p class="author">Hans C. Leier </p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Department of Molecular Microbiology and Immunology, Oregon Health &amp; Science University, Portland, Oregon, USA
          </p>
      </div>
    <div class="quarto-title-meta-contents">
    <p class="author">Ali Rashidforokhi </p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Department of Molecular Microbiology and Immunology, Oregon Health &amp; Science University, Portland, Oregon, USA
          </p>
      </div>
    <div class="quarto-title-meta-contents">
    <p class="author">Veronica Richina </p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Department of Molecular Microbiology and Immunology, Oregon Health &amp; Science University, Portland, Oregon, USA
          </p>
      </div>
    <div class="quarto-title-meta-contents">
    <p class="author">Fabian Pott </p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Department of Molecular Microbiology and Immunology, Oregon Health &amp; Science University, Portland, Oregon, USA
          </p>
      </div>
    <div class="quarto-title-meta-contents">
    <p class="author">Caroline Barisch </p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Molecular Cell Biology Division, Department of Biology and Chemistry, University of Osnabrück, Osnabrück, Germany
          </p>
      </div>
    <div class="quarto-title-meta-contents">
    <p class="author">Joost C.M. Holthuis </p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Molecular Cell Biology Division, Department of Biology and Chemistry, University of Osnabrück, Osnabrück, Germany
          </p>
      </div>
    <div class="quarto-title-meta-contents">
    <p class="author">Fikadu G. Tafesse <a href="mailto:tafesse@ohsu.edu" class="quarto-title-author-email"><i class="bi bi-envelope"></i></a> </p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Department of Molecular Microbiology and Immunology, Oregon Health &amp; Science University, Portland, Oregon, USA
          </p>
      </div>
  </div>

<div class="quarto-title-meta">

      
  
    
  </div>
  


</header>


<div class="page-columns page-full"><p></p><div class="no-row-height column-margin column-container"><span class="margin-aside"><sup>*</sup> Patrick Niekamp and Gaelen Guzman contributed equally to this study. Since Patrick Niekamp started working on the project before Gaelen Guzman, his name is listed first.</span></div></div>
<p>This chapter has been published in mBio at the following: <a href="https://doi.org/10.1128/mbio.03141-20" class="uri">https://doi.org/10.1128/mbio.03141-20</a></p>
<section id="abstract" class="level2" data-number="3.1">
<h2 data-number="3.1" class="anchored" data-anchor-id="abstract"><span class="header-section-number">3.1</span> Abstract</h2>
<p>Phagocytosis by alveolar macrophages is the obligate first step in <em>Mycobacterium tuberculosis</em> (<em>Mtb</em>) infection, yet the mechanism underlying this process is incompletely understood. Here we show that <em>Mtb</em> invasion relies on an intact sphingolipid biosynthetic pathway. Inhibition or knockout of early sphingolipid biosynthetic enzymes greatly reduces <em>Mtb</em> uptake across multiple phagocytic cell types without affecting other forms of endocytosis. While the phagocytic receptor Dectin-1 undergoes normal clustering at the pathogen contact sites, sphingolipid biosynthetic mutant cells fail to segregate the regulatory phosphatase CD45 from the clustered receptors. Blocking sphingolipid production also impairs downstream activation of Rho GTPases, actin dynamics, and phosphoinositide turnover at the nascent phagocytic cup. Moreover, we find that production of sphingomyelin, not glycosphingolipids, is essential for <em>Mtb</em> uptake. Collectively, our data support a critical role of sphingomyelin biosynthesis in an early stage of <em>Mtb</em> infection and provide novel insights into the mechanism underlying phagocytic entry of this pathogen.</p>
</section>
<section id="importance" class="level2" data-number="3.2">
<h2 data-number="3.2" class="anchored" data-anchor-id="importance"><span class="header-section-number">3.2</span> Importance</h2>
<p><em>Mycobacterium tuberculosis</em> (<em>Mtb</em>) invades alveolar macrophages through phagocytosis to establish infection and cause disease. The molecular mechanisms underlying <em>Mtb</em> entry are still poorly understood. Here, we report that an intact sphingolipid biosynthetic pathway is essential for the uptake of <em>Mtb</em> by phagocytes. Disrupting sphingolipid production affects the segregation of the regulatory phosphatase CD45 from the nascent phagosome, a critical step in the progression of phagocytosis. We also show that blocking sphingolipid biosynthesis impairs activation of small GTPases and phosphoinositide turnover at the host-pathogen contact sites. Moreover, production of sphingomyelin, not glycosphingolipids, is critical for the phagocytic uptake of <em>Mtb</em>. These data demonstrate a vital role for sphingomyelin biosynthesis in an early step of <em>Mtb</em> infection, defining a potential target for anti-mycobacterial therapeutics.</p>
</section>
<section id="introduction" class="level2" data-number="3.3">
<h2 data-number="3.3" class="anchored" data-anchor-id="introduction"><span class="header-section-number">3.3</span> Introduction</h2>
<p><em>Mycobacterium tuberculosis</em> (<em>Mtb</em>), the etiologic agent of tuberculosis (TB), remains a significant global health threat. It is estimated that more than two billion people around the world are infected with the latent form of <em>Mtb</em> and about 1.5 million die every year<span class="citation" data-cites="worldhealthorganizationGlobalTuberculosisReport2018"><sup><a href="../references.html#ref-worldhealthorganizationGlobalTuberculosisReport2018" role="doc-biblioref">1</a></sup></span>. The success of <em>Mtb</em> as a pathogen relies on its ability to effectively enter the immune cells and establish its niche by subverting the defense mechanisms of the host<span class="citation" data-cites="korbMycobacteriumTuberculosisManipulator2016 quevalMacrophageDisputedFortress2017"><sup><a href="../references.html#ref-korbMycobacteriumTuberculosisManipulator2016" role="doc-biblioref">2</a>,<a href="../references.html#ref-quevalMacrophageDisputedFortress2017" role="doc-biblioref">3</a></sup></span>. Phagocytosis of <em>Mtb</em> by lung resident alveolar macrophages represents the obligate first step in the <em>Mtb</em> infection cycle. <em>Mtb</em> can proliferate in these cells, escape from the phagosome, and migrate to local lymph nodes to disseminate and spread the infection. Precise knowledge of the mechanism exploited by <em>Mtb</em> to invade immune cells is desirable, as this may yield novel targets and treatment options to combat TB.</p>
<p>Macrophages, along with dendritic cells and neutrophils, serve as professional phagocytic cells of the immune system that are specialized in the recognition, engulfment and elimination of microbial pathogens, thereby contributing to the first line of defense against infection. Phagocytosis is a receptor-initiated process involving a massive reorganization and deformation of the cell membrane coordinated by the underlying actin cytoskeleton to enable the internalization of microbial pathogens<span class="citation" data-cites="levinPhosphoinositidesPhagocytosisMacropinocytosis2015"><sup><a href="../references.html#ref-levinPhosphoinositidesPhagocytosisMacropinocytosis2015" role="doc-biblioref">4</a></sup></span>. To this end, phagocytes are equipped with pattern-recognition receptors (PRRs) that bind ligands on the surface of the pathogen<span class="citation" data-cites="stuartPhagocytosisElegantComplexity2005 flannaganCellBiologyPhagocytosis2012"><sup><a href="../references.html#ref-stuartPhagocytosisElegantComplexity2005" role="doc-biblioref">5</a>,<a href="../references.html#ref-flannaganCellBiologyPhagocytosis2012" role="doc-biblioref">6</a></sup></span>. Prototypical PRRs include Toll-like receptors, scavenger receptors and C-type lectin receptors of the Dectin family. Activation of PPRs results from the clustering of several receptors upon binding to multiple adjacent ligands on the pathogen surface, which brings their cytosolic domains in close apposition. This initiates signaling by non-receptor tyrosine kinases of the Src family and the recruitment of phosphoinositide-modifying enzymes<span class="citation" data-cites="flannaganCellBiologyPhagocytosis2012 goodridgeActivationInnateImmune2011a"><sup><a href="../references.html#ref-flannaganCellBiologyPhagocytosis2012" role="doc-biblioref">6</a>,<a href="../references.html#ref-goodridgeActivationInnateImmune2011a" role="doc-biblioref">7</a></sup></span>. Local alterations in phosphoinositide levels and concomitant activation of Rho family GTPases orchestrate a remodeling of the actin cytoskeleton, which drives an extension of the plasma membrane along the particle to form a phagocytic cup. Further actin polymerization and contractile activity lead to membrane apposition and eventual sealing of the nascent phagosome, which then evolves into a degradative phagolysosome<span class="citation" data-cites="flannaganCellBiologyPhagocytosis2012 goodridgeActivationInnateImmune2011a"><sup><a href="../references.html#ref-flannaganCellBiologyPhagocytosis2012" role="doc-biblioref">6</a>,<a href="../references.html#ref-goodridgeActivationInnateImmune2011a" role="doc-biblioref">7</a></sup></span>. While the precise mechanism by which <em>Mtb</em> invades immune cells remains to be established, a wide variety of PPRs has been implicated in the recognition and phagocytic uptake of mycobacteria<span class="citation" data-cites="tailleuxDCSIGNMajorMycobacterium2003a yadavVglucanReceptorDectin12006 killickReceptormediatedRecognitionMycobacterial2013"><sup><a href="../references.html#ref-tailleuxDCSIGNMajorMycobacterium2003a" role="doc-biblioref">8</a>–<a href="../references.html#ref-killickReceptormediatedRecognitionMycobacterial2013" role="doc-biblioref">10</a></sup></span>. This apparent redundancy in the entry process hampers the development of an effective strategy to target a critical step in the infection cycle of <em>Mtb</em>.</p>
<p>Sphingolipids are essential lipids enriched in the plasma membrane of mammalian cells, where they interact with cholesterol to control vital membrane properties, serve as adhesion sites for extracellular proteins, and play important roles in signal transmission and cell recognition<span class="citation" data-cites="lingwoodNewAspectsRegulation2010 slotteBiologicalFunctionsSphingomyelins2013 hannunSphingolipidsTheirMetabolism2018"><sup><a href="../references.html#ref-lingwoodNewAspectsRegulation2010" role="doc-biblioref">11</a>–<a href="../references.html#ref-hannunSphingolipidsTheirMetabolism2018" role="doc-biblioref">13</a></sup></span>. Sphingolipids have also emerged as active participants in various infectious diseases. Various pathogens, including bacteria, fungi and viruses, modulate host membrane sphingolipids and their metabolites to manipulate host defense, thereby promoting their survival and pathogenicity<span class="citation" data-cites="heungRoleSphingolipidsMicrobial2006a"><sup><a href="../references.html#ref-heungRoleSphingolipidsMicrobial2006a" role="doc-biblioref">14</a></sup></span>. For instance, <em>Neisseria gonorrhoeae</em> and <em>Pseudomonas aeriginosa</em> exploit the hydrolysis of cell surface sphingomyelin (SM) to generate ceramide-rich microdomains that serve as portals for their entry into macrophages<span class="citation" data-cites="grassmeAcidicSphingomyelinaseMediates1997a grassmeHostDefensePseudomonas2003"><sup><a href="../references.html#ref-grassmeAcidicSphingomyelinaseMediates1997a" role="doc-biblioref">15</a>,<a href="../references.html#ref-grassmeHostDefensePseudomonas2003" role="doc-biblioref">16</a></sup></span>. Sphingolipids also play a critical role in the in vivo clearance of the pathogenic fungi <em>Candida albicans</em> through phagocytosis<span class="citation" data-cites="tafesseDisruptionSphingolipidBiosynthesis2015a"><sup><a href="../references.html#ref-tafesseDisruptionSphingolipidBiosynthesis2015a" role="doc-biblioref">17</a></sup></span>. Moreover, mycobacteria utilize host sphingolipids and cholesterol as nutrient source for their growth and survival<span class="citation" data-cites="lovewellChewingFatLipid2016a"><sup><a href="../references.html#ref-lovewellChewingFatLipid2016a" role="doc-biblioref">18</a></sup></span>. Although the role of host membrane cholesterol in the entry of different <em>Mycobacterium</em> species have previously been addressed<span class="citation" data-cites="gatfieldEssentialRoleCholesterol2000a"><sup><a href="../references.html#ref-gatfieldEssentialRoleCholesterol2000a" role="doc-biblioref">19</a></sup></span>, the function of sphingolipids in <em>Mtb</em> invasion has not been explored.</p>
<p>Here, we show that blocking sphingolipid biosynthesis causes a substantial reduction in the uptake of both <em>Mtb</em> and <em>Mycobacterium marinum</em> by a panel of macrophages and dendritic cells without affecting clathrin-mediated endocytosis or micropinocytosis. Time-lapse microscopy revealed that sphingolipid-deficient macrophages are capable of pathogen recognition and actin assembly at the contact site, but impaired in the subsequent remodeling of the actin cytoskeleton necessary to complete internalization of the pathogen. We also show that sphingolipid-deficient macrophages fail to exclude the regulatory tyrosine phosphatase CD45 from the receptor-pathogen contact site, a crucial step in the development of a functional phagocytic synapse. This defect was accompanied by an impaired activation of Rho GTPases and significant reduction in phosphoinositide turnover. Finally, we show that the production of SM, not glycosphingolipids, is essential for efficient <em>Mtb</em> uptake. Our findings offer fresh mechanistic insights into how <em>Mtb</em> invades immune cells and establish a critical role for SM in this process.  </p>
</section>
<section id="results" class="level2" data-number="3.4">
<h2 data-number="3.4" class="anchored" data-anchor-id="results"><span class="header-section-number">3.4</span> Results</h2>
<section id="manipulation-of-sphingolipid-levels-in-mammalian-phagocytes" class="level3" data-number="3.4.1">
<h3 data-number="3.4.1" class="anchored" data-anchor-id="manipulation-of-sphingolipid-levels-in-mammalian-phagocytes"><span class="header-section-number">3.4.1</span> Manipulation of sphingolipid levels in mammalian phagocytes</h3>
<p>As depicted in <a href="#fig-mBioFig1" class="quarto-xref">Figure&nbsp;<span>3.1</span></a>A, the initial and rate limiting step of sphingolipid biosynthesis is the production of a sphingoid base through the condensation of serine and palmitoyl-CoA, a reaction performed by the serine palmitoyltransferase (SPT) enzyme complex<span class="citation" data-cites="hanadaSerinePalmitoyltransferaseKey2003"><sup><a href="../references.html#ref-hanadaSerinePalmitoyltransferaseKey2003" role="doc-biblioref">20</a></sup></span>. Genetic ablation of the SPT subunit Sptlc2 blocks the de novo synthesis of all sphingolipids, as does treatment with the atypical amino acid myriocin (Myr), a potent inhibitor of SPT activity<span class="citation" data-cites="tafesseDisruptionSphingolipidBiosynthesis2015a"><sup><a href="../references.html#ref-tafesseDisruptionSphingolipidBiosynthesis2015a" role="doc-biblioref">17</a></sup></span>. The fungal toxin fumonisin B1 (FB1) acts several steps downstream in the sphingolipid biosynthetic pathway by blocking the activity of ceramide synthases (CerS)<span class="citation" data-cites="tafesseDisruptionSphingolipidBiosynthesis2015a wangInhibitionSphingolipidBiosynthesis1991"><sup><a href="../references.html#ref-tafesseDisruptionSphingolipidBiosynthesis2015a" role="doc-biblioref">17</a>,<a href="../references.html#ref-wangInhibitionSphingolipidBiosynthesis1991" role="doc-biblioref">21</a></sup></span>. As the central nexus of sphingolipid biosynthesis, ceramide is converted into a variety of complex sphingolipids, including sphingomyelin (SM) and glycosphingolipids<span class="citation" data-cites="leierLipidsPathogenicFlaviviruses2018a gaultOverviewSphingolipidMetabolism2010a"><sup><a href="../references.html#ref-leierLipidsPathogenicFlaviviruses2018a" role="doc-biblioref">22</a>,<a href="../references.html#ref-gaultOverviewSphingolipidMetabolism2010a" role="doc-biblioref">23</a></sup></span>. To define the role of sphingolipid biosynthesis in the phagocytic uptake of <em>Mtb</em>, we employed four different phagocytic cell lines: murine RAW246.7 macrophages, murine DC2.4 dendritic cells, human THP-1 and U937 monocytes (<a href="#fig-mBioFig1" class="quarto-xref">Figure&nbsp;<span>3.1</span></a>B).</p>
<div class="border">
<div id="fig-mBioFig1" class="quarto-figure quarto-figure-center quarto-float anchored" data-group="results_PhagocytosisGallery">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mBioFig1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_Mtb_phagocytosis_files/TrueHighResImages/mBio03141-20_Figure_1.png" class="lightbox" data-gallery="results_PhagocytosisGallery" data-glightbox="description: .lightbox-desc-1"><img src="../../parts/results/results_Mtb_phagocytosis_files/TrueHighResImages/mBio03141-20_Figure_1.png" style="height:5in" class="figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-mBioFig1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.1: <strong>Manipulation of cellular sphingolipid levels using genetics and chemical tools.</strong> (A) Schematic outline of the sphingolipid biosynthetic pathway. Inhibitors of sphingolipid biosynthetic enzymes used in this study are in red. Genes deleted using CRISPR/Cas9 are in blue. SPT, serine palmitoyltransferase; KDSR, 3-ketodihydrosphingosine reductase; CerS, ceramide synthase; DegS, δ4-desaturase; GlcCerS, glucosylceramide synthase; SMS, sphingomyelin synthase; FB1, fumonisin B1. (B) Phagocytic cell lines used in this study. (C) DC2.4 and RAW264.7 wild-type (WT) and Sptlc2<sup>−/−</sup> cells were processed for immunoblotting with antibodies against Sptlc2 and calnexin. (D) Sptlc2<sup>−/−</sup> and wild-type RAW264.7 or THP-1 cells were treated with myriocin or fumonisin B1 for 3 days and then metabolically labeled with 3-l-[14C]serine for 4 h. Total lipids were extracted and analyzed by thin-layer chromatography (TLC) and autoradiography. Veh, vehicle treated. (E and F) Quantification of [14C]SM and [14C]PS signals from TLCs shown in panel D. (G) Wild-type, Sptlc2<sup>−/−</sup>, and inhibitor-treated RAW264.7 cells were subjected to total lipid extraction. Sphingosine, ceramide, sphingomyelin, and glucosylceramide (GlucCer) levels were quantified by LC-MS/MS. Data are means ± standard deviations (SD) (n = 3). *, P &lt; 0.05; **, P &lt; 0.01; ***, P &lt; 0.001 (two-tailed unpaired t test).
</figcaption>
</figure>
</div>
</div>
<p>To verify the reported effects of Myr and FB1 on de novo sphingolipid biosynthesis, we monitored the incorporation of 3-L-[14C]-serine into sphingolipid species in RAW246.7, THP-1, DC2.4 and U937 cells after 3 days of drug treatment using thin layer chromatography (TLC) analysis. Across all four cell lines examined, treatment with Myr or FB1 in each case reduced incorporation of the serine isotope into SM by more than 90%; in contrast, incorporation of the isotope into phosphatidylserine (PS) was largely unaffected, (<a href="#fig-mBioFig1" class="quarto-xref">Figure&nbsp;<span>3.1</span></a>D-F and <a href="#suppfig-mBioFigS1" class="quarto-xref">Figure S<span>3.1</span></a>A and B). Genetic ablation of Sptlc2 in RAW246.7 and DC2.4 cells also virtually abolished de novo SM production without affecting PS biosynthesis (<a href="#fig-mBioFig1" class="quarto-xref">Figure&nbsp;<span>3.1</span></a>C-E and <a href="#suppfig-mBioFigS1" class="quarto-xref">Figure S<span>3.1</span></a>A), in line with our previous findings<span class="citation" data-cites="tafesseDisruptionSphingolipidBiosynthesis2015a"><sup><a href="../references.html#ref-tafesseDisruptionSphingolipidBiosynthesis2015a" role="doc-biblioref">17</a></sup></span>.</p>
<p>We next examined the impact of drug treatment and Sptlc2 removal on the steady state sphingolipid levels. Both Myr-treated and Sptlc2<sup>-/-</sup> cells contained significantly (&gt;=3-fold) reduced levels of sphingosine, ceramides, SM and glucosylceramide (GlcCer) in comparison to control cells (<a href="#fig-mBioFig1" class="quarto-xref">Figure&nbsp;<span>3.1</span></a>G). In FB1-treated cells, we observed a reduction in the levels of ceramide, SM and GlcCer concomitant with an increase in sphingosine levels, as expected, (<a href="#fig-mBioFig1" class="quarto-xref">Figure&nbsp;<span>3.1</span></a>G)<span class="citation" data-cites="meivar-levyRoleSphingolipidsMaintenance1997"><sup><a href="../references.html#ref-meivar-levyRoleSphingolipidsMaintenance1997" role="doc-biblioref">24</a></sup></span>. The residual sphingolipid levels detected in Sptlc2<sup>-/-</sup>, Myr- and FB1-treated cells are likely due to the activity of sphingolipid salvage pathways, through which cells may acquire sphingolipids from additives in the culture medium – primarily serum. This enables them to bypass the growth defect that is generally observed when sphingolipid biosynthetic mutants are cultured in serum-depleted medium. Indeed, we observed that RAW264.7 and DC2.4 cells lacking Sptlc2 grew equally well in serum-containing medium as their wild type counterparts (<a href="#suppfig-mBioFigS1" class="quarto-xref">Figure S<span>3.1</span></a>C and D)<span class="citation" data-cites="tafesseDisruptionSphingolipidBiosynthesis2015a"><sup><a href="../references.html#ref-tafesseDisruptionSphingolipidBiosynthesis2015a" role="doc-biblioref">17</a></sup></span>.</p>
</section>
<section id="sphingolipid-deficient-cells-display-a-reduced-capacity-to-phagocytose-mtb" class="level3" data-number="3.4.2">
<h3 data-number="3.4.2" class="anchored" data-anchor-id="sphingolipid-deficient-cells-display-a-reduced-capacity-to-phagocytose-mtb"><span class="header-section-number">3.4.2</span> Sphingolipid-deficient cells display a reduced capacity to phagocytose <em>Mtb</em></h3>
<p>To address whether sphingolipids participate in the phagocytic uptake of <em>Mtb</em>, RAW264.7, THP-1, DC2.4 and U937 cells were treated with Myr and FB1 for 3 days and then infected with a mCherry-expressing strain of <em>Mtb</em>. In brief, cells were infected at a multiplicity of infection (MOI) of 10 for 2 h, fixed and then stained with DyLight 488-conjugated phalloidin and DAPI to visualize filamentous actin (F-actin) and cell nuclei, respectively. Fluorescence microscopy was used to quantify uptake efficiency by automated counting of cell number (blue nuclei) and internalized bacteria (red fluorescent particles within green cell boundary; <a href="#fig-mBioFig2" class="quarto-xref">Figure&nbsp;<span>3.2</span></a>A). Uptake efficiency was defined as the ratio of the total number of internalized <em>Mtb</em> particles against the total number of identified nuclei. The ratio is reported as a percentage of phagocytosis relative to the control cells. In all four cell models analyzed, we observed that Myr and FB1 treatment caused a significant (40-60%) reduction in <em>Mtb</em> uptake (<a href="#fig-mBioFig2" class="quarto-xref">Figure&nbsp;<span>3.2</span></a>B and C; <a href="#suppfig-mBioFigS2" class="quarto-xref">Figure S<span>3.2</span></a>A-D). Similarly, genetic ablation of Sptlc2 in RAW264.7 and DC2.4 cells results in a ~50% reduction in uptake relative to their wild type controls (<a href="#fig-mBioFig2" class="quarto-xref">Figure&nbsp;<span>3.2</span></a>D and E). Collectively, these results indicate that an intact sphingolipid biosynthetic pathway is essential for an efficient uptake of <em>Mtb</em> by professional phagocytes. To test whether the same holds true for other mycobacteria, we used <em>Mycobacterium marinum</em>, a natural pathogen of poikilotherms that is closely related to <em>Mtb</em> and serves as a widely accepted model for TB<span class="citation" data-cites="ramakrishnanMycobacteriumMarinumPersists1994 rocaTNFDuallyMediates2013"><sup><a href="../references.html#ref-ramakrishnanMycobacteriumMarinumPersists1994" role="doc-biblioref">25</a>,<a href="../references.html#ref-rocaTNFDuallyMediates2013" role="doc-biblioref">26</a></sup></span>. Similar to infections with <em>Mtb</em>, Sptlc2<sup>-/-</sup> RAW264.7 and DC2.4 cells showed a ~50% reduction in phagocytic uptake of <em>M. marinum</em> relative to wild type cells (<a href="#fig-mBioFig2" class="quarto-xref">Figure&nbsp;<span>3.2</span></a>F and G).</p>
<div class="border">
<div id="fig-mBioFig2" class="quarto-figure quarto-figure-center quarto-float anchored" data-group="results_PhagocytosisGallery">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mBioFig2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_Mtb_phagocytosis_files/TrueHighResImages/mBio03141-20_Figure_2.png" class="lightbox" data-gallery="results_PhagocytosisGallery" data-glightbox="description: .lightbox-desc-2"><img src="../../parts/results/results_Mtb_phagocytosis_files/TrueHighResImages/mBio03141-20_Figure_2.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-mBioFig2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.2: <strong>Sphingolipid biosynthesis is required for efficient phagocytosis of <em>Mtb</em>.</strong> (A) Control and sphingolipid-depleted phagocytic cells were infected with mCherry-expressing <em>Mtb</em> at an MOI of 10 for 2 h, washed, fixed, stained with phalloidin Alexa Fluor 488 and DAPI, and then analyzed by high-throughput fluorescence imaging. The efficiency of phagocytic uptake was determined by dividing the total number of internalized bacteria by the total number of cells. (B) RAW264.7, DC2.4, THP-1, and U937 cells were grown in the presence or absence of myriocin for 3 days, infected with mCherry-expressing <em>Mtb</em>, and then processed as for panel A. (C) Quantification of <em>Mtb</em> uptake by cells treated as for panel B. (D) Wild-type and Sptlc2<sup>−/−</sup> RAW264.7 and DC2.4 cells were infected with mCherry-expressing <em>Mtb</em> and then processed as for panel A. (E) Quantification of <em>Mtb</em> uptake by cells treated as for panel D. (F) Wild-type and Sptlc2<sup>−/−</sup> RAW264.7 and DC2.4 cells were infected with mCherry-expressing <em>M. marinum</em> at an MOI of 10 for 2 h and then processed as for panel A. (G) Quantification of <em>M. marinum</em> uptake by cells treated as for panel F. (H) Wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells were incubated with zymosan A particles at an MOI of 1 for 1 h and then processed as for panel A. (I) Representative images of wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells infected with mCherry-tagged herpes simplex virus (HSV) at an MOI of 1. At 12 h post-infection, cells were fixed, stained with DAPI, and analyzed by high-throughput fluorescence microscopy to determine the percent infected cells. Bar, 5 μm. Data in panels C, E, and G are means ± SD (n = 3). *, P &lt; 0.05; **, P &lt; 0.01, ***, P &lt; 0.001 (unpaired t test). Data in panel I are means ± standard errors (SE) (n = 2). ns, not significant by two-tailed unpaired t test.
</figcaption>
</figure>
</div>
</div>
</section>
<section id="sptlc2---cells-are-permissive-to-herpes-simplex-virus-entry-and-clathrin-mediated-endocytosis." class="level3" data-number="3.4.3">
<h3 data-number="3.4.3" class="anchored" data-anchor-id="sptlc2---cells-are-permissive-to-herpes-simplex-virus-entry-and-clathrin-mediated-endocytosis."><span class="header-section-number">3.4.3</span> Sptlc2<sup>-/-</sup> cells are permissive to herpes simplex virus entry and clathrin-mediated endocytosis.</h3>
<p>To determine whether an intact sphingolipid biosynthetic pathway is a general prerequisite for the entry of pathogens in professional phagocytes, we performed parallel infections of wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells with a fluorescent reporter strain of herpes simplex virus 1 (HSV-1) encoding a fusion of the viral protein VP26 and mCherry (28). HSV-1 is an enveloped virus that gains entry to host cells through the phagocytosis-like process of micropinocytosis; the viral receptor involved is nectin-1, also known as herpesviruses entry mediator<span class="citation" data-cites="clementNovelRolePhagocytosislike2006a tiwariNonprofessionalPhagocytosisCan2012"><sup><a href="../references.html#ref-clementNovelRolePhagocytosislike2006a" role="doc-biblioref">27</a>,<a href="../references.html#ref-tiwariNonprofessionalPhagocytosisCan2012" role="doc-biblioref">28</a></sup></span>. At 12 h post-infection, we quantified the rates of HSV-1 infection using high-content imaging as described above. Because the VP26-mCherry reporter construct localizes to the nucleus, we did not stain cells using phalloidin and instead used the co-localization of DAPI with mCherry to determine infection efficiency. However, we observed no significant difference in the HSV-1 infection rates between wild-type and Sptlc2<sup>−/−</sup> cells (<a href="#fig-mBioFig2" class="quarto-xref">Figure&nbsp;<span>3.2</span></a>I). We also analyzed Myr-treated and control THP-1 cells for their ability to take up transferrin (Tf), which binds Tf receptors on the cell surface to enter cells via clathrin-mediated endocytosis<span class="citation" data-cites="heEndocytosisFunctionallyEnhanced2015a"><sup><a href="../references.html#ref-heEndocytosisFunctionallyEnhanced2015a" role="doc-biblioref">29</a></sup></span>. As shown in <a href="#suppfig-mBioFigS2" class="quarto-xref">Figure S<span>3.2</span></a>G and H, Myr treatment did not affect the endocytic uptake of transferrin. Collectively, these data indicate that sphingolipid biosynthesis is dispensable for HSV-1 entry and clathrin-mediated endocytosis, thus supporting a more specific role of sphingolipids in the phagocytic uptake of <em>Mtb</em>.</p>
</section>
<section id="sptlc2-cells-display-defects-in-filamentous-actin-dynamics-during-phagocytic-uptake." class="level3" data-number="3.4.4">
<h3 data-number="3.4.4" class="anchored" data-anchor-id="sptlc2-cells-display-defects-in-filamentous-actin-dynamics-during-phagocytic-uptake."><span class="header-section-number">3.4.4</span> Sptlc2<sup>−/−</sup> cells display defects in filamentous actin dynamics during phagocytic uptake.</h3>
<p>Particle ingestion by phagocytosis results from sequential rearrangements of F-actin and the overlying membrane. F-actin polymerization is the force driving membrane extension and phagosome formation. Nucleation of filaments at the site of particle recognition initiates F-actin polymerization and promotes formation of pseudopodia or lamellae. Then, actin filaments depolymerize at the base of the phagocytic cup, while polymerization proceeds in the tips of the lamellae to promote phagosomal membrane extension and closure<span class="citation" data-cites="srinivasanRacCdc42Play2003"><sup><a href="../references.html#ref-srinivasanRacCdc42Play2003" role="doc-biblioref">30</a></sup></span> (<a href="#fig-mBioFig3" class="quarto-xref">Figure&nbsp;<span>3.3</span></a>A). To determine at which step in this process sphingolipids exert their critical role, we monitored F-actin dynamics in wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells expressing LifeAct-mKate during infection with a monomeric GFP (mGFP)-expressing strain of <em>M. marinum</em>. Time-lapse fluorescence microscopy revealed that in wild-type RAW264.7 cells, F-actin readily assembled at sites of pathogen binding (<a href="#fig-mBioFig3" class="quarto-xref">Figure&nbsp;<span>3.3</span></a>B; <a href="#vid-Mtb_phagocytosis_Movie1" class="quarto-xref">Supplementary Movie&nbsp;<span>3.1</span></a>). In nearly all cases (∼89%; n = 46), this led to a successful internalization of the pathogen and clearance of F-actin polymers (<a href="#fig-mBioFig3" class="quarto-xref">Figure&nbsp;<span>3.3</span></a>C). In contrast, while RAW264.7 Sptlc2<sup>−/−</sup> cells readily assembled F-actin at the site of pathogen recognition, in about half of the cases (47%; n = 30) the pathogen failed to enter and F-actin remained assembled at the contact site (<a href="#fig-mBioFig3" class="quarto-xref">Figure&nbsp;<span>3.3</span></a>B and C; <a href="#vid-Mtb_phagocytosis_Movie1" class="quarto-xref">Supplementary Movie&nbsp;<span>3.1</span></a>).</p>
<div class="border">
<div id="fig-mBioFig3" class="quarto-figure quarto-figure-center quarto-float anchored" data-group="results_PhagocytosisGallery">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mBioFig3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_Mtb_phagocytosis_files/TrueHighResImages/mBio03141-20_Figure_3.png" class="lightbox" data-gallery="results_PhagocytosisGallery" data-glightbox="description: .lightbox-desc-3"><img src="../../parts/results/results_Mtb_phagocytosis_files/TrueHighResImages/mBio03141-20_Figure_3.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-mBioFig3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.3: <strong>Sptlc2<sup>−/−</sup> cells display a defect in actin dynamics during phagocytic cup formation.</strong> (A) Schematic outline of F-actin remodeling during phagocytosis. Clustering of pattern recognition receptors (PRRs) at the pathogen interaction site generates second messengers that promote local actin polymerization. Actin continues to be actively polymerized to extend the phagosomal membrane while being remodeled at the base of the phagocytic cup. Further actin polymerization and contractile activity lead to membrane apposition and eventually fusion to complete phagosome formation. (B) Time-lapse confocal images of wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells stably transduced with LifeAct-mKate (red) and infected with GFP-expressing <em>M. marinum</em> (green). Arrows indicate sites where mycobacteria trigger actin assembly, initiating formation of a phagocytic cup. Unlike wild-type cells, Sptlc2<sup>−/−</sup> RAW264.7 cells often failed to disassemble actin at the base of phagocytic cups, thus blocking entry of the mycobacteria. (C) Quantification of the percentage of successful versus failed phagocytic uptake events of <em>M. marinum</em> in wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells. (D) Time-lapse confocal images of wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells stably transduced with LifeAct-GFP (red) and incubated with zymosan A particles (green). Bar, 5 μm. (E) Quantification of the percentage of successful versus failed phagocytic uptake events of zymosan A particles in wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells. Data represent uptake events in &gt;30 cells per condition. Bar, 5 μm.
</figcaption>
</figure>
</div>
</div>
<p>An in-depth microscopic analysis of the role of sphingolipids in the phagocytic uptake of <em>Mtb</em> and <em>M. marinum</em> is hampered by the fact that these pathogens are often internalized as heterologous clusters that comprise multiple bacteria. To overcome this limitation, we also analyzed wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells for their ability to take up zymosan A beads. These pathogen-mimicking particles engage the C-type lectin receptor dectin-1<span class="citation" data-cites="tafesseDisruptionSphingolipidBiosynthesis2015a mansourDectin1ActivationControls2013a underhillMacrophageRecognitionZymosan2003a"><sup><a href="../references.html#ref-tafesseDisruptionSphingolipidBiosynthesis2015a" role="doc-biblioref">17</a>,<a href="../references.html#ref-mansourDectin1ActivationControls2013a" role="doc-biblioref">31</a>,<a href="../references.html#ref-underhillMacrophageRecognitionZymosan2003a" role="doc-biblioref">32</a></sup></span>, which is one of several pathogen recognition receptors on professional phagocytes that recognize <em>Mtb</em> and initiate phagocytic signaling to facilitate host cell entry<span class="citation" data-cites="rothfuchsDectin1InteractionMycobacterium2007a wagenerDectin1SykCARD9SignalingPathway2018a"><sup><a href="../references.html#ref-rothfuchsDectin1InteractionMycobacterium2007a" role="doc-biblioref">33</a>,<a href="../references.html#ref-wagenerDectin1SykCARD9SignalingPathway2018a" role="doc-biblioref">34</a></sup></span>. Sptlc2<sup>−/−</sup> RAW264.7 cells displayed an ∼40% reduction in the phagocytic uptake of zymosan A particles relative to wild-type cells (<a href="#fig-mBioFig2" class="quarto-xref">Figure&nbsp;<span>3.2</span></a>H), similar to the defect in <em>Mtb</em> and <em>M. marinum</em> uptake observed in these cells (<a href="#fig-mBioFig2" class="quarto-xref">Figure&nbsp;<span>3.2</span></a>D to G). Time-lapse microscopy revealed that in wild-type RAW264.7 cells, zymosan A particles initiating F-actin assembly at the recognition site were efficiently internalized, whereby depolymerization of F-actin at the base of the phagocytic cup and subsequent F-actin polymerization in the leading edges of the lamellae could be readily visualized in real time (90%; n = 30) (<a href="#fig-mBioFig3" class="quarto-xref">Figure&nbsp;<span>3.3</span></a>D and E; <a href="#vid-Mtb_phagocytosis_Movie2" class="quarto-xref">Supplementary Movie&nbsp;<span>3.2</span></a>). Again, while RAW264.7 Sptlc2<sup>−/−</sup> cells readily assembled F-actin at the site of particle recognition, in a significant number of cases the particles failed to enter (40%; n = 30) and F-actin polymers remained assembled at the contact site (<a href="#fig-mBioFig3" class="quarto-xref">Figure&nbsp;<span>3.3</span></a>D and E; <a href="#vid-Mtb_phagocytosis_Movie2" class="quarto-xref">Supplementary Movie&nbsp;<span>3.2</span></a>). In a complementary approach, wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells were incubated with IgG-opsonized zymosan A particles for 10 min and cell surface-bound particles were discriminated from internalized particles by immunostaining. Sites of F-actin polymerization were visualized by labeling cells with phalloidin Alexa Fluor 488. As shown in <a href="#suppfig-mBioFigS2" class="quarto-xref">Figure S<span>3.2</span></a>E and F, Sptlc2<sup>−/−</sup> cells displayed a nearly 3-fold reduction in the ratio of internalized to cell surface-bound beads compared to wild-type cells. Together, these results indicate that sphingolipids are dispensable for the initial steps of phagocytosis, including pathogen recognition and the assembly of F-actin polymers at the contact site. Instead, it appears sphingolipids exert their critical role immediately downstream of these events, as Sptlc2<sup>−/−</sup> cells often fail to complete the sequential rearrangements of F-actin necessary for pathogen entry.</p>
</section>
<section id="sptlc2-cells-fail-to-exclude-cd45-from-the-pathogen-contact-site." class="level3" data-number="3.4.5">
<h3 data-number="3.4.5" class="anchored" data-anchor-id="sptlc2-cells-fail-to-exclude-cd45-from-the-pathogen-contact-site."><span class="header-section-number">3.4.5</span> Sptlc2<sup>−/−</sup> cells fail to exclude CD45 from the pathogen contact site.</h3>
<p>The engagement of PPRs with their ligands initiates receptor clustering at the phagocytic cup, followed by a lateral segregation of the inhibitory phosphatase CD45<span class="citation" data-cites="goodridgeActivationInnateImmune2011a"><sup><a href="../references.html#ref-goodridgeActivationInnateImmune2011a" role="doc-biblioref">7</a></sup></span>. Because sphingolipids have been implicated as active participants in the organization of signaling complexes at the cell surface<span class="citation" data-cites="simonsLipidRaftsSignal2000a"><sup><a href="../references.html#ref-simonsLipidRaftsSignal2000a" role="doc-biblioref">35</a></sup></span>, we reasoned that sphingolipid levels at the plasma membrane may be critical for the aggregation of dectin-1 and/or exclusion of CD45 from that pathogen contact site. To experimentally address this idea, we transfected wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells with GFP-tagged dectin-1 and assessed the co-localization of the fluorescent receptor with CD45 following treatment with non-fluorescent zymosan A beads. As control, we also monitored the sub-cellular distribution of GFP-tagged dectin-1 in RAW264.7 cells upon infection with mCherry-expressing M. marinum. Dectin-1 readily accumulated at the particle contact site in M. marinum-exposed cells (<a href="#fig-mBioFig4" class="quarto-xref">Figure&nbsp;<span>3.4</span></a>A; <a href="#suppfig-mBioFigS3" class="quarto-xref">Figure S<span>3.3</span></a>A; <a href="#vid-Mtb_phagocytosis_Movie3" class="quarto-xref">Supplementary Movie&nbsp;<span>3.3</span></a> and <a href="#vid-Mtb_phagocytosis_Movie4" class="quarto-xref">Supplementary Movie&nbsp;<span>3.4</span></a>). Besides supporting a role of dectin-1 in the phagocytic uptake of mycobacteria, these findings point to a significant overlap in the pathways that mediate host cell entry of zymosan A beads and mycobacteria. In both wild-type and Sptlc2<sup>−/−</sup> cells, dectin-1 aggregated at the zymosan A contact site within 2 min after bead exposure, indicating that sphingolipid biosynthesis is dispensable for ligand-induced receptor clustering. As expected, in wild-type cells, the inhibitory phosphatase CD45 was efficiently segregated from the dectin-1 clusters at the particle contact site (<a href="#fig-mBioFig4" class="quarto-xref">Figure&nbsp;<span>3.4</span></a>B and C; <a href="#suppfig-mBioFigS3" class="quarto-xref">Figure S<span>3.3</span></a>B)<span class="citation" data-cites="goodridgeActivationInnateImmune2011a"><sup><a href="../references.html#ref-goodridgeActivationInnateImmune2011a" role="doc-biblioref">7</a></sup></span>. In contrast, CD45 failed to efficiently segregate from the dectin-1 clusters in zymosan A-treated Sptlc2<sup>−/−</sup> cells (<a href="#fig-mBioFig4" class="quarto-xref">Figure&nbsp;<span>3.4</span></a>B and C).</p>
<div class="border">
<div id="fig-mBioFig4" class="quarto-figure quarto-figure-center quarto-float anchored" data-group="results_PhagocytosisGallery">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mBioFig4-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_Mtb_phagocytosis_files/TrueHighResImages/mBio03141-20_Figure_4.png" class="lightbox" data-gallery="results_PhagocytosisGallery" data-glightbox="description: .lightbox-desc-4"><img src="../../parts/results/results_Mtb_phagocytosis_files/TrueHighResImages/mBio03141-20_Figure_4.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-mBioFig4-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.4: <strong>Segregation of CD45 from dectin-1 at the phagocytic cup is affected in Sptlc2<sup>−/−</sup> cells.</strong> (A) Time-lapse confocal images of wild-type RAW264.7 cells transfected with GFP-tagged dectin-1 (green) and infected with mCherry-expressing <em>M. marinum</em> (red). Arrows indicate sites where mycobacteria trigger clustering of dectin-1. Bar, 5 μm. (B) Wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells transfected with GFP-tagged dectin-1 (green) were incubated with zymosan A particles (dashed circles) on ice, warmed to 37°C for 2 min to initiate phagocytosis, fixed, and then stained with antibodies against CD45 (purple). Representative slices of z-stacks are shown. Insets show fluorescence intensity profiles of CD45 and dectin-1-GFP signals in a single slice. Bar, 3 μm. (C) Violin plots showing the level of CD45 segregation from dectin-1-enriched phagocytic cups in wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells, defined as the percentage of dectin-1 volume containing no CD45 voxels. More than 100 individual cups were analyzed per cell line.”****, P = 0.00006 by a two-tailed unpaired t test.
</figcaption>
</figure>
</div>
</div>
<p>To address whether a block in sphingolipid biosynthesis affects the overall surface levels of CD45 and dectin-1, wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells were stained with allophycocyanin (APC)-conjugated anti-CD45 and fluorescein isothiocyanate (FITC)-conjugated anti-dectin-1 antibodies and subjected to analytical flow cytometry. We found no differences in the overall fluorescent signals for both markers between wild-type and Sptlc2<sup>−/−</sup> cells (<a href="#suppfig-mBioFigS3" class="quarto-xref">Figure S<span>3.3</span></a>C to F), indicating that the cell surface levels of endogenous dectin-1 and CD45 are unaffected by blocking sphingolipid biosynthesis. From this, we conclude that an intact sphingolipid biosynthetic pathway is required to enable a proper displacement of CD45 from the site of dectin-1 clustering, a step critical for the initiation of phagocytic synapse formation.</p>
</section>
<section id="activation-of-rho-gtpases-rac1-and-cdc42-is-impaired-in-sptlc2-cells." class="level3" data-number="3.4.6">
<h3 data-number="3.4.6" class="anchored" data-anchor-id="activation-of-rho-gtpases-rac1-and-cdc42-is-impaired-in-sptlc2-cells."><span class="header-section-number">3.4.6</span> Activation of Rho GTPases Rac1 and Cdc42 is impaired in Sptlc2<sup>−/−</sup> cells.</h3>
<p>Phagocytosis is strictly regulated through a signaling network that involves the recruitment of pathogen receptors and Rho GTPases to the particle contact site. Activated Rho GTPases promote the conversion of phosphatidylinositol-4-phosphate (PI4P) to phosphatidylinositol-4,5-bis-phosphate [PI(4,5)P2] by stimulating phosphatidylinositol 5-kinase (PI5K). The local accumulation of PI(4,5)P2 is essential to initiate the polymerization of F-actin, a process that drives extension of the leading edges of the nascent phagosome around the particle<span class="citation" data-cites="schlamPhosphoinositide3kinaseEnables2015"><sup><a href="../references.html#ref-schlamPhosphoinositide3kinaseEnables2015" role="doc-biblioref">36</a></sup></span>. However, to complete the engulfment and internalization of the particle, F-actin must disassemble at the base of the phagocytic cup. This is achieved through a local conversion of PI(4,5)P2 to phosphatidylinositol-3,4,5-tris-phosphate [PI(3,4,5)P3] by the phosphatidylinositol 3-kinase (PI3K) (<a href="#fig-mBioFig5" class="quarto-xref">Figure&nbsp;<span>3.5</span></a>A)<span class="citation" data-cites="levinPhosphoinositidesPhagocytosisMacropinocytosis2015 schlamPhosphoinositide3kinaseEnables2015 ledvinaPhosphatidylinositol3KinaseEffector2018"><sup><a href="../references.html#ref-levinPhosphoinositidesPhagocytosisMacropinocytosis2015" role="doc-biblioref">4</a>,<a href="../references.html#ref-schlamPhosphoinositide3kinaseEnables2015" role="doc-biblioref">36</a>,<a href="../references.html#ref-ledvinaPhosphatidylinositol3KinaseEffector2018" role="doc-biblioref">37</a></sup></span>.</p>
<div class="border">
<div id="fig-mBioFig5" class="quarto-figure quarto-figure-center quarto-float anchored" data-group="results_PhagocytosisGallery">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mBioFig5-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_Mtb_phagocytosis_files/TrueHighResImages/mBio03141-20_Figure_5.png" class="lightbox" data-gallery="results_PhagocytosisGallery" data-glightbox="description: .lightbox-desc-5"><img src="../../parts/results/results_Mtb_phagocytosis_files/TrueHighResImages/mBio03141-20_Figure_5.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-mBioFig5-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.5: <strong>Sptlc2<sup>−/−</sup> cells display an impaired phosphoinositide turnover at the phagocytic cup.</strong> (A) Schematic outline of molecular events underlying phagocytosis. Engagement of pattern recognition receptors (PRRs) with a pathogen initiates receptor clustering at the phagocytic cup, followed by a lateral segregation of the inhibitory CD45 phosphatase. This results in activation of the Rho GTPases Cdc42 and Rac1, which, in turn, stimulate PI5K activity to enable local production of PI(4,5)P2 and promote actin assembly. Next, PI3K orchestrates the turnover of PI(4,5)P2 to PI(3,4,5)P3, leading to actin disassembly at the base of the phagosome, a process required to complete the engulfment and internalization of the pathogen. (B) Time-lapse confocal images of wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells transfected with Cdc42 activity biosensor PAK(PBD)-GFP (green) and incubated with zymosan beads (red) at an MOI of 10. (C) Quantification of the PAK(PBD) fluorescence intensity in the course of phagocytic cup formation in cells treated as for panel B. (D) Time-lapse confocal images of wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells transfected with the PI(4,5)P2 biosensor PH(PLCδ)-GFP (green) and incubated with zymosan beads (red) as for panel B. (E) Quantification of the PH(PLCδ) fluorescence intensity in the course of phagocytic cup formation in cells treated as for panel D. (F) Time-lapse confocal images of wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells transfected with the PI(3,4,5)P3 biosensor PH(AKT)-GFP (green) and incubated with zymosan beads (red) as for panel B. Bar, 5 μm. (G) Quantification of the PH(AKT) fluorescence intensity in the course of phagocytic cup formation in cells treated as for panel F. Note that only a small fraction of the beads recognized by Sptlc2<sup>−/−</sup> mutant cells were internalized, in contrast to wild-type cells. Therefore, for Sptlc2<sup>−/−</sup> mutant cells, the normalized fluorescence intensity was determined at the particle contact site at time points equivalent to those at which particle recognition, engulfment, and internalization took place in wild-type cells. The GFP signal at the particle contact site was quantified and divided by the value for the GFP threshold of the whole cell at the corresponding time point. Average intensity profiles of 5 to 10 cells taken from two independent biological experiments are shown. Data are means ± SD. *, P &lt; 0.05; **, P &lt; 0.01; ns, not significant (two-tailed unpaired t test).
</figcaption>
</figure>
</div>
</div>
<p>To determine whether blocking sphingolipid biosynthesis perturbs phagocytic signaling events downstream of pathogen receptor activation, we first focused on the activation and localization of the Rho GTPases Rac1 and Cdc42 in zymosan A-treated wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells. Visualization of activated Rho GTPases was enabled by transfecting cells with a fluorescent biosensor comprising yellow fluorescent protein (YFP) fused to the p21-binding domain of the p21-activated kinase PAK [PBD(PAK)-YFP]<span class="citation" data-cites="schlamPhosphoinositide3kinaseEnables2015 hoppeCdc42Rac1Rac22004a"><sup><a href="../references.html#ref-schlamPhosphoinositide3kinaseEnables2015" role="doc-biblioref">36</a>,<a href="../references.html#ref-hoppeCdc42Rac1Rac22004a" role="doc-biblioref">38</a></sup></span>. At 24 h post-transfection, cells were incubated with zymosan A beads at an MOI of 10 and subjected to live-cell imaging. In wild-type cells, we observed a sharp increase in PBD(PAK)-YFP fluorescence at the particle contact site. Fluorescence levels peaked during particle engulfment, corresponding to the expected pattern of pathogen-induced Rac1/Cdc42 activation (<a href="#fig-mBioFig5" class="quarto-xref">Figure&nbsp;<span>3.5</span></a>B and C; <a href="#vid-Mtb_phagocytosis_Movie5" class="quarto-xref">Supplementary Movie&nbsp;<span>3.5</span></a>). In Sptlc2<sup>−/−</sup> cells, we also observed an increase in PBD(PAK)-YFP fluorescence upon particle recognition. However, the PBD(PAK)-YFP fluorescence intensity failed to reach the levels detected in zymosan A-treated wild-type cells and faded out prior to a successful particle internalization, suggesting that Sptlc2<sup>−/−</sup> cells are impaired in particle-induced activation of Rho GTPases (<a href="#fig-mBioFig5" class="quarto-xref">Figure&nbsp;<span>3.5</span></a>B and C; <a href="#vid-Mtb_phagocytosis_Movie5" class="quarto-xref">Supplementary Movie&nbsp;<span>3.5</span></a>).</p>
</section>
<section id="phosphoinositide-turnover-is-impaired-in-sptlc2-cells." class="level3" data-number="3.4.7">
<h3 data-number="3.4.7" class="anchored" data-anchor-id="phosphoinositide-turnover-is-impaired-in-sptlc2-cells."><span class="header-section-number">3.4.7</span> Phosphoinositide turnover is impaired in Sptlc2<sup>−/−</sup> cells.</h3>
<p>As PI(4,5)P2 and PI(3,4,5)P3 are key regulators of F-actin dynamics during phagocytosis (<a href="#fig-mBioFig5" class="quarto-xref">Figure&nbsp;<span>3.5</span></a>A), we next monitored the spatial distribution of these phosphoinositides in zymosan A-treated wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells. To this end, cells were first transfected with a GFP fusion of the pleckstrin homology (PH) domain of the phospholipase Cδ [PH(PLCδ)-GFP], which serves as a fluorescent biosensor for PI(4,5)P2<span class="citation" data-cites="levinPhosphoinositidesPhagocytosisMacropinocytosis2015 marshallRestrictedAccumulationPhosphatidylinositol2001"><sup><a href="../references.html#ref-levinPhosphoinositidesPhagocytosisMacropinocytosis2015" role="doc-biblioref">4</a>,<a href="../references.html#ref-marshallRestrictedAccumulationPhosphatidylinositol2001" role="doc-biblioref">39</a></sup></span>. In zymosan A-treated wild-type cells, PH(PLCδ)-GFP was readily concentrated at the particle contact site and preferentially accumulated at the leading edges of the nascent phagosome during particle engulfment (<a href="#fig-mBioFig5" class="quarto-xref">Figure&nbsp;<span>3.5</span></a>D and E; <a href="#vid-Mtb_phagocytosis_Movie6" class="quarto-xref">Supplementary Movie&nbsp;<span>3.6</span></a>). In Sptlc2<sup>−/−</sup> cells, however, PH(PLCδ)-GFP was readily recruited to the contact site but remained associated with the base of the phagocytic cup, analogous to what we observed with the LifeAct-GFP probe (<a href="#fig-mBioFig3" class="quarto-xref">Figure&nbsp;<span>3.3</span></a>C). This suggests that PI(4,5)P2 synthesis is initiated upon zymosan A binding but that PI(4,5)P2 turnover at the base of the phagocytic synapse is impaired in Sptlc2<sup>−/−</sup> cells (<a href="#fig-mBioFig5" class="quarto-xref">Figure&nbsp;<span>3.5</span></a>D and E).</p>
<p>To check whether a block in sphingolipid biosynthesis affects the turnover of PI(4,5)P2 into PI(3,4,5)P3 in zymosan-A-treated cells, wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells were transfected with a GFP fusion of the PH domain of Akt (PH(AKT)-GFP), which serves as a fluorescent biosensor for PI(3,4,5)P3<span class="citation" data-cites="levinPhosphoinositidesPhagocytosisMacropinocytosis2015 marshallRestrictedAccumulationPhosphatidylinositol2001"><sup><a href="../references.html#ref-levinPhosphoinositidesPhagocytosisMacropinocytosis2015" role="doc-biblioref">4</a>,<a href="../references.html#ref-marshallRestrictedAccumulationPhosphatidylinositol2001" role="doc-biblioref">39</a></sup></span>. In wild-type cells, PH(AKT)-GFP was readily recruited to the particle contact site, and its membrane association increased sharply during particle engulfment (<a href="#fig-mBioFig5" class="quarto-xref">Figure&nbsp;<span>3.5</span></a>F and G, upper panels; <a href="#vid-Mtb_phagocytosis_Movie7" class="quarto-xref">Supplementary Movie&nbsp;<span>3.7</span></a>). In contrast, Sptlc2<sup>−/−</sup> cells almost completely failed to mobilize PH(AKT)-GFP at the particle contact site (<a href="#fig-mBioFig5" class="quarto-xref">Figure&nbsp;<span>3.5</span></a>F and G, lower panels; <a href="#vid-Mtb_phagocytosis_Movie7" class="quarto-xref">Supplementary Movie&nbsp;<span>3.7</span></a>). Together, these results indicate that blocking sphingolipid biosynthesis disrupts phosphoinositide metabolism at phagocytic synapses in zymosan A-treated macrophages, notably at the level of PI(4,5)P2-to-PI(3,4,5)P3 conversion, a step known to be critical for the internalization of pathogenic bacteria by professional phagocytes<span class="citation" data-cites="agramonte-heviaGramnegativeBacteriaPhagocytic2002"><sup><a href="../references.html#ref-agramonte-heviaGramnegativeBacteriaPhagocytic2002" role="doc-biblioref">40</a></sup></span>.</p>
</section>
<section id="production-of-sphingomyelin-not-glycosphingolipids-is-critical-for-efficient-mtb-uptake." class="level3" data-number="3.4.8">
<h3 data-number="3.4.8" class="anchored" data-anchor-id="production-of-sphingomyelin-not-glycosphingolipids-is-critical-for-efficient-mtb-uptake."><span class="header-section-number">3.4.8</span> Production of sphingomyelin, not glycosphingolipids, is critical for efficient <em>Mtb</em> uptake.</h3>
<p>Both SM and glycosphingolipids are highly enriched in the exoplasmic leaflet of the plasma membrane of mammalian cells. Therefore, we next addressed whether efficient phagocytic uptake of <em>Mtb</em> relies on the production of SM, glycosphingolipids, or both. Bulk production of SM is mediated by the enzyme sphingomyelin synthase 1 (SGMS1) in the trans-Golgi, whereas glycosphingolipid production is strictly dependent on the enzyme UDP-glucose ceramide glycosyltransferase (UGCG) in the cis-Golgi<span class="citation" data-cites="gaultOverviewSphingolipidMetabolism2010a wegnerUDPglucoseCeramideGlycosyltransferase2018"><sup><a href="../references.html#ref-gaultOverviewSphingolipidMetabolism2010a" role="doc-biblioref">23</a>,<a href="../references.html#ref-wegnerUDPglucoseCeramideGlycosyltransferase2018" role="doc-biblioref">41</a></sup></span>. We used CRISPR (clustered regularly interspaced palindromic repeats)/Cas9 gene editing to ablate UGCG expression in THP-1 cells (<a href="#suppfig-mBioFigS3" class="quarto-xref">Figure S<span>3.3</span></a>G), and the lab of S. V. Winter kindly provided us with SGMS1 knockout U937 cells<span class="citation" data-cites="virreirawinterGenomewideCRISPRScreen2016"><sup><a href="../references.html#ref-virreirawinterGenomewideCRISPRScreen2016" role="doc-biblioref">42</a></sup></span>. Strikingly, SGMS1 knockout resulted in a major (∼60%) reduction in the phagocytic uptake of mCherry-expressing <em>Mtb</em> by U937 cells (<a href="#fig-mBioFig6" class="quarto-xref">Figure&nbsp;<span>3.6</span></a>A and B), similar to what we observed in myriocin- or FB1-treated U937 cells (<a href="#fig-mBioFig2" class="quarto-xref">Figure&nbsp;<span>3.2</span></a>B and C; <a href="#suppfig-mBioFigS2" class="quarto-xref">Figure S<span>3.2</span></a>A to D). In contrast, genetic ablation of UGCG had no impact on <em>Mtb</em> uptake by THP-1 cells (<a href="#fig-mBioFig6" class="quarto-xref">Figure&nbsp;<span>3.6</span></a>C and D), even though phagocytosis of <em>Mtb</em> in THP-1 cells can be readily inhibited by myriocin and FB1 (<a href="#fig-mBioFig2" class="quarto-xref">Figure&nbsp;<span>3.2</span></a>B and C; <a href="#suppfig-mBioFigS2" class="quarto-xref">Figure S<span>3.2</span></a>C and D). Together, these data indicate that efficient uptake of <em>Mtb</em> by professional phagocytes is critically dependent on an intact SM biosynthetic pathway, whereas the production of glycosphingolipids is dispensable for this process.</p>
<div class="border">
<div id="fig-mBioFig6" class="quarto-figure quarto-figure-center quarto-float anchored" data-group="results_PhagocytosisGallery">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mBioFig6-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_Mtb_phagocytosis_files/TrueHighResImages/mBio03141-20_Figure_6.png" class="lightbox" data-gallery="results_PhagocytosisGallery" data-glightbox="description: .lightbox-desc-6"><img src="../../parts/results/results_Mtb_phagocytosis_files/TrueHighResImages/mBio03141-20_Figure_6.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-mBioFig6-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.6: <strong>Production of sphingomyelin, not glycosphingolipids, is critical for efficient <em>Mtb</em> uptake.</strong> (A) Wild-type and two independent SGMS1<sup>−/−</sup> knockout clones of U937 cells were infected with mCherry-<em>Mtb</em> (red) for 2 h, stained with phalloidin Alexa Fluor 488 (green), and then imaged by confocal fluorescence microscopy. (B) Quantification of <em>Mtb</em> uptake by cells treated as for panel A. (C) Wild-type and two independent UGCG<sup>−/−</sup> knockout clones of THP-1 cells were infected with mCherry-expressing <em>Mtb</em> (red) for 2 h, stained with phalloidin Alexa Fluor 488 (green), and then imaged by confocal fluorescence microscopy. Bar, 5 μm. (D) Quantification of <em>Mtb</em> uptake by cells treated as for panel C. U937 and THP-1 cells were activated with 50 nM PMA for 24 h before <em>Mtb</em> infection. Three independent experiments were performed, each in triplicate with at least 1,000 cells quantified per replicate. Data are means ± SD. ***, P &lt; 0.001; ns, not significant (two-tailed unpaired t test).
</figcaption>
</figure>
</div>
</div>
</section>
</section>
<section id="discussion" class="level2" data-number="3.5">
<h2 data-number="3.5" class="anchored" data-anchor-id="discussion"><span class="header-section-number">3.5</span> Discussion</h2>
<p>In this study, we report that the invasion of phagocytes by <em>Mtb</em> is critically dependent on an intact sphingolipid biosynthetic pathway. Blocking sphingolipid production does not impair clathrin-mediated endocytosis or macropinocytosis of herpes simplex virus, indicating that phagocytes remain susceptible to infection by pathogens that utilize other routes of cellular uptake. Strikingly, sphingolipid-deficient macrophages display normal pathogen-induced clustering of phagocytic receptors but fail to fully exclude the regulatory phosphatase CD45 from the phagocytic synapse. We also show that key steps in the downstream signaling cascade, notably Rho GTPase activation and phosphoinositide turnover, are significantly impaired in the mutant cells, leading to perturbations in actin dynamics and defective phagosome formation. Blocking production of SM, not glycosphingolipids, suffices to disrupt phagocytic uptake of <em>Mtb</em>. Collectively, our findings disclose a vital role of SM in an early step of <em>Mtb</em> invasion and establish a novel molecular framework for the underlying cellular process.</p>
<p>A major finding of this work is that macrophages defective in sphingolipid biosynthesis exhibit an impaired segregation of CD45 from pathogen-induced clusters of dectin-1 at the phagocytic cup. Dectin-1 belongs to a family of phagocytic cell surface receptors that participate in the internalization of pathogens, including <em>Mtb</em><span class="citation" data-cites="goodridgeActivationInnateImmune2011a rothfuchsDectin1InteractionMycobacterium2007a hossainPatternRecognitionReceptors2013"><sup><a href="../references.html#ref-goodridgeActivationInnateImmune2011a" role="doc-biblioref">7</a>,<a href="../references.html#ref-rothfuchsDectin1InteractionMycobacterium2007a" role="doc-biblioref">33</a>,<a href="../references.html#ref-hossainPatternRecognitionReceptors2013" role="doc-biblioref">43</a></sup></span> and <em>M. marinum</em> (this study). Upon pathogen engagement, these receptors cluster in the lateral plane of the membrane, bringing their cytosolic domains in close apposition<span class="citation" data-cites="jaumouilleReceptorMobilityCytoskeleton2011"><sup><a href="../references.html#ref-jaumouilleReceptorMobilityCytoskeleton2011" role="doc-biblioref">44</a></sup></span>. The cytosolic domain of dectin-1 contains a hemi-immunoreceptor tyrosine-based activation motif (hemITAM), which becomes phosphorylated by Src and Syk family kinases upon receptor coalescence<span class="citation" data-cites="goodridgeActivationInnateImmune2011a kerriganSykcoupledCtypeLectin2010"><sup><a href="../references.html#ref-goodridgeActivationInnateImmune2011a" role="doc-biblioref">7</a>,<a href="../references.html#ref-kerriganSykcoupledCtypeLectin2010" role="doc-biblioref">45</a></sup></span>. Importantly, segregation of the regulatory tyrosine phosphatase CD45 from the site of dectin-1 clustering is a crucial step in the amplification of a downstream phosphorylation cascade<span class="citation" data-cites="goodridgeActivationInnateImmune2011a"><sup><a href="../references.html#ref-goodridgeActivationInnateImmune2011a" role="doc-biblioref">7</a></sup></span>, which eventually culminates in the recruitment of phosphoinositide-modifying enzymes, including the phosphoinositide 3-kinase (PI3K)<span class="citation" data-cites="levinPhosphoinositidesPhagocytosisMacropinocytosis2015 zhuStructurallyDistinctPhosphatases2008"><sup><a href="../references.html#ref-levinPhosphoinositidesPhagocytosisMacropinocytosis2015" role="doc-biblioref">4</a>,<a href="../references.html#ref-zhuStructurallyDistinctPhosphatases2008" role="doc-biblioref">46</a></sup></span>. Thus, our finding that Sptlc2<sup>−/−</sup> macrophages fail to generate PI(3,4,5)P3 at the phagocytic cup might be due to an impaired phagocyte receptor signaling caused by a defective segregation of CD45. This is consistent with our immunoblot data, which show that sphingolipids are essential for phosphorylation of PI3K during phagocytosis. In particular, we found that blocking the de novo sphingolipid biosynthesis using myriocin in THP-1 cells resulted in a 3.5-fold reduction in phosphorylation of PI3K compared to that in control cells (P &lt; 0.01; n = 3). PI3K activation and formation of PI(3,4,5)P3 at the pathogen engagement site are necessary for late stages of pseudopodium progression, the clearance of actin at the base of the phagocytic cup, and phagosome closure<span class="citation" data-cites="levinPhosphoinositidesPhagocytosisMacropinocytosis2015"><sup><a href="../references.html#ref-levinPhosphoinositidesPhagocytosisMacropinocytosis2015" role="doc-biblioref">4</a></sup></span>. Indeed, time-lapse microscopy revealed that these processes are all significantly compromised in Sptlc2<sup>−/−</sup> macrophages. Collectively, our data suggest that blocking sphingolipid biosynthesis affects the phagocytic uptake of <em>Mtb</em> by precluding a proper activation of phagocyte receptors during pathogen engagement.</p>
<p>Another key observation is that production of SM rather than sphingolipid biosynthesis per se is critical for efficient phagocytic uptake of <em>Mtb</em>. SM is an abundant sphingolipid in the exoplasmic leaflet of the plasma membrane in mammalian cells. Owing to its saturated nature and hydrogen-bonding capacity, SM enables the formation of cholesterol-enriched microdomains<span class="citation" data-cites="slotteBiologicalFunctionsSphingomyelins2013"><sup><a href="../references.html#ref-slotteBiologicalFunctionsSphingomyelins2013" role="doc-biblioref">12</a></sup></span>. In this capacity, SM may play a critical role in the lateral organization and activation of signaling complexes that promote <em>Mtb</em> invasion, consistent with prior reports indicating that cholesterol is essential for <em>Mtb</em> entry into macrophages<span class="citation" data-cites="gatfieldEssentialRoleCholesterol2000a"><sup><a href="../references.html#ref-gatfieldEssentialRoleCholesterol2000a" role="doc-biblioref">19</a></sup></span>. However, an alternative scenario is that the phagocytic uptake of <em>Mtb</em> relies on the local generation of the SM-derived metabolite ceramide. Previous work revealed that hydrolysis of cell surface SM by acid SMase to liberate ceramide mediates the invasion of Gram-negative pathogens like <em>Neisseria gonorrhoeae</em> and <em>Pseudomonas aeruginosa</em> in mammalian cells<span class="citation" data-cites="grassmeAcidicSphingomyelinaseMediates1997a liAcidSphingomyelinaseCeramideSystem2019 smithUnexpectedRoleAcid2008"><sup><a href="../references.html#ref-grassmeAcidicSphingomyelinaseMediates1997a" role="doc-biblioref">15</a>,<a href="../references.html#ref-liAcidSphingomyelinaseCeramideSystem2019" role="doc-biblioref">47</a>,<a href="../references.html#ref-smithUnexpectedRoleAcid2008" role="doc-biblioref">48</a></sup></span>. Activation of acid SMase at the pathogen contact site appears to trigger local assembly of ceramide-enriched signaling platforms that initiate internalization<span class="citation" data-cites="grassmeHostDefensePseudomonas2003"><sup><a href="../references.html#ref-grassmeHostDefensePseudomonas2003" role="doc-biblioref">16</a></sup></span>. While SMases and ceramides have also been implicated in mycobacterial infection<span class="citation" data-cites="speerSurfaceHydrolysisSphingomyelin2015 viswanathanMacrophageSphingolipidsAre2018 wuFunctionSphingomyelinasesMycobacterial2018"><sup><a href="../references.html#ref-speerSurfaceHydrolysisSphingomyelin2015" role="doc-biblioref">49</a>–<a href="../references.html#ref-wuFunctionSphingomyelinasesMycobacterial2018" role="doc-biblioref">51</a></sup></span>, their role in the phagocytic uptake of <em>Mtb</em> remains to be established.</p>
<p>We previously reported that blocking sphingolipid biosynthesis in dendritic DC2.4 cells impairs phagocytosis of Candida albicans by preventing the assembly of F-actin at the particle contact site<span class="citation" data-cites="tafesseDisruptionSphingolipidBiosynthesis2015a"><sup><a href="../references.html#ref-tafesseDisruptionSphingolipidBiosynthesis2015a" role="doc-biblioref">17</a></sup></span>. Moreover, the sphingolipid-deficient cells were unable to bind fungal particles and displayed a reduced number of dectin-1 receptors on their surface in comparison to controls. Here, we found that in sphingolipid-deficient RAW264.7 macrophages, which have normal dectin-1 levels, F-actin readily assembles at the particle contact site but fails to undergo subsequent remodeling to complete engulfment of the particle. Thus, sphingolipids in macrophages are required at a step in the phagocytic pathway downstream of particle recognition and receptor clustering. In dendritic cells, this requirement would be masked by a critical role of sphingolipids in maintaining an adequate number of dectin-1 receptors on their surface to enable actin polymerization at the particle recognition site. Why dectin-1 levels in macrophages are not affected by a loss of sphingolipids remains to be established. However, it is well known that the expression profiles of pattern recognition receptors vary considerably among different types of immune cells<span class="citation" data-cites="erkelensIntestinalMacrophagesBalance2020a"><sup><a href="../references.html#ref-erkelensIntestinalMacrophagesBalance2020a" role="doc-biblioref">52</a></sup></span>, and that macrophages expose a higher number of dectin-1 receptors on their surface than dendritic cells<span class="citation" data-cites="taylorBetaglucanReceptorDectin12002 garfootEng1Exg8Are2017"><sup><a href="../references.html#ref-taylorBetaglucanReceptorDectin12002" role="doc-biblioref">53</a>,<a href="../references.html#ref-garfootEng1Exg8Are2017" role="doc-biblioref">54</a></sup></span>. As these cell types likely also have different lipid compositions, it is conceivable that a block in sphingolipid biosynthesis differentially affects dectin-1 trafficking and/or turnover.</p>
<p>A 2-fold reduction in phagocytic uptake of <em>Mtb</em> caused by a block in sphingolipid biosynthesis reported in here appears rather modest. However, it should be noted that both Sptlc2<sup>−/−</sup> mutant and myriocin-treated phagocytes still contain substantial amounts of SM, i.e., 30 to 40% of the amount found in control cells. As Sptlc2<sup>−/−</sup> mutant cells rely on serum-supplemented medium for growth (our unpublished observation), the residual amount of SM in these cells is likely derived from the serum. Consequently, our data may underestimate the actual functional relevance of SM in the phagocytic uptake of <em>Mtb</em>. While the therapeutic value of blocking sphingolipid production in the treatment of TB remains to be established, an ongoing clinical trial investigating the use of statins (a family of cholesterol-depleting drugs) as a synergistic co-therapeutic among a cocktail of frontline anti-tubercular agents has yielded encouraging results<span class="citation" data-cites="liAcidSphingomyelinaseCeramideSystem2019"><sup><a href="../references.html#ref-liAcidSphingomyelinaseCeramideSystem2019" role="doc-biblioref">47</a></sup></span>.</p>
<p>In sum, we show that <em>Mtb</em> relies on an intact SM biosynthetic pathway to efficiently invade professional phagocytes. While the latter process is critical for the clearance of pathogens, <em>Mtb</em> harnesses phagocytosis to gain entry into host immune cells and establish long-term survival. By offering fresh insights into the underlying mechanisms, our work provides a basis for the development of new strategies to fight this ancient pathogen.</p>
</section>
<section id="materials-and-methods" class="level2" data-number="3.6">
<h2 data-number="3.6" class="anchored" data-anchor-id="materials-and-methods"><span class="header-section-number">3.6</span> Materials and Methods</h2>
<section id="reagents" class="level3" data-number="3.6.1">
<h3 data-number="3.6.1" class="anchored" data-anchor-id="reagents"><span class="header-section-number">3.6.1</span> Reagents</h3>
<p>Myriocin and FB1 were purchased from Cayman Chemical Company. 3-l-[14C]serine was purchased from American Radioactivity. Phorbol 12-myristate 13-acetate (PMA) was purchased from BioLegend. Paraformaldehyde was purchased from Fisher Scientific. DyLight phalloidin 488 was purchased from Cell Signaling Technology, and Alexa Fluor 594- and 647-conjugated zymosan A (<em>Saccharomyces cerevisiae</em>) BioParticles were purchased from Thermo Fisher. Mouse monoclonal antibody against GAPDH (ab8245) was purchased from Abcam. Anti-rabbit IgG horseradish peroxidase (HRP)-linked antibody and anti-mouse IgG HRP-linked antibody were purchased from Cell Signaling. APC-conjugated rat monoclonal antibody against CD45 antibody (clone 30-F11) was purchased from BioLegend. Zymosan A BioParticles opsonization reagent was purchased from Invitrogen (no. Z2850). Cy3 AffiniPure donkey anti-rabbit IgG (heavy plus light chain [H+L]) was purchased from JIR (711-165-15). FITC-conjugated rat monoclonal antibody against dectin-1 (clone 2A11) was purchased from AbD Serotec. A Zombie Aqua fixable viability kit (no. 77143) was purchased from BioLegend.</p>
</section>
<section id="plasmids" class="level3" data-number="3.6.2">
<h3 data-number="3.6.2" class="anchored" data-anchor-id="plasmids"><span class="header-section-number">3.6.2</span> Plasmids</h3>
<p>The plasmids used for generation of lentivirus consists of the lentiviral vector (pCDH-sgRNA, pCW-Cas9 or pInducer20-LifeAct-mKate), the psPAX2 (packaging) and pMD2.G (envelope) plasmids. pCW-Cas9 plasmid was purchased from Addgene (no. 50661). For generation of pInducer20-LifeAct-mKate, DNA fragments encoding LifeAct-mKate (Addgene no. 54697) were created by PCR and inserted via KpnI and NotI sites into pENTR11 (Invitrogen). Using the Gateway cloning system (Thermo Fisher Scientific), the LifeAct-mKate fragment was transferred onto the lentiviral pInducer20 plasmid (Addgene no. 44012). pMXsIP-Dectin-1-GFP was kindly provided by the lab of Hidde Ploegh (Boston Children’s Hospital, Boston, MA) (8). Active (GTP-bound) Rac1/Cdc42 was detected with PAK(PBD)-YFP, a plasmid encoding the PBD of PAK fused to YFP (40) obtained from Addgene (no. 11407). PI(4,5)P2 was detected with PH(PLCδ)-GFP, a plasmid encoding the PH domain of PLCδ fused to GFP. PI(3,4,5)P3 levels were monitored by PH(AKT)-GFP, a plasmid encoding the PH domain of AKT fused to GFP<span class="citation" data-cites="fengRapidlyReversibleChemical2014a"><sup><a href="../references.html#ref-fengRapidlyReversibleChemical2014a" role="doc-biblioref">55</a></sup></span>. These two fusion constructs were kindly provided by the lab of Hidde Ploegh (Boston Children’s Hospital, Boston, MA).</p>
</section>
<section id="cell-culture-and-small-molecule-inhibitor-treatment" class="level3" data-number="3.6.3">
<h3 data-number="3.6.3" class="anchored" data-anchor-id="cell-culture-and-small-molecule-inhibitor-treatment"><span class="header-section-number">3.6.3</span> Cell culture and small-molecule-inhibitor treatment</h3>
<p>Human THP-1 monocytes (ATCC catalog no. TIB-22), human U937 monocytes (ATCC catalog no. CRL-1593.2), and mouse dendritic DC2.4 cells (Millipore catalog no. SCC142) were routinely cultured in RPMI medium supplemented with 10% FBS (Seradigm) and 1% penicillin-streptomycin (Pen/Strep) (Gibco). Mouse RAW264.7 macrophages (ATCC catalog no. TIB-71) were cultured in Dulbecco’s modified Eagle medium (DMEM) supplemented with 10% fetal bovine serum (FBS) and 1% Pen/Strep. All cell lines were cultured at 37°C and 5% CO<sub>2</sub>. Adherent cells (RAW264.7 and DC2.4), were trypsinized using 0.25% trypsin solution (Gibco) and counted using a hemocytometer and trypan blue solution (Gibco) as a counterstain. THP-1 and U937 cells were differentiated to macrophages in RPMI medium supplemented with 10% FBS and 50 ng/ml PMA. After 24 h, cells were washed and treated with 5 μM myriocin or 5 μM FB1 for 3 days in RPMI supplemented with 10% FBS and 1% Pen/Strep. RAW264.7 and DC2.4 cells were treated similarly with myriocin and FB1; however, 24 h before infection, cells were trypsinized, counted, and seeded at an appropriate density in 96-well plates or on coverslips. Biological replicates represent independent treatments with chemical inhibitors and infections on separate days.</p>
</section>
<section id="metabolic-labeling-and-thin-layer-chromatography" class="level3" data-number="3.6.4">
<h3 data-number="3.6.4" class="anchored" data-anchor-id="metabolic-labeling-and-thin-layer-chromatography"><span class="header-section-number">3.6.4</span> Metabolic labeling and thin-layer chromatography</h3>
<p>A total of 5 × 10<sup>5</sup> cells were seeded into a 6-well plate. Cells were labeled with 1 μCi/ml of 3-l-[14C]-serine for 4 h in Opti-MEM supplemented with the appropriate inhibitors. Cells were then washed twice with phosphate-buffered saline (PBS), and lipid extraction was done following the method of Bligh and Dyer<span class="citation" data-cites="blighRAPIDMETHODTOTAL1959"><sup><a href="../references.html#ref-blighRAPIDMETHODTOTAL1959" role="doc-biblioref">56</a></sup></span>.The methanol/chloroform-lipid extracts were dried with nitrogen gas. Dried lipids were redissolved in several drops of chloroform/methanol (1:2, vol/vol) and loaded on a TLC plate. Lipids were separated by developing the TLC plate first in acetone and then in a mixture of chloroform, methanol, and 25% ammonia solution (50:25:6, vol/vol/vol). Radio-labeled lipids were detected on a Storm 825 Phosphor-Imager (GE Healthcare).</p>
</section>
<section id="lipodome-analysis" class="level3" data-number="3.6.5">
<h3 data-number="3.6.5" class="anchored" data-anchor-id="lipodome-analysis"><span class="header-section-number">3.6.5</span> Lipodome analysis</h3>
<p>Liquid chromatography-tandem mass spectrometry (LC-MS/MS) analysis was performed as described previously (59). Lipids were extracted from lysed RAW246.7 cells according to 50 μg of protein by chloroform/methanol extraction. Before the extraction, a standard mix containing sphingosine (d17:1), ceramide (d18:1/17), glycosyl(β) (C12 Cer), and sphingomyelin (17:0) was used to spike each sample for normalization and quantification. The dried lipid films were dissolved in a mixture of mobile phase A (60:40, water/acetonitrile, including 10 mM ammonium formate and 0.1% formic acid) and mobile phase B (88:10:2, 2-propanol/acetonitrile/H2O, including 2 mM ammonium formate and 0.02% formic acid) with a ratio of 65:35. High-performance liquid chromatography (HPLC) analysis was performed using a C30 reverse-phase column (Thermo Acclaim C30; 2.1 by 250 mm; 3 μm; operated at 50°C; Thermo Fisher Scientific) connected to an HP 1100 series HPLC system (Agilent) and a QExactivePLUS Orbitrap mass spectrometer (Thermo Fisher Scientific) equipped with a heated electrospray ionization (HESI) probe. The elution was performed with a gradient of 45 min; during 0 to 3 min, elution started with 40% B and increased to 100% in a linear gradient over 23 min; 100% B was maintained for 3 min. Afterward, solvent B was decreased to 40% and maintained for another 15 min for column re-equilibration. The flow rate was set to 0.1 ml/min. MS spectra of lipids were acquired in full-scan, data-dependent MS2 mode. The maximum injection time for full scans was 100 ms, with a target value of 3,000,000 at a resolution of 70,000 at m/z 200 and a mass range of 200 to 2,000 m/z in both positive and negative mode. The 10 most intense ions from the survey scan were selected and fragmented with high-energy collisional dissociation (HCD) with a normalized collision energy of 30. Target values for MS/MS were set at 100,000, with a maximum injection time of 50 ms at a resolution of 17,500 at m/z 200. To avoid repetitive sequencing, the dynamic exclusion of sequenced lipids was set at 10 s. Peaks were analyzed using the Lipid Search algorithm (MKI, Tokyo, Japan). Peaks were defined through raw files and product ion and precursor ion accurate masses. Candidate molecular species were identified by a database (&gt;1,000,000 entries) search of positive (+H+; +NH4+) ion adducts. Mass tolerance was set to 5 ppm for the precursor mass. Samples were aligned within a time window, and results were combined in a single report. From the intensities of lipid standards and lipid classes, absolute values for each lipid, in picomoles per milligram of protein, were calculated.</p>
</section>
<section id="generation-of-lentiviruses" class="level3" data-number="3.6.6">
<h3 data-number="3.6.6" class="anchored" data-anchor-id="generation-of-lentiviruses"><span class="header-section-number">3.6.6</span> Generation of lentiviruses</h3>
<p>Lentivirus production was performed as described previously<span class="citation" data-cites="tafesseGPR107GproteincoupledReceptor2014"><sup><a href="../references.html#ref-tafesseGPR107GproteincoupledReceptor2014" role="doc-biblioref">57</a></sup></span>. Briefly, lentiviruses were produced by co-transfection of HEK293T cells with lentiviral transfer vectors containing the gene of interest (i.e., pCDH-sgRNA, pCW-Cas9, and pInducer20-LifeAct-mKate) and the packaging plasmids psPAX2 and pMD2.G. Transfection was performed with Lipofectamine 3000 (Thermo Fisher) according to the manufacturer’s instructions. Cells were cultured in DMEM supplemented with 10% FBS, and the growth medium was replaced after 6 h. At 48 h after transfection, lentivirus-containing supernatants were harvested, centrifuged for 5 min at 1,250 rpm, and filtered through a 0.45-μm filter.</p>
</section>
<section id="generation-of-crisprcas9-mediated-knockout-cell-lines" class="level3" data-number="3.6.7">
<h3 data-number="3.6.7" class="anchored" data-anchor-id="generation-of-crisprcas9-mediated-knockout-cell-lines"><span class="header-section-number">3.6.7</span> Generation of CRISPR/Cas9-mediated knockout cell lines</h3>
<p>CRISPR/Cas9-mediated genome editing for genes involved in the sphingolipid pathway were performed as described previously<span class="citation" data-cites="tafesseDisruptionSphingolipidBiosynthesis2015a"><sup><a href="../references.html#ref-tafesseDisruptionSphingolipidBiosynthesis2015a" role="doc-biblioref">17</a></sup></span>. Briefly, to generate the Sptlc2<sup>−/−</sup> knockout, RAW264.7 cells were infected with lentivirus (pCW-Cas9) encoding Cas9 cDNA and were cultured in medium containing 2 μg/ml of puromycin (Sigma-Aldrich). Potential target sequences for CRISPR interference were found with the rules outlined by Mali et al.<span class="citation" data-cites="maliRNAguidedHumanGenome2013"><sup><a href="../references.html#ref-maliRNAguidedHumanGenome2013" role="doc-biblioref">58</a></sup></span>. The following two seed sequences (CRISPR target sequences) preceding PAM motifs found in the open reading frame of Sptlc2 gene were used: Sptlc2 #1, GAACGGCTGCGTCAAGAAC; Sptlc2 #2, AGCAGCACCGCCACCGTCG. The CRISPR target sequence for UGCG was GCTGTGGCTGATGCATTTCA. Potential off-target effects of the seed sequence were evaluated using the NCBI <em>Mus musculus</em> nucleotide BLAST. The CRISPR/Cas9-mediated Sptlc2-knockout RAW264.7 cell line and UGCG-knockout THP-1 cell line were generated as previously described<span class="citation" data-cites="tafesseGPR107GproteincoupledReceptor2014"><sup><a href="../references.html#ref-tafesseGPR107GproteincoupledReceptor2014" role="doc-biblioref">57</a></sup></span>. Briefly, CRISPR gBlock was designed to be cloned into the NheI/BamHI site of pCDH-CMV(−) (SBI; CD515B-1) as follows: cacagtcagacagtgactcaGTGTCACAgctagcTTTCCCATGATTCCTT CATATTTGCATATACGATACAAGGCTGTTAGAGAGATAATTAGAATTAAT TTGACTGTAAACACAAAGATATTAGTACAAAATACGTGACGTAGAAAGTA ATAATTTCTTGGGTAGTTTGCAGTTTTAAAATTATGTTTTAAAATGGACT ATCATATGCTTACCGTAACTTGAAAGTATTTCGATTTCTTGGCTTTATAT ATCTTGTGGAAAGGACGAAACACCGnnnnnnnnnnnnnnnnnnnGTTTTA GAGCTAGAAATAGCAAGTTAAAATAAGGCTAGTCCGTTATCAACTTGAAA AAGTGGCACCGAGTCGGTGCTTTTTTTggatccTGTGCACAgtcagtcac agtcagtctac (n, CRISPR target sequence; lowercase indicates extra nucleotides added to optimize synthesis or restriction enzyme sites). The gBlock was then digested using the restriction enzymes NheI and BamHI and ligated into the pCDH-CMV(−) vector, which had been linearized by digestion with the same restriction enzyme. The Cas9-inducible cells were infected with lentivirus carrying pCDH-CMV(−)-sgRNA and were cultured in medium containing 250 μg/ml of hygromycin B (Life Technology). To induce expression of Cas9, cells were treated with 1 μg/ml of doxycycline (Clontech) for 3 to 5 days. Clonal selection was performed by single-cell dilution on 96-well plates. The individual colonies were collected, and the expression of Sptlc2 was examined by immunoblotting using an anti-Sptlc2 antibody. U937 SGSM1<sup>-/−</sup> cells were kindly provided by Arturo Zychlinsky (Max Planck Institute of Biochemistry, Berlin, Germany), and the CRISPR/Cas9-mediated knockout cell lines were established as described by Virierra Winter et al.<span class="citation" data-cites="virreirawinterGenomewideCRISPRScreen2016"><sup><a href="../references.html#ref-virreirawinterGenomewideCRISPRScreen2016" role="doc-biblioref">42</a></sup></span>.</p>
</section>
<section id="culturing-of-mtb-and-m.-marinum" class="level3" data-number="3.6.8">
<h3 data-number="3.6.8" class="anchored" data-anchor-id="culturing-of-mtb-and-m.-marinum"><span class="header-section-number">3.6.8</span> Culturing of <em>Mtb</em> and <em>M. marinum</em></h3>
<p>Culturing and infection of <em>Mtb</em> were conducted in a biosafety level 3 laboratory following general safety guidelines. <em>Mtb</em> strain H37R, constitutively expressing mCherry, was obtained from the Fortune lab (Ragon Institute, Cambridge, MA). <em>Mtb</em> was cultured with shaking at 150 rpm at 37°C in 7H9 broth medium supplemented with 50 μg/ml hygromycin B. Culturing and infection of <em>M. marinum</em> were conducted in a biosafety level 2 laboratory following general safety guidelines. <em>M. marinum</em> constitutively expressing mCherry was cultured with shaking at 150 rpm in 7H9 medium supplemented with 10% oleic acid-albumin-dextrose-catalase (OADC), 0.05% Tween 80, 0.2% glycerol, and 50 μg/ml hygromycin B at 32°C. To minimize clumping of the bacteria, Erlenmeyer flasks containing 5 mm glass beads were used. For infections, cultures with an optical density at 600 nm (OD600) of 0.8 to 1.0 were used.</p>
</section>
<section id="mtb-and-m.-marinum-infection" class="level3" data-number="3.6.9">
<h3 data-number="3.6.9" class="anchored" data-anchor-id="mtb-and-m.-marinum-infection"><span class="header-section-number">3.6.9</span> <em>Mtb</em> and <em>M. marinum</em> infection</h3>
<p>For infection of THP-1 and U937 cells, the cells were seeded in SensoPlate 96-well glass-bottom plates (Greiner Bio-One) at a density of 1.5 × 10<sup>4</sup> cells per well in RPMI supplemented with 10% FBS, 1% Pen/Strep and 50 ng/ml PMA at 37°C. One day after differentiation, cells were treated for 3 days with 5 μM myriocin, 5 μM FB1, or the vehicle. For infection of RAW264.7 and DC2.4 cells, the cells were cultured in RPMI supplemented with 10% FBS and 1% Pen/Strep and treated for 3 days before infection with 5 μM myriocin, 5 μM FB1, or the vehicle. At 24 h before infection, RAW264.7 and DC2.4 cells were used to seed SensoPlate 96-well glass-bottom plates (Greiner Bio-One) at a density of 2 × 10<sup>4</sup> cells per well. Before infection, the mycobacteria were first passaged through a 5-μm filter (SLSV025LS; Merck) and then passaged 10 times through a 25-gauge needle to avoid clumping. All cell lines and conditions were then infected with a multiplicity of infection (MOI) of 10 in RPMI supplemented with 10% FBS for 2 h at 37°C. Myr, FB1, and vehicle were kept during infection. Cells were washed with PBS and then fixed with 4% paraformaldehyde (PFA) overnight at 4°C.</p>
</section>
<section id="zymosan-a-uptake-assay" class="level3" data-number="3.6.10">
<h3 data-number="3.6.10" class="anchored" data-anchor-id="zymosan-a-uptake-assay"><span class="header-section-number">3.6.10</span> Zymosan A uptake assay</h3>
<p>For infection of RAW264.7 cells, 2.0 × 10<sup>4</sup> cells per well were seeded in SensoPlate 96-well glass-bottom plates (Greiner Bio-One). After 24 h, the cells were incubated with zymosan A beads at an MOI of 10 in RPMI supplemented with 10% FBS for 1 h at 37°C. Cells were washed with PBS and then fixed with 4% PFA overnight at 4°C.</p>
</section>
<section id="phagocytosis-assay" class="level3" data-number="3.6.11">
<h3 data-number="3.6.11" class="anchored" data-anchor-id="phagocytosis-assay"><span class="header-section-number">3.6.11</span> Phagocytosis assay</h3>
<p>The phagocytosis assay was performed as previously described<span class="citation" data-cites="rashidfarrokhiVisualizingEarlyStages2017"><sup><a href="../references.html#ref-rashidfarrokhiVisualizingEarlyStages2017" role="doc-biblioref">59</a></sup></span>. Briefly, fixed cells were washed twice with PBS and incubated for 15 min in permeabilization buffer (0.1% Triton X-100 and 1% bovine serum albumin [BSA]). Cells were stained with phalloidin-Alexa Fluor 488 (Thermo Fisher) at a final concentration of 33 nM in permeabilization buffer for 60 min and washed once with PBS. After the phalloidin staining, cells were stained with DAPI for 10 min and washed 3 times with PBS. Cells were then imaged using a Keyence BZ-X700 microscope with a PlanFluor 20× objective. The parameters for imaging were kept the same for each sample. For image analysis, Keyence BZ-X Analyzer software was used. In brief, the outline of the cells was determined via the Alexa Fluor 488 signal (phalloidin staining), and the number and area of mCherry (<em>Mtb</em> and <em>M. marinum</em>) or Alexa Fluor 594 (zymosan A) signals in the Alexa Fluor 488 signal were detected by the program. In a second round, the number of cells was determined by counting the nuclei via DAPI staining. The number of bacteria or particles (mCherry or A594 signal) inside the cell was divided by the number of cells (DAPI signal) to identify the phagocytosis efficiency.</p>
</section>
<section id="herpes-simplex-virus-1-infection" class="level3" data-number="3.6.12">
<h3 data-number="3.6.12" class="anchored" data-anchor-id="herpes-simplex-virus-1-infection"><span class="header-section-number">3.6.12</span> Herpes simplex virus 1 infection</h3>
<p>HSV-1–mCherry reporter virus was kindly provided by the lab of David Johnson (Oregon Health and Science University, Portland, OR). Twenty-four hours prior to infection, 1 × 10<sup>4</sup> wild-type and Sptlc2 knockout RAW264.7 cells were seeded on glass coverslips in a 12-well plate. Cells were equally infected at an MOI of ∼1 and incubated for 12 h at 37°C and 5% CO<sub>2</sub>. Coverslips were fixed for 20 min in 4% paraformaldehyde, stained with DAPI, and then imaged on a Keyence BZ-X700 microscope. Infection rates were calculated by dividing the mCherry-expressing cells by the total number of cells (DAPI signal).</p>
</section>
<section id="binding-assay-zymosan-a-particles" class="level3" data-number="3.6.13">
<h3 data-number="3.6.13" class="anchored" data-anchor-id="binding-assay-zymosan-a-particles"><span class="header-section-number">3.6.13</span> Binding assay zymosan A particles</h3>
<p>RAW264.7 cells were seeded in SensoPlate 96-well glass-bottom plates at 1.5 × 10<sup>4</sup> cells/well 24 h prior to the experiment. Before the experiment, unlabeled zymosan A particles were incubated with zymosan A opsonizing reagent (rabbit polyclonal IgG; Invitrogen) for 1 h at 37°C. The particles were washed 3 times with PBS and resuspended in RPMI supplemented with 10% FBS. Cell growth medium was replaced with medium containing opsonized zymosan A particles at an MOI of 10. The 96-well plates were centrifuged at 1,000 rpm for 30 s. Cells were incubated at 37°C for 10 min. Next, cells were placed on ice, washed twice with chilled PBS, and stained for 15 min at 4°C for external zymosan A particles with anti-rabbit-Cy3 (JIR). Afterwards, cells were fixed with 4% PFA for 15 min and stained with phalloidin Alexa Fluor 488 for 1 h. The ratio of internalized (blue) beads to attached beads (red beads with clear phalloidin Alexa Fluor 488 signal) was quantified manually.</p>
</section>
<section id="real-time-imaging-of-phagocytosis" class="level3" data-number="3.6.14">
<h3 data-number="3.6.14" class="anchored" data-anchor-id="real-time-imaging-of-phagocytosis"><span class="header-section-number">3.6.14</span> Real time imaging of phagocytosis</h3>
<p>For transient transfection, RAW264.7 cells were seeded in SensoPlate 96-well glass-bottom plates at 1.5 × 10<sup>4</sup> cells/well 24 h prior to live imaging. Transfection was conducted with Lipofectamine 3000 according to manufacturer’s instructions 16 h prior to imaging. Alternatively, RAW 264.7 cells containing a doxycycline-inducible LifeAct-mKate construct were seeded in SensoPlate 96-well glass-bottom plates at 1.5 × 10<sup>4</sup> cells/well 24 h prior to transfection, and the medium was supplemented with 1 μg/ml doxycycline. For imaging, the growth medium was aspirated and replaced with RPMI without phenol red, supplemented with 30 mM HEPES and containing mCherry-expressing <em>M. marinum</em> at an MOI of 25 or zymosan A BioParticles with Alexa Fluor 595 at an MOI of 10. The 96-well plates were centrifuged at 1,000 rpm for 30 s. The cells were imaged with an SDC microscope (Nikon), and 3 or 4 different spots per well were imaged at the same time for 60 min with capture intervals of 15 s. The parameters for imaging were kept the same for each sample.</p>
</section>
<section id="live-imaging-of-dectin-1gfp" class="level3" data-number="3.6.15">
<h3 data-number="3.6.15" class="anchored" data-anchor-id="live-imaging-of-dectin-1gfp"><span class="header-section-number">3.6.15</span> Live imaging of dectin-1–GFP</h3>
<p>RAW264.7 cells were seeded in SensoPlate 96-well glass-bottom plates at 1.5 × 10<sup>4</sup> cells/well 24 h prior to live imaging, and transfection was conducted with Lipofectamine 3000 according to the manufacturer’s instructions 16 h prior to imaging. For imaging, the growth medium was aspirated and replaced with RPMI without phenol red, supplemented with 30 mM HEPES, and containing mCherry-expressing <em>M. marinum</em> at an MOI of 25. The 96-well plates were centrifuged at 1,000 rpm for 30 s. The cells were imaged with an SD microscope (Zeiss), and 3 or 4 different spots per well were imaged at the same time for 60 min with capture intervals of 15 s. The parameters for imaging were kept the same for each sample.</p>
</section>
<section id="live-image-analysis-quantification" class="level3" data-number="3.6.16">
<h3 data-number="3.6.16" class="anchored" data-anchor-id="live-image-analysis-quantification"><span class="header-section-number">3.6.16</span> Live image analysis (quantification)</h3>
<p>Of live cell images, individual images were selected for further analysis if they captured a cell bound to <em>M. marinum</em> or a zymosan A particle while maintaining a healthy morphology throughout the imaging time course. Imaging analysis was performed using FIJI software<span class="citation" data-cites="garfootEng1Exg8Are2017"><sup><a href="../references.html#ref-garfootEng1Exg8Are2017" role="doc-biblioref">54</a></sup></span>. The parameters for image processing were kept constant when different data sets were compared. For <a href="#fig-mBioFig3" class="quarto-xref">Figure&nbsp;<span>3.3</span></a>, all cells were included in the analysis that showed a clear recognition of the bacteria or particles, which was defined by an increase in F-actin signal around the bacterium or particle, and the number of successful versus failed internalizations was quantified. For <a href="#fig-mBioFig5" class="quarto-xref">Figure&nbsp;<span>3.5</span></a>, all cells were included in the analysis that showed a clear recognition of the zymosan A particles, which was defined by an increase in the corresponding reporter signal at the particle contact site. For quantification of <a href="#fig-mBioFig5" class="quarto-xref">Figure&nbsp;<span>3.5</span></a>, the area of the particle contact site was selected, cropped, and opened in a new window. The threshold for the particle contact site was set 1.5 times higher than the whole-cell threshold to exclusively detect an increase in fluorescence signal. The fluorescence intensity was measured at different time points by the analyze-measure function of FIJI. The values for the GFP threshold of the particle contact site area were divided by the values for the GFP threshold of the whole cell at the corresponding time points.</p>
</section>
<section id="dectin-1-and-cd45-displacement-assay" class="level3" data-number="3.6.17">
<h3 data-number="3.6.17" class="anchored" data-anchor-id="dectin-1-and-cd45-displacement-assay"><span class="header-section-number">3.6.17</span> Dectin-1 and CD45 displacement assay</h3>
<p>Raw 264.7 WT and Sptlc2<sup>−/−</sup> macrophages were seeded on glass coverslips in 24-well plates at a density of 50,000 cells per well and grown overnight. The next day, cells were transfected with dectin-1–GFP using Lipofectamine 3000 (Thermo Fisher) according to the manufacturer’s instructions. At 24 h post-transfection, cells were washed three times with ice-cold PBS, chilled on ice for 5 min, and inoculated with a chilled suspension of zymosan A beads (250 μg/ml in serum-free DMEM, prepared with repeated vortexing). After brief centrifugation (3 min at 250 × g and 4°C), plates were incubated at 37°C for 3 min, washed three times with ice-cold PBS, and fixed for 15 min with room-temperature 4% PFA. Cells were permeabilized for 30 min with 0.2% saponin in PBS, incubated for 1 h in blocking buffer (5% normal goat serum, 0.01% saponin in PBS), and stained for 1 h with anti-CD45–APC (BioLegend) diluted 1:100 in blocking buffer. Coverslips were mounted on glass slides with ProLong Glass anti-fade mountant (Thermo Fisher). Super-resolution microscopy was performed on a Zeiss LSM 880 confocal microscope with the Airyscan module using a 63×, 1.4 numerical aperture (NA) oil immersion objective. Airyscan processing was performed with ZEN Black (Zeiss) using the software’s preset parameters. Intensity profiles of dectin-1 and CD45 were created using the 2.5D visualization module. Measurement of CD45 exclusion from phagocytic cups was performed with Imaris 9.5 (Bitplane). All Airyscan images were converted to the .ims file format and subjected to background subtraction using the software’s default settings. The Surfaces module was then used to build three-dimensional (3D) models of phagocytic cups using absolute intensity of dectin-1 signal and a volume cutoff of 1 μm3, followed by masking so that all signals outside the cups were set to zero. All images were fed through this Surfaces pipeline, and statistics files were downloaded in the Microsoft Excel format. Percent CD45 exclusion was calculated for each cup as [1 − (number of CD45 voxels/sum of CD45 and dectin-1 voxels)] × 100.</p>
</section>
<section id="immunoblot-analysis" class="level3" data-number="3.6.18">
<h3 data-number="3.6.18" class="anchored" data-anchor-id="immunoblot-analysis"><span class="header-section-number">3.6.18</span> Immunoblot analysis</h3>
<p>About 1 × 10<sup>6</sup> THP-1 cells were seeded in a 6-well plate and treated for 3 days with 5 μM myriocin or left untreated. Cells were then infected with <em>Mtb</em> (H37Rv) at an MOI of 10. At 10 min post-infection, cells were washed with PBS, lysed, separated by SDS-PAGE, and transferred to a membrane. Levels of PI3K phosphorylation were detected by immunoblot analysis using an anti-phospho-PI3K antibody (p85; clone 19H8; Cell Signaling Technology). Immunostaining with an anti-GAPDH antibody (clone D16H11; Cell Signaling Technology) served as a loading control. Results were from three independent biological replicates and normalized with GAPDH signal. Quantification of the bands was performed using FIJI, and a Student’s t test was performed to show significance.</p>
</section>
<section id="flow-cytometry" class="level3" data-number="3.6.19">
<h3 data-number="3.6.19" class="anchored" data-anchor-id="flow-cytometry"><span class="header-section-number">3.6.19</span> Flow cytometry</h3>
<p>RAW264.7 wild-type and Sptlc2<sup>−/−</sup> cells were rinsed with PBS and lifted from the flask using treatment with trypsin. Fc receptor was blocked with CD16/32 antibody (1:100), and cells were stained with Zombie Aqua live/dead stain (1:200). Cells were then stained with either CD45-APC (1:200) or dectin-1–FITC (1:50) for 30 min. Finally, cells were fixed for 20 min in 4% paraformaldehyde and analyzed using an LSR-II flow cytometer. Cells independently stained with Zombie Aqua, CD45-APC, and dectin-1–FITC were used for compensation. Analysis was performed using FlowJo software.</p>
</section>
<section id="transferrin-uptake-analysis" class="level3" data-number="3.6.20">
<h3 data-number="3.6.20" class="anchored" data-anchor-id="transferrin-uptake-analysis"><span class="header-section-number">3.6.20</span> Transferrin uptake analysis</h3>
<p>Undifferentiated THP-1 cells were counted and plated in a SensoPlate 96-well glass-bottom plates (Greiner Bio-One) at a density of 2.5 × 10<sup>4</sup> in RPMI supplemented with 10% FCS, 1% PS, and 50 ng/ml PMA. Cells were allowed to differentiate at 37°C and 5% CO<sub>2</sub> for 3 days before being treated with 5 μM myriocin for another 3 days (or left untreated). These cells were then starved for 30 min in serum-free RPMI and treated with 100 nM Alexa Fluor 555-conjugated human transferrin (T35352; Thermo Fisher Scientific) resuspended in serum-supplemented RPMI. After a 30-min uptake, cells were washed once with PBS at pH 7.0 and once with PBS acidified to pH 5.0 (to release surface-bound transferrin) and fixed for 1 h at 4°C using 4% paraformaldehyde. Cells were stained with both DyLight 488-conjugated phalloidin (12935S; Cell Signaling Technology) and DAPI according to the manufacturer’s specifications. Uptake comparison between untreated and myriocin treated cells was quantified using high-content imaging collected on a Keyence BZ-X700 microscope using a Super Fluor 40× oil immersion objective. Using Keyence BZ-X Analyzer software, cell interior was demarcated using the phalloidin signal, nuclear area was assessed using the 405 signal, and transferrin internalization was assessed using the 555 signal. Data are reported as the integrated 555 signal per cell, normalized by the 405-positive nuclear area, with three independent biological experiments and four technical replicates each. A competition assay was performed in the presence of additional non-fluorescent human transferrin (T8158-100MG; Sigma-Aldrich) and quantified with a CLARIOstar Plus fluorimeter (BMG Labtech). Fluorescence emission was collected using a spiral detection pattern averaging across 200 reads per well. Resultant data are reported as the raw fluorescence intensities, in arbitrary fluorescent units.</p>
</section>
</section>
<section id="acknowledgements" class="level2" data-number="3.7">
<h2 data-number="3.7" class="anchored" data-anchor-id="acknowledgements"><span class="header-section-number">3.7</span> Acknowledgements</h2>
<p>We thank Arturo Zychlinsky (Max Planck Institute of Biochemistry, Berlin, Germany), and David Johnson (Oregon Health and Science University, Portland, OR) for kindly providing reagents, cell lines, viruses, and plasmid constructs essential to the completion of this study. We also thank the members of the OHSU Advanced Light Microscopy Core and the OHSU Flow Cytometry Shared Resource for assistance and training in data collection and analysis; in particular, we thank Stefanie Kaech Petrie, Matthew Lewis, and Sara Christensen. Finally, we thank Madeleine Faucher, Marie Foss, and Jossef Osborn who kindly and patiently assisted in the editing and preparation of this work.</p>
<p>This work was supported by the Medical Research Foundation and National Institutes of Health (3R21AI124225-01A1 and 1R01AI141549-01A1) to F.G.T., and the Deutsche Forschungsgemeinschaft (projects SFB944-P14 and HO3539/1-1) to J.C.M.H.</p>
<p>We have no conflicts of interest to declare.</p>
</section>
<section id="supplementary-materials" class="level2" data-number="3.8">
<h2 data-number="3.8" class="anchored" data-anchor-id="supplementary-materials"><span class="header-section-number">3.8</span> Supplementary materials</h2>
<div class="border">
<div id="suppfig-mBioFigS1" class="quarto-float" data-group="results_PhagocytosisGallery">
<figure class="quarto-float quarto-float-suppfig figure">
<div aria-describedby="suppfig-mBioFigS1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_Mtb_phagocytosis_files/TrueHighResImages/SuppFig1.png" class="lightbox" data-gallery="results_PhagocytosisGallery" data-glightbox="description: .lightbox-desc-7"><img src="../../parts/results/results_Mtb_phagocytosis_files/TrueHighResImages/SuppFig1.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-suppfig" id="suppfig-mBioFigS1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure S3.1: <strong>Blocking de novo sphingolipid biosynthesis using genetics and chemical tools has no significant impact on cell growth.</strong> Sptlc2<sup>−/−</sup> and wild-type DC2.4 or U937 cells were treated with myriocin or fumonisin B1 for 3 days and then metabolically labelled with 3-l-[14C]serine for 4 h. Total lipids were extracted and analyzed by TLC and autoradiography. Data are representative of 3 independent experiments. Veh, vehicle-treated. (C) Phase-contrast microscopy images of wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells. (D) Numbers of wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells plated out at equal densities and grown for 24 or 48 h in serum-containing medium. Data are means ± SD (n = 3).
</figcaption>
</figure>
</div>
</div>
<div class="border">
<div id="suppfig-mBioFigS2" class="quarto-float" data-group="results_PhagocytosisGallery">
<figure class="quarto-float quarto-float-suppfig figure">
<div aria-describedby="suppfig-mBioFigS2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_Mtb_phagocytosis_files/TrueHighResImages/SuppFig2.png" class="lightbox" data-gallery="results_PhagocytosisGallery" data-glightbox="description: .lightbox-desc-8"><img src="../../parts/results/results_Mtb_phagocytosis_files/TrueHighResImages/SuppFig2.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-suppfig" id="suppfig-mBioFigS2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure S3.2: <strong>Sphingolipid biosynthesis is required for efficient phagocytosis of <em>Mtb</em> but not for receptor-mediated uptake of transferrin.</strong> (A and C) RAW264.7, DC2.4, THP-1, and U937 cells were grown in the presence or absence of fumonisin B1 for 3 days, infected with mCherry-expressing <em>Mtb</em> for 2 h, washed, fixed, stained with phalloidin Alexa Fluor 488 and DAPI, and then imaged by fluorescence microscopy. (B and D) Quantification of <em>Mtb</em> uptake by cells treated as for panel A. (E and F) Sptlc2<sup>−/−</sup> cells bind but fail to internalize pathogen-mimicking beads; (E) Wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells were incubated with rabbit IgG-opsonized zymosan A particles for 5 min at 37°C. Next, cells were washed with chilled PBS and stained at 4°C with donkey anti-rabbit IgG-Cy3 in PBS supplemented with 1% BSA for 15 min to distinguish external beads (red) from internalized beads (blue). After chemical fixation, cells were stained for actin with phalloidin Alexa Fluor 488 (green) and analyzed by confocal fluorescence microscopy. (F) Quantification of the ratio of internalized (blue) over attached beads (red). Three independent experiments were performed, each in triplicate, with at least 100 cells quantified per replicate. (G) Differentiated THP-1 cells were incubated with 100 nM Alexa Fluor 555-conjugated human transferrin (Tf555) in the absence or presence of different concentrations of non-fluorescent transferrin for 30 min. Next, cells were washed in acidic buffer and fixed, and receptor-mediated uptake of Tf555 was quantified using a plate reader fluorimeter. (H) Quantification of receptor-mediated uptake of Tf555 in control or 3-day myriocin-treated THP-1 cells. Data are means ± SD (n = 3). **, P &lt; 0.01; *, P &lt; 0.05; ns, not significant (unpaired t test).
</figcaption>
</figure>
</div>
</div>
<div class="border">
<div id="suppfig-mBioFigS3" class="quarto-float" data-group="results_PhagocytosisGallery">
<figure class="quarto-float quarto-float-suppfig figure">
<div aria-describedby="suppfig-mBioFigS3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_Mtb_phagocytosis_files/TrueHighResImages/SuppFig3.png" class="lightbox" data-gallery="results_PhagocytosisGallery" data-glightbox="description: .lightbox-desc-9"><img src="../../parts/results/results_Mtb_phagocytosis_files/TrueHighResImages/SuppFig3.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-suppfig" id="suppfig-mBioFigS3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure S3.3: (A) WT RAW264.7 cells transfected with GFP-tagged dectin-1 (green) were infected with mCherry-expressing <em>M. marinum</em> (red) as for <a href="#fig-mBioFig4" class="quarto-xref">Figure&nbsp;<span>3.4</span></a>A. Bar, 5 μm. Data are means ± SD (n = 3). (B) No difference in sub-cellular localization of CD45 between WT and Sptlc2<sup>−/−</sup> cells. Wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells transfected with GFP-tagged dectin-1 (green) were incubated with zymosan A particles (blue) on ice, warmed to 37°C for 2 min to initiate phagocytosis, fixed, and then stained with antibodies against CD45 (purple). Boxed areas are magnified at right. Arrowheads mark sites of zymosan A binding. Bar, 5 μm. (C) WT and Sptlc2<sup>−/−</sup> RAW264.7 cells were stained with APC-conjugated anti-CD45 antibody, fixed, and subjected to flow cytometry analysis. (D) Quantitative analysis of median fluorescence intensity of cells treated as for panel C. (E) Wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells were stained with FITC-conjugated anti-dectin-1 antibody, fixed, and subjected to flow cytometry analysis. (F) Quantitative analysis of median fluorescence intensity of cells treated as for panel E. (G) Immunoblot analysis of total cell lysates collected from wild-type and two independent UGCG<sup>−/−</sup> KO clones of THP-1 cells using antibodies against UGCG and GAPDH.
</figcaption>
</figure>
</div>
</div>
<section id="supplementary-movies" class="level3" data-number="3.8.1">
<h3 data-number="3.8.1" class="anchored" data-anchor-id="supplementary-movies"><span class="header-section-number">3.8.1</span> Supplementary Movies</h3>
<p><br></p>
<div class="border">
<div id="vid-Mtb_phagocytosis_Movie1" class="quarto-float" data-group="Mtb_Phagocytosis_Movies">
<figure class="quarto-float quarto-float-vid figure">
<div aria-describedby="vid-Mtb_phagocytosis_Movie1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_Mtb_phagocytosis_files/mbio.03141-20-sm001.mp4" class="lightbox" data-gallery="Mtb_Phagocytosis_Movies" data-glightbox="description: .lightbox-desc-10"><video src="../../parts/results/results_Mtb_phagocytosis_files/mbio.03141-20-sm001.mp4" class="img-fluid" controls=""></video></a><a href="../../parts/results/results_Mtb_phagocytosis_files/mbio.03141-20-sm001.mp4">Video</a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-vid" id="vid-Mtb_phagocytosis_Movie1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Supplementary Movie&nbsp;3.1: Time-lapse confocal images of a wild-type (left) and Sptlc2<sup>-/-</sup> (right) RAW264.7 cell stably transduced with LifeAct-mKate (red) and infected with GFP-expressing <em>M. marinum</em> (green). Images were taken every 15 s. Bar, 5 μm.
</figcaption>
</figure>
</div>
</div>
<p><br></p>
<div class="border">
<div id="vid-Mtb_phagocytosis_Movie2" class="quarto-float" data-group="Mtb_Phagocytosis_Movies">
<figure class="quarto-float quarto-float-vid figure">
<div aria-describedby="vid-Mtb_phagocytosis_Movie2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_Mtb_phagocytosis_files/mbio.03141-20-sm002.mp4" class="lightbox" data-gallery="Mtb_Phagocytosis_Movies" data-glightbox="description: .lightbox-desc-11"><video src="../../parts/results/results_Mtb_phagocytosis_files/mbio.03141-20-sm002.mp4" width="400" height="300" controls=""></video></a><a href="../../parts/results/results_Mtb_phagocytosis_files/mbio.03141-20-sm002.mp4">Video</a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-vid" id="vid-Mtb_phagocytosis_Movie2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Supplementary Movie&nbsp;3.2: Time-lapse confocal images of a wild-type (left) and Sptlc2<sup>-/-</sup> (right) RAW264.7 cell stably transduced with LifeAct-mKate (red) and incubated with zymosan A particles (green). Images were taken every 15 s. Bar, 5 μm.
</figcaption>
</figure>
</div>
</div>
<p><br></p>
<div class="border">
<div id="vid-Mtb_phagocytosis_Movie3" class="quarto-float" data-group="Mtb_Phagocytosis_Movies">
<figure class="quarto-float quarto-float-vid figure">
<div aria-describedby="vid-Mtb_phagocytosis_Movie3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_Mtb_phagocytosis_files/mbio.03141-20-sm003.mp4" class="lightbox" data-gallery="Mtb_Phagocytosis_Movies" data-glightbox="description: .lightbox-desc-12"><video src="../../parts/results/results_Mtb_phagocytosis_files/mbio.03141-20-sm003.mp4" width="400" height="300" controls=""></video></a><a href="../../parts/results/results_Mtb_phagocytosis_files/mbio.03141-20-sm003.mp4">Video</a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-vid" id="vid-Mtb_phagocytosis_Movie3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Supplementary Movie&nbsp;3.3: Time-lapse confocal images of a wild-type RAW264.7 cell transfected with dectin-1-GFP (green) and infected with mCherry-expressing <em>M. marinum</em> (red). Images were taken every 15 seconds. Bar, 5 μm.
</figcaption>
</figure>
</div>
</div>
<p><br></p>
<div class="border">
<div id="vid-Mtb_phagocytosis_Movie4" class="quarto-float" data-group="Mtb_Phagocytosis_Movies">
<figure class="quarto-float quarto-float-vid figure">
<div aria-describedby="vid-Mtb_phagocytosis_Movie4-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_Mtb_phagocytosis_files/mbio.03141-20-sm004.mp4" class="lightbox" data-gallery="Mtb_Phagocytosis_Movies" data-glightbox="description: .lightbox-desc-13"><video src="../../parts/results/results_Mtb_phagocytosis_files/mbio.03141-20-sm004.mp4" width="400" height="300" controls=""></video></a><a href="../../parts/results/results_Mtb_phagocytosis_files/mbio.03141-20-sm004.mp4">Video</a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-vid" id="vid-Mtb_phagocytosis_Movie4-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Supplementary Movie&nbsp;3.4: Time-lapse confocal images of a wild-type RAW264.7 cell transfected with dectin-1-GFP (green) and infected with mCherry-expressing <em>M. marinum</em> (red). Images were taken every 15 s. The imaging is similar to that in <a href="#vid-Mtb_phagocytosis_Movie3" class="quarto-xref">Supplemental Movie&nbsp;<span>3.3</span></a> with a different field.
</figcaption>
</figure>
</div>
</div>
<p><br></p>
<div class="border">
<div id="vid-Mtb_phagocytosis_Movie5" class="quarto-float" data-group="Mtb_Phagocytosis_Movies">
<figure class="quarto-float quarto-float-vid figure">
<div aria-describedby="vid-Mtb_phagocytosis_Movie5-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_Mtb_phagocytosis_files/mbio.03141-20-sm005.mp4" class="lightbox" data-gallery="Mtb_Phagocytosis_Movies" data-glightbox="description: .lightbox-desc-14"><video src="../../parts/results/results_Mtb_phagocytosis_files/mbio.03141-20-sm005.mp4" width="400" height="300" controls=""></video></a><a href="../../parts/results/results_Mtb_phagocytosis_files/mbio.03141-20-sm005.mp4">Video</a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-vid" id="vid-Mtb_phagocytosis_Movie5-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Supplementary Movie&nbsp;3.5: Time-lapse confocal images of a Sptlc2<sup>-/-</sup> RAW264.7 cell transfected with PAK(PBD)-GFP (green) and incubated with zymosan beads (red). Images were taken every 15 s. Bar, 5 μm.
</figcaption>
</figure>
</div>
</div>
<p><br></p>
<div class="border">
<div id="vid-Mtb_phagocytosis_Movie6" class="quarto-float" data-group="Mtb_Phagocytosis_Movies">
<figure class="quarto-float quarto-float-vid figure">
<div aria-describedby="vid-Mtb_phagocytosis_Movie6-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_Mtb_phagocytosis_files/mbio.03141-20-sm006.mp4" class="lightbox" data-gallery="Mtb_Phagocytosis_Movies" data-glightbox="description: .lightbox-desc-15"><video src="../../parts/results/results_Mtb_phagocytosis_files/mbio.03141-20-sm006.mp4" width="400" height="300" controls=""></video></a><a href="../../parts/results/results_Mtb_phagocytosis_files/mbio.03141-20-sm006.mp4">Video</a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-vid" id="vid-Mtb_phagocytosis_Movie6-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Supplementary Movie&nbsp;3.6: Time-lapse confocal images of a Sptlc2<sup>-/-</sup> RAW264.7 cell transfected with PLCd-GFP (green) and incubated with zymosan beads (red). Images were taken every 15 s. Bar, 5 μm.
</figcaption>
</figure>
</div>
</div>
<p><br></p>
<div class="border">
<div id="vid-Mtb_phagocytosis_Movie7" class="quarto-float" data-group="Mtb_Phagocytosis_Movies">
<figure class="quarto-float quarto-float-vid figure">
<div aria-describedby="vid-Mtb_phagocytosis_Movie7-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_Mtb_phagocytosis_files/mbio.03141-20-sm007.mp4" class="lightbox" data-gallery="Mtb_Phagocytosis_Movies" data-glightbox="description: .lightbox-desc-16"><video src="../../parts/results/results_Mtb_phagocytosis_files/mbio.03141-20-sm007.mp4" width="400" height="300" controls=""></video></a><a href="../../parts/results/results_Mtb_phagocytosis_files/mbio.03141-20-sm007.mp4">Video</a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-vid" id="vid-Mtb_phagocytosis_Movie7-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Supplementary Movie&nbsp;3.7: Time-lapse confocal images of a Sptlc2<sup>-/-</sup> RAW264.7 cell transfected with PH(AKT)-GFP (green) and incubated with zymosan beads (red). Images were taken every 15 s.
</figcaption>
</figure>
</div>
</div>


<div class="hidden" aria-hidden="true">
<span class="glightbox-desc lightbox-desc-1">Figure&nbsp;3.1: <strong>Manipulation of cellular sphingolipid levels using genetics and chemical tools.</strong> (A) Schematic outline of the sphingolipid biosynthetic pathway. Inhibitors of sphingolipid biosynthetic enzymes used in this study are in red. Genes deleted using CRISPR/Cas9 are in blue. SPT, serine palmitoyltransferase; KDSR, 3-ketodihydrosphingosine reductase; CerS, ceramide synthase; DegS, δ4-desaturase; GlcCerS, glucosylceramide synthase; SMS, sphingomyelin synthase; FB1, fumonisin B1. (B) Phagocytic cell lines used in this study. (C) DC2.4 and RAW264.7 wild-type (WT) and Sptlc2<sup>−/−</sup> cells were processed for immunoblotting with antibodies against Sptlc2 and calnexin. (D) Sptlc2<sup>−/−</sup> and wild-type RAW264.7 or THP-1 cells were treated with myriocin or fumonisin B1 for 3 days and then metabolically labeled with 3-l-[14C]serine for 4 h. Total lipids were extracted and analyzed by thin-layer chromatography (TLC) and autoradiography. Veh, vehicle treated. (E and F) Quantification of [14C]SM and [14C]PS signals from TLCs shown in panel D. (G) Wild-type, Sptlc2<sup>−/−</sup>, and inhibitor-treated RAW264.7 cells were subjected to total lipid extraction. Sphingosine, ceramide, sphingomyelin, and glucosylceramide (GlucCer) levels were quantified by LC-MS/MS. Data are means ± standard deviations (SD) (n = 3). *, P &lt; 0.05; **, P &lt; 0.01; ***, P &lt; 0.001 (two-tailed unpaired t test).</span>
<span class="glightbox-desc lightbox-desc-2">Figure&nbsp;3.2: <strong>Sphingolipid biosynthesis is required for efficient phagocytosis of <em>Mtb</em>.</strong> (A) Control and sphingolipid-depleted phagocytic cells were infected with mCherry-expressing <em>Mtb</em> at an MOI of 10 for 2 h, washed, fixed, stained with phalloidin Alexa Fluor 488 and DAPI, and then analyzed by high-throughput fluorescence imaging. The efficiency of phagocytic uptake was determined by dividing the total number of internalized bacteria by the total number of cells. (B) RAW264.7, DC2.4, THP-1, and U937 cells were grown in the presence or absence of myriocin for 3 days, infected with mCherry-expressing <em>Mtb</em>, and then processed as for panel A. (C) Quantification of <em>Mtb</em> uptake by cells treated as for panel B. (D) Wild-type and Sptlc2<sup>−/−</sup> RAW264.7 and DC2.4 cells were infected with mCherry-expressing <em>Mtb</em> and then processed as for panel A. (E) Quantification of <em>Mtb</em> uptake by cells treated as for panel D. (F) Wild-type and Sptlc2<sup>−/−</sup> RAW264.7 and DC2.4 cells were infected with mCherry-expressing <em>M. marinum</em> at an MOI of 10 for 2 h and then processed as for panel A. (G) Quantification of <em>M. marinum</em> uptake by cells treated as for panel F. (H) Wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells were incubated with zymosan A particles at an MOI of 1 for 1 h and then processed as for panel A. (I) Representative images of wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells infected with mCherry-tagged herpes simplex virus (HSV) at an MOI of 1. At 12 h post-infection, cells were fixed, stained with DAPI, and analyzed by high-throughput fluorescence microscopy to determine the percent infected cells. Bar, 5 μm. Data in panels C, E, and G are means ± SD (n = 3). *, P &lt; 0.05; **, P &lt; 0.01, ***, P &lt; 0.001 (unpaired t test). Data in panel I are means ± standard errors (SE) (n = 2). ns, not significant by two-tailed unpaired t test.</span>
<span class="glightbox-desc lightbox-desc-3">Figure&nbsp;3.3: <strong>Sptlc2<sup>−/−</sup> cells display a defect in actin dynamics during phagocytic cup formation.</strong> (A) Schematic outline of F-actin remodeling during phagocytosis. Clustering of pattern recognition receptors (PRRs) at the pathogen interaction site generates second messengers that promote local actin polymerization. Actin continues to be actively polymerized to extend the phagosomal membrane while being remodeled at the base of the phagocytic cup. Further actin polymerization and contractile activity lead to membrane apposition and eventually fusion to complete phagosome formation. (B) Time-lapse confocal images of wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells stably transduced with LifeAct-mKate (red) and infected with GFP-expressing <em>M. marinum</em> (green). Arrows indicate sites where mycobacteria trigger actin assembly, initiating formation of a phagocytic cup. Unlike wild-type cells, Sptlc2<sup>−/−</sup> RAW264.7 cells often failed to disassemble actin at the base of phagocytic cups, thus blocking entry of the mycobacteria. (C) Quantification of the percentage of successful versus failed phagocytic uptake events of <em>M. marinum</em> in wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells. (D) Time-lapse confocal images of wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells stably transduced with LifeAct-GFP (red) and incubated with zymosan A particles (green). Bar, 5 μm. (E) Quantification of the percentage of successful versus failed phagocytic uptake events of zymosan A particles in wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells. Data represent uptake events in &gt;30 cells per condition. Bar, 5 μm.</span>
<span class="glightbox-desc lightbox-desc-4">Figure&nbsp;3.4: <strong>Segregation of CD45 from dectin-1 at the phagocytic cup is affected in Sptlc2<sup>−/−</sup> cells.</strong> (A) Time-lapse confocal images of wild-type RAW264.7 cells transfected with GFP-tagged dectin-1 (green) and infected with mCherry-expressing <em>M. marinum</em> (red). Arrows indicate sites where mycobacteria trigger clustering of dectin-1. Bar, 5 μm. (B) Wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells transfected with GFP-tagged dectin-1 (green) were incubated with zymosan A particles (dashed circles) on ice, warmed to 37°C for 2 min to initiate phagocytosis, fixed, and then stained with antibodies against CD45 (purple). Representative slices of z-stacks are shown. Insets show fluorescence intensity profiles of CD45 and dectin-1-GFP signals in a single slice. Bar, 3 μm. (C) Violin plots showing the level of CD45 segregation from dectin-1-enriched phagocytic cups in wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells, defined as the percentage of dectin-1 volume containing no CD45 voxels. More than 100 individual cups were analyzed per cell line.”****, P = 0.00006 by a two-tailed unpaired t test.</span>
<span class="glightbox-desc lightbox-desc-5">Figure&nbsp;3.5: <strong>Sptlc2<sup>−/−</sup> cells display an impaired phosphoinositide turnover at the phagocytic cup.</strong> (A) Schematic outline of molecular events underlying phagocytosis. Engagement of pattern recognition receptors (PRRs) with a pathogen initiates receptor clustering at the phagocytic cup, followed by a lateral segregation of the inhibitory CD45 phosphatase. This results in activation of the Rho GTPases Cdc42 and Rac1, which, in turn, stimulate PI5K activity to enable local production of PI(4,5)P2 and promote actin assembly. Next, PI3K orchestrates the turnover of PI(4,5)P2 to PI(3,4,5)P3, leading to actin disassembly at the base of the phagosome, a process required to complete the engulfment and internalization of the pathogen. (B) Time-lapse confocal images of wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells transfected with Cdc42 activity biosensor PAK(PBD)-GFP (green) and incubated with zymosan beads (red) at an MOI of 10. (C) Quantification of the PAK(PBD) fluorescence intensity in the course of phagocytic cup formation in cells treated as for panel B. (D) Time-lapse confocal images of wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells transfected with the PI(4,5)P2 biosensor PH(PLCδ)-GFP (green) and incubated with zymosan beads (red) as for panel B. (E) Quantification of the PH(PLCδ) fluorescence intensity in the course of phagocytic cup formation in cells treated as for panel D. (F) Time-lapse confocal images of wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells transfected with the PI(3,4,5)P3 biosensor PH(AKT)-GFP (green) and incubated with zymosan beads (red) as for panel B. Bar, 5 μm. (G) Quantification of the PH(AKT) fluorescence intensity in the course of phagocytic cup formation in cells treated as for panel F. Note that only a small fraction of the beads recognized by Sptlc2<sup>−/−</sup> mutant cells were internalized, in contrast to wild-type cells. Therefore, for Sptlc2<sup>−/−</sup> mutant cells, the normalized fluorescence intensity was determined at the particle contact site at time points equivalent to those at which particle recognition, engulfment, and internalization took place in wild-type cells. The GFP signal at the particle contact site was quantified and divided by the value for the GFP threshold of the whole cell at the corresponding time point. Average intensity profiles of 5 to 10 cells taken from two independent biological experiments are shown. Data are means ± SD. *, P &lt; 0.05; **, P &lt; 0.01; ns, not significant (two-tailed unpaired t test).</span>
<span class="glightbox-desc lightbox-desc-6">Figure&nbsp;3.6: <strong>Production of sphingomyelin, not glycosphingolipids, is critical for efficient <em>Mtb</em> uptake.</strong> (A) Wild-type and two independent SGMS1<sup>−/−</sup> knockout clones of U937 cells were infected with mCherry-<em>Mtb</em> (red) for 2 h, stained with phalloidin Alexa Fluor 488 (green), and then imaged by confocal fluorescence microscopy. (B) Quantification of <em>Mtb</em> uptake by cells treated as for panel A. (C) Wild-type and two independent UGCG<sup>−/−</sup> knockout clones of THP-1 cells were infected with mCherry-expressing <em>Mtb</em> (red) for 2 h, stained with phalloidin Alexa Fluor 488 (green), and then imaged by confocal fluorescence microscopy. Bar, 5 μm. (D) Quantification of <em>Mtb</em> uptake by cells treated as for panel C. U937 and THP-1 cells were activated with 50 nM PMA for 24 h before <em>Mtb</em> infection. Three independent experiments were performed, each in triplicate with at least 1,000 cells quantified per replicate. Data are means ± SD. ***, P &lt; 0.001; ns, not significant (two-tailed unpaired t test).</span>
<span class="glightbox-desc lightbox-desc-7">Figure S3.1: <strong>Blocking de novo sphingolipid biosynthesis using genetics and chemical tools has no significant impact on cell growth.</strong> Sptlc2<sup>−/−</sup> and wild-type DC2.4 or U937 cells were treated with myriocin or fumonisin B1 for 3 days and then metabolically labelled with 3-l-[14C]serine for 4 h. Total lipids were extracted and analyzed by TLC and autoradiography. Data are representative of 3 independent experiments. Veh, vehicle-treated. (C) Phase-contrast microscopy images of wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells. (D) Numbers of wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells plated out at equal densities and grown for 24 or 48 h in serum-containing medium. Data are means ± SD (n = 3).</span>
<span class="glightbox-desc lightbox-desc-8">Figure S3.2: <strong>Sphingolipid biosynthesis is required for efficient phagocytosis of <em>Mtb</em> but not for receptor-mediated uptake of transferrin.</strong> (A and C) RAW264.7, DC2.4, THP-1, and U937 cells were grown in the presence or absence of fumonisin B1 for 3 days, infected with mCherry-expressing <em>Mtb</em> for 2 h, washed, fixed, stained with phalloidin Alexa Fluor 488 and DAPI, and then imaged by fluorescence microscopy. (B and D) Quantification of <em>Mtb</em> uptake by cells treated as for panel A. (E and F) Sptlc2<sup>−/−</sup> cells bind but fail to internalize pathogen-mimicking beads; (E) Wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells were incubated with rabbit IgG-opsonized zymosan A particles for 5 min at 37°C. Next, cells were washed with chilled PBS and stained at 4°C with donkey anti-rabbit IgG-Cy3 in PBS supplemented with 1% BSA for 15 min to distinguish external beads (red) from internalized beads (blue). After chemical fixation, cells were stained for actin with phalloidin Alexa Fluor 488 (green) and analyzed by confocal fluorescence microscopy. (F) Quantification of the ratio of internalized (blue) over attached beads (red). Three independent experiments were performed, each in triplicate, with at least 100 cells quantified per replicate. (G) Differentiated THP-1 cells were incubated with 100 nM Alexa Fluor 555-conjugated human transferrin (Tf555) in the absence or presence of different concentrations of non-fluorescent transferrin for 30 min. Next, cells were washed in acidic buffer and fixed, and receptor-mediated uptake of Tf555 was quantified using a plate reader fluorimeter. (H) Quantification of receptor-mediated uptake of Tf555 in control or 3-day myriocin-treated THP-1 cells. Data are means ± SD (n = 3). **, P &lt; 0.01; *, P &lt; 0.05; ns, not significant (unpaired t test).</span>
<span class="glightbox-desc lightbox-desc-9">Figure S3.3: (A) WT RAW264.7 cells transfected with GFP-tagged dectin-1 (green) were infected with mCherry-expressing <em>M. marinum</em> (red) as for <a href="#fig-mBioFig4" class="quarto-xref">Figure&nbsp;<span>3.4</span></a>A. Bar, 5 μm. Data are means ± SD (n = 3). (B) No difference in sub-cellular localization of CD45 between WT and Sptlc2<sup>−/−</sup> cells. Wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells transfected with GFP-tagged dectin-1 (green) were incubated with zymosan A particles (blue) on ice, warmed to 37°C for 2 min to initiate phagocytosis, fixed, and then stained with antibodies against CD45 (purple). Boxed areas are magnified at right. Arrowheads mark sites of zymosan A binding. Bar, 5 μm. (C) WT and Sptlc2<sup>−/−</sup> RAW264.7 cells were stained with APC-conjugated anti-CD45 antibody, fixed, and subjected to flow cytometry analysis. (D) Quantitative analysis of median fluorescence intensity of cells treated as for panel C. (E) Wild-type and Sptlc2<sup>−/−</sup> RAW264.7 cells were stained with FITC-conjugated anti-dectin-1 antibody, fixed, and subjected to flow cytometry analysis. (F) Quantitative analysis of median fluorescence intensity of cells treated as for panel E. (G) Immunoblot analysis of total cell lysates collected from wild-type and two independent UGCG<sup>−/−</sup> KO clones of THP-1 cells using antibodies against UGCG and GAPDH.</span>
<span class="glightbox-desc lightbox-desc-10">Supplementary Movie&nbsp;3.1: Time-lapse confocal images of a wild-type (left) and Sptlc2<sup>-/-</sup> (right) RAW264.7 cell stably transduced with LifeAct-mKate (red) and infected with GFP-expressing <em>M. marinum</em> (green). Images were taken every 15 s. Bar, 5 μm.</span>
<span class="glightbox-desc lightbox-desc-11">Supplementary Movie&nbsp;3.2: Time-lapse confocal images of a wild-type (left) and Sptlc2<sup>-/-</sup> (right) RAW264.7 cell stably transduced with LifeAct-mKate (red) and incubated with zymosan A particles (green). Images were taken every 15 s. Bar, 5 μm.</span>
<span class="glightbox-desc lightbox-desc-12">Supplementary Movie&nbsp;3.3: Time-lapse confocal images of a wild-type RAW264.7 cell transfected with dectin-1-GFP (green) and infected with mCherry-expressing <em>M. marinum</em> (red). Images were taken every 15 seconds. Bar, 5 μm.</span>
<span class="glightbox-desc lightbox-desc-13">Supplementary Movie&nbsp;3.4: Time-lapse confocal images of a wild-type RAW264.7 cell transfected with dectin-1-GFP (green) and infected with mCherry-expressing <em>M. marinum</em> (red). Images were taken every 15 s. The imaging is similar to that in <a href="#vid-Mtb_phagocytosis_Movie3" class="quarto-xref">Supplemental Movie&nbsp;<span>3.3</span></a> with a different field.</span>
<span class="glightbox-desc lightbox-desc-14">Supplementary Movie&nbsp;3.5: Time-lapse confocal images of a Sptlc2<sup>-/-</sup> RAW264.7 cell transfected with PAK(PBD)-GFP (green) and incubated with zymosan beads (red). Images were taken every 15 s. Bar, 5 μm.</span>
<span class="glightbox-desc lightbox-desc-15">Supplementary Movie&nbsp;3.6: Time-lapse confocal images of a Sptlc2<sup>-/-</sup> RAW264.7 cell transfected with PLCd-GFP (green) and incubated with zymosan beads (red). Images were taken every 15 s. Bar, 5 μm.</span>
<span class="glightbox-desc lightbox-desc-16">Supplementary Movie&nbsp;3.7: Time-lapse confocal images of a Sptlc2<sup>-/-</sup> RAW264.7 cell transfected with PH(AKT)-GFP (green) and incubated with zymosan beads (red). Images were taken every 15 s.</span>
</div>
<div id="refs" class="references csl-bib-body" role="list" style="display: none">
<div id="ref-worldhealthorganizationGlobalTuberculosisReport2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">1. </div><div class="csl-right-inline">World Health Organization (2018). <a href="https://apps.who.int/iris/handle/10665/274453">Global tuberculosis report 2018</a> (World Health Organization).</div>
</div>
<div id="ref-korbMycobacteriumTuberculosisManipulator2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">2. </div><div class="csl-right-inline">Korb, V., Chuturgoon, A., and Moodley, D. (2016). Mycobacterium tuberculosis: <span>Manipulator</span> of <span>Protective Immunity</span>. IJMS <em>17</em>, 131. <a href="https://doi.org/10.3390/ijms17030131">10.3390/ijms17030131</a>.</div>
</div>
<div id="ref-quevalMacrophageDisputedFortress2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">3. </div><div class="csl-right-inline">Queval, C.J., Brosch, R., and Simeone, R. (2017). The <span>Macrophage</span>: <span>A Disputed Fortress</span> in the <span>Battle</span> against <span>Mycobacterium</span> tuberculosis. Frontiers in Microbiology <em>8</em>. <a href="https://doi.org/10.3389/fmicb.2017.02284">10.3389/fmicb.2017.02284</a>.</div>
</div>
<div id="ref-levinPhosphoinositidesPhagocytosisMacropinocytosis2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">4. </div><div class="csl-right-inline">Levin, R., Grinstein, S., and Schlam, D. (2015). Phosphoinositides in phagocytosis and macropinocytosis. Biochimica et Biophysica Acta (BBA) - Molecular and Cell Biology of Lipids <em>1851</em>, 805–823. <a href="https://doi.org/10.1016/j.bbalip.2014.09.005">10.1016/j.bbalip.2014.09.005</a>.</div>
</div>
<div id="ref-stuartPhagocytosisElegantComplexity2005" class="csl-entry" role="listitem">
<div class="csl-left-margin">5. </div><div class="csl-right-inline">Stuart, L.M., and Ezekowitz, R.A.B. (2005). Phagocytosis: Elegant complexity. Immunity <em>22</em>, 539–550. <a href="https://doi.org/10.1016/j.immuni.2005.05.002">10.1016/j.immuni.2005.05.002</a>.</div>
</div>
<div id="ref-flannaganCellBiologyPhagocytosis2012" class="csl-entry" role="listitem">
<div class="csl-left-margin">6. </div><div class="csl-right-inline">Flannagan, R.S., Jaumouillé, V., and Grinstein, S. (2012). The <span>Cell Biology</span> of <span>Phagocytosis</span>. Annual Review of Pathology: Mechanisms of Disease <em>7</em>, 61–98. <a href="https://doi.org/10.1146/annurev-pathol-011811-132445">10.1146/annurev-pathol-011811-132445</a>.</div>
</div>
<div id="ref-goodridgeActivationInnateImmune2011a" class="csl-entry" role="listitem">
<div class="csl-left-margin">7. </div><div class="csl-right-inline">Goodridge, H.S., Reyes, C.N., Becker, C. a, Katsumoto, T.R., Ma, J., Wolf, A.J., Bose, N., Chan, A.S.H., Magee, A.S., Danielson, M.E., et al. (2011). Activation of the innate immune receptor <span>Dectin-1</span> upon formation of a <span>“phagocytic synapse.”</span> Nature <em>472</em>, 471–475. <a href="https://doi.org/10.1038/nature10071">10.1038/nature10071</a>.</div>
</div>
<div id="ref-tailleuxDCSIGNMajorMycobacterium2003a" class="csl-entry" role="listitem">
<div class="csl-left-margin">8. </div><div class="csl-right-inline">Tailleux, L., Schwartz, O., Herrmann, J.-L., Pivert, E., Jackson, M., Amara, A., Legres, L., Dreher, D., Nicod, L.P., Gluckman, J.C., et al. (2003). <span>DC-SIGN Is</span> the <span>Major</span> <span><span><em>Mycobacterium</em></span></span><span><span><em> Tuberculosis</em></span></span> <span>Receptor</span> on <span>Human Dendritic Cells</span>. The Journal of Experimental Medicine <em>197</em>, 121–127. <a href="https://doi.org/10.1084/jem.20021468">10.1084/jem.20021468</a>.</div>
</div>
<div id="ref-yadavVglucanReceptorDectin12006" class="csl-entry" role="listitem">
<div class="csl-left-margin">9. </div><div class="csl-right-inline">Yadav, M., and Schorey, J.S. (2006). The β-glucan receptor dectin-1 functions together with <span>TLR2</span> to mediate macrophage activation by mycobacteria. Blood <em>108</em>, 3168–3175. <a href="https://doi.org/10.1182/blood-2006-05-024406">10.1182/blood-2006-05-024406</a>.</div>
</div>
<div id="ref-killickReceptormediatedRecognitionMycobacterial2013" class="csl-entry" role="listitem">
<div class="csl-left-margin">10. </div><div class="csl-right-inline">Killick, K.E., Ní Cheallaigh, C., O’Farrelly, C., Hokamp, K., MacHugh, D.E., and Harris, J. (2013). Receptor-mediated recognition of mycobacterial pathogens: <span>Cellular</span> recognition of mycobacteria. Cell Microbiol <em>15</em>, 1484–1495. <a href="https://doi.org/10.1111/cmi.12161">10.1111/cmi.12161</a>.</div>
</div>
<div id="ref-lingwoodNewAspectsRegulation2010" class="csl-entry" role="listitem">
<div class="csl-left-margin">11. </div><div class="csl-right-inline">Lingwood, C.A., Manis, A., Mahfoud, R., Khan, F., Binnington, B., and Mylvaganam, M. (2010). New aspects of the regulation of glycosphingolipid receptor function. Chemistry and Physics of Lipids <em>163</em>, 27–35. <a href="https://doi.org/10.1016/j.chemphyslip.2009.09.001">10.1016/j.chemphyslip.2009.09.001</a>.</div>
</div>
<div id="ref-slotteBiologicalFunctionsSphingomyelins2013" class="csl-entry" role="listitem">
<div class="csl-left-margin">12. </div><div class="csl-right-inline">Slotte, J.P. (2013). Biological functions of sphingomyelins. Progress in Lipid Research <em>52</em>, 424–437. <a href="https://doi.org/10.1016/j.plipres.2013.05.001">10.1016/j.plipres.2013.05.001</a>.</div>
</div>
<div id="ref-hannunSphingolipidsTheirMetabolism2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">13. </div><div class="csl-right-inline">Hannun, Y.A., and Obeid, L.M. (2018). Sphingolipids and their metabolism in physiology and disease. Nat Rev Mol Cell Biol <em>19</em>, 175–191. <a href="https://doi.org/10.1038/nrm.2017.107">10.1038/nrm.2017.107</a>.</div>
</div>
<div id="ref-heungRoleSphingolipidsMicrobial2006a" class="csl-entry" role="listitem">
<div class="csl-left-margin">14. </div><div class="csl-right-inline">Heung, L.J., Luberto, C., and Del Poeta, M. (2006). Role of sphingolipids in microbial pathogenesis. Infection and Immunity <em>74</em>, 28–39. <a href="https://doi.org/10.1128/IAI.74.1.28-39.2006">10.1128/IAI.74.1.28-39.2006</a>.</div>
</div>
<div id="ref-grassmeAcidicSphingomyelinaseMediates1997a" class="csl-entry" role="listitem">
<div class="csl-left-margin">15. </div><div class="csl-right-inline">Grassmé, H., Gulbins, E., Brenner, B., Ferlinz, K., Sandhoff, K., Harzer, K., Lang, F., and Meyer, T.F. (1997). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9393854">Acidic sphingomyelinase mediates entry of <span>N</span>. Gonorrhoeae into nonphagocytic cells.</a> Cell <em>91</em>, 605–615.</div>
</div>
<div id="ref-grassmeHostDefensePseudomonas2003" class="csl-entry" role="listitem">
<div class="csl-left-margin">16. </div><div class="csl-right-inline">Grassmé, H., Jendrossek, V., Riehle, A., von Kürthy, G., Berger, J., Schwarz, H., Weller, M., Kolesnick, R., and Gulbins, E. (2003). Host defense against <span>Pseudomonas</span> aeruginosa requires ceramide-rich membrane rafts. Nature Medicine <em>9</em>, 322–330. <a href="https://doi.org/10.1038/nm823">10.1038/nm823</a>.</div>
</div>
<div id="ref-tafesseDisruptionSphingolipidBiosynthesis2015a" class="csl-entry" role="listitem">
<div class="csl-left-margin">17. </div><div class="csl-right-inline">Tafesse, F.G., Rashidfarrokhi, A., Schmidt, F.I., Freinkman, E., Dougan, S., Dougan, M., Esteban, A., Maruyama, T., Strijbis, K., and Ploegh, H.L. (2015). Disruption of <span>Sphingolipid Biosynthesis Blocks Phagocytosis</span> of <span>Candida</span> albicans. PLoS pathogens <em>11</em>, e1005188. <a href="https://doi.org/10.1371/journal.ppat.1005188">10.1371/journal.ppat.1005188</a>.</div>
</div>
<div id="ref-lovewellChewingFatLipid2016a" class="csl-entry" role="listitem">
<div class="csl-left-margin">18. </div><div class="csl-right-inline">Lovewell, R.R., Sassetti, C.M., and VanderVen, B.C. (2016). Chewing the fat: Lipid metabolism and homeostasis during <span>M</span>. Tuberculosis infection. Current Opinion in Microbiology <em>29</em>, 30–36. <a href="https://doi.org/10.1016/j.mib.2015.10.002">10.1016/j.mib.2015.10.002</a>.</div>
</div>
<div id="ref-gatfieldEssentialRoleCholesterol2000a" class="csl-entry" role="listitem">
<div class="csl-left-margin">19. </div><div class="csl-right-inline">Gatfield, J., and Pieters, J. (2000). Essential role for cholesterol in entry of mycobacteria into macrophages. Science (New York, N.Y.) <em>288</em>, 1647–1650. <a href="https://doi.org/10.1126/science.288.5471.1647">10.1126/science.288.5471.1647</a>.</div>
</div>
<div id="ref-hanadaSerinePalmitoyltransferaseKey2003" class="csl-entry" role="listitem">
<div class="csl-left-margin">20. </div><div class="csl-right-inline">Hanada, K. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12782147">Serine palmitoyltransferase, a key enzyme of sphingolipid metabolism.</a> Biochimica et biophysica acta <em>1632</em>, 16–30.</div>
</div>
<div id="ref-wangInhibitionSphingolipidBiosynthesis1991" class="csl-entry" role="listitem">
<div class="csl-left-margin">21. </div><div class="csl-right-inline">Wang, E., Norred, W.P., Bacon, C.W., Riley, R.T., and Merrill, A.H. (1991). Inhibition of sphingolipid biosynthesis by fumonisins. <span>Implications</span> for diseases associated with <span>Fusarium</span> moniliforme. J Biol Chem <em>266</em>, 14486–14490.</div>
</div>
<div id="ref-leierLipidsPathogenicFlaviviruses2018a" class="csl-entry" role="listitem">
<div class="csl-left-margin">22. </div><div class="csl-right-inline">Leier, H.C., Messer, W.B., and Tafesse, F.G. (2018). Lipids and pathogenic flaviviruses : <span>An</span> intimate union. 1–7. <a href="https://doi.org/10.1371/journal.ppat.1006952">10.1371/journal.ppat.1006952</a>.</div>
</div>
<div id="ref-gaultOverviewSphingolipidMetabolism2010a" class="csl-entry" role="listitem">
<div class="csl-left-margin">23. </div><div class="csl-right-inline">Gault, C.R., Obeid, L.M., and Hannun, Y.A. (2010). An <span>Overview</span> of <span>Sphingolipid Metabolism</span>: <span>From Synthesis</span> to <span>Breakdown</span>. In Adv <span>Exp Med Biol</span>, pp. 1–23. <a href="https://doi.org/10.1007/978-1-4419-6741-1_1">10.1007/978-1-4419-6741-1_1</a>.</div>
</div>
<div id="ref-meivar-levyRoleSphingolipidsMaintenance1997" class="csl-entry" role="listitem">
<div class="csl-left-margin">24. </div><div class="csl-right-inline">Meivar-Levy, I., Sabanay, H., Bershadsky, A.D., and Futerman, A.H. (1997). The <span>Role</span> of <span>Sphingolipids</span> in the <span>Maintenance</span> of <span>Fibroblast Morphology</span>. Journal of Biological Chemistry <em>272</em>, 1558–1564. <a href="https://doi.org/10.1074/jbc.272.3.1558">10.1074/jbc.272.3.1558</a>.</div>
</div>
<div id="ref-ramakrishnanMycobacteriumMarinumPersists1994" class="csl-entry" role="listitem">
<div class="csl-left-margin">25. </div><div class="csl-right-inline">Ramakrishnan, L., and Falkow, S. (1994). Mycobacterium marinum persists in cultured mammalian cells in a temperature-restricted fashion. Infect Immun <em>62</em>, 3222–3229. <a href="https://doi.org/10.1128/iai.62.8.3222-3229.1994">10.1128/iai.62.8.3222-3229.1994</a>.</div>
</div>
<div id="ref-rocaTNFDuallyMediates2013" class="csl-entry" role="listitem">
<div class="csl-left-margin">26. </div><div class="csl-right-inline">Roca, F.J., and Ramakrishnan, L. (2013). <span>TNF</span> dually mediates resistance and susceptibility to mycobacteria via mitochondrial reactive oxygen species. Cell <em>153</em>, 521–534. <a href="https://doi.org/10.1016/j.cell.2013.03.022">10.1016/j.cell.2013.03.022</a>.</div>
</div>
<div id="ref-clementNovelRolePhagocytosislike2006a" class="csl-entry" role="listitem">
<div class="csl-left-margin">27. </div><div class="csl-right-inline">Clement, C., Tiwari, V., Scanlan, P.M., Valyi-Nagy, T., Yue, B.Y.J.T., and Shukla, D. (2006). A novel role for phagocytosis-like uptake in herpes simplex virus entry. Journal of Cell Biology <em>174</em>, 1009–1021. <a href="https://doi.org/10.1083/jcb.200509155">10.1083/jcb.200509155</a>.</div>
</div>
<div id="ref-tiwariNonprofessionalPhagocytosisCan2012" class="csl-entry" role="listitem">
<div class="csl-left-margin">28. </div><div class="csl-right-inline">Tiwari, V., and Shukla, D. (2012). Nonprofessional <span>Phagocytosis Can Facilitate Herpesvirus Entry</span> into <span>Ocular Cells</span>. Clinical and Developmental Immunology <em>2012</em>, 1–8. <a href="https://doi.org/10.1155/2012/651691">10.1155/2012/651691</a>.</div>
</div>
<div id="ref-heEndocytosisFunctionallyEnhanced2015a" class="csl-entry" role="listitem">
<div class="csl-left-margin">29. </div><div class="csl-right-inline">He, Q., Sun, X., Chu, C., Jiang, Q., Zhu, H., He, Y., Yue, T., Wang, R., Lei, P., and Shen, G. (2015). Endocytosis of a <span>Functionally Enhanced GFP-Tagged Transferrin Receptor</span> in <span>CHO Cells</span>. PLOS ONE <em>10</em>, e0122452. <a href="https://doi.org/10.1371/journal.pone.0122452">10.1371/journal.pone.0122452</a>.</div>
</div>
<div id="ref-srinivasanRacCdc42Play2003" class="csl-entry" role="listitem">
<div class="csl-left-margin">30. </div><div class="csl-right-inline">Srinivasan, S., Wang, F., Glavas, S., Ott, A., Hofmann, F., Aktories, K., Kalman, D., and Bourne, H.R. (2003). Rac and <span>Cdc42</span> play distinct roles in regulating <span>PI</span>(3,4,5)<span>P3</span> and polarity during neutrophil chemotaxis. Journal of Cell Biology <em>160</em>, 375–385. <a href="https://doi.org/10.1083/jcb.200208179">10.1083/jcb.200208179</a>.</div>
</div>
<div id="ref-mansourDectin1ActivationControls2013a" class="csl-entry" role="listitem">
<div class="csl-left-margin">31. </div><div class="csl-right-inline">Mansour, M.K., Tam, J.M., Khan, N.S., Seward, M., Davids, P.J., Puranam, S., Sokolovska, A., Sykes, D.B., Dagher, Z., Becker, C., et al. (2013). Dectin-1 <span>Activation Controls Maturation</span> of β-1,3-<span class="nocase">Glucan-containing Phagosomes</span>. Journal of Biological Chemistry <em>288</em>, 16043–16054. <a href="https://doi.org/10.1074/jbc.M113.473223">10.1074/jbc.M113.473223</a>.</div>
</div>
<div id="ref-underhillMacrophageRecognitionZymosan2003a" class="csl-entry" role="listitem">
<div class="csl-left-margin">32. </div><div class="csl-right-inline">Underhill, D.M. (2003). Macrophage recognition of zymosan particles. Journal of Endotoxin Research <em>9</em>, 176–180. <a href="https://doi.org/10.1179/096805103125001586">10.1179/096805103125001586</a>.</div>
</div>
<div id="ref-rothfuchsDectin1InteractionMycobacterium2007a" class="csl-entry" role="listitem">
<div class="csl-left-margin">33. </div><div class="csl-right-inline">Rothfuchs, A.G., Bafica, A., Feng, C.G., Egen, J.G., Williams, D.L., Brown, G.D., and Sher, A. (2007). Dectin-1 <span>Interaction</span> with <span>Mycobacterium</span> tuberculosis <span>Leads</span> to <span>Enhanced IL-12p40 Production</span> by <span>Splenic Dendritic Cells</span>. The Journal of Immunology <em>179</em>, 3463–3471. <a href="https://doi.org/10.4049/jimmunol.179.6.3463">10.4049/jimmunol.179.6.3463</a>.</div>
</div>
<div id="ref-wagenerDectin1SykCARD9SignalingPathway2018a" class="csl-entry" role="listitem">
<div class="csl-left-margin">34. </div><div class="csl-right-inline">Wagener, M., Hoving, J.C., Ndlovu, H., and Marakalala, M.J. (2018). Dectin-1-<span>Syk-CARD9 Signaling Pathway</span> in <span>TB Immunity</span>. Frontiers in Immunology <em>9</em>. <a href="https://doi.org/10.3389/fimmu.2018.00225">10.3389/fimmu.2018.00225</a>.</div>
</div>
<div id="ref-simonsLipidRaftsSignal2000a" class="csl-entry" role="listitem">
<div class="csl-left-margin">35. </div><div class="csl-right-inline">Simons, K., and Toomre, D. (2000). Lipid rafts and signal transduction. Nature Reviews Molecular Cell Biology <em>1</em>, 31–39. <a href="https://doi.org/10.1038/35036052">10.1038/35036052</a>.</div>
</div>
<div id="ref-schlamPhosphoinositide3kinaseEnables2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">36. </div><div class="csl-right-inline">Schlam, D., Bagshaw, R.D., Freeman, S.A., Collins, R.F., Pawson, T., Fairn, G.D., and Grinstein, S. (2015). Phosphoinositide 3-kinase enables phagocytosis of large particles by terminating actin assembly through <span>Rac</span>/<span class="nocase">Cdc42 GTPase-activating</span> proteins. Nature Communications <em>6</em>, 8623. <a href="https://doi.org/10.1038/ncomms9623">10.1038/ncomms9623</a>.</div>
</div>
<div id="ref-ledvinaPhosphatidylinositol3KinaseEffector2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">37. </div><div class="csl-right-inline">Ledvina, H.E., Kelly, K.A., Eshraghi, A., Plemel, R.L., Peterson, S.B., Lee, B., Steele, S., Adler, M., Kawula, T.H., Merz, A.J., et al. (2018). A <span>Phosphatidylinositol</span> 3-<span>Kinase Effector Alters Phagosomal Maturation</span> to <span>Promote Intracellular Growth</span> of <span>Francisella</span>. Cell Host &amp; Microbe <em>24</em>, 285–295.e8. <a href="https://doi.org/10.1016/j.chom.2018.07.003">10.1016/j.chom.2018.07.003</a>.</div>
</div>
<div id="ref-hoppeCdc42Rac1Rac22004a" class="csl-entry" role="listitem">
<div class="csl-left-margin">38. </div><div class="csl-right-inline">Hoppe, A.D., and Swanson, J.A. (2004). Cdc42, <span>Rac1</span>, and <span>Rac2 Display Distinct Patterns</span> of <span>Activation</span> during <span>Phagocytosis</span>. Molecular Biology of the Cell <em>15</em>, 3509–3519. <a href="https://doi.org/10.1091/mbc.e03-11-0847">10.1091/mbc.e03-11-0847</a>.</div>
</div>
<div id="ref-marshallRestrictedAccumulationPhosphatidylinositol2001" class="csl-entry" role="listitem">
<div class="csl-left-margin">39. </div><div class="csl-right-inline">Marshall, J.G., Booth, J.W., Stambolic, V., Mak, T., Balla, T., Schreiber, A.D., Meyer, T., and Grinstein, S. (2001). Restricted <span>Accumulation</span> of <span>Phosphatidylinositol</span> 3-<span>Kinase Products</span> in a <span>Plasmalemmal Subdomain</span> during <span>Fcγ Receptor-Mediated Phagocytosis</span>. Journal of Cell Biology <em>153</em>, 1369–1380. <a href="https://doi.org/10.1083/jcb.153.7.1369">10.1083/jcb.153.7.1369</a>.</div>
</div>
<div id="ref-agramonte-heviaGramnegativeBacteriaPhagocytic2002" class="csl-entry" role="listitem">
<div class="csl-left-margin">40. </div><div class="csl-right-inline">Agramonte-Hevia, J. (2002). Gram-negative bacteria and phagocytic cell interaction mediated by complement receptor 3. FEMS Immunology and Medical Microbiology <em>34</em>, 255–266. <a href="https://doi.org/10.1016/S0928-8244(02)00408-X">10.1016/S0928-8244(02)00408-X</a>.</div>
</div>
<div id="ref-wegnerUDPglucoseCeramideGlycosyltransferase2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">41. </div><div class="csl-right-inline">Wegner, M.S., Gruber, L., Mattjus, P., Geisslinger, G., and Grösch, S. (2018). The <span class="nocase">UDP-glucose</span> ceramide glycosyltransferase (<span>UGCG</span>) and the link to multidrug resistance protein 1 (<span>MDR1</span>). BMC Cancer <em>18</em>, 1–10. <a href="https://doi.org/10.1186/s12885-018-4084-4">10.1186/s12885-018-4084-4</a>.</div>
</div>
<div id="ref-virreirawinterGenomewideCRISPRScreen2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">42. </div><div class="csl-right-inline">Virreira Winter, S., Zychlinsky, A., and Bardoel, B.W. (2016). Genome-wide <span>CRISPR</span> screen reveals novel host factors required for <span>Staphylococcus</span> aureus α-hemolysin-mediated toxicity. Scientific Reports <em>6</em>, 24242. <a href="https://doi.org/10.1038/srep24242">10.1038/srep24242</a>.</div>
</div>
<div id="ref-hossainPatternRecognitionReceptors2013" class="csl-entry" role="listitem">
<div class="csl-left-margin">43. </div><div class="csl-right-inline">Hossain, Md.M., and Norazmi, M.-N. (2013). Pattern <span>Recognition Receptors</span> and <span>Cytokines</span> in <span>Mycobacterium</span> tuberculosis <span>Infection</span>—<span>The Double-Edged Sword</span>? BioMed Research International <em>2013</em>, 1–18. <a href="https://doi.org/10.1155/2013/179174">10.1155/2013/179174</a>.</div>
</div>
<div id="ref-jaumouilleReceptorMobilityCytoskeleton2011" class="csl-entry" role="listitem">
<div class="csl-left-margin">44. </div><div class="csl-right-inline">Jaumouillé, V., and Grinstein, S. (2011). Receptor mobility, the cytoskeleton, and particle binding during phagocytosis. Current Opinion in Cell Biology <em>23</em>, 22–29. <a href="https://doi.org/10.1016/j.ceb.2010.10.006">10.1016/j.ceb.2010.10.006</a>.</div>
</div>
<div id="ref-kerriganSykcoupledCtypeLectin2010" class="csl-entry" role="listitem">
<div class="csl-left-margin">45. </div><div class="csl-right-inline">Kerrigan, A.M., and Brown, G.D. (2010). Syk-coupled <span class="nocase">C-type</span> lectin receptors that mediate cellular activation via single tyrosine based activation motifs. Immunological Reviews <em>234</em>, 335–352. <a href="https://doi.org/10.1111/j.0105-2896.2009.00882.x">10.1111/j.0105-2896.2009.00882.x</a>.</div>
</div>
<div id="ref-zhuStructurallyDistinctPhosphatases2008" class="csl-entry" role="listitem">
<div class="csl-left-margin">46. </div><div class="csl-right-inline">Zhu, J.W., Brdicka, T., Katsumoto, T.R., Lin, J., and Weiss, A. (2008). Structurally <span>Distinct Phosphatases CD45</span> and <span>CD148 Both Regulate B Cell</span> and <span>Macrophage Immunoreceptor Signaling</span>. Immunity <em>28</em>, 183–196. <a href="https://doi.org/10.1016/j.immuni.2007.11.024">10.1016/j.immuni.2007.11.024</a>.</div>
</div>
<div id="ref-liAcidSphingomyelinaseCeramideSystem2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">47. </div><div class="csl-right-inline">Li, C., Wang, A., Wu, Y., Gulbins, E., Grassmé, H., and Zhao, Z. (2019). Acid <span>Sphingomyelinase-Ceramide System</span> in <span>Bacterial Infections</span>. Cellular physiology and biochemistry : international journal of experimental cellular physiology, biochemistry, and pharmacology <em>52</em>, 280–301. <a href="https://doi.org/10.33594/000000021">10.33594/000000021</a>.</div>
</div>
<div id="ref-smithUnexpectedRoleAcid2008" class="csl-entry" role="listitem">
<div class="csl-left-margin">48. </div><div class="csl-right-inline">Smith, E.L., and Schuchman, E.H. (2008). The unexpected role of acid sphingomyelinase in cell death and the pathophysiology of common diseases. The FASEB Journal <em>22</em>, 3419–3431. <a href="https://doi.org/10.1096/fj.08-108043">10.1096/fj.08-108043</a>.</div>
</div>
<div id="ref-speerSurfaceHydrolysisSphingomyelin2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">49. </div><div class="csl-right-inline">Speer, A., Sun, J., Danilchanka, O., Meikle, V., Rowland, J.L., Walter, K., Buck, B.R., Pavlenok, M., Hölscher, C., Ehrt, S., et al. (2015). Surface hydrolysis of sphingomyelin by the outer membrane protein <span>Rv0888</span> supports replication of <span>M</span> ycobacterium tuberculosis in macrophages. Molecular Microbiology <em>97</em>, 881–897. <a href="https://doi.org/10.1111/mmi.13073">10.1111/mmi.13073</a>.</div>
</div>
<div id="ref-viswanathanMacrophageSphingolipidsAre2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">50. </div><div class="csl-right-inline">Viswanathan, G., Jafurulla, M., Kumar, G.A., Raghunand, T.R., and Chattopadhyay, A. (2018). Macrophage sphingolipids are essential for the entry of mycobacteria. Chemistry and Physics of Lipids <em>213</em>, 25–31. <a href="https://doi.org/10.1016/j.chemphyslip.2018.03.004">10.1016/j.chemphyslip.2018.03.004</a>.</div>
</div>
<div id="ref-wuFunctionSphingomyelinasesMycobacterial2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">51. </div><div class="csl-right-inline">Wu, Y., Gulbins, E., and Grassmé, H. (2018). The function of sphingomyelinases in mycobacterial infections. Biological Chemistry <em>399</em>, 1125–1133. <a href="https://doi.org/10.1515/hsz-2018-0179">10.1515/hsz-2018-0179</a>.</div>
</div>
<div id="ref-erkelensIntestinalMacrophagesBalance2020a" class="csl-entry" role="listitem">
<div class="csl-left-margin">52. </div><div class="csl-right-inline">Erkelens, M.N., Goverse, G., Konijn, T., Molenaar, R., Beijer, M.R., Van Den Bossche, J., De Goede, K.E., Verberk, S.G.S., De Jonge, W.J., Den Haan, J.M.M., et al. (2020). Intestinal <span>Macrophages Balance Inflammatory Expression Profiles</span> via <span>Vitamin A</span> and <span>Dectin-1-Mediated Signaling</span>. Front. Immunol. <em>11</em>, 551. <a href="https://doi.org/10.3389/fimmu.2020.00551">10.3389/fimmu.2020.00551</a>.</div>
</div>
<div id="ref-taylorBetaglucanReceptorDectin12002" class="csl-entry" role="listitem">
<div class="csl-left-margin">53. </div><div class="csl-right-inline">Taylor, P.R., Brown, G.D., Reid, D.M., Willment, J.A., Martinez-Pomares, L., Gordon, S., and Wong, S.Y.C. (2002). The beta-glucan receptor, dectin-1, is predominantly expressed on the surface of cells of the monocyte/macrophage and neutrophil lineages. J Immunol <em>169</em>, 3876–3882. <a href="https://doi.org/10.4049/jimmunol.169.7.3876">10.4049/jimmunol.169.7.3876</a>.</div>
</div>
<div id="ref-garfootEng1Exg8Are2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">54. </div><div class="csl-right-inline">Garfoot, A.L., Dearing, K.L., VanSchoiack, A.D., Wysocki, V.H., and Rappleye, C.A. (2017). Eng1 and <span>Exg8 Are</span> the <span>Major</span> β-<span>Glucanases Secreted</span> by the <span>Fungal Pathogen Histoplasma</span> capsulatum. J Biol Chem <em>292</em>, 4801–4810. <a href="https://doi.org/10.1074/jbc.M116.762104">10.1074/jbc.M116.762104</a>.</div>
</div>
<div id="ref-fengRapidlyReversibleChemical2014a" class="csl-entry" role="listitem">
<div class="csl-left-margin">55. </div><div class="csl-right-inline">Feng, S., Laketa, V., Stein, F., Rutkowska, A., MacNamara, A., Depner, S., Klingmüller, U., Saez-Rodriguez, J., and Schultz, C. (2014). A <span>Rapidly Reversible Chemical Dimerizer System</span> to <span>Study Lipid Signaling</span> in <span>Living Cells</span>. Angewandte Chemie International Edition <em>53</em>, 6720–6723. <a href="https://doi.org/10.1002/anie.201402294">10.1002/anie.201402294</a>.</div>
</div>
<div id="ref-blighRAPIDMETHODTOTAL1959" class="csl-entry" role="listitem">
<div class="csl-left-margin">56. </div><div class="csl-right-inline">Bligh, E.G., and Dyer, W.J. (1959). A <span>RAPID METHOD OF TOTAL LIPID EXTRACTION AND PURIFICATION</span>. Canadian Journal of Biochemistry and Physiology <em>37</em>, 911–917. <a href="https://doi.org/10.1139/o59-099">10.1139/o59-099</a>.</div>
</div>
<div id="ref-tafesseGPR107GproteincoupledReceptor2014" class="csl-entry" role="listitem">
<div class="csl-left-margin">57. </div><div class="csl-right-inline">Tafesse, F.G., Guimaraes, C.P., Maruyama, T., Carette, J.E., Lory, S., Brummelkamp, T.R., and Ploegh, H.L. (2014). <span>GPR107</span>, a <span class="nocase">G-protein-coupled Receptor Essential</span> for <span>Intoxication</span> by <span>Pseudomonas</span> aeruginosa <span>Exotoxin A</span>, <span>Localizes</span> to the <span>Golgi</span> and <span>Is Cleaved</span> by <span>Furin</span>. Journal of Biological Chemistry <em>289</em>, 24005–24018. <a href="https://doi.org/10.1074/jbc.M114.589275">10.1074/jbc.M114.589275</a>.</div>
</div>
<div id="ref-maliRNAguidedHumanGenome2013" class="csl-entry" role="listitem">
<div class="csl-left-margin">58. </div><div class="csl-right-inline">Mali, P., Yang, L., Esvelt, K.M., Aach, J., Guell, M., DiCarlo, J.E., Norville, J.E., and Church, G.M. (2013). <span class="nocase">RNA-guided</span> human genome engineering via <span>Cas9</span>. Science (New York, N.Y.) <em>339</em>, 823–826. <a href="https://doi.org/10.1126/science.1232033">10.1126/science.1232033</a>.</div>
</div>
<div id="ref-rashidfarrokhiVisualizingEarlyStages2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">59. </div><div class="csl-right-inline">Rashidfarrokhi, A., Richina, V., and Tafesse, F.G. (2017). Visualizing the <span>Early Stages</span> of <span>Phagocytosis</span>. Journal of Visualized Experiments. <a href="https://doi.org/10.3791/54646">10.3791/54646</a>.</div>
</div>
</div>
</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleGiscusIfUsed = (isAlternate, darkModeDefault) => {
    const baseTheme = document.querySelector('#giscus-base-theme')?.value ?? 'light';
    const alternateTheme = document.querySelector('#giscus-alt-theme')?.value ?? 'dark';
    let newTheme = '';
    if(darkModeDefault) {
      newTheme = isAlternate ? baseTheme : alternateTheme;
    } else {
      newTheme = isAlternate ? alternateTheme : baseTheme;
    }
    const changeGiscusTheme = () => {
      // From: https://github.com/giscus/giscus/issues/336
      const sendMessage = (message) => {
        const iframe = document.querySelector('iframe.giscus-frame');
        if (!iframe) return;
        iframe.contentWindow.postMessage({ giscus: message }, 'https://giscus.app');
      }
      sendMessage({
        setConfig: {
          theme: newTheme
        }
      });
    }
    const isGiscussLoaded = window.document.querySelector('iframe.giscus-frame') !== null;
    if (isGiscussLoaded) {
      changeGiscusTheme();
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  const darkModeDefault = false;
  let localAlternateSentinel = darkModeDefault ? 'alternate' : 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
    toggleGiscusIfUsed(toAlternate, darkModeDefault);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } else {
    toggleColorMode(false);
  }
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../../parts/results.html" class="pagination-link  aria-label=" main="" results"="">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text">Main Results</span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="../../parts/results/results_Mtb_Membrane_Damage_V2.html" class="pagination-link" aria-label="<span class='chapter-number'>4</span>&nbsp; <span class='chapter-title'>Sealing the Gaps</span>">
        <span class="nav-page-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Sealing the Gaps</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
<p>Copyright 2024, Gaelen Guzman</p>
</div>   
    <div class="nav-footer-center">
      &nbsp;
    </div>
    <div class="nav-footer-right">
      &nbsp;
    </div>
  </div>
</footer>
<script>var lightboxQuarto = GLightbox({"loop":true,"selector":".lightbox","descPosition":"bottom","openEffect":"fade","closeEffect":"fade"});
window.onload = () => {
  lightboxQuarto.on('slide_before_load', (data) => {
    const { slideIndex, slideNode, slideConfig, player, trigger } = data;
    const href = trigger.getAttribute('href');
    if (href !== null) {
      const imgEl = window.document.querySelector(`a[href="${href}"] img`);
      if (imgEl !== null) {
        const srcAttr = imgEl.getAttribute("src");
        if (srcAttr && srcAttr.startsWith("data:")) {
          slideConfig.href = srcAttr;
        }
      }
    } 
  });

  lightboxQuarto.on('slide_after_load', (data) => {
    const { slideIndex, slideNode, slideConfig, player, trigger } = data;
    if (window.Quarto?.typesetMath) {
      window.Quarto.typesetMath(slideNode);
    }
  });

};
          </script>




</body></html>