<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.4.549">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Gaelen Guzman">
<meta name="author" content="Scotland E. Farley">
<meta name="author" content="Frank Stein">
<meta name="author" content="Per Haberkant">
<meta name="author" content="Carsten Schultz">
<meta name="author" content="Fikadu G. Tafesse">

<title>TB &amp; The Sphinx - 6&nbsp; An Affinity for Sphingolipids</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../">
<link href="../../parts/addenda.html" rel="next">
<link href="../../parts/results/results_Mtb_lipidomics_v2.html" rel="prev">
<script src="../../site_libs/quarto-html/quarto.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../../site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script src="../../site_libs/quarto-contrib/glightbox/glightbox.min.js"></script>
<link href="../../site_libs/quarto-contrib/glightbox/glightbox.min.css" rel="stylesheet">
<link href="../../site_libs/quarto-contrib/glightbox/lightbox.css" rel="stylesheet">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="../../styles.css">
</head>

<body class="nav-sidebar floating nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="../../index.html">
    <span class="navbar-title">TB and the Sphinx</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="https://drive.google.com/file/d/1Cdb-DDU9dsD4KLxv5WlcxFEM6ViPRuXl/view?usp=share_link" target="_blank"> <i class="bi bi-file-pdf" role="img" aria-label="Download .pdf">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://docs.google.com/document/d/1L0fe7AC-k4hhSJYj_NG2i_aBLd4QHr9T/edit?usp=share_link&amp;ouid=107014449567027741510&amp;rtpof=true&amp;sd=true" target="_blank"> <i class="bi bi-file-word" role="img" aria-label="Download .docx">
</i> 
<span class="menu-text"></span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
          <div class="quarto-navbar-tools">
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../../parts/results.html">Main Results</a></li><li class="breadcrumb-item"><a href="../../parts/results/results_trifunctionalProteomics.html"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">An Affinity for Sphingolipids</span></a></li></ol></nav>
        <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Certificate of Approval</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Acknowledgements.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Acknowledgements</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Glossary.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Glossary</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../parts/introduction.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Introduction: TB and the Sphinx</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/introduction/intro_Layperson.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Introduction Part 1</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/introduction/intro_Scientific.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Introduction Part 2</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../parts/results.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Main Results</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/results/results_Mtb_phagocytosis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Breaking and Entering</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/results/results_Mtb_Membrane_Damage_V2.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Sealing the Gaps</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/results/results_Mtb_lipidomics_v2.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Living Off the Fat of the Cell</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/results/results_trifunctionalProteomics.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">An Affinity for Sphingolipids</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../parts/addenda.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Additional work</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Addenda/addenda_SLs_GeneticTools.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Genetic tools to study sphingolipids in disease</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Addenda/addenda_SLs_Fungus.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">The Squeaky Yeast Gets Greased</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Addenda/addenda_MethodsForVisualizingPhagocytosis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Rapid visualization and quantification of phagocytosis in neutrophils</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Addenda/addenda_Celegans_SMS_family.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Systematic analysis of the sphingomyelin synthase family in <em>C. elegans</em></span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../parts/discussionInterstitial.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Final thoughts &amp; future directions</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-4" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Discussion/discussion.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Discussion</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/references.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">References</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/about.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">About</span></a>
  </div>
</li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#abstract" id="toc-abstract" class="nav-link active" data-scroll-target="#abstract"><span class="header-section-number">6.1</span> Abstract</a></li>
  <li><a href="#introduction" id="toc-introduction" class="nav-link" data-scroll-target="#introduction"><span class="header-section-number">6.2</span> Introduction</a></li>
  <li><a href="#results" id="toc-results" class="nav-link" data-scroll-target="#results"><span class="header-section-number">6.3</span> Results</a>
  <ul class="collapse">
  <li><a href="#trifunctional-lipid-probes-offer-unique-utility-in-assessing-functions-localization-and-interaction-partners-of-a-given-lipid-species" id="toc-trifunctional-lipid-probes-offer-unique-utility-in-assessing-functions-localization-and-interaction-partners-of-a-given-lipid-species" class="nav-link" data-scroll-target="#trifunctional-lipid-probes-offer-unique-utility-in-assessing-functions-localization-and-interaction-partners-of-a-given-lipid-species"><span class="header-section-number">6.3.1</span> Trifunctional lipid probes offer unique utility in assessing functions, localization, and interaction partners of a given lipid species</a></li>
  <li><a href="#trifunctional-probes-reveal-protein-lipid-interactions-of-the-thp-1-macrophage" id="toc-trifunctional-probes-reveal-protein-lipid-interactions-of-the-thp-1-macrophage" class="nav-link" data-scroll-target="#trifunctional-probes-reveal-protein-lipid-interactions-of-the-thp-1-macrophage"><span class="header-section-number">6.3.2</span> Trifunctional probes reveal protein-lipid interactions of the THP-1 macrophage</a></li>
  <li><a href="#proteomics-of-trifunctional-probes-reveals-protein-sphingolipid-interactions-during-pathogenic-mtb-infection" id="toc-proteomics-of-trifunctional-probes-reveals-protein-sphingolipid-interactions-during-pathogenic-mtb-infection" class="nav-link" data-scroll-target="#proteomics-of-trifunctional-probes-reveals-protein-sphingolipid-interactions-during-pathogenic-mtb-infection"><span class="header-section-number">6.3.3</span> Proteomics of trifunctional probes reveals protein-sphingolipid interactions during pathogenic <em>Mtb</em> infection</a></li>
  </ul></li>
  <li><a href="#pathogenic-versus-attenuated-mtb-infections-induce-highly-similar-changes-to-the-tf-sph-interactome" id="toc-pathogenic-versus-attenuated-mtb-infections-induce-highly-similar-changes-to-the-tf-sph-interactome" class="nav-link" data-scroll-target="#pathogenic-versus-attenuated-mtb-infections-induce-highly-similar-changes-to-the-tf-sph-interactome"><span class="header-section-number">6.4</span> Pathogenic versus attenuated <em>Mtb</em> infections induce highly similar changes to the tf-Sph interactome</a></li>
  <li><a href="#mtb-protein-rv3466-is-enriched-by-tf-sphingosine-during-infection" id="toc-mtb-protein-rv3466-is-enriched-by-tf-sphingosine-during-infection" class="nav-link" data-scroll-target="#mtb-protein-rv3466-is-enriched-by-tf-sphingosine-during-infection"><span class="header-section-number">6.5</span> <em>Mtb</em> protein Rv3466 is enriched by tf-Sphingosine during infection</a></li>
  <li><a href="#discussion" id="toc-discussion" class="nav-link" data-scroll-target="#discussion"><span class="header-section-number">6.6</span> Discussion</a></li>
  <li><a href="#materials-and-methods" id="toc-materials-and-methods" class="nav-link" data-scroll-target="#materials-and-methods"><span class="header-section-number">6.7</span> Materials and Methods</a>
  <ul class="collapse">
  <li><a href="#thp-1-cell-culturing-and-plating" id="toc-thp-1-cell-culturing-and-plating" class="nav-link" data-scroll-target="#thp-1-cell-culturing-and-plating"><span class="header-section-number">6.7.1</span> THP-1 cell culturing and plating</a></li>
  <li><a href="#mycobacterium-tuberculosis-culturing-and-infection" id="toc-mycobacterium-tuberculosis-culturing-and-infection" class="nav-link" data-scroll-target="#mycobacterium-tuberculosis-culturing-and-infection"><span class="header-section-number">6.7.2</span> Mycobacterium tuberculosis culturing and infection</a></li>
  <li><a href="#sec-TrifunctionalProteomics-Methods-ProbeTreatment" id="toc-sec-TrifunctionalProteomics-Methods-ProbeTreatment" class="nav-link" data-scroll-target="#sec-TrifunctionalProteomics-Methods-ProbeTreatment"><span class="header-section-number">6.7.3</span> Trifunctionalized lipid treatments, uncaging, crosslinking</a></li>
  <li><a href="#cell-lysis-and-protein-collection" id="toc-cell-lysis-and-protein-collection" class="nav-link" data-scroll-target="#cell-lysis-and-protein-collection"><span class="header-section-number">6.7.4</span> Cell lysis and protein collection</a></li>
  <li><a href="#sec-TrifunctionalProteomics-Methods-IGFClick" id="toc-sec-TrifunctionalProteomics-Methods-IGFClick" class="nav-link" data-scroll-target="#sec-TrifunctionalProteomics-Methods-IGFClick"><span class="header-section-number">6.7.5</span> Fluorescence click reaction &amp; In-gel/on-blot fluorescence</a></li>
  <li><a href="#affinity-purification-of-probe-bound-proteins" id="toc-affinity-purification-of-probe-bound-proteins" class="nav-link" data-scroll-target="#affinity-purification-of-probe-bound-proteins"><span class="header-section-number">6.7.6</span> Affinity purification of probe-bound proteins</a></li>
  <li><a href="#liquid-chromatography-and-tandem-mass-spectrometry-analysis-lc-msms" id="toc-liquid-chromatography-and-tandem-mass-spectrometry-analysis-lc-msms" class="nav-link" data-scroll-target="#liquid-chromatography-and-tandem-mass-spectrometry-analysis-lc-msms"><span class="header-section-number">6.7.7</span> Liquid chromatography and tandem mass spectrometry analysis (LC-MS/MS)</a></li>
  <li><a href="#data-analysis" id="toc-data-analysis" class="nav-link" data-scroll-target="#data-analysis"><span class="header-section-number">6.7.8</span> Data analysis</a></li>
  </ul></li>
  <li><a href="#contributions" id="toc-contributions" class="nav-link" data-scroll-target="#contributions"><span class="header-section-number">6.8</span> Contributions</a></li>
  <li><a href="#supplemental-data" id="toc-supplemental-data" class="nav-link" data-scroll-target="#supplemental-data"><span class="header-section-number">6.9</span> Supplemental Data</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../../parts/results.html">Main Results</a></li><li class="breadcrumb-item"><a href="../../parts/results/results_trifunctionalProteomics.html"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">An Affinity for Sphingolipids</span></a></li></ol></nav>
<div class="quarto-title">
<h1 class="title"><span id="sec-TrifunctionalProteomics" class="quarto-section-identifier"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">An Affinity for Sphingolipids</span></span></h1>
<p class="subtitle lead">Trifunctional lipid probes reveal protein-sphingolipid interactions during <em>Mycobacterium tuberculosis</em> infection</p>
</div>


<div class="quarto-title-meta-author">
  <div class="quarto-title-meta-heading">Authors</div>
  <div class="quarto-title-meta-heading">Affiliations</div>
  
    <div class="quarto-title-meta-contents">
    <p class="author">Gaelen Guzman </p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Department of Molecular Microbiology and Immunology, Oregon Health &amp; Science University, Portland, Oregon, USA
          </p>
      </div>
    <div class="quarto-title-meta-contents">
    <p class="author">Scotland E. Farley </p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Department of Molecular Microbiology and Immunology, Oregon Health &amp; Science University, Portland, Oregon, USA
          </p>
      </div>
    <div class="quarto-title-meta-contents">
    <p class="author">Frank Stein </p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Proteomics Core, European Molecular Biology Laboratory, Heidelberg, Germany
          </p>
      </div>
    <div class="quarto-title-meta-contents">
    <p class="author">Per Haberkant </p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Proteomics Core, European Molecular Biology Laboratory, Heidelberg, Germany
          </p>
      </div>
    <div class="quarto-title-meta-contents">
    <p class="author">Carsten Schultz </p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Department of Chemical Physiology and Biochemistry, Oregon Health &amp; Science University, Portland, Oregon, USA
          </p>
      </div>
    <div class="quarto-title-meta-contents">
    <p class="author">Fikadu G. Tafesse <a href="mailto:tafesse@ohsu.edu" class="quarto-title-author-email"><i class="bi bi-envelope"></i></a> </p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Department of Molecular Microbiology and Immunology, Oregon Health &amp; Science University, Portland, Oregon, USA
          </p>
      </div>
  </div>

<div class="quarto-title-meta">

      
  
    
  </div>
  


</header>


<section id="abstract" class="level2" data-number="6.1">
<h2 data-number="6.1" class="anchored" data-anchor-id="abstract"><span class="header-section-number">6.1</span> Abstract</h2>
<p>Understanding the molecular interactions underlying <em>Mtb</em>’s capacity for immune subversion and hijacking of the host cell can provide a unique insight into potential new routes of antitubercular therapy. Because <em>Mtb</em> is a pathogen with a unique sensitivity to the host lipid compartment, and because sphingolipids are a lipid family with emerging roles in regulating antimicrobial functions in macrophages, we sought to characterize the sphingolipid interactome of an <em>Mtb</em>-infected cell. To this end, we utilized trifunctionalized lipid analogs as affinity handles for enrichment proteomics. We found that <em>Mtb</em> infection induces a significant reprogramming of proteins that interact with sphingosine. These changes included a downregulation of interactions among proteins involved in secretory trafficking and lysosomal maturation and an upregulation of interactions with cytoskeletal proteins. We further identify a candidate <em>Mtb</em> protein that selectively interacts with trifunctional sphingosine. This analysis is the first of its kind and represents a crucial step in characterizing how sphingolipids mediate the cellular response to intracellular infection.</p>
<p><em>To accompany the main body of this report, we present an interactive format to explore the results of this study at the following:</em> <a href="https://gaelen-guzman.shinyapps.io/LipidInteractomicsViewer/" target="_blank">Trifunctional Proteomics Shiny App</a>.</p>
</section>
<section id="introduction" class="level2" data-number="6.2">
<h2 data-number="6.2" class="anchored" data-anchor-id="introduction"><span class="header-section-number">6.2</span> Introduction</h2>
<p>As discussed in <a href="../introduction/intro_Scientific.html" class="quarto-xref"><span>Chapter 2</span></a>, <em>Mycobacterium tuberculosis</em> (<em>Mtb</em>) is a globally significant human pathogen. In the face of spreading antibiotic-resistant strains of <em>Mtb</em>, there is a growing need for novel host-directed therapy (HDT). Given their extensive roles in regulating mammalian cell survival and apoptosis, we and others have proposed sphingolipids as a strong candidate for anti-tubercular HDT (<a href="results_Mtb_phagocytosis.html" class="quarto-xref"><span>Chapter 3</span></a> and <a href="results_Mtb_Membrane_Damage_V2.html" class="quarto-xref"><span>Chapter 4</span></a>)<span class="citation" data-cites="sharmaSphingolipidsAreDual2017 mohammedSphingolipidsMycobacteriaHost2022"><sup><a href="../references.html#ref-sharmaSphingolipidsAreDual2017" role="doc-biblioref">1</a>,<a href="../references.html#ref-mohammedSphingolipidsMycobacteriaHost2022" role="doc-biblioref">2</a></sup></span>. Following this line of study, we sought to address whether the interactions between sphingolipids and proteins in the host cell are potential intervention points in host-directed therapy.</p>
<p>Lipid dynamics are a significant determinant of <em>Mtb</em> biology, as discussed in Barisch and Soldati’s 2017 review<span class="citation" data-cites="barischBreakingFatHow2017"><sup><a href="../references.html#ref-barischBreakingFatHow2017" role="doc-biblioref">3</a></sup></span>. We and others have shown that host lipids, such as diacyl- and triacylglycerides, and lipid storage structures, such as lipid droplets, are substantially upregulated in response to <em>Mtb</em> infection (<a href="results_Mtb_lipidomics_v2.html" class="quarto-xref"><span>Chapter 5</span></a>)<span class="citation" data-cites="barischBreakingFatHow2017"><sup><a href="../references.html#ref-barischBreakingFatHow2017" role="doc-biblioref">3</a></sup></span>. Previous authors have postulated that the sphingolipid pathway contains numerous potential targets for anti-<em>Mtb</em> HDT<span class="citation" data-cites="mohammedSphingolipidsMycobacteriaHost2022 malikCuttingEdgeMycobacterium2003a sharmaSphingolipidsAreDual2017"><sup><a href="../references.html#ref-sharmaSphingolipidsAreDual2017" role="doc-biblioref">1</a>,<a href="../references.html#ref-mohammedSphingolipidsMycobacteriaHost2022" role="doc-biblioref">2</a>,<a href="../references.html#ref-malikCuttingEdgeMycobacterium2003a" role="doc-biblioref">4</a></sup></span>. However, there are significant open questions regarding whether the interactions between sphingolipids and proteins in the host cell are potential intervention points in host-directed therapy. We use a trifunctionalized sphingosine analog to explore the interactions between sphingolipids and proteins during <em>Mtb</em> infection. We expect that the <em>Mtb</em>-host interface will be highly dependent on protein-lipid interactions – but, historically, few tools have existed which allow for identifying the protein interactors of a given lipid species.</p>
<p>The advent of trifunctionalized lipids has circumvented many of the long-standing challenges of lipid studies. These challenges have been discussed at length elsewhere<span class="citation" data-cites="schultzChemicalToolsLipid2023"><sup><a href="../references.html#ref-schultzChemicalToolsLipid2023" role="doc-biblioref">5</a></sup></span>. However, they include 1. the small size and potent hydrophobicity of lipids, which preclude many chemical modifications and exogenous supplementation or add-back; 2. the rapid interconversion of lipid species along complex and convoluted metabolic pathways; and 3. the transience of interactions between lipids and proteins<span class="citation" data-cites="farleyTrifunctionalSphinganineNew2024 hoglingerTrifunctionalLipidProbes2017 schultzChemicalToolsLipid2023"><sup><a href="../references.html#ref-schultzChemicalToolsLipid2023" role="doc-biblioref">5</a>–<a href="../references.html#ref-hoglingerTrifunctionalLipidProbes2017" role="doc-biblioref">7</a></sup></span>. In particular, trifunctional sphingosine (tf-Sph) has enabled significant breakthroughs in studying the roles of bioactive sphingolipids in the cell. The native form of sphingosine is a transient and potent signaling lipid known to induce profound effects in the cell[<span class="citation" data-cites="weigertSphingosine1PhosphateMacrophageBiology2019"><sup><a href="../references.html#ref-weigertSphingosine1PhosphateMacrophageBiology2019" role="doc-biblioref">8</a></sup></span>; ]. In 2015 and 2017, Höglinger showed that trifunctional sphingosine induces rapid calcium release from the lysosome and eventual lysophagy (targeted autophagy of the lysosome)<span class="citation" data-cites="hoglingerIntracellularSphingosineReleases2015a hoglingerTrifunctionalLipidProbes2017"><sup><a href="../references.html#ref-hoglingerTrifunctionalLipidProbes2017" role="doc-biblioref">7</a>,<a href="../references.html#ref-hoglingerIntracellularSphingosineReleases2015a" role="doc-biblioref">9</a></sup></span>. Our recent report described a novel sphingolipid probe’s synthesis and biological behavior: trifunctional sphinganine (tf-Spa)<span class="citation" data-cites="farleyTrifunctionalSphinganineNew2024"><sup><a href="../references.html#ref-farleyTrifunctionalSphinganineNew2024" role="doc-biblioref">6</a></sup></span>. Tf-Spa Trifunctional lipids allow for nuanced investigation into the protein interaction partners of select sphingolipid species in various biological contexts.</p>
<p>In the present study, we utilize trifunctional sphingosine to explore selective interactions between proteins and sphingolipids in human THP-1 macrophages during infection with pathogenic and attenuated strains of <em>Mycobacterium tuberculosis</em>. To assess the specificity of identified interactions, we compare the interaction partners of tf-Sph with those of several other previously reported trifunctional probes. We find that, at baseline, sphingosine interacts with several lipid-modifying proteins such as phosphatidylinositol transfer protein beta (PITPNB) and lysosomal proteins such as Cathepsin D (CTSD) and lysosomal Pro-X carboxypeptidase (PRCP), among others. When applied to the <em>Mtb</em>-infected cell, we find that tf-Sph experiences a dramatically altered interactome – the many up- and down-regulated interactions reflect altered lysosomal activity, protein synthesis, and histone modifications, among others. We additionally identify modest enrichment of a single <em>Mtb</em> protein to tf-Sph, the uncharacterized Rv3466. We hope that revealing these alterations to the sphingosine-protein interactome will reveal novel avenues for manipulating the host-pathogen interface during <em>Mtb</em> infection.</p>
</section>
<section id="results" class="level2" data-number="6.3">
<h2 data-number="6.3" class="anchored" data-anchor-id="results"><span class="header-section-number">6.3</span> Results</h2>
<section id="trifunctional-lipid-probes-offer-unique-utility-in-assessing-functions-localization-and-interaction-partners-of-a-given-lipid-species" class="level3" data-number="6.3.1">
<h3 data-number="6.3.1" class="anchored" data-anchor-id="trifunctional-lipid-probes-offer-unique-utility-in-assessing-functions-localization-and-interaction-partners-of-a-given-lipid-species"><span class="header-section-number">6.3.1</span> Trifunctional lipid probes offer unique utility in assessing functions, localization, and interaction partners of a given lipid species</h3>
<p>The lipid biologist’s toolbox was recently greatly expanded by adding trifunctional lipid analogs (trifunctional probes). These synthetic molecules maintain high structural similarities to their natural counterparts but have been modified with three chemical domains that allow for nuanced and tightly controlled dissection of a given lipid’s cellular trafficking, metabolism, and interactions (<a href="#fig-TrifunctionalProteomics-Fig1" class="quarto-xref">Figure&nbsp;<span>6.1</span></a>). We and others have previously reported on their synthesis and utility, including Farley et al.<span class="citation" data-cites="farleyTrifunctionalSphinganineNew2024"><sup><a href="../references.html#ref-farleyTrifunctionalSphinganineNew2024" role="doc-biblioref">6</a></sup></span>, Haberkant et al.<span class="citation" data-cites="haberkantBifunctionalSphingosineCellBased2016a haberkantFatFabulousBifunctional2014 haberkantProteinsphingolipidInteractionsCellular2008"><sup><a href="../references.html#ref-haberkantBifunctionalSphingosineCellBased2016a" role="doc-biblioref">10</a>–<a href="../references.html#ref-haberkantProteinsphingolipidInteractionsCellular2008" role="doc-biblioref">12</a></sup></span>, and Höglinger et al.<span class="citation" data-cites="hoglingerIntracellularSphingosineReleases2015a hoglingerTrifunctionalLipidProbes2017"><sup><a href="../references.html#ref-hoglingerTrifunctionalLipidProbes2017" role="doc-biblioref">7</a>,<a href="../references.html#ref-hoglingerIntracellularSphingosineReleases2015a" role="doc-biblioref">9</a></sup></span>. While many such trifunctional probes are now available, we consigned our focus to four probes in this study: trifunctional-Sphingosine (tf-Sph), tf-Sphinganine (tf-Spa), trifunctional-8:3-Fatty acid (tf-8:3-FA), and trifunctional-1:10-Fatty acid (tf-1:10-FA), as previously described by Farley et al.&nbsp;in 2023<span class="citation" data-cites="farleyTrifunctionalSphinganineNew2024"><sup><a href="../references.html#ref-farleyTrifunctionalSphinganineNew2024" role="doc-biblioref">6</a></sup></span>.</p>
<div class="border">
<div id="fig-TrifunctionalProteomics-Fig1" class="quarto-figure quarto-figure-center quarto-float anchored" data-group="results_TrifunctionalProteomics">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-TrifunctionalProteomics-Fig1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_TrifunctionalProteomics/300ppi/Figure1_ProbeOverview.png" class="lightbox" data-glightbox="description: .lightbox-desc-1" data-gallery="results_TrifunctionalProteomics"><img src="../../parts/results/results_TrifunctionalProteomics/300ppi/Figure1_ProbeOverview.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-TrifunctionalProteomics-Fig1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;6.1: <strong>Lipid probe overview &amp; workflows</strong> (A) The chemical structure of native sphinganine, sphingosine, and free fatty acid, alongside their synthetic trifunctionalized analogs. The functional moieties are highlighted and described in the color-coordinated boxes. (B) A simplified overview of several trifunctional probe workflows. See <a href="#sec-TrifunctionalProteomics-Methods-ProbeTreatment" class="quarto-xref">Methods Section&nbsp;<span>6.7.3</span></a> for a detailed description of the workflow. Briefly, plated cells are treated with lipid probes for 30 minutes (1), uncaged via irradiation with blue-wavelength light (2), and then covalently crosslinked to nearby molecules via irradiation with UV-wavelength light (3). Next, the probe may be clicked to a fluorophore and stained with cellular markers for fluorescence microscopy (4) or lysed and protein collected for in-gel fluorescence or proteomics analysis (5). (C) Example uses of trifunctional probes: (left panel) fluorescence microscopy using tf-Spa; (middle panel) in-gel fluorescence using tf-Spa; (right panel) cartoon depiction of an MS2 scan of a peptide tagged with a tandem mass tag.
</figcaption>
</figure>
</div>
</div>
<p>Each of these probes is expected to elicit a distinct cellular response, traffic to a distinct cellular compartment, and interact with a distinct cohort of proteins. For example, the two fatty acid probes differ only in the position of the diazirine ring: tf-8:3-FA is expected to preferentially bind to proteins that penetrate deep into the membrane, such as transmembrane domain-containing proteins. In contrast, the tf-1:10-FA probe would be expected to enrich more transiently membrane-associated proteins because the diazirine crosslinking motif is nearer to the hydrophilic headgroup. We expect these fatty acid probes to be incorporated into a wide array of lipids, including phospholipids, glycerolipids, and sphingolipids via transacylation.</p>
<p>In contrast, the trifunctionalized sphingolipid probes are expected to be incorporated into the sphingolipid metabolic pathway primarily. These probes differ only in a single double carbon bond at the C4-C5 position: sphinganine (or dihydrosphingosine) is produced early in the sphingolipid synthesis pathway and is incorporated into dihydroceramide by the ceramide synthases before being desaturated by dihydroceramide Δ4-desaturase<span class="citation" data-cites="gaultOverviewSphingolipidMetabolism2010a"><sup><a href="../references.html#ref-gaultOverviewSphingolipidMetabolism2010a" role="doc-biblioref">13</a></sup></span> and rarely accumulates in a typical cell. In contrast, sphingosine is a well-characterized signaling lipid produced primarily via the degradation of ceramide in response to signaling stimuli<span class="citation" data-cites="gaultOverviewSphingolipidMetabolism2010a"><sup><a href="../references.html#ref-gaultOverviewSphingolipidMetabolism2010a" role="doc-biblioref">13</a></sup></span>. The production of sphingosine stimulates profound immediate effects in the cell<span class="citation" data-cites="hoglingerIntracellularSphingosineReleases2015a"><sup><a href="../references.html#ref-hoglingerIntracellularSphingosineReleases2015a" role="doc-biblioref">9</a></sup></span> and can also be used in ceramide synthesis as a part of the sphingolipid salvage pathway<span class="citation" data-cites="gaultOverviewSphingolipidMetabolism2010a"><sup><a href="../references.html#ref-gaultOverviewSphingolipidMetabolism2010a" role="doc-biblioref">13</a></sup></span>. We previously reported on the subcellular localization of both trifunctional sphinganine and sphingosine<span class="citation" data-cites="farleyTrifunctionalSphinganineNew2024"><sup><a href="../references.html#ref-farleyTrifunctionalSphinganineNew2024" role="doc-biblioref">6</a></sup></span>. In this report, Farley found that both probes are trafficked through the endoplasmic reticulum and the Golgi, aligning with the well-described sphingolipid synthesis pathway<span class="citation" data-cites="farleyTrifunctionalSphinganineNew2024 gaultOverviewSphingolipidMetabolism2010a"><sup><a href="../references.html#ref-farleyTrifunctionalSphinganineNew2024" role="doc-biblioref">6</a>,<a href="../references.html#ref-gaultOverviewSphingolipidMetabolism2010a" role="doc-biblioref">13</a></sup></span>. In the same report, Farley et al.&nbsp;also showed that tf-Sph retains its biological fidelity: it induces lysosomal calcium release, as previously shown<span class="citation" data-cites="hoglingerIntracellularSphingosineReleases2015a"><sup><a href="../references.html#ref-hoglingerIntracellularSphingosineReleases2015a" role="doc-biblioref">9</a></sup></span>. Notably, however, tf-Spa fails to induce calcium release. These results indicate that tf-Sph retains its activity as a signaling lipid, whereas tf-Spa is primarily a sphingolipid precursor. We thus expect that the products of tf-Spa metabolism will interact with many proteins and that tf-Sph will initially interact with proteins involved in lipid signaling before being trafficked to the ER<span class="citation" data-cites="haberkantBifunctionalSphingosineCellBased2016a farleyTrifunctionalSphinganineNew2024"><sup><a href="../references.html#ref-farleyTrifunctionalSphinganineNew2024" role="doc-biblioref">6</a>,<a href="../references.html#ref-haberkantBifunctionalSphingosineCellBased2016a" role="doc-biblioref">10</a></sup></span>.</p>
<div class="border">
<div id="fig-TrifunctionalProteomics-Fig2" class="quarto-figure quarto-figure-center quarto-float anchored" data-group="results_TrifunctionalProteomics">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-TrifunctionalProteomics-Fig2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_TrifunctionalProteomics/300ppi/Figure2_SphinganineMIcroscopy.png" class="lightbox" data-glightbox="description: .lightbox-desc-2" data-gallery="results_TrifunctionalProteomics"><img src="../../parts/results/results_TrifunctionalProteomics/300ppi/Figure2_SphinganineMIcroscopy.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-TrifunctionalProteomics-Fig2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;6.2: <strong>Fluorescence microscopy reveals intracellular trafficking of trifunctional sphinganine</strong> THP-1 cells were treated with trifunctional sphinganine, uncaged via irradiation with blue light, and incubated for the denoted periods before UV-induced crosslinking and methanol fixation. Cells were stained with anti-Giantin antibody (to visualize the Golgi apparatus) and DAPI (to visualize the nucleus). Cells were imaged using a Zeiss LSM 900 confocal microscope using AiryScan super-resolution and a 63x oil-immersion objective. Prior to imaging, cells were stained with 1:500 mouse anti-Giantin primary antibody (abcam #ab37266) and 1:1000 goat anti-mouse Alexa488-conjugated secondary antibody (Invitrogen #A21042) (See <a href="results_Mtb_Membrane_Damage_V2.html#sec-MtbDamage-Methods-MicroscopyStaining" class="quarto-xref">Methods Section&nbsp;<span>4.5.5</span></a> for staining protocol).
</figcaption>
</figure>
</div>
</div>
<p>To validate these trafficking patterns in THP-1 cells, we visualized the subcellular localization of tf-Spa in THP-1 cells at increasing times after photo-uncaging (<a href="#fig-TrifunctionalProteomics-Fig2" class="quarto-xref">Figure&nbsp;<span>6.2</span></a>). We observe similar patterns in these cells as has been previously reported by Farley et al.: tf-Spa experiences a time-dependent enrichment to the Golgi apparatus. These images serve primarily as validation of the utility of trifunctional probes.</p>
</section>
<section id="trifunctional-probes-reveal-protein-lipid-interactions-of-the-thp-1-macrophage" class="level3" data-number="6.3.2">
<h3 data-number="6.3.2" class="anchored" data-anchor-id="trifunctional-probes-reveal-protein-lipid-interactions-of-the-thp-1-macrophage"><span class="header-section-number">6.3.2</span> Trifunctional probes reveal protein-lipid interactions of the THP-1 macrophage</h3>
<p>To identify the proteins that selectively interact with sphingolipid precursors in activated THP-1 macrophages, we performed an initial proteomics analysis to determine the protein interaction partners of the four trifunctionalized lipid probes: tf-Sph, tf-Spa, tf-8:3-FA, and tf-1:10-FA. Previous studies have used these or similar probes in human cervical adenocarcinoma cells (HeLa) and human liver epithelial cells (Huh7)<span class="citation" data-cites="hoglingerIntracellularSphingosineReleases2015a hoglingerTrifunctionalLipidProbes2017 farleyTrifunctionalSphinganineNew2024"><sup><a href="../references.html#ref-farleyTrifunctionalSphinganineNew2024" role="doc-biblioref">6</a>,<a href="../references.html#ref-hoglingerTrifunctionalLipidProbes2017" role="doc-biblioref">7</a>,<a href="../references.html#ref-hoglingerIntracellularSphingosineReleases2015a" role="doc-biblioref">9</a></sup></span>. However, human monocyte-derived macrophages (THP-1) have not been subjected to such lipid interaction analysis. Given that macrophages have dramatically distinct roles from epithelial cells and adenocarcinoma cells, we expected to identify a unique cohort of lipid-interacting proteins in these cells – including, presumably, (immuno)proteasome subunits, lysosomal resident proteins, and proteins involved in free radical production. The probe treatment and affinity enrichment procedures are depicted in <a href="#fig-TrifunctionalProteomics-Fig3" class="quarto-xref">Figure&nbsp;<span>6.3</span></a>A and described in detail in <a href="#sec-TrifunctionalProteomics-Methods-ProbeTreatment" class="quarto-xref">Methods Section&nbsp;<span>6.7.3</span></a>. All treatment and sample preparation were performed as previously described<span class="citation" data-cites="farleyTrifunctionalSphinganineNew2024 haberkantBifunctionalSphingosineCellBased2016a hoglingerTrifunctionalLipidProbes2017"><sup><a href="../references.html#ref-farleyTrifunctionalSphinganineNew2024" role="doc-biblioref">6</a>,<a href="../references.html#ref-hoglingerTrifunctionalLipidProbes2017" role="doc-biblioref">7</a>,<a href="../references.html#ref-haberkantBifunctionalSphingosineCellBased2016a" role="doc-biblioref">10</a></sup></span>. As a negative control, we prepared samples treated with each probe and uncaged but not treated with UV (-UV) and thus lacked crosslinking-stabilized interactions during enrichment.</p>
<p>We performed quantitative proteomics analysis using the TMT-16 isotopic labeling platform and LC-MS/MS. PCA analysis showed a distinct separation between +UV and -UV samples for each probe (<a href="#suppfig-TrifunctionalProteomics-FigS1" class="quarto-xref">Figure S<span>6.1</span></a> A). We used variance stabilization to normalize the raw signal intensity of each channel (<a href="#suppfig-TrifunctionalProteomics-FigS1" class="quarto-xref">Figure S<span>6.1</span></a> B). We identified 869 unique proteins with at least two unique peptides in at least two replicates. To identify the fold-change in the enrichment of proteins via diazirine crosslinking, we took the +UV versus -UV ratio for each probe and log2 transformed (Log2 fold-change, Log2FC). These results are depicted in <a href="#fig-TrifunctionalProteomics-Fig3" class="quarto-xref">Figure&nbsp;<span>6.3</span></a>B.</p>
<div class="border">
<div id="fig-TrifunctionalProteomics-Fig3" class="quarto-figure quarto-figure-center quarto-float anchored" data-group="results_TrifunctionalProteomics">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-TrifunctionalProteomics-Fig3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_TrifunctionalProteomics/300ppi/Figure3_BaselineProteomics.png" class="lightbox" data-glightbox="description: .lightbox-desc-3" data-gallery="results_TrifunctionalProteomics"><img src="../../parts/results/results_TrifunctionalProteomics/300ppi/Figure3_BaselineProteomics.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-TrifunctionalProteomics-Fig3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;6.3: <strong>Affinity enrichment proteomics reveals the protein interactors of trifunctional lipid probes</strong> (A) Schematic depiction of experimental design. Briefly, THP-1 cells were treated with the denoted probes described in <a href="#sec-TrifunctionalProteomics-Methods-ProbeTreatment" class="quarto-xref">Methods Section&nbsp;<span>6.7.3</span></a>. Cells were lysed, and proteins were extracted; probe-protein conjugates were enriched using azide-coated beads and subjected to mass spectrometry analysis. (B) Volcano plots depict the relative enrichment of proteins to each probe compared to probe-matched no-crosslinking (-UV) control lysate. Enrichment is defined as follows: Depleted = Log2FC &lt; 0, Partially enriched = Log2FC &gt; 0 &amp; p-value &gt; 0.05, Significantly enriched = Log2FC &gt; 1 &amp; p-value &lt; 0.05.
</figcaption>
</figure>
</div>
</div>
<p>We applied Limma analysis<span class="citation" data-cites="ritchieLimmaPowersDifferential2015"><sup><a href="../references.html#ref-ritchieLimmaPowersDifferential2015" role="doc-biblioref">14</a></sup></span> to attain statistical significance for each protein between the +UV and -UV conditions. We applied an arbitrary cutoff for significance: LogFC &gt; 1 and p-value &lt; 0.05. Using these significance thresholds, we found that UV-crosslinking induced the enrichment of six proteins to the tf-Sph probe, twenty proteins to the tf-Spa probe, eight proteins to the tf-8:3-FA probe, and six proteins to the tf-1:10-FA probe. The relative enrichment of each significant protein among each probe (+UV versus -UV) is depicted in <a href="#fig-TrifunctionalProteomics-Fig4" class="quarto-xref">Figure&nbsp;<span>6.4</span></a>A. Some proteins, such as VDAC2 and PITPNB, are enriched to more than one probe; other proteins, such as ATP6V1A, are uniquely enriched to a single probe.</p>
<p>We manually clustered each of the 35 enriched proteins into the following groups: proteins containing transmembrane domains, proteins with reported membrane association, secreted proteins, and proteins localized to the mitochondria, nucleus, or ribosome. Many of these putative lipid-interacting proteins are transmembrane domain-containing or peripheral membrane proteins. Relatively few are cytosolic or secreted.</p>
<div class="border">
<div id="fig-TrifunctionalProteomics-Fig4" class="quarto-figure quarto-figure-center quarto-float anchored" data-group="results_TrifunctionalProteomics">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-TrifunctionalProteomics-Fig4-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_TrifunctionalProteomics/300ppi/Figure4-VDAC-OBF.png" class="lightbox" data-glightbox="description: .lightbox-desc-4" data-gallery="results_TrifunctionalProteomics"><img src="../../parts/results/results_TrifunctionalProteomics/300ppi/Figure4-VDAC-OBF.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-TrifunctionalProteomics-Fig4-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;6.4: <strong>Proteins significantly enriched to lipid probes are known interactors of lipids</strong> (A) Bar plots depict the Log2FC enrichment for each of the 26 significantly enriched proteins across all lipid probes (+UV versus -UV). Significance is denoted by fill coloring; probe is denoted by bar color. (B) Pie charts show the relative enrichment for transmembrane and membrane-associating proteins among the four lipid probes. (C) GO Term analysis of partially and significantly enriched proteins across each lipid probe. (D) Example on-blot fluorescence depicting simultaneous visualization of probe-crosslinked and antibody-stained VDAC proteins. (Red = Azide647-clicked probe; blue = anti-VDAC antibody).
</figcaption>
</figure>
</div>
</div>
<p>Our recent report investigated the sphingosine and sphinganine interacting proteins in Huh7 cells and characterized the similarities and differences between these two probes<span class="citation" data-cites="farleyTrifunctionalSphinganineNew2024"><sup><a href="../references.html#ref-farleyTrifunctionalSphinganineNew2024" role="doc-biblioref">6</a></sup></span>. For both probes, we find significant overlap among significantly enriched hits. In the case of tf-Sphinganine, proteins such as NUC1 and NUCB2 (also named HEL-S-109), Cathepsin D (CTSD), ECH1, PITPNB, VDAC 1 and VDAC2 were enriched in both Huh7 and THP-1 cells. In contrast, we see several proteins that appear to be unique to THP-1 monocytes compared to previously reported interactors; these include Prosaposin (PSAP) and Neudesin (NENF).</p>
<p>Among these enriched hits are several interesting proteins with previously reported lipid interactions. Among these are the mitochondrial voltage-dependent anion channel isoforms VDAC1, VDAC2, and VDAC3. Previous studies have shown that VDAC2 contains a <em>bona fide</em> ceramide binding site and that interaction with ceramide is a potent trigger of apoptosis<span class="citation" data-cites="dadsenaCeramidesBindVDAC22019a"><sup><a href="../references.html#ref-dadsenaCeramidesBindVDAC22019a" role="doc-biblioref">15</a></sup></span>. Further, Jahn et al.&nbsp;showed that VDAC1 and VDAC2 form a heterodimer which acts as a scramblase that transfers phospholipids synthesized in the ER into the inner mitochondrial membrane<span class="citation" data-cites="jahnPhospholipidsAreImported2023"><sup><a href="../references.html#ref-jahnPhospholipidsAreImported2023" role="doc-biblioref">16</a></sup></span>.</p>
<p>Notably, these enriched proteins include several Cathepsin aspartic peptidases. Cathepsins A and D are enriched to both tf-Sphingosine and tf-Sphinganine. Cathepsins L and Z are enriched to tf-8:3-FA and tf-1:10-FA, respectively. There are many reported interactions between Cathepsins and sphingolipids. For example, CTSB is activated by lysosomal ceramide generated via aSMase, a process involved in NK and T cell apoptosis<span class="citation" data-cites="taniguchiLysosomalCeramideGenerated2015"><sup><a href="../references.html#ref-taniguchiLysosomalCeramideGenerated2015" role="doc-biblioref">17</a></sup></span>. Similarly, sphingolipids are known regulators of CTSD activity<span class="citation" data-cites="heinrichCathepsinTargetedAcid2000 zebrakovskaComplexModulationPeptidolytic2011"><sup><a href="../references.html#ref-heinrichCathepsinTargetedAcid2000" role="doc-biblioref">18</a>,<a href="../references.html#ref-zebrakovskaComplexModulationPeptidolytic2011" role="doc-biblioref">19</a></sup></span>.</p>
<p>We also note the enrichment of Prosaposin to sphinganine and sphingosine. PSAP is a precursor polypeptide and is proteolytically cleaved into four small heat-stable glycoproteins: Saposins A, B, C, and D<span class="citation" data-cites="kishimotoSaposinsStructureFunction1992a"><sup><a href="../references.html#ref-kishimotoSaposinsStructureFunction1992a" role="doc-biblioref">20</a></sup></span>. Each of these subunits is approximately 80 amino acids, with conserved glycosylation sites and identically placed cysteines<span class="citation" data-cites="morimotoSaposinSecondCerebrosidase1989 kishimotoSaposinsStructureFunction1992a"><sup><a href="../references.html#ref-kishimotoSaposinsStructureFunction1992a" role="doc-biblioref">20</a>,<a href="../references.html#ref-morimotoSaposinSecondCerebrosidase1989" role="doc-biblioref">21</a></sup></span>. Each Saposin appears to bind to a distinct cohort of higher sphingolipids. For example, Saposin A binds to glycosphingolipids, including glucocerebroside and galactocerebroside – and further, Saposin A stimulates acid β-glucosidase<span class="citation" data-cites="kishimotoSaposinsStructureFunction1992a"><sup><a href="../references.html#ref-kishimotoSaposinsStructureFunction1992a" role="doc-biblioref">20</a></sup></span>. aSMase activity is regulated by the binding of Saposin<span class="citation" data-cites="xiongStructureHumanAcid2016"><sup><a href="../references.html#ref-xiongStructureHumanAcid2016" role="doc-biblioref">22</a></sup></span>. Notably, Saposins have been shown to play critical roles in regulating lysosomal activity and immune function. Confusingly, there appears to be a cyclic regulation between Cathepsins and PSAP<span class="citation" data-cites="kimLysosomalCeramidesRegulate2022"><sup><a href="../references.html#ref-kimLysosomalCeramidesRegulate2022" role="doc-biblioref">23</a></sup></span>. Cathepsins are the primary lysosomal peptidases responsible for cleaving PSAP into the mature Saposin subunits, and yet Saposins are widely reported for their capacity to stimulate CTSD activity<span class="citation" data-cites="kimLysosomalCeramidesRegulate2022 heinrichCathepsinTargetedAcid2000"><sup><a href="../references.html#ref-heinrichCathepsinTargetedAcid2000" role="doc-biblioref">18</a>,<a href="../references.html#ref-kimLysosomalCeramidesRegulate2022" role="doc-biblioref">23</a></sup></span>. What comes first? We find it intriguing that, in this proteomics analysis of the baseline interactions in THP-1 cells, we see “significant” enrichment of PSAP in sphinganine – however, PSAP was just below the Log2FC significance threshold.</p>
<p>These results reveal a diversity of lipid-interacting proteins in THP-1 monocyte-derived macrophages. Ongoing work seeks to validate and clarify these interactions’ biological impacts within the cell. For the remainder of this study, we used these results to inform a subsequent proteomics analysis on THP-1 macrophages infected with pathogenic and attenuated strains of <em>Mycobacterium tuberculosis</em>.</p>
</section>
<section id="proteomics-of-trifunctional-probes-reveals-protein-sphingolipid-interactions-during-pathogenic-mtb-infection" class="level3" data-number="6.3.3">
<h3 data-number="6.3.3" class="anchored" data-anchor-id="proteomics-of-trifunctional-probes-reveals-protein-sphingolipid-interactions-during-pathogenic-mtb-infection"><span class="header-section-number">6.3.3</span> Proteomics of trifunctional probes reveals protein-sphingolipid interactions during pathogenic <em>Mtb</em> infection</h3>
<p>The main goal of this study was to analyze sphingosine interactions during <em>Mtb</em> infection. To achieve this, we infected cells with the pathogenic <em>Mtb</em> strain H37Rv for 24 and 96 hours, as depicted in <a href="#fig-TrifunctionalProteomics-Fig5" class="quarto-xref">Figure&nbsp;<span>6.5</span></a>. Different multiplicities of infection were used for the two time points to maximize the infection rate at 24 hours and minimize cell cytotoxicity at 96 hours<a href="#suppfig-TrifunctionalProteomics-FigS3" class="quarto-xref">Figure S<span>6.3</span></a>. At the desired time points, cells were treated with tf-Sphingosine as described in <a href="#sec-TrifunctionalProteomics-Methods-ProbeTreatment" class="quarto-xref"><span>Section 6.7.3</span></a>. We prepared a parallel experiment using the tf-8:3-Fatty acid probe as a control for non-specific lipid-binding proteins. After UV-induced crosslinking, cell lysates were collected, and a portion was used for in-gel fluorescence to validate sample quality (<a href="#suppfig-TrifunctionalProteomics-FigS4" class="quarto-xref">Figure S<span>6.4</span></a> A). The remainder of each sample was clicked to azide-coated beads, and lipid-conjugated proteins were enriched. Samples were digested off the beads using trypsin and labeled with TMT-16 mass tags for LC-MS/MS proteomics analysis (one full TMT-16-plex for each probe). As above, data were normalized using variance stabilization (<a href="#suppfig-TrifunctionalProteomics-FigS4" class="quarto-xref">Figure S<span>6.4</span></a> C). PCA analysis shows a wide separation between infected and uninfected samples for both lipid probes (<a href="#suppfig-TrifunctionalProteomics-FigS3" class="quarto-xref">Figure S<span>6.3</span></a> B).</p>
<p>In total, there were 1169 unique proteins identified in the tf-Sphingosine-treated samples and 1411 unique proteins identified in the tf-8:3-FA-treated samples. Of these, 1039 were identified in both experiments, and 502 were identified in only one. We calculated the log2 fold-change of each protein for each possible comparison within TMT-16-plexes(<a href="#suppfig-TrifunctionalProteomics-FigS5" class="quarto-xref">Figure S<span>6.5</span></a>). We again performed statistical testing using Limma analysis. However, for this experiment, we applied a more stringent threshold for significance: a log2 fold-change &gt; 2, a false-discovery rate &lt; 0.05, and a p-value &lt; 0.01. The enrichment annotation of the baseline UV experiment was applied to the proteins identified in this infection experiment. As a notable exception, we included PSAP as a “Significant” protein in our UV annotation, as discussed below.</p>
<div class="border">
<div id="tbl-TrifunctionalProteomics-EnrichmentOverview" class="striped hover quarto-float anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-TrifunctionalProteomics-EnrichmentOverview-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;6.1: <strong>Overview of statistically significant proteins</strong> Hit = Log2FC &gt; 1, p-value &gt; 0.01, &amp; FDR &lt; 0.05.
</figcaption>
<div aria-describedby="tbl-TrifunctionalProteomics-EnrichmentOverview-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<table class="table-striped table-hover table">
<colgroup>
<col style="width: 14%">
<col style="width: 26%">
<col style="width: 10%">
<col style="width: 23%">
<col style="width: 23%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Probe</th>
<th style="text-align: left;">Infection (versus mock)</th>
<th style="text-align: center;">Timepoint</th>
<th style="text-align: center;"># Hits enriched</th>
<th style="text-align: center;"># Hits depleted</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">tf-Sph</td>
<td style="text-align: left;">H37Rv</td>
<td style="text-align: center;">24 hrs</td>
<td style="text-align: center;">31</td>
<td style="text-align: center;">120</td>
</tr>
<tr class="even">
<td style="text-align: center;">tf-Sph</td>
<td style="text-align: left;">H37Rv</td>
<td style="text-align: center;">96 hrs</td>
<td style="text-align: center;">35</td>
<td style="text-align: center;">109</td>
</tr>
<tr class="odd">
<td style="text-align: center;">tf-Sph</td>
<td style="text-align: left;">H37Rv-ΔRD1</td>
<td style="text-align: center;">24 hrs</td>
<td style="text-align: center;">19</td>
<td style="text-align: center;">78</td>
</tr>
<tr class="even">
<td style="text-align: center;">tf-Sph</td>
<td style="text-align: left;">H37Rv-ΔRD1</td>
<td style="text-align: center;">96 hrs</td>
<td style="text-align: center;">35</td>
<td style="text-align: center;">101</td>
</tr>
<tr class="odd">
<td style="text-align: center;"></td>
<td style="text-align: left;"></td>
<td style="text-align: center;"></td>
<td style="text-align: center;"></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: center;">tf-FA</td>
<td style="text-align: left;">H37Rv</td>
<td style="text-align: center;">24 hrs</td>
<td style="text-align: center;">30</td>
<td style="text-align: center;">140</td>
</tr>
<tr class="odd">
<td style="text-align: center;">tf-FA</td>
<td style="text-align: left;">H37Rv</td>
<td style="text-align: center;">96 hrs</td>
<td style="text-align: center;">23</td>
<td style="text-align: center;">86</td>
</tr>
<tr class="even">
<td style="text-align: center;">tf-FA</td>
<td style="text-align: left;">H37Rv-ΔRD1</td>
<td style="text-align: center;">24 hrs</td>
<td style="text-align: center;">34</td>
<td style="text-align: center;">149</td>
</tr>
<tr class="odd">
<td style="text-align: center;">tf-FA</td>
<td style="text-align: left;">H37Rv-ΔRD1</td>
<td style="text-align: center;">96 hrs</td>
<td style="text-align: center;">25</td>
<td style="text-align: center;">118</td>
</tr>
</tbody>
</table>
</div>
</figure>
</div>
</div>
<p>In assessing these data, there are several initial observations to be made. First, we see many statistically “significant” differentially enriched proteins for infection versus mock comparisons (<a href="#tbl-TrifunctionalProteomics-EnrichmentOverview" class="quarto-xref">Table&nbsp;<span>6.1</span></a>). There is a propensity for <em>decreased</em> interaction with trifunctional probes, as many more proteins are significantly depleted during infection.</p>
<p>We must emphasize that these results represent the relative abundance of each protein after affinity enrichment. While this measure is intended to reflect the rate of interaction between each protein and the trifunctional probe, it also encompasses changes in abundance that result from altered transcription, translation, trafficking, and degradation of each protein. To determine whether the transcriptional response to infection impacted these results, we cross-referenced our results with the 2023 findings of Jani et al., who investigated the transcriptomes of cells infected with H37Rv (<a href="#suppfig-TrifunctionalProteomics-FigS6" class="quarto-xref">Figure S<span>6.6</span></a>)<span class="citation" data-cites="janiTLR2NonredundantPopulation2023"><sup><a href="../references.html#ref-janiTLR2NonredundantPopulation2023" role="doc-biblioref">24</a></sup></span>. Several proteins are concomitantly enriched between Jani et al.’s data and our own (e.g., the interferon-induced gene MX1). Notably, however, most significantly downregulated interactions among our findings were not significantly changed at the transcription level. Public data on the whole cell proteome of H37Rv infected cells are not publicly available, and we cannot comment on the overall protein abundance for our differentially enriched hits.</p>
<div class="border">
<div id="fig-TrifunctionalProteomics-Fig5" class="quarto-figure quarto-figure-center quarto-float anchored" data-group="results_TrifunctionalProteomics">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-TrifunctionalProteomics-Fig5-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_TrifunctionalProteomics/300ppi/Figure5-MtbVolcanoPlots.png" class="lightbox" data-glightbox="description: .lightbox-desc-5" data-gallery="results_TrifunctionalProteomics"><img src="../../parts/results/results_TrifunctionalProteomics/300ppi/Figure5-MtbVolcanoPlots.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-TrifunctionalProteomics-Fig5-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;6.5: <strong><em>Mtb</em> infection reprograms the sphingosine interactome</strong> (A) A schematic overview of infection design and timeline. (B) Volcano plots depict the differential enrichment of proteins to the tf-Sphingosine and tf-8:3-Fatty Acid probes in H37Rv infection versus uninfected control. Statistical significance is as denoted (Partially significant, purple = Log2FC &gt; 1 &amp; p-value &lt; 0.05; Significant, orange = Log2FC &gt; 2, FDR &lt; 0.05, &amp; p-value &lt; 0.01; No significance in black).
</figcaption>
</figure>
</div>
</div>
<p>Focusing on 24-hour H37Rv infection, we observe several immunity-related genes among the protein interactions enriched during infection. These include MX1, IKBKB, and CTLA. We also observe several proteins related to motility and cytoskeletal trafficking, including Kinesin light chain 1 (KLC1), Tubulin, FLII, Microtubule-associated protein 4, Coronin-7, Myosin heavy chains 9 and 14 (MYH9 and MYH14), and Myosin light chain 12A (MYL12A). Intriguingly, we see a depletion of the activated leukocyte cell adhesion molecule (ALCAM), which is responsible for the extravasation of monocytes across the blood-brain barrier<span class="citation" data-cites="lyckALCAMCD166Involved2017"><sup><a href="../references.html#ref-lyckALCAMCD166Involved2017" role="doc-biblioref">25</a></sup></span>. Some evidence suggests that BCG infection induces a transcriptional downregulation of ALCAM in PBMCs<span class="citation" data-cites="fletcherTranscriptionalProfilingMycobacterial2009"><sup><a href="../references.html#ref-fletcherTranscriptionalProfilingMycobacterial2009" role="doc-biblioref">26</a></sup></span>. However, we do not observe this in our comparison to recent transcriptomics data reported by Jani et al.<span class="citation" data-cites="janiTLR2NonredundantPopulation2023"><sup><a href="../references.html#ref-janiTLR2NonredundantPopulation2023" role="doc-biblioref">24</a></sup></span>.</p>
<p>Among the downregulated interactions, we see a preponderance of proteins in the secretory/endosomal compartment. These include the Coatomer subunit epsilon (COPE), GOLGA2, Vescicle-associated membrane protein-associated protein A (VAPA), Rab11b, PSAP, Cathepsin A, Cathepsin B, and Cathepsin Z (and Cathepsin D, though it was only partially significant). We see many cytoskeletal proteins among the upregulated interactions, including Tubulin, FLII, Myosin-9 and Myosin-14, Microtubule-associated protein 4, and Coronin-7. We postulate that these results indicate dysregulation of endosomal trafficking.</p>
<p>At 24 hours post-infection, we observe differential enrichment of several proteins of high interest in the secretory/endosomal compartment. Among these are Cathepsins A, B, and Z (CTSA, CTSB, and CTSZ; and CTSD, though it was only partially significant), Prosaposin (PSAP), Coatomer subunit epsilon (COPE), Vescicle-associated membrane protein-associated protein A (VAPA), Rab11b. We also see several calcium-binding proteins among these downregulated interactions: Calumenin, Reticulocalbin-1, and Calmodulin-1 (CALM1).</p>
<p>As noted above, there are prior reports of Prosaposin being involved in <em>Mtb</em> restriction, including the 2020 report by Shepelkova that showed that Saposin D is essential in the restriction of <em>Mtb</em> growth<span class="citation" data-cites="shepelkovaSaposinActingMacrophage2020"><sup><a href="../references.html#ref-shepelkovaSaposinActingMacrophage2020" role="doc-biblioref">27</a></sup></span>. An intriguing hit is PH4P, which is significantly depleted at 24 hours. There are many reported functions of the protein P4HB in the cell. Among these, it is a structural subunit of the microsomal triacylglycerol transfer protein (MTTP). MTTP complex loads triglycerides, cholesteryl esters, and phospholipids onto apolipoprotein B (apoB)<span class="citation" data-cites="hussainMicrosomalTriglycerideTransfer2003"><sup><a href="../references.html#ref-hussainMicrosomalTriglycerideTransfer2003" role="doc-biblioref">28</a></sup></span>. This complex is essential for presenting lipid antigens to NK T cells via CD1d<span class="citation" data-cites="douganMicrosomalTriglycerideTransfer2005"><sup><a href="../references.html#ref-douganMicrosomalTriglycerideTransfer2005" role="doc-biblioref">29</a></sup></span>. In 2014, De Libero and Mori discussed that lipid antigens are essential for the invariant Natural Killer T cell response during <em>Mtb</em> infection<span class="citation" data-cites="deliberoTCellResponseLipid2014"><sup><a href="../references.html#ref-deliberoTCellResponseLipid2014" role="doc-biblioref">30</a></sup></span>.</p>
<p>In comparing the enrichment of proteins to the tf-Sph and the tf-8:3-FA probe, we find a high degree of alignment among the co-identified protein at both time points (<a href="#suppfig-TrifunctionalProteomics-tfSphvstfFA" class="quarto-xref">Figure S<span>6.7</span></a>). Linear regression analysis shows an adjusted R<sup>2</sup> value of 0.80 for the 24-hour comparisons versus mock infection, suggesting a robust correlation between the datasets. More analytical work will be needed to distinguish proteins that interact more robustly with tf-Sph than tf-8:3-FA.</p>
</section>
</section>
<section id="pathogenic-versus-attenuated-mtb-infections-induce-highly-similar-changes-to-the-tf-sph-interactome" class="level2" data-number="6.4">
<h2 data-number="6.4" class="anchored" data-anchor-id="pathogenic-versus-attenuated-mtb-infections-induce-highly-similar-changes-to-the-tf-sph-interactome"><span class="header-section-number">6.4</span> Pathogenic versus attenuated <em>Mtb</em> infections induce highly similar changes to the tf-Sph interactome</h2>
<p>A primary goal of this study was to compare the effect of pathogenic and attenuated <em>Mtb</em> infections on the sphingosine interactome. To this end, we prepared parallel samples of cells infected with the attenuated <em>Mtb</em> strain H37Rv-ΔRD1 (this strain was discussed extensively in <a href="results_Mtb_lipidomics_v2.html" class="quarto-xref"><span>Chapter 5</span></a>). In comparing the protein interactors of tf-Sph and tf-8:3-FA in these ΔRD1-infected cells versus mock-infected cells, we see many of the same hits as were differentially enriched in the H37Rv versus mock comparison (<a href="#suppfig-TrifunctionalProteomics-FigS7" class="quarto-xref">Figure S<span>6.8</span></a>). This is particularly evident when the log2 fold-changes of H37Rv versus mock are plotted against those of H37Rv-ΔRD1 versus mock – linear regression shows that these datasets have an adjusted R<sup>2</sup> value of 0.87 (<a href="#suppfig-TrifunctionalProteomics-FigS8" class="quarto-xref">Figure S<span>6.9</span></a> Upper panels).</p>
<div class="border">
<div id="fig-TrifunctionalProteomics-Fig7" class="quarto-figure quarto-figure-center quarto-float anchored" data-group="results_TrifunctionalProteomics">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-TrifunctionalProteomics-Fig7-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_TrifunctionalProteomics/300ppi/Figure7-H37RvvsRD1.png" class="lightbox" data-glightbox="description: .lightbox-desc-6" data-gallery="results_TrifunctionalProteomics"><img src="../../parts/results/results_TrifunctionalProteomics/300ppi/Figure7-H37RvvsRD1.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-TrifunctionalProteomics-Fig7-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;6.6: <strong>Proteins enriched during H37Rv infection are highly similar to those enriched during H37Rv-ΔRD1 infection</strong> Volcano plots depict the enrichment of proteins in H37Rv infection in comparison to H37Rv-ΔRD1 infection for both tf-Sphingosine and tf-Fatty acid probe at 24- and 96-hours post-infection. Point size and shape denote the enrichment of each protein in the baseline proteomics experiment (+/-UV); point color denotes the statistical annotation among <em>Mtb</em> infection experimental results (as denoted).
</figcaption>
</figure>
</div>
</div>
<p>Despite this, we continued our assessment of H37Rv versus H37Rv-ΔRD1 by plotting the log2 fold-change of proteins during each infection. Again, very few proteins are substantively different between pathogenic H37Rv infection and attenuated H37Rv-ΔRD1 infection (<a href="#fig-TrifunctionalProteomics-Fig7" class="quarto-xref">Figure&nbsp;<span>6.6</span></a>). Among these few proteins of partial significance, we see PSME3 enriched toward H37Rv infection at 24 hours; this protein is a component of the immunoproteasome critical for antigen presentation and correlated with antigen presentation<span class="citation" data-cites="yangSinglecellProfilingReveals2023"><sup><a href="../references.html#ref-yangSinglecellProfilingReveals2023" role="doc-biblioref">31</a></sup></span>. At 96 hours post-infection, we see a mild enrichment of several Type I Interferon-regulating genes, including poly(ADP-ribose) polymerase family member 9 (PARP9), 2’-5’-Oligoadenylate Synthetase 3 (OAS3), and MX Dynamin Like GTPases 1 and 2 (MX1 and MX2). Interestingly, PARP9 is involved in protection against <em>Mtb</em><span class="citation" data-cites="thirunavukkarasuPolyADPribosePolymerase2023"><sup><a href="../references.html#ref-thirunavukkarasuPolyADPribosePolymerase2023" role="doc-biblioref">32</a></sup></span>, OAS3 restricts intracellular Mtb replication and enhances cytokine secretion<span class="citation" data-cites="leischingOAS1OAS2OAS32019"><sup><a href="../references.html#ref-leischingOAS1OAS2OAS32019" role="doc-biblioref">33</a></sup></span>, and MX1 and MX2 are well known interferon-stimulated genes transcriptionally upregulated during <em>Mtb</em> infection<span class="citation" data-cites="janiTLR2NonredundantPopulation2023 yiTranscriptionalProfilingHuman2021"><sup><a href="../references.html#ref-janiTLR2NonredundantPopulation2023" role="doc-biblioref">24</a>,<a href="../references.html#ref-yiTranscriptionalProfilingHuman2021" role="doc-biblioref">34</a></sup></span>.</p>
<p>The only notable enriched gene for tf-8:3-FA was RSL1D1, enriched at 96 hours; this gene is known to suppress autophagy<span class="citation" data-cites="liuRSL1D1PromotesProgression2022"><sup><a href="../references.html#ref-liuRSL1D1PromotesProgression2022" role="doc-biblioref">35</a></sup></span>. There is no reported connection to <em>Mycobacterium tuberculosis</em>.</p>
</section>
<section id="mtb-protein-rv3466-is-enriched-by-tf-sphingosine-during-infection" class="level2" data-number="6.5">
<h2 data-number="6.5" class="anchored" data-anchor-id="mtb-protein-rv3466-is-enriched-by-tf-sphingosine-during-infection"><span class="header-section-number">6.5</span> <em>Mtb</em> protein Rv3466 is enriched by tf-Sphingosine during infection</h2>
<p>The enrichment of mycobacterial proteins to either trifunctional sphingosine would be of profound significance – this would suggest direct evidence that mycobacteria utilize host sphingolipids during infection. The standard, though strict, criteria for confident protein identification above (at least two unique peptides identified in at least two replicates) yields no <em>Mtb</em> proteins. That we fail to identify <em>Mtb</em> proteins with high confidence may be expected in retrospect, given the disparity in total protein mass between an infected cell and an infecting bacillus. However, <em>Mtb</em> candidate peptides were identified aligning with the <em>Mtb</em> proteins accD2, Rv1910, and Rv3466 (<a href="#fig-TrifunctionalProteomics-Fig8" class="quarto-xref">Figure&nbsp;<span>6.7</span></a>A). accD2 was only identified in the tf-8:3-FA TMT-16-plex, Rv3466 was only identified in the tf-Sph TMT-16-plex, and the same Rv1910c peptide was identified in both TMT-16-plexes. The relative enrichment of each protein toward H37Rv or H37Rv-ΔRD1 at both time points is depicted in <a href="#fig-TrifunctionalProteomics-Fig8" class="quarto-xref">Figure&nbsp;<span>6.7</span></a>B. Among these three hits, only Rv3466 is enriched compared to mock infection for both strains at both time points.</p>
<div class="border">
<div id="fig-TrifunctionalProteomics-Fig8" class="quarto-figure quarto-figure-center quarto-float anchored" data-group="results_TrifunctionalProteomics">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-TrifunctionalProteomics-Fig8-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_TrifunctionalProteomics/300ppi/Figure8-MtbProteins.png" class="lightbox" data-glightbox="description: .lightbox-desc-7" data-gallery="results_TrifunctionalProteomics"><img src="../../parts/results/results_TrifunctionalProteomics/300ppi/Figure8-MtbProteins.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-TrifunctionalProteomics-Fig8-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;6.7: <strong>Potential <em>Mtb</em> protein identification: Rv3466</strong> A summary of low-confidence identifications of three <em>Mtb</em> proteins accD2, RV1910c, and Rv3466. (A) Enrichment of each protein in infection versus mock comparisons. Blue bars are infections with pathogenic strain H37Rv; pink bars are infections with attenuated strain H37Rv-ΔRD1. (B) A depiction of the percentage of coverage for each protein. (C) Prediction of RV3466 structure using AlphaFold. Blue-coded sequence denotes high-confidence structure prediction; red- and yellow-coded sequence denotes low confidence prediction.
</figcaption>
</figure>
</div>
</div>
<p>The total knowledge regarding Rv3466 can be summarized in a few words: it is an uncharacterized protein with no functional annotation. Benq et al.&nbsp;report that Rv3466 lies in an MT-complex-specific genomic island – the genes that distinguish environmental mycobacteria from pathogenic <em>M. tuberculosis</em> complex members<span class="citation" data-cites="becqContributionHorizontallyAcquired2007"><sup><a href="../references.html#ref-becqContributionHorizontallyAcquired2007" role="doc-biblioref">36</a></sup></span>. A 2019 genome-wide transposon mutagenesis study by Minato et al.&nbsp;showed that this protein is not essential for <em>in vitro</em> growth in MtbYM-rich medium<span class="citation" data-cites="minatoGenomewideAssessmentMycobacterium2019"><sup><a href="../references.html#ref-minatoGenomewideAssessmentMycobacterium2019" role="doc-biblioref">37</a></sup></span>; its expression is significantly enriched following culture with the broad-spectrum antibiotic levofloxacin<span class="citation" data-cites="boshoffTranscriptionalResponsesMycobacterium2004"><sup><a href="../references.html#ref-boshoffTranscriptionalResponsesMycobacterium2004" role="doc-biblioref">38</a></sup></span>; no protein domains are annotated in the sequence of Rv3466. Rv3466 is enriched to the lipid fraction in studies of the total <em>Mtb</em> proteome<span class="citation" data-cites="galaganMycobacteriumTuberculosisRegulatory2013 lewTubercuList10Years2011"><sup><a href="../references.html#ref-galaganMycobacteriumTuberculosisRegulatory2013" role="doc-biblioref">39</a>,<a href="../references.html#ref-lewTubercuList10Years2011" role="doc-biblioref">40</a></sup></span>. This gene is close genomic neighbors with rmlB and rmlC, two sugar-modifying enzymes involved in synthesizing peptidoglycan – suggesting a role for Rv3466 in peptidoglycan synthesis at the bacterial membrane. Rv3466 shares high sequence homology with another uncharacterized <em>Mtb</em> protein, Rv1588c.</p>
<p>Further mass spectrometry analysis will determine whether these or other <em>Mtb</em> protein interactors of sphingosine can be identified.</p>
</section>
<section id="discussion" class="level2" data-number="6.6">
<h2 data-number="6.6" class="anchored" data-anchor-id="discussion"><span class="header-section-number">6.6</span> Discussion</h2>
<p>In this study, we expand the application of trifunctionalized probes to assess the impact of intracellular bacterial infection on the sphingolipid interactome. We first identified the proteins that selectively interact with trifunctionalized sphinganine and sphingosine (as opposed to a pair of control trifunctionalized fatty acid probes). We identify a suite of differentially enriched protein interactors of trifunctional sphingosine upon infection with the pathogenic <em>Mtb</em> strain H37Rv. Most differentially enriched proteins do not align with published transcriptional signatures of <em>Mtb</em> infection, suggesting that these proteins’ stability, localization, or function is altered during infection. We then compared the enriched interactions of cells infected with pathogenic and attenuated strains of <em>Mtb</em>. Finally, we hone in on a candidate <em>Mtb</em> protein enriched during infection (though its identification is below the confidence threshold).</p>
<p>Trifunctional probes are a precious resource for studying lipid biology. In this study, we primarily utilized these synthetic analogs of sphinganine and sphingosine as affinity handles to identify the protein interactors of these lipids in monocyte-derived macrophages. We note that a substantive majority of the enriched proteins either contain a transmembrane domain or are annotated as peripheral membrane proteins in the literature; very few are cytosolic or secreted. We identify several interactors reported in previous studies, such as Farley et al.&nbsp;2024<span class="citation" data-cites="farleyTrifunctionalSphinganineNew2024"><sup><a href="../references.html#ref-farleyTrifunctionalSphinganineNew2024" role="doc-biblioref">6</a></sup></span>. Such previously verified hits include ECH1, CTSD, VDACs 1 and 2, PITPNB, PRCP, and PSAP. We identify several intriguing, novel binding partners that appear to be unique to macrophages – these include NENF, CSTA, SCPDH, and HSPA5, among others. More work is needed to validate these interactions and to determine the biological effects of these interactions.</p>
<p>Before we can discuss the results of the <em>Mtb</em> infection interactomics experiments, we must make a technical note about +/-UV proteomics: we would have expected a dramatic disparity between the total protein input of +UV samples and -UV samples due to the stringency of sample wash. Notably, there are proteins substantially enriched to the -UV sample condition for tf-Sphingosine and tf-Sphinganine, though we expect little biological significance among these proteins. These proteins are likely the result of spurious interactions, spontaneous crosslinking, or artifacts of data normalization. Notably, the depth of coverage in the +/-UV results hinders its utility in filtering out candidates – there were over 300 proteins identified in the <em>Mtb</em> infection samples treated with tf-Sph and 542 proteins identified in the samples treated with tf-8:3-FA. It is unclear how to consider these proteins which have no annotation in the baseline interactome.</p>
<p>With the baseline interaction analysis in mind, we may now discuss the main results of this study. We hoped to determine whether <em>Mtb</em> infection induces changes in the protein-sphingolipid interactions of the cell. We found that a 24-hour infection with H37Rv induces significant differential enrichment of 151 proteins (31 were enriched during infection, and 120 were depleted). Our comparison to publicly available transcriptomics data shows that while some transcriptionally upregulated proteins are reflected by positive interaction enrichment, little change is reported for many of the most trustworthy hits. While this comparison does not preclude changes in the total abundance of proteins during <em>Mtb</em> infection, it serves to decouple our interaction analysis from the transcriptional response of the cell.</p>
<p>There are many potentially exciting hits for follow-up. There appear to be several pathways enriched in the up- and downregulated interactions. These include an upregulation of immunity related proteins (e.g., MX1, IKBKB, and CTLA), cytoskeleton and motility proteins (e.g., myosin heavy and light chains, kinesin light chain, tubulin, and Coronin-7). Among the downregulated pathways are numerous secretory or vescicular trafficking proteins (e.g., COPE, GOLGA2, VAPA, and Rab11b). These results appear to suggest a significant dysregulation of cellular trafficking and endosomal protein composition. We also see a depletion of several calcium-binding proteins; this is intriguing because uncaging tf-sphingosine has been previously shown to induce the release of calcium from the lysosome<span class="citation" data-cites="hoglingerIntracellularSphingosineReleases2015a"><sup><a href="../references.html#ref-hoglingerIntracellularSphingosineReleases2015a" role="doc-biblioref">9</a></sup></span> – are these infected cells unable to trigger sphingosine-mediated calcium signaling? Finally, we report the low-confidence identification of a candidate <em>Mtb</em> protein selectively enriched to tf-Sphingosine during infection: Rv3466. This protein is not expected to contain a transmembrane domain and has no known function. Follow-up experiments are needed to validate this hit and to explore the possibility of further yet-undiscovered <em>Mtb</em> proteins that selectively interact with host sphingolipids.</p>
<p>While much work remains on this project, this study is among the first to utilize trifunctionalized lipids in an affinity enrichment of <em>Mtb</em>-infected cells. These mark crucial first steps into establishing a toehold into the waters of sphingolipid-mediated host-pathogen interactions. These results may yield new insights into the molecular mechanisms underlying the pathogenicity of <em>Mycobacterium tuberculosis</em>.</p>
</section>
<section id="materials-and-methods" class="level2" data-number="6.7">
<h2 data-number="6.7" class="anchored" data-anchor-id="materials-and-methods"><span class="header-section-number">6.7</span> Materials and Methods</h2>
<section id="thp-1-cell-culturing-and-plating" class="level3" data-number="6.7.1">
<h3 data-number="6.7.1" class="anchored" data-anchor-id="thp-1-cell-culturing-and-plating"><span class="header-section-number">6.7.1</span> THP-1 cell culturing and plating</h3>
<p>Human THP-1 monocytes were sourced from American Type Culture Collection (ATCC catalog no. TIB-22) were maintained in culture using RPMI 1640 medium (Thermo Fischer Scientific #11875119) supplemented with 10% fetal bovine serum (Seradigm), 1x Gibco penicillin-streptomycin (Thermo Fischer Scientific #15-140-163), and 1x Gibco Non-Essential Amino Acids (Thermo Fischer Scientific #11140050). Cells were grown in suspension at 37 °C with 5% CO<sub>2</sub>. Before experimental use, THP-1 monocytes were differentiated into macrophages via 48-hour treatment with 50 nM phorbol 12-myristate 13-acetate (PMA; Sigma-Aldrich P8139-1MG). All cell counting was achieved using a hemocytometer. For microscopy, cells were either plated in a 96-well plate at a density of 2.5 x 10<sup>4</sup> cells per well or on glass coverslips at 1.0 x 10 <sup>5</sup> cells per well. For proteomics sample collection, cells were plated at a density of 4 x 10<sup>6</sup> cells per 10 cm dish.</p>
</section>
<section id="mycobacterium-tuberculosis-culturing-and-infection" class="level3" data-number="6.7.2">
<h3 data-number="6.7.2" class="anchored" data-anchor-id="mycobacterium-tuberculosis-culturing-and-infection"><span class="header-section-number">6.7.2</span> Mycobacterium tuberculosis culturing and infection</h3>
<p>Wildtype historical Mycobacterium tuberculosis strains H37Rv and H37Rv-ΔRD1 were gifts from the laboratory of Dr.&nbsp;David Lewinsohn at Oregon Health &amp; Science University and Portland Veterans Association. Both Mtb strains were cultured in Difco<sup>TM</sup> Middlebrook 7H9 medium (Beckton Dickinson, Cat# 271310) supplemented with BBL<sup>TM</sup> Middlebrook OADC Enrichment (Beckton Dickinson, Cat# L12240) and 0.1% Tween-80. Bacterial cultures were carefully monitored to guarantee they did not overgrow (OD<sub>600</sub> &gt; 1.5) and were kept at a low passage before experimental use.</p>
<p>Before infection, THP-1 monocyte-derived macrophages were plated in 10 cm dishes at a density of 4 x 10<sup>6</sup> cells per dish and activated 48 hrs in 50 nM PMA. Mycobacterium tuberculosis (wildtype H37Rv and H37RvΔRD1) was cultured as above in 7H9 to mid-log phase and pelleted via 5 min spin at 2000 xg. Bacterial pellets were washed once with PBS and resuspended in RPMI. We prepared a single-cell suspension by passaging bacteria 25x through an 18-gauge blunt-tipped needle and then through a 5 μm filter. Bacterial concentration was estimated as a proportion of the OD<sub>600</sub> (assuming 2.5 x 10<sup>8</sup> CFU/mL at an OD<sub>600</sub> of 1.0). For 24-hour infection samples, cells were infected overnight at an MOI of 3. For 96-hour infection samples, cells were infected overnight at an MOI of 1 and washed 2x with PBS before an additional 72-hour incubation. Infections were repeated in 3 biological replicates on independent days. We validated a minimum infection rate of ~70% by plating parallel coverslips of activated THP-1 cells, which we infected with both strains of Mtb and at both MOIs and inspected via fluorescence microscopy.</p>
</section>
<section id="sec-TrifunctionalProteomics-Methods-ProbeTreatment" class="level3" data-number="6.7.3">
<h3 data-number="6.7.3" class="anchored" data-anchor-id="sec-TrifunctionalProteomics-Methods-ProbeTreatment"><span class="header-section-number">6.7.3</span> Trifunctionalized lipid treatments, uncaging, crosslinking</h3>
<p>After a 48-hour activation with 50 nM PMA, THP-1 cells were treated with a trifunctional lipid probe at the below concentrations in <a href="#tbl-BaselineInteractomics_LipidProbeTreatments" class="quarto-xref">Table&nbsp;<span>6.2</span></a>. Cells were incubated with probes for 30 min at 37 °C to allow probe incorporation into cellular membranes. Lipid probes were then exposed to blue wavelength light (405 nm) for 5 min to release the coumarin cage and incubated for the times depicted below (<a href="#tbl-BaselineInteractomics_LipidProbeTreatments" class="quarto-xref">Table&nbsp;<span>6.2</span></a>). Probes were then diazirine crosslinked via 5-minute exposure to ultraviolet wavelength light (355 nm). For proteomics experiments, control cells were treated with trifunctional probes and uncaged but were left without UV-induced diazirine crosslinking (-UV conditions)</p>
<div class="border">
<div id="tbl-BaselineInteractomics_LipidProbeTreatments" class="striped hover quarto-float anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-BaselineInteractomics_LipidProbeTreatments-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;6.2: <strong>Trifunctional lipid probe treatments</strong>
</figcaption>
<div aria-describedby="tbl-BaselineInteractomics_LipidProbeTreatments-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<table class="table-striped table-hover table">
<thead>
<tr class="header">
<th style="text-align: left;">Probe</th>
<th style="text-align: center;">Final treatment concentration</th>
<th style="text-align: center;">Incubation post-uncaging</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Tf-8:3-Sphingosine</td>
<td style="text-align: center;">2 mM</td>
<td style="text-align: center;">15 min</td>
</tr>
<tr class="even">
<td style="text-align: left;">Tf-8:3-Sphinganine</td>
<td style="text-align: center;">2 mM</td>
<td style="text-align: center;">15 min</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Tf-8:3-Fatty Acid</td>
<td style="text-align: center;">50 mM</td>
<td style="text-align: center;">60 min</td>
</tr>
<tr class="even">
<td style="text-align: left;">Tf-1:10-Fatty Acid</td>
<td style="text-align: center;">50 mM</td>
<td style="text-align: center;">60 min</td>
</tr>
</tbody>
</table>
</div>
</figure>
</div>
</div>
</section>
<section id="cell-lysis-and-protein-collection" class="level3" data-number="6.7.4">
<h3 data-number="6.7.4" class="anchored" data-anchor-id="cell-lysis-and-protein-collection"><span class="header-section-number">6.7.4</span> Cell lysis and protein collection</h3>
<p>After probe crosslinking, plates were washed 2x with ice-cold PBS, and cells were scraped into PBS using a cell scraper; for each sample, 2x 10 cm plates were combined during scraping for a total of 8 x 10<sup>6</sup> cells per sample. Cells were pelleted via 5 min spin at 500 xg and resuspended in a lysis buffer consisting of 8 M urea, 1x RIPA buffer (EMD Millipore, Cat# 20-188), and 1x HALT protease inhibitor cocktail (Thermo Fisher Scientific, Cat# 87786). Cell pellets were first broken up using repeated passage through an 18-gauge blunt-tipped needle, then passage through a lysis filter cartridge (from the Minute<sup>TM</sup> Plasma Membrane Protein Isolation and Cell Fractionation Kit, Invent Biotech, Cat# SM-005) to achieve a solution of lysed cellular proteins and intact nuclei. All lysis steps were performed on ice or in a pre-cooled centrifuge.</p>
<p>Finally, in accommodation of OHSU biosafety level 3 protocol, cell lysates from Mtb-infected samples (and corresponding mock-infected controls) were spun through a 0.5 μm centrifugal filter (Ultrafree-MC GV Centrifugal Filter, EMD Millipore, Cat# UFC30GV25) via 20-minute spin at 10,000 xg.</p>
</section>
<section id="sec-TrifunctionalProteomics-Methods-IGFClick" class="level3" data-number="6.7.5">
<h3 data-number="6.7.5" class="anchored" data-anchor-id="sec-TrifunctionalProteomics-Methods-IGFClick"><span class="header-section-number">6.7.5</span> Fluorescence click reaction &amp; In-gel/on-blot fluorescence</h3>
<p>In-gel fluorescence was used as a quality control metric for infected samples. 5% of each sample was split off and conjugated with CalFluor647-azide via CuAAC. 50 μL of each sample was incubated with 100 μM tris(benzyltriazolylmethyl)amine (TBTA), 1 mM cupric sulfate (CuSO<sub>4</sub>), 1 mM sodium ascorbate, and 20 μM CalFluor 647-azide (Click Chemistry Tools, Cat. #1372-1). Click reaction mixtures were incubated at room temperature for 60 minutes in the dark.</p>
<p>Following the click reaction, Laemmli sample buffer containing β-mercaptoethanol was added, and samples were heated to 42 °C for 30 minutes. Samples were spun at 10000 xg for 10 minutes. Each sample was loaded and was run on a 12.5% SDS-PAGE gel until the dye front had wholly run off the bottom of the gel – this removes much of the free, non-clicked 647-azide dye in each sample. After running, gels were left for at least 30 minutes in diH<sub>2</sub>O to allow any remaining free dye to diffuse out of the gel. Gels were then imaged using the 647nm filter of a Sapphire™ FL biomolecular fluorescence imager (Azure Biosystems). Contrast and brightness were globally adjusted in FIJI.</p>
<p>For validation of hits using on-blot fluorescence, samples were fluorescently labeled and separated via SDS-PAGE as above, and gels were transferred to polyvinylidene difluoride blotting membranes via overnight wet transfer. Blots were blocked using 1% BSA and stained with denoted primary antibodies. Blots were then stained against the appropriate primary background with a secondary antibody conjugated to Alexa Fluor 555 fluorophore. Finally, blots were imaged as above on a Sapphire™ FL biomolecular fluorescence imager using the 555 nm and 647 nm filters. Contrast and brightness were globally adjusted in FIJI.</p>
</section>
<section id="affinity-purification-of-probe-bound-proteins" class="level3" data-number="6.7.6">
<h3 data-number="6.7.6" class="anchored" data-anchor-id="affinity-purification-of-probe-bound-proteins"><span class="header-section-number">6.7.6</span> Affinity purification of probe-bound proteins</h3>
<p>The remaining 95% of each sample was used as input for affinity purification via CuAAC to azide beads. First, Picolyl Azide Agarose resin (Click Chemistry Tools, Cat. #1408-2; 50% slurry) was transferred to new Eppendorf tubes and gently spun to pellet; storage solution supernatant was aspirated and beads were washed with diH<sub>2</sub>O to remove any residual storage solution. CuAAC/click reaction was then initiated through the addition of 100 μM TBTA, 1 mM sodium ascorbate, 1 mM CuSO4, washed picolyl-azide agarose resin, and the remaining volume of probe-treated lysates. The final 1.25 mL reaction proceeded under vigorous shaking for 1 hour at room temperature.</p>
<section id="bead-wash-digestion-procedure" class="level4" data-number="6.7.6.1">
<h4 data-number="6.7.6.1" class="anchored" data-anchor-id="bead-wash-digestion-procedure"><span class="header-section-number">6.7.6.1</span> Bead wash &amp; digestion procedure</h4>
<p>Bead/Click reaction mix transferred to a 2 mL gravity flow column (Pierce™ Centrifuge Columns 2 mL, Cat# 89896). Flow through was collected for each stage of the washing. Beads were washed a total of 25x with the buffers denoted below in <a href="#tbl-TrifunctionalProteomics-BeadWashes" class="quarto-xref">Table&nbsp;<span>6.3</span></a>.</p>
<div id="tbl-TrifunctionalProteomics-BeadWashes" class="striped hover quarto-float anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-TrifunctionalProteomics-BeadWashes-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;6.3: <strong>Details of bead washes in affinity enrichment</strong>
</figcaption>
<div aria-describedby="tbl-TrifunctionalProteomics-BeadWashes-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<table class="table-striped table-hover table">
<colgroup>
<col style="width: 25%">
<col style="width: 46%">
<col style="width: 28%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;"></th>
<th style="text-align: left;">Composition</th>
<th style="text-align: center;"># of washes (volume per wash)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Wash Buffer #1</td>
<td style="text-align: left;">100 mM tris-HCl, 250 mM NaCl, 5 mM ethylenediaminetetraacetic acid (EDTA), 1% sodium dodecyl sulfate (SDS) (pH = 8.0)</td>
<td style="text-align: center;">5 (1 mL)</td>
</tr>
<tr class="even">
<td style="text-align: left;">Wash Buffer #2</td>
<td style="text-align: left;">100 mM tris-HCl, 8 M urea (pH = 8.0)</td>
<td style="text-align: center;">10 (2 mL)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Wash Buffer #3</td>
<td style="text-align: left;">20 % acetonitrile, 80 % diH<sub>2</sub>O (volume/volume)</td>
<td style="text-align: center;">10 (2 mL)</td>
</tr>
</tbody>
</table>
</div>
</figure>
</div>
<p>Beads were transferred to a new Eppendorf tube and pelleted. Bead-conjugated proteins were reduced via treatment with 1 mM dithiothreitol for 30 minutes at 42 °C. Beads were again pelleted and resuspended in an alkylating solution of 40 mM iodoacetamide for 30 minutes at room temperature (in the dark). Beads were then pelleted, and the alkylating solution was aspirated.</p>
<p>Finally, the beads were resuspended in 40 μL of a tryptic digestion solution consisting of 100 mM tris-HCl, 2 mM CaCl<sub>2</sub>, and 10% acetonitrile. A total of 1 μg of trypsin (Promega Trypsin/Lys-C Mix, Mass Spec Grade, Cat. #V5071) was added to each sample for a 1:25 ratio of [LC-MS-grade trypsin/Lys C : substrate] (assuming 1mg/mL protein on beads). Beads were rocked vigorously at 37 °C overnight to digest bead-conjugated proteins.</p>
</section>
<section id="sample-desalting" class="level4" data-number="6.7.6.2">
<h4 data-number="6.7.6.2" class="anchored" data-anchor-id="sample-desalting"><span class="header-section-number">6.7.6.2</span> Sample desalting</h4>
<p>Following digestion from beads, the digestion solution was washed and desalted using a reverse phase C18 desalting spin column (BioPureSPN MIDI, PROTO 300 C18, Fisher Scientific Cat #NC1678004). At all stages of desalting, columns were spun at 110 xg for 1 minute. First, columns were conditioned with acetonitrile (ACN) and flushed with H<sub>2</sub>O; columns were equilibrated with 2% ACN and blotted dry. The bead solution (including beads) was then transferred to the spin column and washed 2x with 2 % ACN to remove trace salts. The sample was released twice with an elution buffer of 80% ACN and 0.1 % trifluoroacetic acid. Finally, the eluted peptide solution was dried using a speedvac and stored at -80 °C.</p>
</section>
</section>
<section id="liquid-chromatography-and-tandem-mass-spectrometry-analysis-lc-msms" class="level3" data-number="6.7.7">
<h3 data-number="6.7.7" class="anchored" data-anchor-id="liquid-chromatography-and-tandem-mass-spectrometry-analysis-lc-msms"><span class="header-section-number">6.7.7</span> Liquid chromatography and tandem mass spectrometry analysis (LC-MS/MS)</h3>
<p>Mass spectrometric analyses were performed as previously described in Farley et al.&nbsp;2024<span class="citation" data-cites="farleyTrifunctionalSphinganineNew2024"><sup><a href="../references.html#ref-farleyTrifunctionalSphinganineNew2024" role="doc-biblioref">6</a></sup></span> and performed at the European Molecular Biology Laboratories Proteomics Core facility. Dried peptides were shipped to EMBL, labeled with isobaric mass tags using the TMT-16-plex system, and analyzed by LC-MS/MS on an Orbitrap Fusion Lumos mass spectrometer (Thermo Scientific).</p>
<p>Peptides were separated by reverse phase chromatography on an Ultimate 3000 nano RLSC system (Dionex) through a trapping cartridge (Precolumn C18 PepMap100, 5 mm, 300 μm i.d., 5 μm, 100 Å) and an analytical column (Acclaim PepMap 100, 75 x 50 cm C18, 3mm, 100 Å) connected to a nanospray-Flex ion source. The peptides were loaded onto the trap column at 30 μL/min using Solvent A (0.1% formic acid in H<sub>2</sub>O) and eluted using a gradient from 2 to 80% Solvent B (0.1% formic acid in acetonitrile) over 2 hr at 0.3 μL/min (all solvents were LC-MS grade). The Orbitrap Fusion Lumos was operated in positive ion mode with a spray voltage of 2.2 kV and a capillary temperature of 275 °C. Full scan MS spectra with a mass range of 375-1500 m/z were acquired in profile mode using a resolution of 120,000 with a maximum injection time of 50 ms, AGC operated in standard mode, and an RF lens setting of 30%. Fragmentation was triggered for 3 s cycle time for peptide-like features with a charge state of 2-7 on the MS scan (data-dependent acquisition).</p>
<p>Fragmentation was triggered for 3 second cycle time for peptide-like features with charge states of 2-7 on the MS scan (in data dependent acquisition). Precursors were isolated using the quadrupole with a window of 0.7 m/z and fragmented with a normalized collision energy of 34%. Fragment mass spectra were acquired in profile mode and with a resolution of 30,000. The maximum injection time was set to 94 ms and AGC target set to custom. The dynamic exclusion was set to 60 s<span class="citation" data-cites="farleyTrifunctionalSphinganineNew2024"><sup><a href="../references.html#ref-farleyTrifunctionalSphinganineNew2024" role="doc-biblioref">6</a></sup></span>.</p>
</section>
<section id="data-analysis" class="level3" data-number="6.7.8">
<h3 data-number="6.7.8" class="anchored" data-anchor-id="data-analysis"><span class="header-section-number">6.7.8</span> Data analysis</h3>
<p>Dr.&nbsp;Frank Stein prepared the data analysis pipeline at the European Molecular Biosciences Laboratories in Heidelberg, Germany and performed the analysis as previously reported in Farley et al., 2024<span class="citation" data-cites="farleyTrifunctionalSphinganineNew2024"><sup><a href="../references.html#ref-farleyTrifunctionalSphinganineNew2024" role="doc-biblioref">6</a></sup></span>. The acquired MS data were analyzed using IsobarQuant<span class="citation" data-cites="mateusThermalProteomeProfiling2016"><sup><a href="../references.html#ref-mateusThermalProteomeProfiling2016" role="doc-biblioref">41</a></sup></span> and Mascot V.24 (Matrix Science) using a reverse UniProt FASTA database including common contaminants; <em>Mtb</em> infection data were analyzed using a concatenated UniProt FASTA database including both human and <em>Mycobacterium tuberculosis</em> protein sequences. Only proteins with at least two unique peptides with lenght of at least 7 amino acids and a false-discovery rate below 0.01 were kept for protein-level analysis. The acquired spectra were scanned for the following peptide modifications: Carbamidomethyl (C, fixed), TMT-16-plex (K, fixed), Acetyl (N-term, variable), Oxidation (M, variable), TMT-16-plex (N-term, variable). The mass error tolerance for full-scan MS spectra was set to 10 ppm and for MS/MS spectra to 0.02 Da. A maximum of 2 missed cleavage sites was allowed. Batch effects were removed between replicates, a variance stabilization normalization was applied on the log2-transformed raw data<span class="citation" data-cites="huberVarianceStabilizationApplied2002"><sup><a href="../references.html#ref-huberVarianceStabilizationApplied2002" role="doc-biblioref">42</a></sup></span>, and changes in abundance of proteins in the experimental condition versus control condition were calculated using LIMMA analysis<span class="citation" data-cites="ritchieLimmaPowersDifferential2015"><sup><a href="../references.html#ref-ritchieLimmaPowersDifferential2015" role="doc-biblioref">14</a></sup></span>.</p>
<p>For the +/-UV experiment of baseline protein-lipid probe interactions, a protein was designated “significantly enriched” if it was enriched toward the +UV condition by at least a log2 fold-change of 1 and a p-value &lt; 0.05. Proteins enriched toward the +UV condition but with p-value &gt; 0.05 were designated “partially enriched”. Due to the larger number of differentially enriched proteins in the <em>Mtb</em> infection affinity proteomics experiment, we used a modified designation of significance: proteins were deemed “significant” if they had a log2 fold-change enrichment of magnitude &gt; 2 (both positive and negative), a p-value &lt; 0.01 and a false-discovery rate &lt; 0.05; proteins were deemed “partially significant” if the magnitude of their log2 fold-change enrichment was greater than 1 (in either the positive or negative direction) and a p-value &lt; 0.05.</p>
<p>Data visualization was performed using R. Packages included: Bioconductor, ggplot2, dplyr, vsn, limma, MSnbase, gplots, fdrtool, biobroom, tidyverse, ggrepel.</p>
<p>Comparative transcriptomic data was found using the GEO database at the National Center for Biotechnology Information, initially reported by Jani et al.&nbsp;in 2023<span class="citation" data-cites="janiTLR2NonredundantPopulation2023"><sup><a href="../references.html#ref-janiTLR2NonredundantPopulation2023" role="doc-biblioref">24</a></sup></span>.</p>
</section>
</section>
<section id="contributions" class="level2" data-number="6.8">
<h2 data-number="6.8" class="anchored" data-anchor-id="contributions"><span class="header-section-number">6.8</span> Contributions</h2>
<p>GG designed the study, performed sample collection and analyzed results, prepared visualizations, prepared the figures, and wrote the manuscript. SF assisted in designing mass spectrometry experiments, designed and synthesized lipid probes, assisted in data visualization, and edited the manuscript. PH &amp; FS assisted in designing mass spectrometry experiments, performed mass spectrometry and protein identification, and assisted in the interpretation of results. CS assisted in designing mass spectrometry experiments, oversaw the design and synthesis of lipid probes, provided reagents, and gave expert advice on study design. FT assisted in the study design and oversaw the study.</p>
</section>
<section id="supplemental-data" class="level2" data-number="6.9">
<h2 data-number="6.9" class="anchored" data-anchor-id="supplemental-data"><span class="header-section-number">6.9</span> Supplemental Data</h2>
<div class="border">
<div id="suppfig-TrifunctionalProteomics-FigS1" class="quarto-float" data-group="results_TrifunctionalProteomics">
<figure class="quarto-float quarto-float-suppfig figure">
<div aria-describedby="suppfig-TrifunctionalProteomics-FigS1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_TrifunctionalProteomics/300ppi/FigureS1-BaselineQC.png" class="lightbox" data-glightbox="description: .lightbox-desc-8" data-gallery="results_TrifunctionalProteomics"><img src="../../parts/results/results_TrifunctionalProteomics/300ppi/FigureS1-BaselineQC.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-suppfig" id="suppfig-TrifunctionalProteomics-FigS1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure S6.1: <strong>Sample quality control for baseline probe interactomics experiment</strong> (A) Principle component analysis of protein abundance in each sample after batch effect removal and normalization. Shapes and colors denote conditions. (B) Box plots depict the normalization of MS data. Left panel: raw intensities of each sample for each condition; middle panel: batch-corrected intensities; right panel: median-normalized MS intensities. Color denotes replicate number.
</figcaption>
</figure>
</div>
</div>
<p><br></p>
<div class="border">
<div id="suppfig-TrifunctionalProteomics-FigS2" class="quarto-float" data-group="results_TrifunctionalProteomics">
<figure class="quarto-float quarto-float-suppfig figure">
<div aria-describedby="suppfig-TrifunctionalProteomics-FigS2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_TrifunctionalProteomics/300ppi/FigureS2-BaselineMA.png" class="lightbox" data-glightbox="description: .lightbox-desc-9" data-gallery="results_TrifunctionalProteomics"><img src="../../parts/results/results_TrifunctionalProteomics/300ppi/FigureS2-BaselineMA.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-suppfig" id="suppfig-TrifunctionalProteomics-FigS2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure S6.2: <strong>Baseline THP-1 probe proteomics, MA plots</strong> MA plots depict the distribution of mean protein abundances between denoted conditions (x-axis) and the log2-transformed ratio of the denoted conditions (y-axis). Point size, shape, and color denote the enrichment annotation of each protein.
</figcaption>
</figure>
</div>
</div>
<p><br></p>
<div class="border">
<div id="suppfig-TrifunctionalProteomics-FigS3" class="quarto-float" data-group="results_TrifunctionalProteomics">
<figure class="quarto-float quarto-float-suppfig figure">
<div aria-describedby="suppfig-TrifunctionalProteomics-FigS3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_TrifunctionalProteomics/300ppi/FigureS3-MtbInfectMOI.png" class="lightbox" data-glightbox="description: .lightbox-desc-10" data-gallery="results_TrifunctionalProteomics"><img src="../../parts/results/results_TrifunctionalProteomics/300ppi/FigureS3-MtbInfectMOI.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-suppfig" id="suppfig-TrifunctionalProteomics-FigS3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure S6.3: <strong>H37Rv and H37Rv-ΔRD1 MOI Optimization</strong> THP-1 cells were infected with <em>Mtb</em> strains H37Rv (A) and H37Rv-ΔRD1 (B) at increasing multiplicities of infection (MOI) for 24 and 96 hours to determine optimal infection rates. At denoted time points, cells were fixed and stained with rhodamine (bacterial cell wall) and DAPI (nuclei) to determine how many cells were infected.
</figcaption>
</figure>
</div>
</div>
<p><br></p>
<div class="border">
<div id="suppfig-TrifunctionalProteomics-FigS4" class="quarto-float" data-group="results_TrifunctionalProteomics">
<figure class="quarto-float quarto-float-suppfig figure">
<div aria-describedby="suppfig-TrifunctionalProteomics-FigS4-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_TrifunctionalProteomics/300ppi/FigureS4-MtbInfectQC.png" class="lightbox" data-glightbox="description: .lightbox-desc-11" data-gallery="results_TrifunctionalProteomics"><img src="../../parts/results/results_TrifunctionalProteomics/300ppi/FigureS4-MtbInfectQC.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-suppfig" id="suppfig-TrifunctionalProteomics-FigS4-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure S6.4: <strong>Sample quality control for Mtb infection interactomics experiment</strong> (A) Representative in-gel fluorescence quality control of proteins isolated from <em>Mtb</em>-infected THP-1 cells treated with tf-Sphingosine or tf-8:3-Fatty acid. (B) Principle component analysis of protein abundance in each sample after batch effect removal and normalization. Shapes and colors denote infection conditions. (C) Box plots depict the normalization of MS data. Left panel: raw intensities of each sample for each condition; middle panel: batch-corrected intensities; right panel: median-normalized MS intensities. Color denotes replicate number.
</figcaption>
</figure>
</div>
</div>
<p><br></p>
<div class="border">
<div id="suppfig-TrifunctionalProteomics-FigS5" class="quarto-float" data-group="results_TrifunctionalProteomics">
<figure class="quarto-float quarto-float-suppfig figure">
<div aria-describedby="suppfig-TrifunctionalProteomics-FigS5-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_TrifunctionalProteomics/300ppi/FigureS5-MtbInfectMA.png" class="lightbox" data-glightbox="description: .lightbox-desc-12" data-gallery="results_TrifunctionalProteomics"><img src="../../parts/results/results_TrifunctionalProteomics/300ppi/FigureS5-MtbInfectMA.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-suppfig" id="suppfig-TrifunctionalProteomics-FigS5-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure S6.5: <strong><em>Mtb</em> infection proteomics data quality control, MA plots</strong> MA plots depict the distribution of average protein abundances between denoted conditions (x-axis) and the log2-transformed ratio of denoted conditions (y-axis). Point color denotes the statistical annotation of each protein (as denoted). Point size and shape denote enrichment toward tf-Sphingosine in baseline proteomics experiment (as denoted).
</figcaption>
</figure>
</div>
</div>
<p><br></p>
<div class="border">
<div id="suppfig-TrifunctionalProteomics-FigS6" class="quarto-float" data-group="results_TrifunctionalProteomics">
<figure class="quarto-float quarto-float-suppfig figure">
<div aria-describedby="suppfig-TrifunctionalProteomics-FigS6-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_TrifunctionalProteomics/300ppi/FigureS8-ProteomicsVStranscriptomics.png" class="lightbox" data-glightbox="description: .lightbox-desc-13" data-gallery="results_TrifunctionalProteomics"><img src="../../parts/results/results_TrifunctionalProteomics/300ppi/FigureS8-ProteomicsVStranscriptomics.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-suppfig" id="suppfig-TrifunctionalProteomics-FigS6-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure S6.6: <strong>Comparison against publicly-available transcriptomics data</strong> Transcriptomics data collected by Jani et al.<span class="citation" data-cites="janiTLR2NonredundantPopulation2023"><sup><a href="../references.html#ref-janiTLR2NonredundantPopulation2023" role="doc-biblioref">24</a></sup></span> (publicly available at GEO accession GSE227851) were compared against log2-transformed ratios of H37Rv-versus-Mock infected cells treated with tf-Sphingosine to assess correlations between transcriptional regulation and interaction enrichment. Point size and shape denote the enrichment of each protein during the baseline proteomics experiment (+/-UV); point color denotes statistical annotation among <em>Mtb</em> infection data (as denoted).
</figcaption>
</figure>
</div>
</div>
<p><br></p>
<div class="border">
<div id="suppfig-TrifunctionalProteomics-tfSphvstfFA" class="quarto-float" data-group="results_TrifunctionalProteomics">
<figure class="quarto-float quarto-float-suppfig figure">
<div aria-describedby="suppfig-TrifunctionalProteomics-tfSphvstfFA-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_TrifunctionalProteomics/300ppi/Figure6-WT_SphvsFA.png" class="lightbox" data-glightbox="description: .lightbox-desc-14" data-gallery="results_TrifunctionalProteomics"><img src="../../parts/results/results_TrifunctionalProteomics/300ppi/Figure6-WT_SphvsFA.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-suppfig" id="suppfig-TrifunctionalProteomics-tfSphvstfFA-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure S6.7: <strong>Distinct protein enrichment to tf-Sph and tf-8:3-FA</strong> Dot plots depict the comparison between protein enrichment for tf-Sph (x-axes) and tf-8:3-FA (y-axes) for the H37Rv versus Mock ratios at 24 (left panel) and 96 hours (right panel). Point color denotes whether each protein was significantly enriched along the x-axis, the y-axis, or both axes. The red line denotes a linear regression analysis on proteins identified in both experiments; red text describes the formula and fit parameters of the line of best fit between the two comparisons.
</figcaption>
</figure>
</div>
</div>
<p><br></p>
<div class="border">
<div id="suppfig-TrifunctionalProteomics-FigS7" class="quarto-float" data-group="results_TrifunctionalProteomics">
<figure class="quarto-float quarto-float-suppfig figure">
<div aria-describedby="suppfig-TrifunctionalProteomics-FigS7-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_TrifunctionalProteomics/300ppi/FigureS6-RD1vsMock.png" class="lightbox" data-glightbox="description: .lightbox-desc-15" data-gallery="results_TrifunctionalProteomics"><img src="../../parts/results/results_TrifunctionalProteomics/300ppi/FigureS6-RD1vsMock.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-suppfig" id="suppfig-TrifunctionalProteomics-FigS7-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure S6.8: <strong>Lipid:protein interactions enriched during infection with H37Rv-ΔRD1</strong> Volcano plots depict the enrichment of proteins toward tf-Sphingosine or tf-8:3-Fatty acid at 24- and 96-hours post-infection with <em>Mtb</em> strain H37Rv-ΔRD1 versus mock-infected control. Point size and shape denote the enrichment of each protein during the baseline proteomics experiment (+/-UV); point color denotes statistical annotation among <em>Mtb</em> infection experimental results (as denoted).
</figcaption>
</figure>
</div>
</div>
<p><br></p>
<div class="border">
<div id="suppfig-TrifunctionalProteomics-FigS8" class="quarto-float" data-group="results_TrifunctionalProteomics">
<figure class="quarto-float quarto-float-suppfig figure">
<div aria-describedby="suppfig-TrifunctionalProteomics-FigS8-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/results/results_TrifunctionalProteomics/300ppi/FigureS6-SupplementalvsPlots.png" class="lightbox" data-glightbox="description: .lightbox-desc-16" data-gallery="results_TrifunctionalProteomics"><img src="../../parts/results/results_TrifunctionalProteomics/300ppi/FigureS6-SupplementalvsPlots.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-suppfig" id="suppfig-TrifunctionalProteomics-FigS8-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure S6.9: <strong>Infection with H37Rv-ΔRD1 has minimal effects on the tf-Sph interactome</strong> Scatterplots depict the log2 fold-change of the denoted conditions against each other, enabling comparison of enriched proteins in control conditions. Point color and size denote the relative statistical annotation of each protein between the comparison conditions depicted on the x- and y-axes (hit = Log2FC &gt; 1 &amp; FDR &lt; 0.05). The Red dashed line denotes the line of best fit between the axes (determined via linear regression); the red text describes the formula for a line of best fit and the R<sup>2</sup> value.
</figcaption>
</figure>
</div>
</div>


<div class="hidden" aria-hidden="true">
<span class="glightbox-desc lightbox-desc-1">Figure&nbsp;6.1: <strong>Lipid probe overview &amp; workflows</strong> (A) The chemical structure of native sphinganine, sphingosine, and free fatty acid, alongside their synthetic trifunctionalized analogs. The functional moieties are highlighted and described in the color-coordinated boxes. (B) A simplified overview of several trifunctional probe workflows. See <a href="#sec-TrifunctionalProteomics-Methods-ProbeTreatment" class="quarto-xref">Methods Section&nbsp;<span>6.7.3</span></a> for a detailed description of the workflow. Briefly, plated cells are treated with lipid probes for 30 minutes (1), uncaged via irradiation with blue-wavelength light (2), and then covalently crosslinked to nearby molecules via irradiation with UV-wavelength light (3). Next, the probe may be clicked to a fluorophore and stained with cellular markers for fluorescence microscopy (4) or lysed and protein collected for in-gel fluorescence or proteomics analysis (5). (C) Example uses of trifunctional probes: (left panel) fluorescence microscopy using tf-Spa; (middle panel) in-gel fluorescence using tf-Spa; (right panel) cartoon depiction of an MS2 scan of a peptide tagged with a tandem mass tag.</span>
<span class="glightbox-desc lightbox-desc-2">Figure&nbsp;6.2: <strong>Fluorescence microscopy reveals intracellular trafficking of trifunctional sphinganine</strong> THP-1 cells were treated with trifunctional sphinganine, uncaged via irradiation with blue light, and incubated for the denoted periods before UV-induced crosslinking and methanol fixation. Cells were stained with anti-Giantin antibody (to visualize the Golgi apparatus) and DAPI (to visualize the nucleus). Cells were imaged using a Zeiss LSM 900 confocal microscope using AiryScan super-resolution and a 63x oil-immersion objective. Prior to imaging, cells were stained with 1:500 mouse anti-Giantin primary antibody (abcam #ab37266) and 1:1000 goat anti-mouse Alexa488-conjugated secondary antibody (Invitrogen #A21042) (See <a href="results_Mtb_Membrane_Damage_V2.html#sec-MtbDamage-Methods-MicroscopyStaining" class="quarto-xref">Methods Section&nbsp;<span>4.5.5</span></a> for staining protocol).</span>
<span class="glightbox-desc lightbox-desc-3">Figure&nbsp;6.3: <strong>Affinity enrichment proteomics reveals the protein interactors of trifunctional lipid probes</strong> (A) Schematic depiction of experimental design. Briefly, THP-1 cells were treated with the denoted probes described in <a href="#sec-TrifunctionalProteomics-Methods-ProbeTreatment" class="quarto-xref">Methods Section&nbsp;<span>6.7.3</span></a>. Cells were lysed, and proteins were extracted; probe-protein conjugates were enriched using azide-coated beads and subjected to mass spectrometry analysis. (B) Volcano plots depict the relative enrichment of proteins to each probe compared to probe-matched no-crosslinking (-UV) control lysate. Enrichment is defined as follows: Depleted = Log2FC &lt; 0, Partially enriched = Log2FC &gt; 0 &amp; p-value &gt; 0.05, Significantly enriched = Log2FC &gt; 1 &amp; p-value &lt; 0.05.</span>
<span class="glightbox-desc lightbox-desc-4">Figure&nbsp;6.4: <strong>Proteins significantly enriched to lipid probes are known interactors of lipids</strong> (A) Bar plots depict the Log2FC enrichment for each of the 26 significantly enriched proteins across all lipid probes (+UV versus -UV). Significance is denoted by fill coloring; probe is denoted by bar color. (B) Pie charts show the relative enrichment for transmembrane and membrane-associating proteins among the four lipid probes. (C) GO Term analysis of partially and significantly enriched proteins across each lipid probe. (D) Example on-blot fluorescence depicting simultaneous visualization of probe-crosslinked and antibody-stained VDAC proteins. (Red = Azide647-clicked probe; blue = anti-VDAC antibody).</span>
<span class="glightbox-desc lightbox-desc-5">Figure&nbsp;6.5: <strong><em>Mtb</em> infection reprograms the sphingosine interactome</strong> (A) A schematic overview of infection design and timeline. (B) Volcano plots depict the differential enrichment of proteins to the tf-Sphingosine and tf-8:3-Fatty Acid probes in H37Rv infection versus uninfected control. Statistical significance is as denoted (Partially significant, purple = Log2FC &gt; 1 &amp; p-value &lt; 0.05; Significant, orange = Log2FC &gt; 2, FDR &lt; 0.05, &amp; p-value &lt; 0.01; No significance in black).</span>
<span class="glightbox-desc lightbox-desc-6">Figure&nbsp;6.6: <strong>Proteins enriched during H37Rv infection are highly similar to those enriched during H37Rv-ΔRD1 infection</strong> Volcano plots depict the enrichment of proteins in H37Rv infection in comparison to H37Rv-ΔRD1 infection for both tf-Sphingosine and tf-Fatty acid probe at 24- and 96-hours post-infection. Point size and shape denote the enrichment of each protein in the baseline proteomics experiment (+/-UV); point color denotes the statistical annotation among <em>Mtb</em> infection experimental results (as denoted).</span>
<span class="glightbox-desc lightbox-desc-7">Figure&nbsp;6.7: <strong>Potential <em>Mtb</em> protein identification: Rv3466</strong> A summary of low-confidence identifications of three <em>Mtb</em> proteins accD2, RV1910c, and Rv3466. (A) Enrichment of each protein in infection versus mock comparisons. Blue bars are infections with pathogenic strain H37Rv; pink bars are infections with attenuated strain H37Rv-ΔRD1. (B) A depiction of the percentage of coverage for each protein. (C) Prediction of RV3466 structure using AlphaFold. Blue-coded sequence denotes high-confidence structure prediction; red- and yellow-coded sequence denotes low confidence prediction.</span>
<span class="glightbox-desc lightbox-desc-8">Figure S6.1: <strong>Sample quality control for baseline probe interactomics experiment</strong> (A) Principle component analysis of protein abundance in each sample after batch effect removal and normalization. Shapes and colors denote conditions. (B) Box plots depict the normalization of MS data. Left panel: raw intensities of each sample for each condition; middle panel: batch-corrected intensities; right panel: median-normalized MS intensities. Color denotes replicate number.</span>
<span class="glightbox-desc lightbox-desc-9">Figure S6.2: <strong>Baseline THP-1 probe proteomics, MA plots</strong> MA plots depict the distribution of mean protein abundances between denoted conditions (x-axis) and the log2-transformed ratio of the denoted conditions (y-axis). Point size, shape, and color denote the enrichment annotation of each protein.</span>
<span class="glightbox-desc lightbox-desc-10">Figure S6.3: <strong>H37Rv and H37Rv-ΔRD1 MOI Optimization</strong> THP-1 cells were infected with <em>Mtb</em> strains H37Rv (A) and H37Rv-ΔRD1 (B) at increasing multiplicities of infection (MOI) for 24 and 96 hours to determine optimal infection rates. At denoted time points, cells were fixed and stained with rhodamine (bacterial cell wall) and DAPI (nuclei) to determine how many cells were infected.</span>
<span class="glightbox-desc lightbox-desc-11">Figure S6.4: <strong>Sample quality control for Mtb infection interactomics experiment</strong> (A) Representative in-gel fluorescence quality control of proteins isolated from <em>Mtb</em>-infected THP-1 cells treated with tf-Sphingosine or tf-8:3-Fatty acid. (B) Principle component analysis of protein abundance in each sample after batch effect removal and normalization. Shapes and colors denote infection conditions. (C) Box plots depict the normalization of MS data. Left panel: raw intensities of each sample for each condition; middle panel: batch-corrected intensities; right panel: median-normalized MS intensities. Color denotes replicate number.</span>
<span class="glightbox-desc lightbox-desc-12">Figure S6.5: <strong><em>Mtb</em> infection proteomics data quality control, MA plots</strong> MA plots depict the distribution of average protein abundances between denoted conditions (x-axis) and the log2-transformed ratio of denoted conditions (y-axis). Point color denotes the statistical annotation of each protein (as denoted). Point size and shape denote enrichment toward tf-Sphingosine in baseline proteomics experiment (as denoted).</span>
<span class="glightbox-desc lightbox-desc-13">Figure S6.6: <strong>Comparison against publicly-available transcriptomics data</strong> Transcriptomics data collected by Jani et al.<span class="citation" data-cites="janiTLR2NonredundantPopulation2023"><sup><a href="../references.html#ref-janiTLR2NonredundantPopulation2023" role="doc-biblioref">24</a></sup></span> (publicly available at GEO accession GSE227851) were compared against log2-transformed ratios of H37Rv-versus-Mock infected cells treated with tf-Sphingosine to assess correlations between transcriptional regulation and interaction enrichment. Point size and shape denote the enrichment of each protein during the baseline proteomics experiment (+/-UV); point color denotes statistical annotation among <em>Mtb</em> infection data (as denoted).</span>
<span class="glightbox-desc lightbox-desc-14">Figure S6.7: <strong>Distinct protein enrichment to tf-Sph and tf-8:3-FA</strong> Dot plots depict the comparison between protein enrichment for tf-Sph (x-axes) and tf-8:3-FA (y-axes) for the H37Rv versus Mock ratios at 24 (left panel) and 96 hours (right panel). Point color denotes whether each protein was significantly enriched along the x-axis, the y-axis, or both axes. The red line denotes a linear regression analysis on proteins identified in both experiments; red text describes the formula and fit parameters of the line of best fit between the two comparisons.</span>
<span class="glightbox-desc lightbox-desc-15">Figure S6.8: <strong>Lipid:protein interactions enriched during infection with H37Rv-ΔRD1</strong> Volcano plots depict the enrichment of proteins toward tf-Sphingosine or tf-8:3-Fatty acid at 24- and 96-hours post-infection with <em>Mtb</em> strain H37Rv-ΔRD1 versus mock-infected control. Point size and shape denote the enrichment of each protein during the baseline proteomics experiment (+/-UV); point color denotes statistical annotation among <em>Mtb</em> infection experimental results (as denoted).</span>
<span class="glightbox-desc lightbox-desc-16">Figure S6.9: <strong>Infection with H37Rv-ΔRD1 has minimal effects on the tf-Sph interactome</strong> Scatterplots depict the log2 fold-change of the denoted conditions against each other, enabling comparison of enriched proteins in control conditions. Point color and size denote the relative statistical annotation of each protein between the comparison conditions depicted on the x- and y-axes (hit = Log2FC &gt; 1 &amp; FDR &lt; 0.05). The Red dashed line denotes the line of best fit between the axes (determined via linear regression); the red text describes the formula for a line of best fit and the R<sup>2</sup> value.</span>
</div>
<div id="refs" class="references csl-bib-body" role="list" style="display: none">
<div id="ref-sharmaSphingolipidsAreDual2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">1. </div><div class="csl-right-inline">Sharma, L., and Prakash, H. (2017). Sphingolipids are dual specific drug targets for the management of pulmonary infections: <span>Perspective</span>. Frontiers in Immunology <em>8</em>. <a href="https://doi.org/10.3389/fimmu.2017.00378">10.3389/fimmu.2017.00378</a>.</div>
</div>
<div id="ref-mohammedSphingolipidsMycobacteriaHost2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">2. </div><div class="csl-right-inline">Mohammed, S.A., Saini, R.V., Jha, A.K., Hadda, V., Singh, A.K., and Prakash, H. (2022). Sphingolipids, mycobacteria and host: <span>Unraveling</span> the tug of war. Front. Immunol. <em>13</em>, 1003384. <a href="https://doi.org/10.3389/fimmu.2022.1003384">10.3389/fimmu.2022.1003384</a>.</div>
</div>
<div id="ref-barischBreakingFatHow2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">3. </div><div class="csl-right-inline">Barisch, C., and Soldati, T. (2017). Breaking fat! <span>How</span> mycobacteria and other intracellular pathogens manipulate host lipid droplets. Biochimie <em>141</em>, 54–61. <a href="https://doi.org/10.1016/j.biochi.2017.06.001">10.1016/j.biochi.2017.06.001</a>.</div>
</div>
<div id="ref-malikCuttingEdgeMycobacterium2003a" class="csl-entry" role="listitem">
<div class="csl-left-margin">4. </div><div class="csl-right-inline">Malik, Z.A., Thompson, C.R., Hashimi, S., Porter, B., Iyer, S.S., and Kusner, D.J. (2003). Cutting <span>Edge</span>: <span>Mycobacterium</span> tuberculosis <span>Blocks Ca2</span>+ <span>Signaling</span> and <span>Phagosome Maturation</span> in <span>Human Macrophages Via Specific Inhibition</span> of <span>Sphingosine Kinase</span>. The Journal of Immunology <em>170</em>, 2811–2815. <a href="https://doi.org/10.4049/jimmunol.170.6.2811">10.4049/jimmunol.170.6.2811</a>.</div>
</div>
<div id="ref-schultzChemicalToolsLipid2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">5. </div><div class="csl-right-inline">Schultz, C. (2023). Chemical <span>Tools</span> for <span>Lipid Cell Biology</span>. Acc. Chem. Res. <em>56</em>, 1168–1177. <a href="https://doi.org/10.1021/acs.accounts.2c00851">10.1021/acs.accounts.2c00851</a>.</div>
</div>
<div id="ref-farleyTrifunctionalSphinganineNew2024" class="csl-entry" role="listitem">
<div class="csl-left-margin">6. </div><div class="csl-right-inline">Farley, S., Stein, F., Haberkant, P., Tafesse, F.G., and Schultz, C. (2024). Trifunctional <span>Sphinganine</span>: <span>A New Tool</span> to <span>Dissect Sphingolipid Function</span>. ACS Chem. Biol. <a href="https://doi.org/10.1021/acschembio.3c00554">10.1021/acschembio.3c00554</a>.</div>
</div>
<div id="ref-hoglingerTrifunctionalLipidProbes2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">7. </div><div class="csl-right-inline">Höglinger, D., Nadler, A., Haberkant, P., Kirkpatrick, J., Schifferer, M., Stein, F., Hauke, S., Porter, F.D., and Schultz, C. (2017). Trifunctional lipid probes for comprehensive studies of single lipid species in living cells. Proceedings of the National Academy of Sciences <em>114</em>, 1566–1571. <a href="https://doi.org/10.1073/pnas.1611096114">10.1073/pnas.1611096114</a>.</div>
</div>
<div id="ref-weigertSphingosine1PhosphateMacrophageBiology2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">8. </div><div class="csl-right-inline">Weigert, A., Olesch, C., and Brüne, B. (2019). Sphingosine-1-<span>Phosphate</span> and <span>Macrophage Biology</span>—<span>How</span> the <span>Sphinx Tames</span> the <span>Big Eater</span>. Frontiers in Immunology <em>10</em>, 1–19. <a href="https://doi.org/10.3389/fimmu.2019.01706">10.3389/fimmu.2019.01706</a>.</div>
</div>
<div id="ref-hoglingerIntracellularSphingosineReleases2015a" class="csl-entry" role="listitem">
<div class="csl-left-margin">9. </div><div class="csl-right-inline">Höglinger, D., Haberkant, P., Aguilera-Romero, A., Riezman, H., Porter, F.D., Platt, F.M., Galione, A., and Schultz, C. (2015). Intracellular sphingosine releases calcium from lysosomes. eLife <em>4</em>. <a href="https://doi.org/10.7554/eLife.10616">10.7554/eLife.10616</a>.</div>
</div>
<div id="ref-haberkantBifunctionalSphingosineCellBased2016a" class="csl-entry" role="listitem">
<div class="csl-left-margin">10. </div><div class="csl-right-inline">Haberkant, P., Stein, F., Höglinger, D., Gerl, M.J., Brügger, B., Van Veldhoven, P.P., Krijgsveld, J., Gavin, A.-C.C., and Schultz, C. (2016). Bifunctional <span>Sphingosine</span> for <span>Cell-Based Analysis</span> of <span>Protein-Sphingolipid Interactions</span>. ACS Chemical Biology <em>11</em>, 222–230. <a href="https://doi.org/10.1021/acschembio.5b00810">10.1021/acschembio.5b00810</a>.</div>
</div>
<div id="ref-haberkantFatFabulousBifunctional2014" class="csl-entry" role="listitem">
<div class="csl-left-margin">11. </div><div class="csl-right-inline">Haberkant, P., and Holthuis, J.C.M. (2014). Fat &amp; fabulous: <span>Bifunctional</span> lipids in the spotlight. Biochimica et Biophysica Acta - Molecular and Cell Biology of Lipids <em>1841</em>, 1022–1030. <a href="https://doi.org/10.1016/j.bbalip.2014.01.003">10.1016/j.bbalip.2014.01.003</a>.</div>
</div>
<div id="ref-haberkantProteinsphingolipidInteractionsCellular2008" class="csl-entry" role="listitem">
<div class="csl-left-margin">12. </div><div class="csl-right-inline">Haberkant, P., Schmitt, O., Contreras, F.-X., Thiele, C., Hanada, K., Sprong, H., Reinhard, C., Wieland, F.T., and Brügger, B. (2008). Protein-sphingolipid interactions within cellular membranes. Journal of Lipid Research <em>49</em>, 251–262. <a href="https://doi.org/10.1194/jlr.D700023-JLR200">10.1194/jlr.D700023-JLR200</a>.</div>
</div>
<div id="ref-gaultOverviewSphingolipidMetabolism2010a" class="csl-entry" role="listitem">
<div class="csl-left-margin">13. </div><div class="csl-right-inline">Gault, C.R., Obeid, L.M., and Hannun, Y.A. (2010). An <span>Overview</span> of <span>Sphingolipid Metabolism</span>: <span>From Synthesis</span> to <span>Breakdown</span>. In Adv <span>Exp Med Biol</span>, pp. 1–23. <a href="https://doi.org/10.1007/978-1-4419-6741-1_1">10.1007/978-1-4419-6741-1_1</a>.</div>
</div>
<div id="ref-ritchieLimmaPowersDifferential2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">14. </div><div class="csl-right-inline">Ritchie, M.E., Phipson, B., Wu, D., Hu, Y., Law, C.W., Shi, W., and Smyth, G.K. (2015). Limma powers differential expression analyses for <span class="nocase">RNA-sequencing</span> and microarray studies. Nucleic Acids Research <em>43</em>, e47–e47. <a href="https://doi.org/10.1093/nar/gkv007">10.1093/nar/gkv007</a>.</div>
</div>
<div id="ref-dadsenaCeramidesBindVDAC22019a" class="csl-entry" role="listitem">
<div class="csl-left-margin">15. </div><div class="csl-right-inline">Dadsena, S., Bockelmann, S., Mina, J.G.M., Hassan, D.G., Korneev, S., Razzera, G., Jahn, H., Niekamp, P., Müller, D., Schneider, M., et al. (2019). Ceramides bind <span>VDAC2</span> to trigger mitochondrial apoptosis. Nature Communications <em>10</em>, 1832. <a href="https://doi.org/10.1038/s41467-019-09654-4">10.1038/s41467-019-09654-4</a>.</div>
</div>
<div id="ref-jahnPhospholipidsAreImported2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">16. </div><div class="csl-right-inline">Jahn, H., Bartoš, L., Dearden, G.I., Dittman, J.S., Holthuis, J.C.M., Vácha, R., and Menon, A.K. (2023). Phospholipids are imported into mitochondria by <span>VDAC</span>, a dimeric beta barrel scramblase. Nat Commun <em>14</em>, 8115. <a href="https://doi.org/10.1038/s41467-023-43570-y">10.1038/s41467-023-43570-y</a>.</div>
</div>
<div id="ref-taniguchiLysosomalCeramideGenerated2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">17. </div><div class="csl-right-inline">Taniguchi, M., Ogiso, H., Takeuchi, T., Kitatani, K., Umehara, H., and Okazaki, T. (2015). Lysosomal ceramide generated by acid sphingomyelinase triggers cytosolic cathepsin <span class="nocase">B-mediated</span> degradation of <span class="nocase">X-linked</span> inhibitor of apoptosis protein in natural killer/<span>T</span> lymphoma cell apoptosis. Cell Death Dis <em>6</em>, e1717–e1717. <a href="https://doi.org/10.1038/cddis.2015.82">10.1038/cddis.2015.82</a>.</div>
</div>
<div id="ref-heinrichCathepsinTargetedAcid2000" class="csl-entry" role="listitem">
<div class="csl-left-margin">18. </div><div class="csl-right-inline">Heinrich, M., Wickel, M., Schneider-Brachert, W., Sandberg, C., Gahr, J., Schwandner, R., Weber, T., Brunner, J., Krönke, M., and Schütze, S. (2000). Cathepsin <span>D</span> targeted by acid sphingomyelinase-derived ceramide. EMBO J <em>19</em>, 315–315. <a href="https://doi.org/10.1038/sj.emboj.7592140">10.1038/sj.emboj.7592140</a>.</div>
</div>
<div id="ref-zebrakovskaComplexModulationPeptidolytic2011" class="csl-entry" role="listitem">
<div class="csl-left-margin">19. </div><div class="csl-right-inline">Žebrakovská, I., Máša, M., Srp, J., Horn, M., Vávrová, K., and Mareš, M. (2011). Complex modulation of peptidolytic activity of cathepsin <span>D</span> by sphingolipids. Biochimica et Biophysica Acta (BBA) - Molecular and Cell Biology of Lipids <em>1811</em>, 1097–1104. <a href="https://doi.org/10.1016/j.bbalip.2011.09.005">10.1016/j.bbalip.2011.09.005</a>.</div>
</div>
<div id="ref-kishimotoSaposinsStructureFunction1992a" class="csl-entry" role="listitem">
<div class="csl-left-margin">20. </div><div class="csl-right-inline">Kishimoto, Y., Hiraiwa, M., and O’Brien, J.S. (1992). <a href="http://www.ncbi.nlm.nih.gov/pubmed/1402395">Saposins: Structure, function, distribution, and molecular genetics.</a> Journal of lipid research <em>33</em>, 1255–1267.</div>
</div>
<div id="ref-morimotoSaposinSecondCerebrosidase1989" class="csl-entry" role="listitem">
<div class="csl-left-margin">21. </div><div class="csl-right-inline">Morimoto, S., Martin, B.M., Yamamoto, Y., Kretz, K.A., O’Brien, J.S., and Kishimoto, Y. (1989). Saposin <span>A</span>: Second cerebrosidase activator protein. Proc. Natl. Acad. Sci. U.S.A. <em>86</em>, 3389–3393. <a href="https://doi.org/10.1073/pnas.86.9.3389">10.1073/pnas.86.9.3389</a>.</div>
</div>
<div id="ref-xiongStructureHumanAcid2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">22. </div><div class="csl-right-inline">Xiong, Z.J., Huang, J., Poda, G., Pomès, R., and Privé, G.G. (2016). Structure of <span>Human Acid Sphingomyelinase Reveals</span> the <span>Role</span> of the <span>Saposin Domain</span> in <span>Activating Substrate Hydrolysis</span>. Journal of Molecular Biology <em>428</em>, 3026–3042. <a href="https://doi.org/10.1016/j.jmb.2016.06.012">10.1016/j.jmb.2016.06.012</a>.</div>
</div>
<div id="ref-kimLysosomalCeramidesRegulate2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">23. </div><div class="csl-right-inline">Kim, M.J., Jeong, H., and Krainc, D. (2022). Lysosomal ceramides regulate cathepsin <span class="nocase">B-mediated</span> processing of saposin <span>C</span> and glucocerebrosidase activity. Human Molecular Genetics <em>31</em>, 2424–2437. <a href="https://doi.org/10.1093/hmg/ddac047">10.1093/hmg/ddac047</a>.</div>
</div>
<div id="ref-janiTLR2NonredundantPopulation2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">24. </div><div class="csl-right-inline">Jani, C., Solomon, S.L., Peters, J.M., Pringle, S.C., Hinman, A.E., Boucau, J., Bryson, B.D., and Barczak, A.K. (2023). <span>TLR2</span> is non-redundant in the population and subpopulation responses to <span><span><em>Mycobacterium</em></span></span><span><span><em> Tuberculosis</em></span></span> in macrophages and <span><span><em>in Vivo</em></span></span>. mSystems, e00052–23. <a href="https://doi.org/10.1128/msystems.00052-23">10.1128/msystems.00052-23</a>.</div>
</div>
<div id="ref-lyckALCAMCD166Involved2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">25. </div><div class="csl-right-inline">Lyck, R., Lécuyer, M.-A., Abadier, M., Wyss, C.B., Matti, C., Rosito, M., Enzmann, G., Zeis, T., Michel, L., García Martín, A.B., et al. (2017). <span>ALCAM</span> (<span>CD166</span>) is involved in extravasation of monocytes rather than <span>T</span> cells across the blood–brain barrier. J Cereb Blood Flow Metab <em>37</em>, 2894–2909. <a href="https://doi.org/10.1177/0271678X16678639">10.1177/0271678X16678639</a>.</div>
</div>
<div id="ref-fletcherTranscriptionalProfilingMycobacterial2009" class="csl-entry" role="listitem">
<div class="csl-left-margin">26. </div><div class="csl-right-inline">Fletcher, H.A., Keyser, A., Bowmaker, M., Sayles, P.C., Kaplan, G., Hussey, G., Hill, A.V., and Hanekom, W.A. (2009). Transcriptional profiling of mycobacterial antigen-induced responses in infants vaccinated with <span>BCG</span> at birth. BMC Med Genomics <em>2</em>, 10. <a href="https://doi.org/10.1186/1755-8794-2-10">10.1186/1755-8794-2-10</a>.</div>
</div>
<div id="ref-shepelkovaSaposinActingMacrophage2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">27. </div><div class="csl-right-inline">Shepelkova, G.S., Evstifeev, V.V., Ergeshov, A.E., and Yeremeev, V.V. (2020). Saposin <span>D</span> acting on macrophage bacteriostatic function in experimental tuberculosis infection. Russian Journal of Infection and Immunity <em>11</em>, 473–480. <a href="https://doi.org/10.15789/2220-7619-TEO-1386">10.15789/2220-7619-TEO-1386</a>.</div>
</div>
<div id="ref-hussainMicrosomalTriglycerideTransfer2003" class="csl-entry" role="listitem">
<div class="csl-left-margin">28. </div><div class="csl-right-inline">Hussain, M.M., Shi, J., and Dreizen, P. (2003). Microsomal triglyceride transfer protein and its role in <span class="nocase">apoB-lipoprotein</span> assembly. Journal of Lipid Research <em>44</em>, 22–32. <a href="https://doi.org/10.1194/jlr.R200014-JLR200">10.1194/jlr.R200014-JLR200</a>.</div>
</div>
<div id="ref-douganMicrosomalTriglycerideTransfer2005" class="csl-entry" role="listitem">
<div class="csl-left-margin">29. </div><div class="csl-right-inline">Dougan, S.K., Salas, A., Rava, P., Agyemang, A., Kaser, A., Morrison, J., Khurana, A., Kronenberg, M., Johnson, C., Exley, M., et al. (2005). Microsomal triglyceride transfer protein lipidation and control of <span>CD1d</span> on antigen-presenting cells. The Journal of Experimental Medicine <em>202</em>, 529–539. <a href="https://doi.org/10.1084/jem.20050183">10.1084/jem.20050183</a>.</div>
</div>
<div id="ref-deliberoTCellResponseLipid2014" class="csl-entry" role="listitem">
<div class="csl-left-margin">30. </div><div class="csl-right-inline">De Libero, G., and Mori, L. (2014). The <span>T-Cell Response</span> to <span>Lipid Antigens</span> of <span>Mycobacterium</span> tuberculosis. Front. Immunol. <em>5</em>. <a href="https://doi.org/10.3389/fimmu.2014.00219">10.3389/fimmu.2014.00219</a>.</div>
</div>
<div id="ref-yangSinglecellProfilingReveals2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">31. </div><div class="csl-right-inline">Yang, X., Yan, J., Xue, Y., Sun, Q., Zhang, Y., Guo, R., Wang, C., Li, X., Liang, Q., Wu, H., et al. (2023). Single-cell profiling reveals distinct immune response landscapes in tuberculous pleural effusion and non-<span>TPE</span>. Front. Immunol. <em>14</em>, 1191357. <a href="https://doi.org/10.3389/fimmu.2023.1191357">10.3389/fimmu.2023.1191357</a>.</div>
</div>
<div id="ref-thirunavukkarasuPolyADPribosePolymerase2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">32. </div><div class="csl-right-inline">Thirunavukkarasu, S., Ahmed, M., Rosa, B.A., Boothby, M., Cho, S.H., Rangel-Moreno, J., Mbandi, S.K., Schreiber, V., Gupta, A., Zuniga, J., et al. (2023). Poly(<span>ADP-ribose</span>) polymerase 9 mediates early protection against <span>Mycobacterium</span> tuberculosis infection by regulating type <span>I IFN</span> production. Journal of Clinical Investigation <em>133</em>, e158630. <a href="https://doi.org/10.1172/JCI158630">10.1172/JCI158630</a>.</div>
</div>
<div id="ref-leischingOAS1OAS2OAS32019" class="csl-entry" role="listitem">
<div class="csl-left-margin">33. </div><div class="csl-right-inline">Leisching, G., Cole, V., Ali, A.T., and Baker, B. (2019). <span>OAS1</span>, <span>OAS2</span> and <span>OAS3</span> restrict intracellular <span>M</span>. Tb replication and enhance cytokine secretion. International Journal of Infectious Diseases <em>80</em>, S77–S84. <a href="https://doi.org/10.1016/j.ijid.2019.02.029">10.1016/j.ijid.2019.02.029</a>.</div>
</div>
<div id="ref-yiTranscriptionalProfilingHuman2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">34. </div><div class="csl-right-inline">Yi, F., Hu, J., Zhu, X., Wang, Y., Yu, Q., Deng, J., Huang, X., Ma, Y., and Xie, Y. (2021). Transcriptional <span>Profiling</span> of <span>Human Peripheral Blood Mononuclear Cells Stimulated</span> by <span>Mycobacterium</span> tuberculosis <span>PPE57 Identifies Characteristic Genes Associated With Type I Interferon Signaling</span>. Front. Cell. Infect. Microbiol. <em>11</em>, 716809. <a href="https://doi.org/10.3389/fcimb.2021.716809">10.3389/fcimb.2021.716809</a>.</div>
</div>
<div id="ref-liuRSL1D1PromotesProgression2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">35. </div><div class="csl-right-inline">Liu, X., Chen, J., Long, X., Lan, J., Liu, X., Zhou, M., Zhang, S., and Zhou, J. (2022). <span>RSL1D1</span> promotes the progression of colorectal cancer through <span class="nocase">RAN-mediated</span> autophagy suppression. Cell Death Dis <em>13</em>, 43. <a href="https://doi.org/10.1038/s41419-021-04492-z">10.1038/s41419-021-04492-z</a>.</div>
</div>
<div id="ref-becqContributionHorizontallyAcquired2007" class="csl-entry" role="listitem">
<div class="csl-left-margin">36. </div><div class="csl-right-inline">Becq, J., Gutierrez, M.C., Rosas-Magallanes, V., Rauzier, J., Gicquel, B., Neyrolles, O., and Deschavanne, P. (2007). Contribution of <span>Horizontally Acquired Genomic Islands</span> to the <span>Evolution</span> of the <span>Tubercle Bacilli</span>. Molecular Biology and Evolution <em>24</em>, 1861–1871. <a href="https://doi.org/10.1093/molbev/msm111">10.1093/molbev/msm111</a>.</div>
</div>
<div id="ref-minatoGenomewideAssessmentMycobacterium2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">37. </div><div class="csl-right-inline">Minato, Y., Gohl, D.M., Thiede, J.M., Chacón, J.M., Harcombe, W.R., Maruyama, F., and Baughn, A.D. (2019). Genomewide <span>Assessment</span> of <span>Mycobacterium</span> tuberculosis <span>Conditionally Essential Metabolic Pathways</span>. mSystems <em>4</em>, e00070–19. <a href="https://doi.org/10.1128/mSystems.00070-19">10.1128/mSystems.00070-19</a>.</div>
</div>
<div id="ref-boshoffTranscriptionalResponsesMycobacterium2004" class="csl-entry" role="listitem">
<div class="csl-left-margin">38. </div><div class="csl-right-inline">Boshoff, H.I.M., Myers, T.G., Copp, B.R., McNeil, M.R., Wilson, M.A., and Barry, C.E. (2004). The <span>Transcriptional Responses</span> of <span>Mycobacterium</span> tuberculosis to <span>Inhibitors</span> of <span>Metabolism</span>. Journal of Biological Chemistry <em>279</em>, 40174–40184. <a href="https://doi.org/10.1074/jbc.M406796200">10.1074/jbc.M406796200</a>.</div>
</div>
<div id="ref-galaganMycobacteriumTuberculosisRegulatory2013" class="csl-entry" role="listitem">
<div class="csl-left-margin">39. </div><div class="csl-right-inline">Galagan, J.E., Minch, K., Peterson, M., Lyubetskaya, A., Azizi, E., Sweet, L., Gomes, A., Rustad, T., Dolganov, G., Glotova, I., et al. (2013). The <span>Mycobacterium</span> tuberculosis regulatory network and hypoxia. Nature <em>499</em>, 178–183. <a href="https://doi.org/10.1038/nature12337">10.1038/nature12337</a>.</div>
</div>
<div id="ref-lewTubercuList10Years2011" class="csl-entry" role="listitem">
<div class="csl-left-margin">40. </div><div class="csl-right-inline">Lew, J.M., Kapopoulou, A., Jones, L.M., and Cole, S.T. (2011). <span>TubercuList</span> – 10 years after. Tuberculosis <em>91</em>, 1–7. <a href="https://doi.org/10.1016/j.tube.2010.09.008">10.1016/j.tube.2010.09.008</a>.</div>
</div>
<div id="ref-mateusThermalProteomeProfiling2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">41. </div><div class="csl-right-inline">Mateus, A., Määttä, T.A., and Savitski, M.M. (2016). Thermal proteome profiling: Unbiased assessment of protein state through heat-induced stability changes. Proteome Sci <em>15</em>, 13. <a href="https://doi.org/10.1186/s12953-017-0122-4">10.1186/s12953-017-0122-4</a>.</div>
</div>
<div id="ref-huberVarianceStabilizationApplied2002" class="csl-entry" role="listitem">
<div class="csl-left-margin">42. </div><div class="csl-right-inline">Huber, W., Von Heydebreck, A., Sültmann, H., Poustka, A., and Vingron, M. (2002). Variance stabilization applied to microarray data calibration and to the quantification of differential expression. Bioinformatics <em>18</em>, S96–S104. <a href="https://doi.org/10.1093/bioinformatics/18.suppl_1.S96">10.1093/bioinformatics/18.suppl_1.S96</a>.</div>
</div>
</div>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleGiscusIfUsed = (isAlternate, darkModeDefault) => {
    const baseTheme = document.querySelector('#giscus-base-theme')?.value ?? 'light';
    const alternateTheme = document.querySelector('#giscus-alt-theme')?.value ?? 'dark';
    let newTheme = '';
    if(darkModeDefault) {
      newTheme = isAlternate ? baseTheme : alternateTheme;
    } else {
      newTheme = isAlternate ? alternateTheme : baseTheme;
    }
    const changeGiscusTheme = () => {
      // From: https://github.com/giscus/giscus/issues/336
      const sendMessage = (message) => {
        const iframe = document.querySelector('iframe.giscus-frame');
        if (!iframe) return;
        iframe.contentWindow.postMessage({ giscus: message }, 'https://giscus.app');
      }
      sendMessage({
        setConfig: {
          theme: newTheme
        }
      });
    }
    const isGiscussLoaded = window.document.querySelector('iframe.giscus-frame') !== null;
    if (isGiscussLoaded) {
      changeGiscusTheme();
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  const darkModeDefault = false;
  let localAlternateSentinel = darkModeDefault ? 'alternate' : 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
    toggleGiscusIfUsed(toAlternate, darkModeDefault);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } else {
    toggleColorMode(false);
  }
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../../parts/results/results_Mtb_lipidomics_v2.html" class="pagination-link  aria-label=" &lt;span="" off="" the="" fat="" of="" cell&lt;="" span&gt;"="">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Living Off the Fat of the Cell</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="../../parts/addenda.html" class="pagination-link" aria-label="Additional work">
        <span class="nav-page-text">Additional work</span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
<p>Copyright 2024, Gaelen Guzman</p>
</div>   
    <div class="nav-footer-center">
      &nbsp;
    </div>
    <div class="nav-footer-right">
      &nbsp;
    </div>
  </div>
</footer>
<script>var lightboxQuarto = GLightbox({"selector":".lightbox","loop":true,"descPosition":"bottom","openEffect":"fade","closeEffect":"fade"});
window.onload = () => {
  lightboxQuarto.on('slide_before_load', (data) => {
    const { slideIndex, slideNode, slideConfig, player, trigger } = data;
    const href = trigger.getAttribute('href');
    if (href !== null) {
      const imgEl = window.document.querySelector(`a[href="${href}"] img`);
      if (imgEl !== null) {
        const srcAttr = imgEl.getAttribute("src");
        if (srcAttr && srcAttr.startsWith("data:")) {
          slideConfig.href = srcAttr;
        }
      }
    } 
  });

  lightboxQuarto.on('slide_after_load', (data) => {
    const { slideIndex, slideNode, slideConfig, player, trigger } = data;
    if (window.Quarto?.typesetMath) {
      window.Quarto.typesetMath(slideNode);
    }
  });

};
          </script>




</body></html>