<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.4.549">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>TB &amp; The Sphinx - 11&nbsp; Discussion</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../">
<link href="../../parts/references.html" rel="next">
<link href="../../parts/discussionInterstitial.html" rel="prev">
<script src="../../site_libs/quarto-html/quarto.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../../site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script src="../../site_libs/quarto-contrib/glightbox/glightbox.min.js"></script>
<link href="../../site_libs/quarto-contrib/glightbox/glightbox.min.css" rel="stylesheet">
<link href="../../site_libs/quarto-contrib/glightbox/lightbox.css" rel="stylesheet">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="../../styles.css">
</head>

<body class="nav-sidebar floating nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="../../index.html">
    <span class="navbar-title">TB and the Sphinx</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="https://drive.google.com/file/d/1Cdb-DDU9dsD4KLxv5WlcxFEM6ViPRuXl/view?usp=share_link" target="_blank"> <i class="bi bi-file-pdf" role="img" aria-label="Download .pdf">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://docs.google.com/document/d/1L0fe7AC-k4hhSJYj_NG2i_aBLd4QHr9T/edit?usp=share_link&amp;ouid=107014449567027741510&amp;rtpof=true&amp;sd=true" target="_blank"> <i class="bi bi-file-word" role="img" aria-label="Download .docx">
</i> 
<span class="menu-text"></span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
          <div class="quarto-navbar-tools">
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../../parts/discussionInterstitial.html">Final thoughts &amp; future directions</a></li><li class="breadcrumb-item"><a href="../../parts/Discussion/discussion.html"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Discussion</span></a></li></ol></nav>
        <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Certificate of Approval</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Acknowledgements.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Acknowledgements</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Glossary.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Glossary</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../parts/introduction.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Introduction: TB and the Sphinx</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/introduction/intro_Layperson.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Introduction Part 1</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/introduction/intro_Scientific.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Introduction Part 2</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../parts/results.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Main Results</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/results/results_Mtb_phagocytosis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Breaking and Entering</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/results/results_Mtb_Membrane_Damage_V2.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Sealing the Gaps</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/results/results_Mtb_lipidomics_v2.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Living Off the Fat of the Cell</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/results/results_trifunctionalProteomics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">An Affinity for Sphingolipids</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../parts/addenda.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Additional work</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Addenda/addenda_SLs_GeneticTools.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Genetic tools to study sphingolipids in disease</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Addenda/addenda_SLs_Fungus.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">The Squeaky Yeast Gets Greased</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Addenda/addenda_MethodsForVisualizingPhagocytosis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Rapid visualization and quantification of phagocytosis in neutrophils</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Addenda/addenda_Celegans_SMS_family.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Systematic analysis of the sphingomyelin synthase family in <em>C. elegans</em></span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../parts/discussionInterstitial.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Final thoughts &amp; future directions</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-4" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Discussion/discussion.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Discussion</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/references.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">References</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/about.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">About</span></a>
  </div>
</li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#are-sphingolipids-a-double-edged-sword-during-mtb-infection" id="toc-are-sphingolipids-a-double-edged-sword-during-mtb-infection" class="nav-link active" data-scroll-target="#are-sphingolipids-a-double-edged-sword-during-mtb-infection"><span class="header-section-number">11.1</span> Are sphingolipids a double-edged sword during <em>Mtb</em> infection?</a></li>
  <li><a href="#a-lipid-map-of-the-tug-of-war-between-mtb-and-the-macrophage" id="toc-a-lipid-map-of-the-tug-of-war-between-mtb-and-the-macrophage" class="nav-link" data-scroll-target="#a-lipid-map-of-the-tug-of-war-between-mtb-and-the-macrophage"><span class="header-section-number">11.2</span> A lipid map of the tug-of-war between <em>Mtb</em> and the macrophage</a></li>
  <li><a href="#interacting-with-the-sphinx-does-mtb-infection-alter-the-sphingosine-interactome" id="toc-interacting-with-the-sphinx-does-mtb-infection-alter-the-sphingosine-interactome" class="nav-link" data-scroll-target="#interacting-with-the-sphinx-does-mtb-infection-alter-the-sphingosine-interactome"><span class="header-section-number">11.3</span> Interacting with the Sphinx: does <em>Mtb</em> infection alter the sphingosine interactome?</a></li>
  <li><a href="#thinking-forward" id="toc-thinking-forward" class="nav-link" data-scroll-target="#thinking-forward"><span class="header-section-number">11.4</span> Thinking forward:</a>
  <ul class="collapse">
  <li><a href="#chapter-1-does-particle-size-affect-the-requirement-for-sphingolipids-during-phagocytosis" id="toc-chapter-1-does-particle-size-affect-the-requirement-for-sphingolipids-during-phagocytosis" class="nav-link" data-scroll-target="#chapter-1-does-particle-size-affect-the-requirement-for-sphingolipids-during-phagocytosis"><span class="header-section-number">11.4.1</span> Chapter 1 – Does particle size affect the requirement for sphingolipids during phagocytosis?</a></li>
  <li><a href="#chapter-2-autophagy-ansmase-what-happens-to-a-chronically-damaged-lysosome" id="toc-chapter-2-autophagy-ansmase-what-happens-to-a-chronically-damaged-lysosome" class="nav-link" data-scroll-target="#chapter-2-autophagy-ansmase-what-happens-to-a-chronically-damaged-lysosome"><span class="header-section-number">11.4.2</span> Chapter 2 – Autophagy &amp; a/nSMase: what happens to a chronically-damaged lysosome?</a></li>
  <li><a href="#chapter-3-dg-tg-lysolipids-plasmalogens-lysoplasmalogens" id="toc-chapter-3-dg-tg-lysolipids-plasmalogens-lysoplasmalogens" class="nav-link" data-scroll-target="#chapter-3-dg-tg-lysolipids-plasmalogens-lysoplasmalogens"><span class="header-section-number">11.4.3</span> Chapter 3 – DG &amp; TG? Lysolipids, Plasmalogens, &amp; Lysoplasmalogens?</a></li>
  <li><a href="#chapter-4-biological-impact-of-interactions-mtb-sphingolipid-interactome" id="toc-chapter-4-biological-impact-of-interactions-mtb-sphingolipid-interactome" class="nav-link" data-scroll-target="#chapter-4-biological-impact-of-interactions-mtb-sphingolipid-interactome"><span class="header-section-number">11.4.4</span> Chapter 4 – Biological impact of interactions? <em>Mtb</em> sphingolipid interactome?</a></li>
  </ul></li>
  <li><a href="#concluding-remarks" id="toc-concluding-remarks" class="nav-link" data-scroll-target="#concluding-remarks"><span class="header-section-number">11.5</span> Concluding remarks</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../../parts/discussionInterstitial.html">Final thoughts &amp; future directions</a></li><li class="breadcrumb-item"><a href="../../parts/Discussion/discussion.html"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Discussion</span></a></li></ol></nav>
<div class="quarto-title">
<h1 class="title"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Discussion</span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<p>I am now tasked with the ambitious project of summarizing the findings of this dissertation work. In our studies on the role of sphingolipids during <em>Mycobacterium tuberculosis</em> infection, we find evidence of a double-edged sword: sphingolipids enable <em>Mtb</em> to enter into host cells via phagocytosis, and yet they are a significant factor in restricting their growth within the infected cell. The total content of sphingolipids such as sphingomyelin and ceramide does not significantly change during pathogenic or attenuated <em>Mtb</em> infections. Still, we observe many intriguing changes among other lipid families. Finally, we find that the protein-sphingolipid interactions of a macrophage are dramatically altered during <em>Mtb</em> infection. Some of these differentially enriched interactions may be spurious results from lysosomal dysfunction and global changes to the proteome, but many are potentially biologically relevant observations.</p>
<section id="are-sphingolipids-a-double-edged-sword-during-mtb-infection" class="level2" data-number="11.1">
<h2 data-number="11.1" class="anchored" data-anchor-id="are-sphingolipids-a-double-edged-sword-during-mtb-infection"><span class="header-section-number">11.1</span> Are sphingolipids a double-edged sword during <em>Mtb</em> infection?</h2>
<p>As mentioned above, the results presented in <a href="../results/results_Mtb_phagocytosis.html" class="quarto-xref"><span>Chapter 3</span></a> show that sphingolipids are <em>bad</em> for us during infection because they are required to initiate <em>Mtb</em> infection. In contrast, our findings in <a href="../results/results_Mtb_Membrane_Damage_V2.html" class="quarto-xref"><span>Chapter 4</span></a> show that sphingolipids are essential for restricting the intracellular growth of <em>Mtb</em>. So which effect is more important? The latter findings seem a more viable route toward antitubercular therapy – though an emerging story may complicate what this therapy may look like. Below, we will envisage several therapeutic options that our reports suggest and weigh the pros and cons of each.</p>
<p>One may imagine an antitubercular prophylactic therapy that blocks sphingolipid production to restrict the uptake of <em>Mtb</em> bacilli in the alveolar space. One must then ask, what happens to these bacilli if they are not internalized? This is not as clear as it may initially seem. Some have suggested that <em>Mtb</em> is an obligate intracellular pathogen<span class="citation" data-cites="agarwalMycobacteriumTuberculosisH37Rv2018"><sup><a href="../references.html#ref-agarwalMycobacteriumTuberculosisH37Rv2018" role="doc-biblioref">1</a></sup></span>. However, this is undoubtedly untrue: more accurately, it is a facultative intracellular pathogen capable of replication without host factors. It is not immediately clear what these non-phagocytosed bacilli may do within the alveolar space, but previous authors have reviewed infection of non-phagocytic cells such as fibroblasts, pneumocytes, and mucosal epithelial cells<span class="citation" data-cites="garcia-perezInternalizationMycobacteriumTuberculosis2003 scordoAlveolarEpithelialCells2016"><sup><a href="../references.html#ref-garcia-perezInternalizationMycobacteriumTuberculosis2003" role="doc-biblioref">2</a>,<a href="../references.html#ref-scordoAlveolarEpithelialCells2016" role="doc-biblioref">3</a></sup></span>. The infection of these cells appears in part dependent on macropinocytosis, a process involving much of the same signaling and cytoskeletal architecture as phagocytosis<span class="citation" data-cites="flannaganCellBiologyPhagocytosis2012 limMacropinocytosisEndocyticPathway2011"><sup><a href="../references.html#ref-flannaganCellBiologyPhagocytosis2012" role="doc-biblioref">4</a>,<a href="../references.html#ref-limMacropinocytosisEndocyticPathway2011" role="doc-biblioref">5</a></sup></span>. It is not yet known whether sphingolipid inhibition would block macropinocytosis in addition to phagocytosis.</p>
<p>However, if one puts aside the question of what may happen if <em>Mtb</em> in the airway is not phagocytosed, other questions arise. What about the other environmental pathogens we encounter daily? What about dead or damaged cells in the alveolar space? What about effete material in the nervous system? One may expect that inhibiting total phagocytosis throughout the body would result in dramatic immune suppression and disease<span class="citation" data-cites="nagreAlveolarMacrophagePhagocytosis2019 podlesny-drabiniokMicroglialPhagocytosisDiseaseAssociated2020"><sup><a href="../references.html#ref-nagreAlveolarMacrophagePhagocytosis2019" role="doc-biblioref">6</a>,<a href="../references.html#ref-podlesny-drabiniokMicroglialPhagocytosisDiseaseAssociated2020" role="doc-biblioref">7</a></sup></span>.</p>
<p>In contrast, utilizing sphingolipids to restrict intracellular <em>Mtb</em> replication is much more appealing. Our results in <a href="../results/results_Mtb_Membrane_Damage_V2.html" class="quarto-xref"><span>Chapter 4</span></a> suggest that more sphingolipids would further limit bacterial growth. Here, too, there are questions about the effects of increasing sphingolipid content in cells. Sphingolipidosis (the harmful accumulation of sphingolipids) is associated with numerous diseases, including many lysosomal storage disorders<span class="citation" data-cites="bradyTherapySphingolipidoses1998 sunLysosomalStorageDisease2018"><sup><a href="../references.html#ref-bradyTherapySphingolipidoses1998" role="doc-biblioref">8</a>,<a href="../references.html#ref-sunLysosomalStorageDisease2018" role="doc-biblioref">9</a></sup></span>. Further, our results have yet to clarify which sphingolipid is primarily responsible for the antimycobacterial restriction we observe – does sphingomyelin play only a minor role in this process, outdone by a select glycosphingolipid species? Much work remains in addressing this question.</p>
<p>There is one final wrench to throw in these perspectives: an emerging body of work suggests that antimicrobial immune activity may drive the formation of persisters during bacterial infection and thereby induce antibiotic-resistant bacterial populations. Stapels et al.&nbsp;report that <em>Salmonella enterica</em> persisters undermine antibiotic therapy and immune response<span class="citation" data-cites="stapelsSalmonellaPersistersUndermine2018"><sup><a href="../references.html#ref-stapelsSalmonellaPersistersUndermine2018" role="doc-biblioref">10</a></sup></span>. Persister bacteria are non-growing, antibiotic-tolerant bacterial cells that form upon sensing antimicrobial effectors – though ongoing work seeks to identify the specific drivers of persister formation in different bacterial families. Candidate signals include acidic pH, diauxic shift, and iron sequestration<span class="citation" data-cites="fisherPersistentBacterialInfections2017"><sup><a href="../references.html#ref-fisherPersistentBacterialInfections2017" role="doc-biblioref">11</a></sup></span>. These authors describe a grey zone of antimicrobial activity, in which many or most bacteria are killed, but the infection is not wholly sterilized. These surviving bacteria, the persisters, have a <em>de facto</em> resistance to antibiotics through their absence of metabolic activity. They are also prime candidates for developing <em>bona fide</em> antibiotic resistance, as they survive sub-lethal doses of antibiotics. These conditions appear highly relevant to <em>Mtb</em> infection: a primary reason underlying the need for 6-9 months of antitubercular antibiotic therapy is the resistance of slow-growing persisters in the infection<span class="citation" data-cites="leeTransientDrugtolerancePermanent2019"><sup><a href="../references.html#ref-leeTransientDrugtolerancePermanent2019" role="doc-biblioref">12</a></sup></span>.</p>
<p>A logical, if frightening, question arises: Can we enhance the efficacy of frontline antibiotics by suppressing immune function? Several authors have already sought to address this question in <em>Staphylococcus aureus</em> infection. Beam et al.&nbsp;showed in 2022 that acute immunosuppression significantly reduced the numbers of antibiotic-tolerant bacteria in BMDM infection<span class="citation" data-cites="beamUseAcuteImmunosuppressive2022"><sup><a href="../references.html#ref-beamUseAcuteImmunosuppressive2022" role="doc-biblioref">13</a></sup></span>. Sphingolipid synthesis inhibitors such as myriocin and fumonisin B1 are used in humans<span class="citation" data-cites="homansTherapeuticEffectsMyriocin2022 bonezziSphingolipidSynthesisInhibition2019 jiIncreasedNovoCeramide2017 makTransplantation2006"><sup><a href="../references.html#ref-homansTherapeuticEffectsMyriocin2022" role="doc-biblioref">14</a>–<a href="../references.html#ref-makTransplantation2006" role="doc-biblioref">17</a></sup></span>, though only rarely because there are many reported side effects, including hepatotoxicity, and immunosuppression<span class="citation" data-cites="makTransplantation2006 salaunMyriocinPreventsMuscle2016"><sup><a href="../references.html#ref-makTransplantation2006" role="doc-biblioref">17</a>,<a href="../references.html#ref-salaunMyriocinPreventsMuscle2016" role="doc-biblioref">18</a></sup></span>. In this context, the side effects of sphingolipid inhibition would be the desired effect.</p>
<p>So, is it preferable to give a TB patient immunosuppressive chemotherapy to shorten the length of their antibiotic therapy? Of course, this regimen would be risky. If the patient is infected with an antibiotic-resistant strain of <em>Mtb</em>, they may deteriorate rapidly and be exceedingly susceptible to other environmental pathogens such as <em>C. albicans</em> (as Tafesse et al.&nbsp;2015 showed that sphingolipid inhibition significantly reduced survival in mice infected with <em>C. albicans</em>). Much more work must be done to answer these questions before such therapies can be implemented in humans, especially given the historical disparities in TB epidemiology and treatment.</p>
</section>
<section id="a-lipid-map-of-the-tug-of-war-between-mtb-and-the-macrophage" class="level2" data-number="11.2">
<h2 data-number="11.2" class="anchored" data-anchor-id="a-lipid-map-of-the-tug-of-war-between-mtb-and-the-macrophage"><span class="header-section-number">11.2</span> A lipid map of the tug-of-war between <em>Mtb</em> and the macrophage</h2>
<p>With <a href="../results/results_Mtb_lipidomics_v2.html" class="quarto-xref"><span>Chapter 5</span></a>, we venture beyond the sphingolipid family to investigate how <em>Mtb</em> infection alters the totality of the host lipidome. In this report, we decided to focus on three distinct phenotypes of infection: the significant induction of diacylglycerol among pathogenic and attenuated <em>Mtb</em> infections and several pathogen-associated molecular patterns, the significant upregulation of triacylglycerol only in the pathogenic infection, and the significantly differential regulation of lysolipids and plasmalogens in pathogenic infection versus attenuated infection.</p>
<p>We find that diacylglycerol production is significantly upregulated in response to both pathogenic and attenuated <em>Mtb</em> infections, as well as in response to treatment with several PAMPs assayed in this study (namely, lipopolysaccharide, lipoarabinomannan, and 1-tuberculosinyladenosine). These findings may be described as a generalized antimicrobial program initiated via the sensing of mycobacteria. However, triacylglycerol production is only significantly stimulated due to pathogenic <em>Mtb</em> infection. This would support previous findings, including the 2023 report by Costa et al.&nbsp;regarding lipid droplet accumulation in H37Rv-infected cells<span class="citation" data-cites="costaMycobacteriumTuberculosisInduces2023"><sup><a href="../references.html#ref-costaMycobacteriumTuberculosisInduces2023" role="doc-biblioref">19</a></sup></span>. Combined with the significant increase in acylcarnitine abundance, these data would suggest that fatty acid β-oxidation is upregulated in these infected cells, also in support of previous reports<span class="citation" data-cites="yangInterceptionHostFatty2021a chandraInhibitionFattyAcid2020a"><sup><a href="../references.html#ref-yangInterceptionHostFatty2021a" role="doc-biblioref">20</a>,<a href="../references.html#ref-chandraInhibitionFattyAcid2020a" role="doc-biblioref">21</a></sup></span>.</p>
<p>We also observe somewhat unique patterns among phospholipids in these results. These include the differential accumulation of lysophospholipids and plasmalogens between pathogenic and attenuated <em>Mtb</em> infections. In 2022, Jurkowitz et al.&nbsp;found that lysoplasmalogens have a direct antimycobacterial effect <em>in vitro</em>, and that a <em>Mtb</em> phospholipase A<sub>2</sub>(MtbYhhN) in H37Rv degrades lysoplasmalogens<span class="citation" data-cites="jurkowitzMycobacteriumTuberculosisEncodes2022"><sup><a href="../references.html#ref-jurkowitzMycobacteriumTuberculosisEncodes2022" role="doc-biblioref">22</a></sup></span>. The expression of this enzyme in <em>M. smegmatis</em> (a non-pathogenic environmental mycobacteria) confers resistance to lysoplasmalogen-induced bacterial death<span class="citation" data-cites="jurkowitzMycobacteriumTuberculosisEncodes2022"><sup><a href="../references.html#ref-jurkowitzMycobacteriumTuberculosisEncodes2022" role="doc-biblioref">22</a></sup></span>. No prior reports have shown a disparity between lysophospholipid accumulation in H37Rv and H37Rv- ΔRD1 infections.</p>
<p>Before moving on, we must briefly return to sphingolipids. In these infection results, we see we see minor but consistent changes in the SL composition of <em>Mtb</em> infected cells - an increase in SM species and a decrease in Cer species, and intriguingly, the opposite for dihydrosphingomyelin and dihydroceramide. The accumulation of these dihydrosphingolipids is intriguing, though subtle. Dihydrosphingolipids are a category of sphingolipid that was long ignored as inactive and biologically irrelevant – a spurious sphingolipid precursor which was somehow missed by the ceramide Δ4-desaturase, DES1<span class="citation" data-cites="magayeRoleDihydrosphingolipidsDisease2019 babiyDihydrosphingolipidsAreAssociated2023"><sup><a href="../references.html#ref-magayeRoleDihydrosphingolipidsDisease2019" role="doc-biblioref">23</a>,<a href="../references.html#ref-babiyDihydrosphingolipidsAreAssociated2023" role="doc-biblioref">24</a></sup></span>. However, the upregulation of these lipids has been associated with several diseases involving dyslipidemia, particularly in diabetes<span class="citation" data-cites="babiyDihydrosphingolipidsAreAssociated2023"><sup><a href="../references.html#ref-babiyDihydrosphingolipidsAreAssociated2023" role="doc-biblioref">24</a></sup></span>. Reports by Turpin et al.&nbsp;and Reali et al.&nbsp;verified that the accumulation of dihydroceramide promotes weight gain and insulin intolerance in mice and found that dihydroceramide is enriched in human adipose tissue in obese humans<span class="citation" data-cites="turpinObesityInducedCerS6DependentC162014 realiMechanisticInterplayCeramide2017"><sup><a href="../references.html#ref-turpinObesityInducedCerS6DependentC162014" role="doc-biblioref">25</a>,<a href="../references.html#ref-realiMechanisticInterplayCeramide2017" role="doc-biblioref">26</a></sup></span>. We omitted these findings in the main report for this chapter because the differential regulations of sphingolipids and dihydrosphingolipids were not statistically significant. However, we see a consistent pattern relating to the pathogenicity of the infecting bacteria: ΔRD1 infection results in depletion of dihydroceramide. This raises interesting questions about pathogenic <em>Mtb</em> infection inducing generalized dyslipidemia: Does this accumulation of dihydrosphingolipids during pathogenic infection coincide with upregulated fatty acid synthesis and lipid droplet formation?</p>
</section>
<section id="interacting-with-the-sphinx-does-mtb-infection-alter-the-sphingosine-interactome" class="level2" data-number="11.3">
<h2 data-number="11.3" class="anchored" data-anchor-id="interacting-with-the-sphinx-does-mtb-infection-alter-the-sphingosine-interactome"><span class="header-section-number">11.3</span> Interacting with the Sphinx: does <em>Mtb</em> infection alter the sphingosine interactome?</h2>
<p>In the report presented in <a href="../results/results_trifunctionalProteomics.html" class="quarto-xref"><span>Chapter 6</span></a>, we seek to uncover the proteins that selectively interact with sphingosine during <em>Mtb</em> infection. To this end, we apply trifunctional sphingolipid analogs and perform affinity enrichment proteomics. An initial foray into using these probes in uninfected cells revealed a narrow cohort of 35 proteins significantly enriched to at least one lipid probe. Intriguingly, twenty proteins were enriched to the sphingolipid precursor sphinganine, and only six proteins were enriched to the signaling sphingolipid sphingosine. Among these enriched proteins, half are reported membrane-associated or peripheral membrane proteins, and many include transmembrane domains. We additionally used two trifunctional fatty acid probes as a control, identifying generalized lipid-binding proteins among our sphingolipid-enriched hits.</p>
<p>Among our identified baseline interactors were several previously reported sphingolipid-interacting proteins, including VDAC1, VDAC2, VDAC3, CTSD, PSAP, and ECH1.</p>
<p>We used these interactomics results of activated THP-1 cells at baseline to inform a follow-up experiment in which cells infected with pathogenic and attenuated <em>Mtb</em> strains were analyzed for differential protein-sphingosine interactions. This analysis produced many differentially enriched proteins – some were upregulated during infection, but the majority were downregulated. By comparing these results to publicly available transcriptomics data, we find that most differentially enriched proteins do not coincide with differential transcriptional regulation – suggesting that these interaction results are the product of altered protein localization, degradation, or function within the infected cell.</p>
<p>Most of the proteins flagged as significantly or partially enriched in the baseline experiment are depleted during infection. Some of these hits may reflect biological function. For example, prior reports show that membrane-bound (inactive) Cathepsin D is activated upon interaction with ceramide in the lysosome and is released from the membrane into the lysosomal lumen<span class="citation" data-cites="dimentCathepsinMembraneassociatedMacrophage1988 heinrichCathepsinTargetedAcid2000 zebrakovskaComplexModulationPeptidolytic2011"><sup><a href="../references.html#ref-dimentCathepsinMembraneassociatedMacrophage1988" role="doc-biblioref">27</a>–<a href="../references.html#ref-zebrakovskaComplexModulationPeptidolytic2011" role="doc-biblioref">29</a></sup></span>. Other interactions have a less clear connection to either sphingosine or <em>Mtb</em> infection.</p>
<p>HMGN1 is one such example. This protein is among the most down-regulated interactors at 24 hours of infection with pathogenic <em>Mtb</em>. While sphingosine is a known inhibitor of several calmodulin-binding proteins, such as HMGB1, no prior reports suggest sphingosine interaction with HMGN1. These proteins are both chromatin remodeling factors activated through their calmodulin binding site<span class="citation" data-cites="kaurCalmodulindependentNuclearImport2010"><sup><a href="../references.html#ref-kaurCalmodulindependentNuclearImport2010" role="doc-biblioref">30</a></sup></span>. No previous reports suggest HMGN1 plays a role in <em>Mtb</em> infection, though the highly related HMGN2 is an antimicrobial peptide that has been shown to regulate mycobacterial survival in host cells<span class="citation" data-cites="ghanaviRoleInterferongammaInterferongamma2021 wangHMGN2RegulatesNon2019"><sup><a href="../references.html#ref-ghanaviRoleInterferongammaInterferongamma2021" role="doc-biblioref">31</a>,<a href="../references.html#ref-wangHMGN2RegulatesNon2019" role="doc-biblioref">32</a></sup></span>.</p>
<p>There are also several intriguing proteins among the few differentially enriched hits between pathogenic H37Rv infection and attenuated H37Rv-ΔRD1 infection. The 20S proteasome subunit PSMA5 and the 20S proteasome subunit activator PSME3 are the third and first most enriched toward the pathogenic infection, respectively. No reports have tied these genes with <em>Mtb</em> infection. Why would proteasome subunits be enriched toward pathogenic infection and away from attenuated infection? Counter to these enrichments, the mitochondrial peptidase LONP1 is enriched toward attenuated infection. Do these findings indicate biologically relevant changes to the immunoproteasome? There were no proteins that were statistically significant in the comparison between H37Rv and H37Rv-ΔRD1, but there may still be interesting patterns in these results.</p>
</section>
<section id="thinking-forward" class="level2" data-number="11.4">
<h2 data-number="11.4" class="anchored" data-anchor-id="thinking-forward"><span class="header-section-number">11.4</span> Thinking forward:</h2>
<p>Where am I leaving this project? What directions will these stories go next? In the sections below, I summarize my thoughts on these four projects and detail the most apparent next steps for these works.</p>
<section id="chapter-1-does-particle-size-affect-the-requirement-for-sphingolipids-during-phagocytosis" class="level3" data-number="11.4.1">
<h3 data-number="11.4.1" class="anchored" data-anchor-id="chapter-1-does-particle-size-affect-the-requirement-for-sphingolipids-during-phagocytosis"><span class="header-section-number">11.4.1</span> Chapter 1 – Does particle size affect the requirement for sphingolipids during phagocytosis?</h3>
<p>In a brief side project, I found that sphingolipid depletion similarly reduced <em>Salmonella enterica</em> Typhimurium uptake by macrophages and dendritic cells (<a href="#fig-Discussion-Fig1" class="quarto-xref">Figure&nbsp;<span>11.1</span></a> A-C). We decided not to pursue this as a project due to its overarching similarity to our previous publications in 2021<span class="citation" data-cites="niekampSphingomyelinBiosynthesisEssential2021"><sup><a href="../references.html#ref-niekampSphingomyelinBiosynthesisEssential2021" role="doc-biblioref">33</a></sup></span> and 2015<span class="citation" data-cites="tafesseDisruptionSphingolipidBiosynthesis2015a"><sup><a href="../references.html#ref-tafesseDisruptionSphingolipidBiosynthesis2015a" role="doc-biblioref">34</a></sup></span>. However, these data suggest an intriguing question that partially undercuts the findings of our prior publications: Do sphingolipids play a generalized role in phagocytosis?</p>
<div class="border">
<div id="fig-Discussion-Fig1" class="quarto-figure quarto-figure-center quarto-float anchored" data-group="gallery_Discussion">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-Discussion-Fig1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/Discussion/Discussion_Figures/300ppi/Discussion_Figure1_Salmonella.png" class="lightbox" data-gallery="gallery_Discussion" data-glightbox="description: .lightbox-desc-1"><img src="../../parts/Discussion/Discussion_Figures/300ppi/Discussion_Figure1_Salmonella.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-Discussion-Fig1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;11.1: <strong>Sphingolipid inhibition reduces uptake of <em>S. enterica</em> along with several other pathogen-particles</strong> (A-C) Unpublished data depicting the phagocytosis scores for <em>Salmonella enterica</em> uptake by RAW 264.7 cells (A), DC2.4 cells (B), and THP-1 cells (C) under the denoted means of sphingolipid depletion. Displayed are all technical replicates of 2 biological replicates (A), 3 biological replicates (B), and 1 biological replicate (C). (D) Published results from Tafesse et al.&nbsp;2015<span class="citation" data-cites="tafesseDisruptionSphingolipidBiosynthesis2015a"><sup><a href="../references.html#ref-tafesseDisruptionSphingolipidBiosynthesis2015a" role="doc-biblioref">34</a></sup></span> and Niekamp et al.&nbsp;2021<span class="citation" data-cites="niekampSphingomyelinBiosynthesisEssential2021"><sup><a href="../references.html#ref-niekampSphingomyelinBiosynthesisEssential2021" role="doc-biblioref">33</a></sup></span> depicting the phagocytosis scores for <em>Candida albicans</em>, <em>Mycobacterium tuberculosis</em>, and Herpes Simplex Virus-1 (HSV) following sphingolipid inhibition. Numbers below the plots are the range of reported sizes for each pathogen-particle.
</figcaption>
</figure>
</div>
</div>
<p>I previously hypothesized that sphingolipids influence phagocytosis efficiency in proportion to the phagocytosed particle. In <a href="#fig-Discussion-Fig1" class="quarto-xref">Figure&nbsp;<span>11.1</span></a> E, I depict our previously published results showing the magnitude of impact that sphingolipid depletion has on phagocytosis score. In the three projects along this vein of research, sphingolipids were absolutely essential for uptake of <em>Candida albicans</em><span class="citation" data-cites="tafesseDisruptionSphingolipidBiosynthesis2015a"><sup><a href="../references.html#ref-tafesseDisruptionSphingolipidBiosynthesis2015a" role="doc-biblioref">34</a></sup></span>, required for <em>efficient</em> uptake of <em>Mtb</em><span class="citation" data-cites="niekampSphingomyelinBiosynthesisEssential2021 viswanathanMacrophageSphingolipidsAre2018"><sup><a href="../references.html#ref-niekampSphingomyelinBiosynthesisEssential2021" role="doc-biblioref">33</a>,<a href="../references.html#ref-viswanathanMacrophageSphingolipidsAre2018" role="doc-biblioref">35</a></sup></span>, and dispensable for receptor-mediated endocytosis<span class="citation" data-cites="niekampSphingomyelinBiosynthesisEssential2021"><sup><a href="../references.html#ref-niekampSphingomyelinBiosynthesisEssential2021" role="doc-biblioref">33</a></sup></span>. The effect size (percent inhibition of phagocytosis rate) is nearly linearly proportional to the size of the particle in question. However, references greatly vary on the estimated sizes of each pathogen particle. Plotting each particle’s (rough) size against the effect size of sphingolipid depletion on phagocytosis scores gives a nearly linear relationship between effect size and particle size (data not shown due to unreliability of reported particle size). Saharan et al.&nbsp;review our data and other similar reports to describe just such a postulate: that sphingolipids are intricately involved in phagocytosis<span class="citation" data-cites="saharanPhagocytosisSphingoLipid2022"><sup><a href="../references.html#ref-saharanPhagocytosisSphingoLipid2022" role="doc-biblioref">36</a></sup></span>.</p>
<p>Given these preliminary and somewhat subjective data, an interesting future project may look into the membrane dynamics of a sphingolipid depletion cell. If sphingolipids at the plasma membrane enable efficient uptake of large particles, there may be implications for pathologies resulting from deficient particle clearance. Silicosis comes to mind, in which phagocytes become “frustrated” and fail to fully internalize a particle – inadvertently releasing lysosomal contents into the extracellular space<span class="citation" data-cites="gilbertiPhagocytosisCrystallineSilica2008 mularskiObservingFrustratedPhagocytosis2018"><sup><a href="../references.html#ref-gilbertiPhagocytosisCrystallineSilica2008" role="doc-biblioref">37</a>,<a href="../references.html#ref-mularskiObservingFrustratedPhagocytosis2018" role="doc-biblioref">38</a></sup></span>. However, this project is neatly wrapped up with its publication in mBio and I find these questions less intriguing than those remaining from Chapter 2, discussed below.</p>
</section>
<section id="chapter-2-autophagy-ansmase-what-happens-to-a-chronically-damaged-lysosome" class="level3" data-number="11.4.2">
<h3 data-number="11.4.2" class="anchored" data-anchor-id="chapter-2-autophagy-ansmase-what-happens-to-a-chronically-damaged-lysosome"><span class="header-section-number">11.4.2</span> Chapter 2 – Autophagy &amp; a/nSMase: what happens to a chronically-damaged lysosome?</h3>
<p>Following chronic, low-level lysosomal damage, the ESCRT membrane repair pathway recruits components of the autophagy pathway to culminate in lysophagy (the targeted autophagy of the lysosome)<span class="citation" data-cites="maejimaAutophagySequestersDamaged2013"><sup><a href="../references.html#ref-maejimaAutophagySequestersDamaged2013" role="doc-biblioref">39</a></sup></span>. However, previous studies have shown that autophagy in <em>Mtb</em>-infected cells is inhibited<span class="citation" data-cites="dereticAutophagyTuberculosis2014a siregarAutophagyresistantMycobacteriumTuberculosis2022"><sup><a href="../references.html#ref-dereticAutophagyTuberculosis2014a" role="doc-biblioref">40</a>,<a href="../references.html#ref-siregarAutophagyresistantMycobacteriumTuberculosis2022" role="doc-biblioref">41</a></sup></span>. Previous authors have also found that autophagy is essential in restricting <em>Mtb</em> infection<span class="citation" data-cites="typasAutophagyCounteractsMycobacterium2023"><sup><a href="../references.html#ref-typasAutophagyCounteractsMycobacterium2023" role="doc-biblioref">42</a></sup></span>. The counterbalance of these effects is intriguing and suggests a complex interplay between host and <em>Mtb</em> factors. The role of sphingolipids in this process is currently undefined, though a ripe avenue for future research. This is a clear direction for future work.</p>
<p>Similarly, <a href="../results/results_Mtb_Membrane_Damage_V2.html" class="quarto-xref"><span>Chapter 4</span></a> does not explicitly address the role of sphingomyelin degradation in our observations. We intentionally excluded data regarding acid and neutral sphingomyelinase inhibition from this report because, in a word, they are confusing. Some of these results are depicted below in <a href="#fig-Discussion-Fig2" class="quarto-xref">Figure&nbsp;<span>11.2</span></a>.</p>
<div class="border">
<div id="fig-Discussion-Fig2" class="quarto-figure quarto-figure-center quarto-float anchored" data-group="gallery_Discussion">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-Discussion-Fig2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/Discussion/Discussion_Figures/300ppi/Discussion_Figure2_SMase.png" class="lightbox" data-gallery="gallery_Discussion" data-glightbox="description: .lightbox-desc-2"><img src="../../parts/Discussion/Discussion_Figures/300ppi/Discussion_Figure2_SMase.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-Discussion-Fig2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;11.2: <strong>Acid and neutral sphingomyelinase inhibitors induce opposing effects on <em>Mtb</em> intracellular growth</strong> (A) Representative growth curves from a single biological replicate (including the same data as presented in <a href="../results/results_Mtb_Membrane_Damage_V2.html" class="quarto-xref"><span>Chapter 4</span></a>, <a href="../results/results_Mtb_Membrane_Damage_V2.html#fig-Mtb-MembraneDamage-V2-Fig4" class="quarto-xref">Figure&nbsp;<span>4.4</span></a> E). Green squares depict technical replicates of cells treated with neutral sphingomyelinase inhibitor GW4869. Hexagons depict cells treated with acid sphingomyelinase inhibitors Amitriptyline (blue) and Desipramine (purple). (B) Areas under the curve were calculated as a measure of bacterial growth. Each point depicts a technical replicate from a single biological replicate. Statistics: Unpaired t-test; *: p &lt; 0.05, **: p &lt; 0.01, ***: p &lt; 0.001, ****: p &lt; 0.0001.
</figcaption>
</figure>
</div>
</div>
<p>The fact that we see a minor increase in pLux growth following neutral sphingomyelinase inhibition via GW4869 treatment (<a href="#fig-Discussion-Fig2" class="quarto-xref">Figure&nbsp;<span>11.2</span></a>, green points) aligns with the larger story involving neutral sphingomyelinase in an ESCRT-independent membrane repair process, as reported by Niekamp et al.&nbsp;(2022)<span class="citation" data-cites="niekampCa2ActivatedSphingomyelin2022a"><sup><a href="../references.html#ref-niekampCa2ActivatedSphingomyelin2022a" role="doc-biblioref">43</a></sup></span>. These authors showed that lysosomal calcium is involved in this incomplete repair mechanism. As will be touched on below, we find a depletion of several calcium-interacting proteins in our sphingosine interaction proteomics data. There is likely an intriguing story to be fleshed out between these results. It must be noted that the underlying mechanism of the observations in Niekamp et al.&nbsp;2022 have yet to be characterized, and there are many remaining open questions regarding how this supposedly-spontaneous repair process occurs.</p>
<p>Additionally, we observe significant restriction of pLux growth following inhibition with the acid sphingomyelinase inhibitors amitriptyline, though not following treatment with the highly related desipramine(<a href="#fig-Discussion-Fig2" class="quarto-xref">Figure&nbsp;<span>11.2</span></a>, light blue and purple points). It must be noted that the mechanisms of action for the nSMase inhibitor and the aSMase inhibitors are distinct. GW4869 is a lipid-competitive inhibitor of nSMase that selectively blocks enzymatic activity (though, interestingly, it does not compete with the substrate SM; it competes with phosphatidylserine)<span class="citation" data-cites="lubertoInhibitionTumorNecrosis2002 airolaStructureHumanNSMase22017"><sup><a href="../references.html#ref-lubertoInhibitionTumorNecrosis2002" role="doc-biblioref">44</a>,<a href="../references.html#ref-airolaStructureHumanNSMase22017" role="doc-biblioref">45</a></sup></span>. In contrast, amitriptyline and desipramine do not directly inhibit aSMase activity but rather induce the dissociation of aSMase from the lysosomal membrane and thereby drive its proteolytic degradation<span class="citation" data-cites="beckmannInhibitionAcidSphingomyelinase2014a"><sup><a href="../references.html#ref-beckmannInhibitionAcidSphingomyelinase2014a" role="doc-biblioref">46</a></sup></span>. Amitriptyline and desipramine belong to a family of psychotropics widely used for their antidepressant effects<span class="citation" data-cites="hoertelAssociationFIASMAPsychotropic2022"><sup><a href="../references.html#ref-hoertelAssociationFIASMAPsychotropic2022" role="doc-biblioref">47</a></sup></span>. These functional inhibitors of acid sphingomyelinase (FIASMAs) confer significant resistance to severe COVID-19 disease<span class="citation" data-cites="hoertelAssociationFIASMAPsychotropic2022"><sup><a href="../references.html#ref-hoertelAssociationFIASMAPsychotropic2022" role="doc-biblioref">47</a></sup></span>, <em>Staphylococcus aureus</em>-induced edema and sepsis<span class="citation" data-cites="pengAcidSphingomyelinaseInhibition2015"><sup><a href="../references.html#ref-pengAcidSphingomyelinaseInhibition2015" role="doc-biblioref">48</a></sup></span>, and <em>Escherichia coli</em>-induced apoptosis of dendritic cells during sepsis<span class="citation" data-cites="falconeActivationAcidSphingomyelinase2004"><sup><a href="../references.html#ref-falconeActivationAcidSphingomyelinase2004" role="doc-biblioref">49</a></sup></span>. The data presented in <a href="#fig-Discussion-Fig2" class="quarto-xref">Figure&nbsp;<span>11.2</span></a> are very preliminary, and we first need to verify the effects of amitriptyline treatment on <em>Mtb</em> replication. We also need to investigate the relative inhibitory capacity of these two related FIASMAs – we need to find the appropriate treatment concentrations of these compounds and adjust accordingly (it is highly probable that the difference apparent in these preliminary results is a simple product of different degrees of inhibitor activity).</p>
<p>Despite these caveats, these data would strongly support future work investigating aSMase as a potential target for antitubercular HDT. To my knowledge, there have been no observational reports investigating a link between FIASMA psychiatric use and TB incidence. A highly intriguing start may be an observational study comparing TB incidence rates of people taking these FIASMA antidepressants versus individuals not taking these drugs. An obvious challenge to this would be the fact that the underlying social determinants which enable access to clinical psychiatric treatment may be associated with inherently reduced TB incidence rates.</p>
</section>
<section id="chapter-3-dg-tg-lysolipids-plasmalogens-lysoplasmalogens" class="level3" data-number="11.4.3">
<h3 data-number="11.4.3" class="anchored" data-anchor-id="chapter-3-dg-tg-lysolipids-plasmalogens-lysoplasmalogens"><span class="header-section-number">11.4.3</span> Chapter 3 – DG &amp; TG? Lysolipids, Plasmalogens, &amp; Lysoplasmalogens?</h3>
<p>This project will need some follow-up before it is ready for publication. Ideally, we will select a vein of interest from the main findings delineated above and validate them directly. Each of the conclusions described may elicit an exciting project.</p>
<p>The role of diacylglycerol production has been observed before, but few reports have sought to characterize how DG signaling affects the intracellular stages of an <em>Mtb</em> infection. A previous project in the lab applied several small molecule inhibitors of DG synthesis to investigate the role of DG during phagocytosis of <em>Mtb</em> in a direct parallel to the report in <a href="../results/results_Mtb_phagocytosis.html" class="quarto-xref"><span>Chapter 3</span></a>. This project was discontinued, though perhaps too soon – we have yet to investigate the effect of these DG inhibitors on the intracellular replication of <em>Mtb</em>. The data reported here would suggest that DG plays a role in the antimicrobial response to mycobacterial infection, and a possible hypothesis would be that these DG inhibitors would release restriction on the bacteria in a similar fashion to sphingolipid inhibition.</p>
<p>Similarly, the role of TG synthesis in pathogenic infection is highly appealing: what factors distinguish the TG effects induced by pathogenic H37Rv strain? Is one of the pathogenicity factors in the RD1 region responsible for triggering lipid droplet accumulation, or is the <em>Mtb</em> Type VII secretion system required to translocate this mysterious lipid droplet-inducer into the cytosol? Will inhibition of TG synthesis restrict bacterial growth?</p>
<p>Finally, as mentioned above, the differential enrichment of lysolipids, plasmalogens, and dihydroceramide are each independently potential candidates for future research. How is the cell upregulating the production of these lipids? How is pathogenic <em>Mtb</em> attenuating the accumulation of these lipids?</p>
<p>Among these exciting observations and potential follow-up, we still await the results of an <em>in situ</em> spatial lipidomics analysis on <em>Mtb</em> infected cells. This analysis brought together the strengths of fluorescence microscopy and MALDI-TOF mass spectrometry and has been used to show profound asymmetries in the lipid composition of tissues<span class="citation" data-cites="kayaSpatialLipidomicsReveals2023"><sup><a href="../references.html#ref-kayaSpatialLipidomicsReveals2023" role="doc-biblioref">50</a></sup></span>. These data would be massively helpful in determining if any upregulated TGs are localized to the <em>Mtb</em> or lipid droplets. Future experiments with lipid droplet staining tools such as Bodipy would coincide beautifully with our findings in the global lipidomics study.</p>
</section>
<section id="chapter-4-biological-impact-of-interactions-mtb-sphingolipid-interactome" class="level3" data-number="11.4.4">
<h3 data-number="11.4.4" class="anchored" data-anchor-id="chapter-4-biological-impact-of-interactions-mtb-sphingolipid-interactome"><span class="header-section-number">11.4.4</span> Chapter 4 – Biological impact of interactions? <em>Mtb</em> sphingolipid interactome?</h3>
<p>This project is far from publication as it stands. While the proteomics experiments yielded exciting results, a large body of follow-up work is needed to validate them and demonstrate their relevance to <em>Mtb</em> infection. In the shortest possible summary, there are three basic needs for these results to be published: validation, annotation, and characterization. There is also a fourth vein of intriguing work on protein-lipid interactions in cultured <em>Mtb</em>.</p>
<section id="validate-the-baseline-interactions-from-the--uv-experiment." class="level4" data-number="11.4.4.1">
<h4 data-number="11.4.4.1" class="anchored" data-anchor-id="validate-the-baseline-interactions-from-the--uv-experiment."><span class="header-section-number">11.4.4.1</span> Validate the baseline interactions from the +/-UV experiment.</h4>
<p>In the last two years, Dr.&nbsp;Scotland Farley and I have sought to optimize a reverse-pulldown protocol to enrich proteins crosslinked to trifunctional lipids by utilizing a bi-valent azide click handle, which comprise both a fluorophore and a cleavable biotin. In this protocol, the cells will be initially treated much as they were in the proteomics experiments presented in <a href="../results/results_trifunctionalProteomics.html" class="quarto-xref"><span>Chapter 6</span></a>, though instead of performing click reactions with azide-coated beads, we will click the lysates to the bi-valent azide handle. We can thus use the same input lysate for two fluorescent Western blots: First, we can pull down using the biotin-conjugated trifunctional lipids to prepare blots much like those depicted in <a href="../results/results_trifunctionalProteomics.html#fig-TrifunctionalProteomics-Fig2" class="quarto-xref">Figure&nbsp;<span>6.2</span></a>, though these may be further stained for protein targets of interest (some of which are listed below). Secondly, the lysate may be used in a protein G pulldown using antibodies against targets of interest, and the fluorescently-labeled trifunctional lipids co-enriched to the target can be visualized on a Western blotting membrane. These efforts are still underway for the 35 significantly-enriched proteins identified in <a href="../results/results_trifunctionalProteomics.html#fig-TrifunctionalProteomics-Fig3" class="quarto-xref">Figure&nbsp;<span>6.3</span></a> and <a href="../results/results_trifunctionalProteomics.html#fig-TrifunctionalProteomics-Fig4" class="quarto-xref">Figure&nbsp;<span>6.4</span></a>. Detection of fluorescence among each pulldown modality will serve as validation of our proteomics data.</p>
<p>Using this methodology, it is highly achievable to perform a reverse pulldown to orthogonally verify that a protein of interest interacts with the trifunctional lipid probe. As a starting point, we will apply this technique to the proteins identified as sphingosine and sphinganine binders in the absence of infection. For sphingosine, these hits include Cathepsin D (CTSD), Prosaposin (PSAP), phosphatidylinositol transfer protein beta (PITPNB), prolylcarboxypeptidase (PRCP), and phosphoribosylformylglycinamidine synthase (PFAS). For sphinganine-interacting proteins, we see all of the sphingosine significant hits and additional hits of interest, including the three voltage dependent anion channels (VDAC 1, 2, and 3), nucleobindins 1 and 2 (NUCB1 and NUCB2), and saccharopine dehydrogenase-like oxidoreductase (SCCPDH). Each of these proteins are involved in intriguing biological pathways – and several are involved in host response to pathogenic viral or bacterial infection – thus, each presents a possible story in which selective lipid interactions enable biological function.</p>
<p>A major challenge to these proposed pulldowns is the overall abundance of each target. In the event that no fluorescence can be detected for some hits, it may be necessary to use ectopic expression of tagged proteins as affinity targets (e.g.&nbsp;FLAG- or HA-tags). A remaining hurdle may be clarifying the specificity of interaction – utilizing treatment with a bolus of non-clickable lipid may serve to validate that a given interaction is selective.</p>
</section>
<section id="determine-how-mtb-infection-changes-the-global-proteome-of-the-cell." class="level4" data-number="11.4.4.2">
<h4 data-number="11.4.4.2" class="anchored" data-anchor-id="determine-how-mtb-infection-changes-the-global-proteome-of-the-cell."><span class="header-section-number">11.4.4.2</span> Determine how <em>Mtb</em> infection changes the global proteome of the cell.</h4>
<p>In order to understand whether our observed differential enrichment of interacting proteins is the product of true changes to the sphingosine interactome or the simple result of altered proteome content of the cell, we must measure the total proteome content of the <em>Mtb</em> infected cell. This would be performed at identical time points and with the same strains as used in the reported study. These results are essential for determining which hits are valid, biologically relevant interactions affected by <em>Mtb</em> infection and which hits are up- or down-regulated at the protein level. We have proposed preparing such global proteomics samples and cross-referencing our findings as I did with the publicly-available transcriptomics data presented in <a href="../results/results_trifunctionalProteomics.html#suppfig-TrifunctionalProteomics-FigS6" class="quarto-xref">Figure S<span>6.6</span></a>. Other groups have performed such an analysis, but their data have yet to be made publicly available. Annotating our interaction data as the product of differential proteomic content is critical in determining which hits are biologically relevant.</p>
<p>With a stronger understanding of the global effects of <em>Mtb</em> infection on the host proteome, we can begin characterizing protein interactions which are dysregulated in a manner that is uncoupled from the total abundance of the protein – we are interested in understanding why the interaction between sphingosine and a given target protein is differentially enriched or depleted during <em>Mtb</em> infection.</p>
</section>
<section id="uncover-the-biological-relevance-of-protein-lipid-interactions-during-mtb-infection." class="level4" data-number="11.4.4.3">
<h4 data-number="11.4.4.3" class="anchored" data-anchor-id="uncover-the-biological-relevance-of-protein-lipid-interactions-during-mtb-infection."><span class="header-section-number">11.4.4.3</span> Uncover the biological relevance of protein-lipid interactions during <em>Mtb</em> infection.</h4>
<p>Finally, we must determine if differentially-enriched interactions during Mtb infection influence the course of infection: up-/down-regulation of bacterial growth. We have proposed selecting 10-20 candidates and screening for their effect on infection via CRISPR knockout and high-throughput growth assays. However, notable challenges include the selection of high-confidence candidates for analysis (our data include broad changes which may simply reflect altered lysosomal function, immunoproteasome activity, and mitochondrial health), the feasibility of CRISPR knockout for these hits (sgRNA selection, validation, maintenance), and the feasibility of high-throughput analysis of intracellular growth (H37Rv-pLux is remarkably noisy, H37Rv-Live/Dead is not incredibly high throughput, colony-forming units are both noisy and incredibly low-throughput). Additionally, we may expect that <em>Mtb</em> pathogenicity factors drive the altered protein:lipid interactions – in which case, comparisons between pathogenic and attenuated <em>Mtb</em> strains would be essential.</p>
<p>The combined understandings of these three objectives will allow us to investigate the biological functions of our differentially-enriched hits. For example, we see a depletion of interaction between sphingosine and the COPI coat complex subunit ε, COPE, at 24 hours post-infection. Because the COPI coatomer protein complex is required for vesicular budding from the Golgi, we may hypothesize that the underlying lipid dynamics of this crucial cellular process are altered during infection. If the total abundance of this protein is unchanged, the question would become: where does COPE traffic during <em>Mtb</em> infection? Is vesicular trafficking globally affected by <em>Mtb</em>? Is this observation dependent on <em>Mtb</em> pathogenicity factors?</p>
</section>
<section id="identify-protein-lipid-interactions-in-purified-mtb-cultures." class="level4" data-number="11.4.4.4">
<h4 data-number="11.4.4.4" class="anchored" data-anchor-id="identify-protein-lipid-interactions-in-purified-mtb-cultures."><span class="header-section-number">11.4.4.4</span> Identify protein-lipid interactions in purified <em>Mtb</em> cultures.</h4>
<p>Though likely not a strict necessity for publishing the results I have prepared in <a href="../results/results_trifunctionalProteomics.html" class="quarto-xref"><span>Chapter 6</span></a>, an additional and highly intriguing possible future direction is to use trifunctional probes to identify lipid-binding proteins within the <em>Mtb</em> proteome. However, identifying direct sphingolipid-binding proteins in the <em>Mtb</em> proteome would be highly impactful. An obvious continuation of this project will be to directly assay for <em>Mtb</em> proteins which selectively bind to tf-Sphingosine or tf-Sphinganine.</p>
<p>I wrote previously that <em>Mtb</em> encodes many lipid-metabolizing enzymes which enable the uptake, modification, and catabolism of many host lipids (<a href="../introduction/intro_Scientific.html" class="quarto-xref"><span>Chapter 2</span></a>). In this regard, it is somewhat disappointing that we did not confidently identify any such proteins in our interactomics analysis (<a href="../results/results_trifunctionalProteomics.html" class="quarto-xref"><span>Chapter 6</span></a>). However, as I discussed in <a href="../results/results_trifunctionalProteomics.html" class="quarto-xref"><span>Chapter 6</span></a>, it is somewhat unsurprising that we did not identify any <em>Mtb</em> proteins with high confidence; among many possible reasons, the lack of <em>Mtb</em> protein identification may have been due to the disparity between total host cellular mass and the expected mycobacterial mass within the prepared samples. Similarly, it may be expected that many of the lipid-binding proteins in both the host cell and the bacterium would be hydrophobic and predisposed to forming insoluble aggregates (even though we identified many membrane-spanning proteins among the host proteome, it is impossible to speculate how many interaction partners were lost in sample preparation due to insolubility).</p>
<p>I have performed some preliminary explorations into optimizing sample preparation and protein collection in cultured <em>Mtb</em> (<a href="#fig-Discussion-Fig3" class="quarto-xref">Figure&nbsp;<span>11.3</span></a>). These data show crosslinked protein enrichment in the tf-8:3-Fatty acid +UV sample. This is not surprising, given that it is widely known that <em>Mtb</em> utilizes fatty acid as an energy source and in its own lipid synthesis. Nonetheless, the identity of these proteins would be of great interest. This gel image does not show any apparent enrichment of tf-Sphingosine-interacting proteins. It must be noted, however, that this protocol is still being optimized, and there may yet be sphingosine-interacting proteins in the <em>Mtb</em> proteome. Future work on this project will certainly include the identification and characterization of proteins which selectively interact with sphingolipids, as these proteins would be of substantial interest as pathogenicity factors; as has been discussed previously in this document (<a href="../introduction/intro_Scientific.html" class="quarto-xref"><span>Chapter 2</span></a>), <em>Mtb</em> does not produce its own sphingolipids, and it may be expected that any protein which selectively interacts with this lipid class may be doing so as part of pathogenic action or as a source of carbon for replication or energy.</p>
<div class="border">
<div id="fig-Discussion-Fig3" class="quarto-figure quarto-figure-center quarto-float anchored" data-group="gallery_Discussion">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-Discussion-Fig3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/Discussion/Discussion_Figures/300ppi/Discussion_Figure3_MtbIGF.png" class="lightbox" data-gallery="gallery_Discussion" data-glightbox="description: .lightbox-desc-3"><img src="../../parts/Discussion/Discussion_Figures/300ppi/Discussion_Figure3_MtbIGF.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-Discussion-Fig3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;11.3: <strong>Preliminary enrichment of lipid-interacting proteins in <em>Mtb</em> culture</strong> (A) A simple schematic of experimental procedure. Briefly, I treated cultured <em>Mtb</em> (strain H37Rv) with trifunctional sphingosine or 8:3-Fatty acid, uncaged, and crosslinked interactions. Control samples were left uncrosslinked. I then lysed the bacteria via chloroform:methanol wash, collected the protein fraction, and clicked crosslinked proteins with a bifunctional azide handle (containing both a fluorophore and a cleavable biotin tag). Using streptavidin beads, I enriched crosslinked proteins and cleaved the biotin tag using hydrazine. Finally, I ran unenriched lysate, bead supernatant, and bead elution on a polyacrylamide gel and visualized fluorescent proteins using a Sapphire fluorescent imager (Azure Bioscience). (B) In-gel fluorescence images of samples described above. Upper panel: raw lysate; middle panel: bead supernatant; lower panel: bead elution.
</figcaption>
</figure>
</div>
</div>
<p>An interesting candidate may be Rv0888, tne <em>Mtb</em> enzyme known to direclty hydrolyze SM and confer the capacity to subsist on SM-enriched minimal media<span class="citation" data-cites="speerSurfaceHydrolysisSphingomyelin2015"><sup><a href="../references.html#ref-speerSurfaceHydrolysisSphingomyelin2015" role="doc-biblioref">51</a></sup></span> – though the specificity for SM elicits several challenges given that trifunctional sphingomyelin has not yet been synthesized in the synthetic lipid analog boom. Nonetheless, the data shown in <a href="#fig-Discussion-Fig3" class="quarto-xref">Figure&nbsp;<span>11.3</span></a> suggest that the described techniques ought to reveal sphingolipid-interacting proteins among the <em>Mtb</em> proteome.</p>
</section>
</section>
</section>
<section id="concluding-remarks" class="level2" data-number="11.5">
<h2 data-number="11.5" class="anchored" data-anchor-id="concluding-remarks"><span class="header-section-number">11.5</span> Concluding remarks</h2>
<p>At long last, I must conclude this document. This is an odd feeling because its completion has less to do with the finality of my thoughts or a loss for words than it has to do with the sand having run out of my hourglass. This document has lived in my mind for the last six years, and there are infinite tweaks and rewrites that I could do. If any readers made it this far, thank you for your attention – I hope some of my thoughts on these topics have been interesting.</p>
<p>The PhD project presented herein sought to uncover and clarify the ways that host sphingolipids (and host lipids in general) influence and are influenced by <em>Mycobacterium tuberculosis</em> infection. We found many interconnecting manners in which <em>Mtb</em> is highly dependent on the nuanced balance of lipid metabolism in host cells – subtle changes to the host lipidome have outsize effects on <em>Mtb</em> entry, replication, and survival. Not everything in this document, by a far margin, has been published – I look forward to seeing where these data take us in the future! There is an irony to be found in practicing science: in answering one question, or even a hundred questions, one does not reduce the number of questions in the universe; instead, one finds that every question answered reveals an infinitude of new questions in a massive fractal spiraling down toward the great glowing coils of the universe.</p>
<p>The projects I’ve etched here have been a significant piece of my adult life. The last six years and six months have been a blur and an eternity. In that time, I have learned quite a lot about <em>Mycobacterium tuberculosis</em>. Despite hundreds of entries into the BSL-3, I have avoided learning what it feels like to have tuberculosis. I have also learned quite a lot about lipids in this time. However, I’ve mainly learned that there is too much to know about lipids for any one person to know everything about them. I have learned a lot about myself over this process. I have made lifelong friends and lived in a place I love. I can’t begin to explain how grateful I feel to all the friends and family I have in my life, as none of this would have been possible without them. In some ways, writing a 300 page dissertation feels like screaming into the void – what use is a document that is traditionally never read once the degree is conferred? If even a single person reads any of this document (aside from my lovely committee, who read it by force), it will be a resounding validation of my effort over these years. If anyone is reading this, I hope you found some interesting something somewhere in this dissertation.</p>


<div class="hidden" aria-hidden="true">
<span class="glightbox-desc lightbox-desc-1">Figure&nbsp;11.1: <strong>Sphingolipid inhibition reduces uptake of <em>S. enterica</em> along with several other pathogen-particles</strong> (A-C) Unpublished data depicting the phagocytosis scores for <em>Salmonella enterica</em> uptake by RAW 264.7 cells (A), DC2.4 cells (B), and THP-1 cells (C) under the denoted means of sphingolipid depletion. Displayed are all technical replicates of 2 biological replicates (A), 3 biological replicates (B), and 1 biological replicate (C). (D) Published results from Tafesse et al.&nbsp;2015<span class="citation" data-cites="tafesseDisruptionSphingolipidBiosynthesis2015a"><sup><a href="../references.html#ref-tafesseDisruptionSphingolipidBiosynthesis2015a" role="doc-biblioref">34</a></sup></span> and Niekamp et al.&nbsp;2021<span class="citation" data-cites="niekampSphingomyelinBiosynthesisEssential2021"><sup><a href="../references.html#ref-niekampSphingomyelinBiosynthesisEssential2021" role="doc-biblioref">33</a></sup></span> depicting the phagocytosis scores for <em>Candida albicans</em>, <em>Mycobacterium tuberculosis</em>, and Herpes Simplex Virus-1 (HSV) following sphingolipid inhibition. Numbers below the plots are the range of reported sizes for each pathogen-particle.</span>
<span class="glightbox-desc lightbox-desc-2">Figure&nbsp;11.2: <strong>Acid and neutral sphingomyelinase inhibitors induce opposing effects on <em>Mtb</em> intracellular growth</strong> (A) Representative growth curves from a single biological replicate (including the same data as presented in <a href="../results/results_Mtb_Membrane_Damage_V2.html" class="quarto-xref"><span>Chapter 4</span></a>, <a href="../results/results_Mtb_Membrane_Damage_V2.html#fig-Mtb-MembraneDamage-V2-Fig4" class="quarto-xref">Figure&nbsp;<span>4.4</span></a> E). Green squares depict technical replicates of cells treated with neutral sphingomyelinase inhibitor GW4869. Hexagons depict cells treated with acid sphingomyelinase inhibitors Amitriptyline (blue) and Desipramine (purple). (B) Areas under the curve were calculated as a measure of bacterial growth. Each point depicts a technical replicate from a single biological replicate. Statistics: Unpaired t-test; *: p &lt; 0.05, **: p &lt; 0.01, ***: p &lt; 0.001, ****: p &lt; 0.0001.</span>
<span class="glightbox-desc lightbox-desc-3">Figure&nbsp;11.3: <strong>Preliminary enrichment of lipid-interacting proteins in <em>Mtb</em> culture</strong> (A) A simple schematic of experimental procedure. Briefly, I treated cultured <em>Mtb</em> (strain H37Rv) with trifunctional sphingosine or 8:3-Fatty acid, uncaged, and crosslinked interactions. Control samples were left uncrosslinked. I then lysed the bacteria via chloroform:methanol wash, collected the protein fraction, and clicked crosslinked proteins with a bifunctional azide handle (containing both a fluorophore and a cleavable biotin tag). Using streptavidin beads, I enriched crosslinked proteins and cleaved the biotin tag using hydrazine. Finally, I ran unenriched lysate, bead supernatant, and bead elution on a polyacrylamide gel and visualized fluorescent proteins using a Sapphire fluorescent imager (Azure Bioscience). (B) In-gel fluorescence images of samples described above. Upper panel: raw lysate; middle panel: bead supernatant; lower panel: bead elution.</span>
</div>
<div id="refs" class="references csl-bib-body" role="list" style="display: none">
<div id="ref-agarwalMycobacteriumTuberculosisH37Rv2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">1. </div><div class="csl-right-inline">Agarwal, S., Ghosh, S., Sharma, S., Kaur, K., and Verma, I. (2018). Mycobacterium tuberculosis <span>H37Rv</span> expresses differential proteome during intracellular survival within alveolar epithelial cells compared with macrophages. Pathogens and Disease <em>76</em>. <a href="https://doi.org/10.1093/femspd/fty058">10.1093/femspd/fty058</a>.</div>
</div>
<div id="ref-garcia-perezInternalizationMycobacteriumTuberculosis2003" class="csl-entry" role="listitem">
<div class="csl-left-margin">2. </div><div class="csl-right-inline">Garcı́a-Pérez, B.E., Mondragón-Flores, R., and Luna-Herrera, J. (2003). Internalization of <span>Mycobacterium</span> tuberculosis by macropinocytosis in non-phagocytic cells. Microbial Pathogenesis <em>35</em>, 49–55. <a href="https://doi.org/10.1016/S0882-4010(03)00089-5">10.1016/S0882-4010(03)00089-5</a>.</div>
</div>
<div id="ref-scordoAlveolarEpithelialCells2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">3. </div><div class="csl-right-inline">Scordo, J.M., Knoell, D.L., and Torrelles, J.B. (2016). Alveolar <span>Epithelial Cells</span> in <span><strong><span><em>Mycobacterium</em></span></strong></span><span><strong><span><em> Tuberculosis</em></span></strong></span> <span>Infection</span>: <span>Active Players</span> or <span>Innocent Bystanders</span>? J Innate Immun <em>8</em>, 3–14. <a href="https://doi.org/10.1159/000439275">10.1159/000439275</a>.</div>
</div>
<div id="ref-flannaganCellBiologyPhagocytosis2012" class="csl-entry" role="listitem">
<div class="csl-left-margin">4. </div><div class="csl-right-inline">Flannagan, R.S., Jaumouillé, V., and Grinstein, S. (2012). The <span>Cell Biology</span> of <span>Phagocytosis</span>. Annual Review of Pathology: Mechanisms of Disease <em>7</em>, 61–98. <a href="https://doi.org/10.1146/annurev-pathol-011811-132445">10.1146/annurev-pathol-011811-132445</a>.</div>
</div>
<div id="ref-limMacropinocytosisEndocyticPathway2011" class="csl-entry" role="listitem">
<div class="csl-left-margin">5. </div><div class="csl-right-inline">Lim, J.P., and Gleeson, P.A. (2011). Macropinocytosis: <span>An</span> endocytic pathway for internalising large gulps. Immunology and Cell Biology <em>89</em>, 836–843. <a href="https://doi.org/10.1038/icb.2011.20">10.1038/icb.2011.20</a>.</div>
</div>
<div id="ref-nagreAlveolarMacrophagePhagocytosis2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">6. </div><div class="csl-right-inline">Nagre, N., Cong, X., Pearson, A.C., and Zhao, X. (2019). Alveolar <span>Macrophage Phagocytosis</span> and <span>Bacteria Clearance</span> in <span>Mice</span>. JoVE, 59088. <a href="https://doi.org/10.3791/59088">10.3791/59088</a>.</div>
</div>
<div id="ref-podlesny-drabiniokMicroglialPhagocytosisDiseaseAssociated2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">7. </div><div class="csl-right-inline">Podleśny-Drabiniok, A., Marcora, E., and Goate, A.M. (2020). Microglial <span>Phagocytosis</span>: <span>A Disease-Associated Process Emerging</span> from <span>Alzheimer</span>’s <span>Disease Genetics</span>. Trends in Neurosciences <em>43</em>, 965–979. <a href="https://doi.org/10.1016/j.tins.2020.10.002">10.1016/j.tins.2020.10.002</a>.</div>
</div>
<div id="ref-bradyTherapySphingolipidoses1998" class="csl-entry" role="listitem">
<div class="csl-left-margin">8. </div><div class="csl-right-inline">Brady, R.O. (1998). Therapy for the <span>Sphingolipidoses</span>. Arch Neurol <em>55</em>, 1055. <a href="https://doi.org/10.1001/archneur.55.8.1055">10.1001/archneur.55.8.1055</a>.</div>
</div>
<div id="ref-sunLysosomalStorageDisease2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">9. </div><div class="csl-right-inline">Sun, A. (2018). Lysosomal storage disease overview. Ann. Transl. Med <em>6</em>, 476.–476. <a href="https://doi.org/10.21037/atm.2018.11.39">10.21037/atm.2018.11.39</a>.</div>
</div>
<div id="ref-stapelsSalmonellaPersistersUndermine2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">10. </div><div class="csl-right-inline">Stapels, D.A.C., Hill, P.W.S., Westermann, A.J., Fisher, R.A., Thurston, T.L., Saliba, A.-E., Blommestein, I., Vogel, J., and Helaine, S. (2018). <span><em>Salmonella</em></span> persisters undermine host immune defenses during antibiotic treatment. Science <em>362</em>, 1156–1160. <a href="https://doi.org/10.1126/science.aat7148">10.1126/science.aat7148</a>.</div>
</div>
<div id="ref-fisherPersistentBacterialInfections2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">11. </div><div class="csl-right-inline">Fisher, R.A., Gollan, B., and Helaine, S. (2017). Persistent bacterial infections and persister cells. Nat Rev Microbiol <em>15</em>, 453–464. <a href="https://doi.org/10.1038/nrmicro.2017.42">10.1038/nrmicro.2017.42</a>.</div>
</div>
<div id="ref-leeTransientDrugtolerancePermanent2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">12. </div><div class="csl-right-inline">Lee, J.J., Lee, S.-K., Song, N., Nathan, T.O., Swarts, B.M., Eum, S.-Y., Ehrt, S., Cho, S.-N., and Eoh, H. (2019). Transient drug-tolerance and permanent drug-resistance rely on the trehalose-catalytic shift in <span>Mycobacterium</span> tuberculosis. Nat Commun <em>10</em>, 2928. <a href="https://doi.org/10.1038/s41467-019-10975-7">10.1038/s41467-019-10975-7</a>.</div>
</div>
<div id="ref-beamUseAcuteImmunosuppressive2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">13. </div><div class="csl-right-inline">Beam, J.E., Maiocchi, S., Cartaya, A., Rowe, S.E., Bahnson, E.S.M., and Conlon, B.P. (2022). The <span>Use</span> of <span>Acute Immunosuppressive Therapy</span> to <span>Improve Antibiotic Efficacy</span> against <span>Intracellular Staphylococcus</span> aureus. Microbiol Spectr <em>10</em>, e00858–22. <a href="https://doi.org/10.1128/spectrum.00858-22">10.1128/spectrum.00858-22</a>.</div>
</div>
<div id="ref-homansTherapeuticEffectsMyriocin2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">14. </div><div class="csl-right-inline">Homans, C., Yalcin, E.B., Tong, M., Gallucci, G., Bautista, D., Moriel, N., and Monte, S.D.L. (2022). Therapeutic <span>Effects</span> of <span>Myriocin</span> in <span>Experimental Alcohol-Related Neurobehavioral Dysfunction</span> and <span>Frontal Lobe White Matter Biochemical Pathology</span>. JBBS <em>12</em>, 23–42. <a href="https://doi.org/10.4236/jbbs.2022.122003">10.4236/jbbs.2022.122003</a>.</div>
</div>
<div id="ref-bonezziSphingolipidSynthesisInhibition2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">15. </div><div class="csl-right-inline">Bonezzi, F., Piccoli, M., Dei Cas, M., Paroni, R., Mingione, A., Monasky, M.M., Caretti, A., Riganti, C., Ghidoni, R., Pappone, C., et al. (2019). Sphingolipid <span>Synthesis Inhibition</span> by <span>Myriocin Administration Enhances Lipid Consumption</span> and <span>Ameliorates Lipid Response</span> to <span>Myocardial Ischemia Reperfusion Injury</span>. Front. Physiol. <em>10</em>, 986. <a href="https://doi.org/10.3389/fphys.2019.00986">10.3389/fphys.2019.00986</a>.</div>
</div>
<div id="ref-jiIncreasedNovoCeramide2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">16. </div><div class="csl-right-inline">Ji, R., Akashi, H., Drosatos, K., Liao, X., Jiang, H., Kennel, P.J., Brunjes, D.L., Castillero, E., Zhang, X., Deng, L.Y., et al. (2017). Increased de novo ceramide synthesis and accumulation in failing myocardium. JCI Insight <em>2</em>, e96203. <a href="https://doi.org/10.1172/jci.insight.96203">10.1172/jci.insight.96203</a>.</div>
</div>
<div id="ref-makTransplantation2006" class="csl-entry" role="listitem">
<div class="csl-left-margin">17. </div><div class="csl-right-inline">Mak, T.W., and Saunders, M.E. (2006). Transplantation. In The <span>Immune Response</span> (Elsevier), pp. 873–921. <a href="https://doi.org/10.1016/B978-012088451-3.50029-6">10.1016/B978-012088451-3.50029-6</a>.</div>
</div>
<div id="ref-salaunMyriocinPreventsMuscle2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">18. </div><div class="csl-right-inline">Salaun, E., Lefeuvre-Orfila, L., Cavey, T., Martin, B., Turlin, B., Ropert, M., Loreal, O., and Derbré, F. (2016). Myriocin prevents muscle ceramide accumulation but not muscle fiber atrophy during short-term mechanical unloading. Journal of Applied Physiology <em>120</em>, 178–187. <a href="https://doi.org/10.1152/japplphysiol.00720.2015">10.1152/japplphysiol.00720.2015</a>.</div>
</div>
<div id="ref-costaMycobacteriumTuberculosisInduces2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">19. </div><div class="csl-right-inline">Costa, M.F.D.S., Pereira‐Dutra, F., Deboosere, N., Jouny, S., Song, O., Iack, G., Souza, A.L., Roma, E.H., Delorme, V., Bozza, P.T., et al. (2023). <span><em>Mycobacterium Tuberculosis</em></span> induces delayed lipid droplet accumulation in dendritic cells depending on bacterial viability and virulence. Molecular Microbiology <em>119</em>, 224–236. <a href="https://doi.org/10.1111/mmi.15023">10.1111/mmi.15023</a>.</div>
</div>
<div id="ref-yangInterceptionHostFatty2021a" class="csl-entry" role="listitem">
<div class="csl-left-margin">20. </div><div class="csl-right-inline">Yang, H., Wang, F., Guo, X., Liu, F., Liu, Z., Wu, X., Zhao, M., Ma, M., Liu, H., Qin, L., et al. (2021). Interception of host fatty acid metabolism by mycobacteria under hypoxia to suppress anti-<span>TB</span> immunity. Cell Discov <em>7</em>, 90. <a href="https://doi.org/10.1038/s41421-021-00301-1">10.1038/s41421-021-00301-1</a>.</div>
</div>
<div id="ref-chandraInhibitionFattyAcid2020a" class="csl-entry" role="listitem">
<div class="csl-left-margin">21. </div><div class="csl-right-inline">Chandra, P., He, L., Zimmerman, M., Yang, G., Köster, S., Ouimet, M., Wang, H., Moore, K.J., Dartois, V., Schilling, J.D., et al. (2020). Inhibition of <span>Fatty Acid Oxidation Promotes Macrophage Control</span> of <span>Mycobacterium</span> tuberculosis. mBio <em>11</em>, e01139–20. <a href="https://doi.org/10.1128/mBio.01139-20">10.1128/mBio.01139-20</a>.</div>
</div>
<div id="ref-jurkowitzMycobacteriumTuberculosisEncodes2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">22. </div><div class="csl-right-inline">Jurkowitz, M.S., Azad, A.K., Monsma, P.C., Keiser, T.L., Kanyo, J., Lam, T.T., Bell, C.E., and Schlesinger, L.S. (2022). Mycobacterium tuberculosis encodes a <span>YhhN</span> family membrane protein with lysoplasmalogenase activity that protects against toxic host lysolipids. Journal of Biological Chemistry <em>298</em>, 101849. <a href="https://doi.org/10.1016/j.jbc.2022.101849">10.1016/j.jbc.2022.101849</a>.</div>
</div>
<div id="ref-magayeRoleDihydrosphingolipidsDisease2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">23. </div><div class="csl-right-inline">Magaye, Ruth.R., Savira, F., Hua, Y., Kelly, D.J., Reid, C., Flynn, B., Liew, D., and Wang, B.H. (2019). The role of dihydrosphingolipids in disease. Cell. Mol. Life Sci. <em>76</em>, 1107–1134. <a href="https://doi.org/10.1007/s00018-018-2984-8">10.1007/s00018-018-2984-8</a>.</div>
</div>
<div id="ref-babiyDihydrosphingolipidsAreAssociated2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">24. </div><div class="csl-right-inline">Babiy, B., Ramos-Molina, B., Ocaña, L., Sacristán, S., Burgos-Santamaría, D., Martínez-Botas, J., Busto, R., Perna, C., Frutos, M.D., Albillos, A., et al. (2023). Dihydrosphingolipids are associated with steatosis and increased fibrosis damage in non-alcoholic fatty liver disease. Biochimica et Biophysica Acta (BBA) - Molecular and Cell Biology of Lipids <em>1868</em>, 159318. <a href="https://doi.org/10.1016/j.bbalip.2023.159318">10.1016/j.bbalip.2023.159318</a>.</div>
</div>
<div id="ref-turpinObesityInducedCerS6DependentC162014" class="csl-entry" role="listitem">
<div class="csl-left-margin">25. </div><div class="csl-right-inline">Turpin, S.M., Nicholls, H.T., Willmes, D.M., Mourier, A., Brodesser, S., Wunderlich, C.M., Mauer, J., Xu, E., Hammerschmidt, P., Brönneke, H.S., et al. (2014). Obesity-<span>Induced CerS6-Dependent C16</span>:0 <span>Ceramide Production Promotes Weight Gain</span> and <span>Glucose Intolerance</span>. Cell Metabolism <em>20</em>, 678–686. <a href="https://doi.org/10.1016/j.cmet.2014.08.002">10.1016/j.cmet.2014.08.002</a>.</div>
</div>
<div id="ref-realiMechanisticInterplayCeramide2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">26. </div><div class="csl-right-inline">Reali, F., Morine, M.J., Kahramanoğulları, O., Raichur, S., Schneider, H.-C., Crowther, D., and Priami, C. (2017). Mechanistic interplay between ceramide and insulin resistance. Sci Rep <em>7</em>, 41231. <a href="https://doi.org/10.1038/srep41231">10.1038/srep41231</a>.</div>
</div>
<div id="ref-dimentCathepsinMembraneassociatedMacrophage1988" class="csl-entry" role="listitem">
<div class="csl-left-margin">27. </div><div class="csl-right-inline">Diment, S., Leech, M.S., and Stahl, P.D. (1988). Cathepsin <span>D</span> is membrane-associated in macrophage endosomes. J Biol Chem <em>263</em>, 6901–6907.</div>
</div>
<div id="ref-heinrichCathepsinTargetedAcid2000" class="csl-entry" role="listitem">
<div class="csl-left-margin">28. </div><div class="csl-right-inline">Heinrich, M., Wickel, M., Schneider-Brachert, W., Sandberg, C., Gahr, J., Schwandner, R., Weber, T., Brunner, J., Krönke, M., and Schütze, S. (2000). Cathepsin <span>D</span> targeted by acid sphingomyelinase-derived ceramide. EMBO J <em>19</em>, 315–315. <a href="https://doi.org/10.1038/sj.emboj.7592140">10.1038/sj.emboj.7592140</a>.</div>
</div>
<div id="ref-zebrakovskaComplexModulationPeptidolytic2011" class="csl-entry" role="listitem">
<div class="csl-left-margin">29. </div><div class="csl-right-inline">Žebrakovská, I., Máša, M., Srp, J., Horn, M., Vávrová, K., and Mareš, M. (2011). Complex modulation of peptidolytic activity of cathepsin <span>D</span> by sphingolipids. Biochimica et Biophysica Acta (BBA) - Molecular and Cell Biology of Lipids <em>1811</em>, 1097–1104. <a href="https://doi.org/10.1016/j.bbalip.2011.09.005">10.1016/j.bbalip.2011.09.005</a>.</div>
</div>
<div id="ref-kaurCalmodulindependentNuclearImport2010" class="csl-entry" role="listitem">
<div class="csl-left-margin">30. </div><div class="csl-right-inline">Kaur, G., Delluc-Clavieres, A., Poon, I.K.H., Forwood, J.K., Glover, D.J., and Jans, D.A. (2010). Calmodulin-dependent nuclear import of <span class="nocase">HMG-box</span> family nuclear factors: Importance of the role of <span>SRY</span> in sex reversal. Biochemical Journal <em>430</em>, 39–48. <a href="https://doi.org/10.1042/BJ20091758">10.1042/BJ20091758</a>.</div>
</div>
<div id="ref-ghanaviRoleInterferongammaInterferongamma2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">31. </div><div class="csl-right-inline">Ghanavi, J., Farnia, P., Farnia, P., and Velayati, A. (2021). The role of interferon-gamma and interferon-gamma receptor in tuberculosis and nontuberculous mycobacterial infections. Int J Mycobacteriol <em>10</em>, 349. <a href="https://doi.org/10.4103/ijmy.ijmy_186_21">10.4103/ijmy.ijmy_186_21</a>.</div>
</div>
<div id="ref-wangHMGN2RegulatesNon2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">32. </div><div class="csl-right-inline">Wang, X., Chen, S., Ren, H., Chen, J., Li, J., Wang, Y., Hua, Y., Wang, X., and Huang, N. (2019). <span>HMGN2</span> regulates non‐tuberculous mycobacteria survival via modulation of <span>M1</span> macrophage polarization. J Cellular Molecular Medi <em>23</em>, 7985–7998. <a href="https://doi.org/10.1111/jcmm.14599">10.1111/jcmm.14599</a>.</div>
</div>
<div id="ref-niekampSphingomyelinBiosynthesisEssential2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">33. </div><div class="csl-right-inline">Niekamp, P., Guzman, G., Leier, H.C., Rashidfarrokhi, A., Richina, V., Pott, F., Barisch, C., Holthuis, J.C.M.M., and Tafesse, F.G. (2021). Sphingomyelin <span>Biosynthesis Is Essential</span> for <span>Phagocytic Signaling</span> during <span>Mycobacterium</span> tuberculosis <span>Host Cell Entry</span>. mBio <em>12</em>, 1–19. <a href="https://doi.org/10.1128/mBio.03141-20">10.1128/mBio.03141-20</a>.</div>
</div>
<div id="ref-tafesseDisruptionSphingolipidBiosynthesis2015a" class="csl-entry" role="listitem">
<div class="csl-left-margin">34. </div><div class="csl-right-inline">Tafesse, F.G., Rashidfarrokhi, A., Schmidt, F.I., Freinkman, E., Dougan, S., Dougan, M., Esteban, A., Maruyama, T., Strijbis, K., and Ploegh, H.L. (2015). Disruption of <span>Sphingolipid Biosynthesis Blocks Phagocytosis</span> of <span>Candida</span> albicans. PLoS pathogens <em>11</em>, e1005188. <a href="https://doi.org/10.1371/journal.ppat.1005188">10.1371/journal.ppat.1005188</a>.</div>
</div>
<div id="ref-viswanathanMacrophageSphingolipidsAre2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">35. </div><div class="csl-right-inline">Viswanathan, G., Jafurulla, M., Kumar, G.A., Raghunand, T.R., and Chattopadhyay, A. (2018). Macrophage sphingolipids are essential for the entry of mycobacteria. Chemistry and Physics of Lipids <em>213</em>, 25–31. <a href="https://doi.org/10.1016/j.chemphyslip.2018.03.004">10.1016/j.chemphyslip.2018.03.004</a>.</div>
</div>
<div id="ref-saharanPhagocytosisSphingoLipid2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">36. </div><div class="csl-right-inline">Saharan, O., Mehendale, N., and Kamat, S.S. (2022). Phagocytosis: <span>A</span> (<span>Sphingo</span>)<span>Lipid Story</span>. Current Research in Chemical Biology <em>2</em>, 100030. <a href="https://doi.org/10.1016/j.crchbi.2022.100030">10.1016/j.crchbi.2022.100030</a>.</div>
</div>
<div id="ref-gilbertiPhagocytosisCrystallineSilica2008" class="csl-entry" role="listitem">
<div class="csl-left-margin">37. </div><div class="csl-right-inline">Gilberti, R.M., Joshi, G.N., and Knecht, D.A. (2008). The <span>Phagocytosis</span> of <span>Crystalline Silica Particles</span> by <span>Macrophages</span>. American Journal of Respiratory Cell and Molecular Biology <em>39</em>, 619–627. <a href="https://doi.org/10.1165/rcmb.2008-0046OC">10.1165/rcmb.2008-0046OC</a>.</div>
</div>
<div id="ref-mularskiObservingFrustratedPhagocytosis2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">38. </div><div class="csl-right-inline">Mularski, A., Marie-Anaïs, F., Mazzolini, J., and Niedergang, F. (2018). Observing <span>Frustrated Phagocytosis</span> and <span>Phagosome Formation</span> and <span>Closure Using Total Internal Reflection Fluorescence Microscopy</span> (<span>TIRFM</span>). In Macrophages, G. Rousselet, ed. (Springer New York), pp. 165–175. <a href="https://doi.org/10.1007/978-1-4939-7837-3_16">10.1007/978-1-4939-7837-3_16</a>.</div>
</div>
<div id="ref-maejimaAutophagySequestersDamaged2013" class="csl-entry" role="listitem">
<div class="csl-left-margin">39. </div><div class="csl-right-inline">Maejima, I., Takahashi, A., Omori, H., Kimura, T., Takabatake, Y., Saitoh, T., Yamamoto, A., Hamasaki, M., Noda, T., Isaka, Y., et al. (2013). Autophagy sequesters damaged lysosomes to control lysosomal biogenesis and kidney injury. EMBO J <em>32</em>, 2336–2347. <a href="https://doi.org/10.1038/emboj.2013.171">10.1038/emboj.2013.171</a>.</div>
</div>
<div id="ref-dereticAutophagyTuberculosis2014a" class="csl-entry" role="listitem">
<div class="csl-left-margin">40. </div><div class="csl-right-inline">Deretic, V. (2014). Autophagy in <span>Tuberculosis</span>. Cold Spring Harbor Perspectives in Medicine <em>4</em>, a018481–a018481. <a href="https://doi.org/10.1101/cshperspect.a018481">10.1101/cshperspect.a018481</a>.</div>
</div>
<div id="ref-siregarAutophagyresistantMycobacteriumTuberculosis2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">41. </div><div class="csl-right-inline">Siregar, T.A.P., Prombutara, P., Kanjanasirirat, P., Kunkaew, N., Tubsuwan, A., Boonmee, A., Palaga, T., Khumpanied, T., Borwornpinyo, S., Chaiprasert, A., et al. (2022). The autophagy-resistant <span><span><em>Mycobacterium</em></span></span><span><span><em> Tuberculosis</em></span></span> <span>Beijing</span> strain upregulates <span>KatG</span> to evade starvation-induced autophagic restriction. Pathogens and Disease <em>80</em>, ftac004. <a href="https://doi.org/10.1093/femspd/ftac004">10.1093/femspd/ftac004</a>.</div>
</div>
<div id="ref-typasAutophagyCounteractsMycobacterium2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">42. </div><div class="csl-right-inline">Typas, D. (2023). Autophagy counteracts <span>Mycobacterium</span> tuberculosis infection at early stages. Nat Struct Mol Biol <em>30</em>, 720–720. <a href="https://doi.org/10.1038/s41594-023-01024-5">10.1038/s41594-023-01024-5</a>.</div>
</div>
<div id="ref-niekampCa2ActivatedSphingomyelin2022a" class="csl-entry" role="listitem">
<div class="csl-left-margin">43. </div><div class="csl-right-inline">Niekamp, P., Scharte, F., Sokoya, T., Vittadello, L., Kim, Y., Deng, Y., Südhoff, E., Hilderink, A., Imlau, M., Clarke, C.J., et al. (2022). Ca2+-activated sphingomyelin scrambling and turnover mediate <span class="nocase">ESCRT-independent</span> lysosomal repair. Nature Communications <em>13</em>. <a href="https://doi.org/10.1038/s41467-022-29481-4">10.1038/s41467-022-29481-4</a>.</div>
</div>
<div id="ref-lubertoInhibitionTumorNecrosis2002" class="csl-entry" role="listitem">
<div class="csl-left-margin">44. </div><div class="csl-right-inline">Luberto, C., Hassler, D.F., Signorelli, P., Okamoto, Y., Sawai, H., Boros, E., Hazen-Martin, D.J., Obeid, L.M., Hannun, Y.A., and Smith, G.K. (2002). Inhibition of <span class="nocase">Tumor Necrosis Factor-induced Cell Death</span> in <span>MCF7</span> by a <span>Novel Inhibitor</span> of <span>Neutral Sphingomyelinase</span>. Journal of Biological Chemistry <em>277</em>, 41128–41139. <a href="https://doi.org/10.1074/jbc.M206747200">10.1074/jbc.M206747200</a>.</div>
</div>
<div id="ref-airolaStructureHumanNSMase22017" class="csl-entry" role="listitem">
<div class="csl-left-margin">45. </div><div class="csl-right-inline">Airola, M.V., Shanbhogue, P., Shamseddine, A.A., Guja, K.E., Senkal, C.E., Maini, R., Bartke, N., Wu, B.X., Obeid, L.M., Garcia-Diaz, M., et al. (2017). Structure of human <span class="nocase">nSMase2</span> reveals an interdomain allosteric activation mechanism for ceramide generation. Proc. Natl. Acad. Sci. U.S.A. <em>114</em>. <a href="https://doi.org/10.1073/pnas.1705134114">10.1073/pnas.1705134114</a>.</div>
</div>
<div id="ref-beckmannInhibitionAcidSphingomyelinase2014a" class="csl-entry" role="listitem">
<div class="csl-left-margin">46. </div><div class="csl-right-inline">Beckmann, N., Sharma, D., Gulbins, E., Becker, K.A., and Edelmann, B. (2014). Inhibition of acid sphingomyelinase by tricyclic antidepressants and analogons. Front. Physiol. <em>5</em>. <a href="https://doi.org/10.3389/fphys.2014.00331">10.3389/fphys.2014.00331</a>.</div>
</div>
<div id="ref-hoertelAssociationFIASMAPsychotropic2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">47. </div><div class="csl-right-inline">Hoertel, N., Sánchez-Rico, M., Gulbins, E., Kornhuber, J., Carpinteiro, A., Abellán, M., De La Muela, P., Vernet, R., Beeker, N., Neuraz, A., et al. (2022). Association between <span>FIASMA</span> psychotropic medications and reduced risk of intubation or death in individuals with psychiatric disorders hospitalized for severe <span>COVID-19</span>: An observational multicenter study. Transl Psychiatry <em>12</em>, 90. <a href="https://doi.org/10.1038/s41398-022-01804-5">10.1038/s41398-022-01804-5</a>.</div>
</div>
<div id="ref-pengAcidSphingomyelinaseInhibition2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">48. </div><div class="csl-right-inline">Peng, H., Li, C., Kadow, S., Henry, B.D., Steinmann, J., Becker, K.A., Riehle, A., Beckmann, N., Wilker, B., Li, P.-L., et al. (2015). Acid sphingomyelinase inhibition protects mice from lung edema and lethal <span>Staphylococcus</span> aureus sepsis. J Mol Med <em>93</em>, 675–689. <a href="https://doi.org/10.1007/s00109-014-1246-y">10.1007/s00109-014-1246-y</a>.</div>
</div>
<div id="ref-falconeActivationAcidSphingomyelinase2004" class="csl-entry" role="listitem">
<div class="csl-left-margin">49. </div><div class="csl-right-inline">Falcone, S., Perrotta, C., De Palma, C., Pisconti, A., Sciorati, C., Capobianco, A., Rovere-Querini, P., Manfredi, A.A., and Clementi, E. (2004). Activation of <span>Acid Sphingomyelinase</span> and <span>Its Inhibition</span> by the <span>Nitric Oxide</span>/<span>Cyclic Guanosine</span> 3′,5′-<span>Monophosphate Pathway</span>: <span>Key Events</span> in <span><span><em>Escherichia</em></span></span><span><span><em> Coli-</em></span></span> <span>Elicited Apoptosis</span> of <span>Dendritic Cells</span>. The Journal of Immunology <em>173</em>, 4452–4463. <a href="https://doi.org/10.4049/jimmunol.173.7.4452">10.4049/jimmunol.173.7.4452</a>.</div>
</div>
<div id="ref-kayaSpatialLipidomicsReveals2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">50. </div><div class="csl-right-inline">Kaya, I., Nilsson, A., Luptáková, D., He, Y., Vallianatou, T., Bjärterot, P., Svenningsson, P., Bezard, E., and Andrén, P.E. (2023). Spatial lipidomics reveals brain region-specific changes of sulfatides in an experimental <span>MPTP Parkinson</span>’s disease primate model. npj Parkinsons Dis. <em>9</em>, 118. <a href="https://doi.org/10.1038/s41531-023-00558-1">10.1038/s41531-023-00558-1</a>.</div>
</div>
<div id="ref-speerSurfaceHydrolysisSphingomyelin2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">51. </div><div class="csl-right-inline">Speer, A., Sun, J., Danilchanka, O., Meikle, V., Rowland, J.L., Walter, K., Buck, B.R., Pavlenok, M., Hölscher, C., Ehrt, S., et al. (2015). Surface hydrolysis of sphingomyelin by the outer membrane protein <span>Rv0888</span> supports replication of <span>M</span> ycobacterium tuberculosis in macrophages. Molecular Microbiology <em>97</em>, 881–897. <a href="https://doi.org/10.1111/mmi.13073">10.1111/mmi.13073</a>.</div>
</div>
</div>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleGiscusIfUsed = (isAlternate, darkModeDefault) => {
    const baseTheme = document.querySelector('#giscus-base-theme')?.value ?? 'light';
    const alternateTheme = document.querySelector('#giscus-alt-theme')?.value ?? 'dark';
    let newTheme = '';
    if(darkModeDefault) {
      newTheme = isAlternate ? baseTheme : alternateTheme;
    } else {
      newTheme = isAlternate ? alternateTheme : baseTheme;
    }
    const changeGiscusTheme = () => {
      // From: https://github.com/giscus/giscus/issues/336
      const sendMessage = (message) => {
        const iframe = document.querySelector('iframe.giscus-frame');
        if (!iframe) return;
        iframe.contentWindow.postMessage({ giscus: message }, 'https://giscus.app');
      }
      sendMessage({
        setConfig: {
          theme: newTheme
        }
      });
    }
    const isGiscussLoaded = window.document.querySelector('iframe.giscus-frame') !== null;
    if (isGiscussLoaded) {
      changeGiscusTheme();
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  const darkModeDefault = false;
  let localAlternateSentinel = darkModeDefault ? 'alternate' : 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
    toggleGiscusIfUsed(toAlternate, darkModeDefault);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } else {
    toggleColorMode(false);
  }
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../../parts/discussionInterstitial.html" class="pagination-link  aria-label=" final="" thoughts="" &amp;="" future="" directions"="">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text">Final thoughts &amp; future directions</span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="../../parts/references.html" class="pagination-link" aria-label="References">
        <span class="nav-page-text">References</span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
<p>Copyright 2024, Gaelen Guzman</p>
</div>   
    <div class="nav-footer-center">
      &nbsp;
    </div>
    <div class="nav-footer-right">
      &nbsp;
    </div>
  </div>
</footer>
<script>var lightboxQuarto = GLightbox({"openEffect":"fade","closeEffect":"fade","loop":true,"descPosition":"bottom","selector":".lightbox"});
window.onload = () => {
  lightboxQuarto.on('slide_before_load', (data) => {
    const { slideIndex, slideNode, slideConfig, player, trigger } = data;
    const href = trigger.getAttribute('href');
    if (href !== null) {
      const imgEl = window.document.querySelector(`a[href="${href}"] img`);
      if (imgEl !== null) {
        const srcAttr = imgEl.getAttribute("src");
        if (srcAttr && srcAttr.startsWith("data:")) {
          slideConfig.href = srcAttr;
        }
      }
    } 
  });

  lightboxQuarto.on('slide_after_load', (data) => {
    const { slideIndex, slideNode, slideConfig, player, trigger } = data;
    if (window.Quarto?.typesetMath) {
      window.Quarto.typesetMath(slideNode);
    }
  });

};
          </script>




</body></html>