<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.4.549">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Gaelen Guzman">
<meta name="author" content="Fikadu G. Tafesse">

<title>TB &amp; The Sphinx - 9&nbsp; Rapid visualization and quantification of phagocytosis in neutrophils</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../">
<link href="../../parts/Addenda/addenda_Celegans_SMS_family.html" rel="next">
<link href="../../parts/Addenda/addenda_SLs_Fungus.html" rel="prev">
<script src="../../site_libs/quarto-html/quarto.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../../site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script src="../../site_libs/quarto-contrib/glightbox/glightbox.min.js"></script>
<link href="../../site_libs/quarto-contrib/glightbox/glightbox.min.css" rel="stylesheet">
<link href="../../site_libs/quarto-contrib/glightbox/lightbox.css" rel="stylesheet">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="../../styles.css">
</head>

<body class="nav-sidebar floating nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="../../index.html">
    <span class="navbar-title">TB and the Sphinx</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="https://drive.google.com/file/d/1Cdb-DDU9dsD4KLxv5WlcxFEM6ViPRuXl/view?usp=share_link" target="_blank"> <i class="bi bi-file-pdf" role="img" aria-label="Download .pdf">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://docs.google.com/document/d/1L0fe7AC-k4hhSJYj_NG2i_aBLd4QHr9T/edit?usp=share_link&amp;ouid=107014449567027741510&amp;rtpof=true&amp;sd=true" target="_blank"> <i class="bi bi-file-word" role="img" aria-label="Download .docx">
</i> 
<span class="menu-text"></span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
          <div class="quarto-navbar-tools">
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../../parts/addenda.html">Additional work</a></li><li class="breadcrumb-item"><a href="../../parts/Addenda/addenda_MethodsForVisualizingPhagocytosis.html"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Rapid visualization and quantification of phagocytosis in neutrophils</span></a></li></ol></nav>
        <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Certificate of Approval</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Acknowledgements.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Acknowledgements</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Glossary.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Glossary</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../parts/introduction.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Introduction: TB and the Sphinx</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/introduction/intro_Layperson.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Introduction Part 1</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/introduction/intro_Scientific.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Introduction Part 2</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../parts/results.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Main Results</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/results/results_Mtb_phagocytosis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Breaking and Entering</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/results/results_Mtb_Membrane_Damage_V2.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Sealing the Gaps</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/results/results_Mtb_lipidomics_v2.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Living Off the Fat of the Cell</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/results/results_trifunctionalProteomics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">An Affinity for Sphingolipids</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../parts/addenda.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Additional work</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Addenda/addenda_SLs_GeneticTools.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Genetic tools to study sphingolipids in disease</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Addenda/addenda_SLs_Fungus.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">The Squeaky Yeast Gets Greased</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Addenda/addenda_MethodsForVisualizingPhagocytosis.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Rapid visualization and quantification of phagocytosis in neutrophils</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Addenda/addenda_Celegans_SMS_family.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Systematic analysis of the sphingomyelin synthase family in <em>C. elegans</em></span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../parts/discussionInterstitial.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Final thoughts &amp; future directions</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-4" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/Discussion/discussion.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Discussion</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/references.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">References</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../parts/about.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">About</span></a>
  </div>
</li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#sec-PhagoMethod" id="toc-sec-PhagoMethod" class="nav-link active" data-scroll-target="#sec-PhagoMethod"><span class="header-section-number">9.1</span> Abstract</a></li>
  <li><a href="#key-words" id="toc-key-words" class="nav-link" data-scroll-target="#key-words"><span class="header-section-number">9.2</span> Key words</a></li>
  <li><a href="#introduction" id="toc-introduction" class="nav-link" data-scroll-target="#introduction"><span class="header-section-number">9.3</span> Introduction</a></li>
  <li><a href="#sec-Visualization-materials" id="toc-sec-Visualization-materials" class="nav-link" data-scroll-target="#sec-Visualization-materials"><span class="header-section-number">9.4</span> Materials</a></li>
  <li><a href="#sec-Visualization-methods" id="toc-sec-Visualization-methods" class="nav-link" data-scroll-target="#sec-Visualization-methods"><span class="header-section-number">9.5</span> Methods</a>
  <ul class="collapse">
  <li><a href="#differentiation-of-hl-60-cells-into-granulocytic-neutrophils" id="toc-differentiation-of-hl-60-cells-into-granulocytic-neutrophils" class="nav-link" data-scroll-target="#differentiation-of-hl-60-cells-into-granulocytic-neutrophils"><span class="header-section-number">9.5.1</span> Differentiation of HL-60 Cells into Granulocytic Neutrophils</a></li>
  <li><a href="#immobilization-of-dhl-60-cells-to-glass-bottom-plate-or-coverslip" id="toc-immobilization-of-dhl-60-cells-to-glass-bottom-plate-or-coverslip" class="nav-link" data-scroll-target="#immobilization-of-dhl-60-cells-to-glass-bottom-plate-or-coverslip"><span class="header-section-number">9.5.2</span> Immobilization of dHL-60 Cells to Glass-Bottom Plate or Coverslip</a></li>
  <li><a href="#phagocytosis-of-phagocytic-particles-by-dhl-60" id="toc-phagocytosis-of-phagocytic-particles-by-dhl-60" class="nav-link" data-scroll-target="#phagocytosis-of-phagocytic-particles-by-dhl-60"><span class="header-section-number">9.5.3</span> Phagocytosis of Phagocytic Particles by dHL-60</a></li>
  <li><a href="#fluorescent-cell-staining" id="toc-fluorescent-cell-staining" class="nav-link" data-scroll-target="#fluorescent-cell-staining"><span class="header-section-number">9.5.4</span> Fluorescent Cell Staining</a></li>
  <li><a href="#image-acquisition" id="toc-image-acquisition" class="nav-link" data-scroll-target="#image-acquisition"><span class="header-section-number">9.5.5</span> Image Acquisition</a></li>
  <li><a href="#image-analysis-quantification" id="toc-image-analysis-quantification" class="nav-link" data-scroll-target="#image-analysis-quantification"><span class="header-section-number">9.5.6</span> Image Analysis (Quantification)</a></li>
  </ul></li>
  <li><a href="#footnotes" id="toc-footnotes" class="nav-link" data-scroll-target="#footnotes"><span class="header-section-number">9.6</span> Footnotes</a>
  <ul class="collapse">
  <li><a href="#sec-FootNote1-HTML" id="toc-sec-FootNote1-HTML" class="nav-link" data-scroll-target="#sec-FootNote1-HTML"><span class="header-section-number">9.6.1</span> Footnote 1</a></li>
  <li><a href="#sec-FootNote2-HTML" id="toc-sec-FootNote2-HTML" class="nav-link" data-scroll-target="#sec-FootNote2-HTML"><span class="header-section-number">9.6.2</span> Footnote 2</a></li>
  <li><a href="#sec-FootNote3-HTML" id="toc-sec-FootNote3-HTML" class="nav-link" data-scroll-target="#sec-FootNote3-HTML"><span class="header-section-number">9.6.3</span> Footnote 3</a></li>
  <li><a href="#sec-FootNote4-HTML" id="toc-sec-FootNote4-HTML" class="nav-link" data-scroll-target="#sec-FootNote4-HTML"><span class="header-section-number">9.6.4</span> Footnote 4</a></li>
  <li><a href="#sec-FootNote5-HTML" id="toc-sec-FootNote5-HTML" class="nav-link" data-scroll-target="#sec-FootNote5-HTML"><span class="header-section-number">9.6.5</span> Footnote 5</a></li>
  <li><a href="#sec-FootNote6-HTML" id="toc-sec-FootNote6-HTML" class="nav-link" data-scroll-target="#sec-FootNote6-HTML"><span class="header-section-number">9.6.6</span> Footnote 6</a></li>
  <li><a href="#sec-FootNote7-HTML" id="toc-sec-FootNote7-HTML" class="nav-link" data-scroll-target="#sec-FootNote7-HTML"><span class="header-section-number">9.6.7</span> Footnote 7</a></li>
  <li><a href="#sec-FootNote8-HTML" id="toc-sec-FootNote8-HTML" class="nav-link" data-scroll-target="#sec-FootNote8-HTML"><span class="header-section-number">9.6.8</span> Footnote 8</a></li>
  <li><a href="#sec-FootNote9-HTML" id="toc-sec-FootNote9-HTML" class="nav-link" data-scroll-target="#sec-FootNote9-HTML"><span class="header-section-number">9.6.9</span> Footnote 9</a></li>
  <li><a href="#sec-FootNote10-HTML" id="toc-sec-FootNote10-HTML" class="nav-link" data-scroll-target="#sec-FootNote10-HTML"><span class="header-section-number">9.6.10</span> Footnote 10</a></li>
  <li><a href="#sec-FootNote11-HTML" id="toc-sec-FootNote11-HTML" class="nav-link" data-scroll-target="#sec-FootNote11-HTML"><span class="header-section-number">9.6.11</span> Footnote 11</a></li>
  </ul></li>
  <li><a href="#contributions" id="toc-contributions" class="nav-link" data-scroll-target="#contributions"><span class="header-section-number">9.7</span> Contributions</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../../parts/addenda.html">Additional work</a></li><li class="breadcrumb-item"><a href="../../parts/Addenda/addenda_MethodsForVisualizingPhagocytosis.html"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Rapid visualization and quantification of phagocytosis in neutrophils</span></a></li></ol></nav>
<div class="quarto-title">
<h1 class="title"><span id="sec-RapidVisualizationofPhagocytosis" class="quarto-section-identifier"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Rapid visualization and quantification of phagocytosis in neutrophils</span></span></h1>
</div>


<div class="quarto-title-meta-author">
  <div class="quarto-title-meta-heading">Authors</div>
  <div class="quarto-title-meta-heading">Affiliation</div>
  
    <div class="quarto-title-meta-contents">
    <p class="author">Gaelen Guzman </p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Department of Molecular Microbiology and Immunology, Oregon Health &amp; Science University, Portland, Oregon, USA
          </p>
      </div>
    <div class="quarto-title-meta-contents">
    <p class="author">Fikadu G. Tafesse <a href="mailto:tafesse@ohsu.edu" class="quarto-title-author-email"><i class="bi bi-envelope"></i></a> </p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Department of Molecular Microbiology and Immunology, Oregon Health &amp; Science University, Portland, Oregon, USA
          </p>
      </div>
  </div>

<div class="quarto-title-meta">

      
  
    
  </div>
  


</header>


<p><em>This methods article was published in Neutrophils: Methods and Protocols and can be found here: DOI: <a href="https://doi.org/10.1007/978-1-0716-0154-9_11" class="uri">https://doi.org/10.1007/978-1-0716-0154-9_11</a></em></p>
<section id="sec-PhagoMethod" class="level2" data-number="9.1">
<h2 data-number="9.1" class="anchored" data-anchor-id="sec-PhagoMethod"><span class="header-section-number">9.1</span> Abstract</h2>
<p>Phagocytosis by phagocytes such as neutrophils is a crucial part of the host innate immune response against invading pathogens. Phagocytosis is a complex process that initiates with the binding of the particles on the cell surface of the phagocytes through the interaction of pattern recognition receptors with ligands on the surface of the pathogens. During this process, phagocytes undergo extensive membrane reorganization and cytoskeleton rearrangement at their cell surface. To gain better insight about the molecular mechanisms of this dynamic cellular process, visualization and quantification in a high-throughput manner is essential. Here, we describe a microscope-based method to visualize and quantify phagocytic uptake of pathogens (such as bacteria and fungi) and model particulates that are larger than 0.5 μm (such as Zymosan A and IgG-coated beads).</p>
</section>
<section id="key-words" class="level2" data-number="9.2">
<h2 data-number="9.2" class="anchored" data-anchor-id="key-words"><span class="header-section-number">9.2</span> Key words</h2>
<p>Neutrophils, phagocytosis, fluorescence microscopy, high-throughput, phenotypic analysis</p>
</section>
<section id="introduction" class="level2" data-number="9.3">
<h2 data-number="9.3" class="anchored" data-anchor-id="introduction"><span class="header-section-number">9.3</span> Introduction</h2>
<p>Neutrophils represent a first line of defense when the body encounters a pathogen. These cells are capable of rapid chemotaxis to sites of bodily damage – they are some of the fastest cells in the mammalian body, and are commonly referred to as “the first responders” upon the initiation of an inflammatory signal<span class="citation" data-cites="aulakhNeutrophilsLungFirst2018 hauertDifferentiatedHL60Cells2002"><sup><a href="../references.html#ref-aulakhNeutrophilsLungFirst2018" role="doc-biblioref">1</a>,<a href="../references.html#ref-hauertDifferentiatedHL60Cells2002" role="doc-biblioref">2</a></sup></span>. Once at an inflamed site, neutrophils are highly versatile in their anti-pathogenic response: they produce cytokines to induce a hostile environment for invading pathogens, they play roles in the activation and direction of the adaptive immune response, they modulate inflammation, and they directly act with potent microbicidal effect<span class="citation" data-cites="aulakhNeutrophilsLungFirst2018 hauertDifferentiatedHL60Cells2002 cassatellaHumanMatureNeutrophils2017a leePhagocytosisNeutrophils2003 koupNeutrophilsAcquireCapacity2017"><sup><a href="../references.html#ref-aulakhNeutrophilsLungFirst2018" role="doc-biblioref">1</a>–<a href="../references.html#ref-koupNeutrophilsAcquireCapacity2017" role="doc-biblioref">5</a></sup></span>. Understanding the mechanisms through which neutrophils respond to pathogenic invasion can give insight into how an infection is resolved – and understanding these mechanisms may allow for the implementation of therapeutic tools to enhance or direct the neutrophil response.</p>
<p>Of particular importance, neutrophils wield phagocytosis to destroy would-be pathogens, as well as effete cells and debris<span class="citation" data-cites="herantMechanicsNeutrophilPhagocytosis2006"><sup><a href="../references.html#ref-herantMechanicsNeutrophilPhagocytosis2006" role="doc-biblioref">6</a></sup></span>. Phagocytosis is a form of receptor-mediated cellular uptake that allows a cell to ingest particles greater than 0.5 μm in size<span class="citation" data-cites="stuartPhagocytosisElegantComplexity2005"><sup><a href="../references.html#ref-stuartPhagocytosisElegantComplexity2005" role="doc-biblioref">7</a></sup></span>. This cellular process has been well studied, but there remain open questions regarding the mechanisms and signaling events that drive the dramatic changes in membrane reorganization, cell morphology, cytoskeleton remodeling, and membrane lipid composition that must ensue for the complete ingestion of a pathogen or particulate. A host of techniques allow for the assessment of phagocytic capacity, but many are limited to gross overviews of internalization (e.g., antibiotic protection assays) or lack the speed necessary for high-throughput phenotypic screens.</p>
<p>Here, we present a methodology for rapidly visualizing and quantifying the phagocytic capacity of neutrophils using high-content imaging microscopy (<a href="#fig-PhagocytosisMethodsFigure1" class="quarto-xref">Figure&nbsp;<span>9.1</span></a>). As the HL-60 neutrophil-like cell line is a classical staple of neutrophil study<span class="citation" data-cites="hauertDifferentiatedHL60Cells2002 fleckUseHL60Cell2005 collinsHL60PromyelocyticLeukemia1987"><sup><a href="../references.html#ref-hauertDifferentiatedHL60Cells2002" role="doc-biblioref">2</a>,<a href="../references.html#ref-fleckUseHL60Cell2005" role="doc-biblioref">8</a>,<a href="../references.html#ref-collinsHL60PromyelocyticLeukemia1987" role="doc-biblioref">9</a></sup></span>, we focus on this line herein—however, these techniques are equally applicable to the study of primary neutrophils isolated from mouse or human peripheral blood . Similarly, any number of fluorophore-conjugated model particle or fluorescent reporter strain of pathogenic microbe may be used under this paradigm<span class="citation" data-cites="tafesseDisruptionSphingolipidBiosynthesis2015a rashidfarrokhiVisualizingEarlyStages2017"><sup><a href="../references.html#ref-tafesseDisruptionSphingolipidBiosynthesis2015a" role="doc-biblioref">10</a>,<a href="../references.html#ref-rashidfarrokhiVisualizingEarlyStages2017" role="doc-biblioref">11</a></sup></span>. Through the use of fluorescent phagocytic particles and a cell membrane stain, one may rapidly acquire visual data sufficient to calculate the rate of phagocytic uptake of many hundreds or thousands of cells across experimental conditions. Combining a well-plate format and automated, high-content image acquisition, this technique allows for high-throughput phenotypic analysis following genetic or chemical screens.</p>
<div class="border">
<div id="fig-PhagocytosisMethodsFigure1" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-PhagocytosisMethodsFigure1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="../../parts/Addenda/MethodsWorkflow2.png" class="lightbox" data-gallery="quarto-lightbox-gallery-1" data-glightbox="description: .lightbox-desc-1"><img src="../../parts/Addenda/MethodsWorkflow2.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-PhagocytosisMethodsFigure1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;9.1: <strong>An overview of the experimental workflow described in this methodology.</strong> (a) Induce polymorphonuclear differentiation of cultured HL-60 cells using 5-day treatment with one of the following: DMSO, ATRA, or PMA . Plate on polylysine-coated imaging surface. (b) Add fluorescent phagocytic particles to cells at a multiplicity of infection of approximately 1:10. (c) Wash unbound particles, fix using 4% paraformaldehyde, and stain using fluorescent cell boundary marker and nuclear stain. (d) Collect high-content images of “infected” cells using fluorescence microscopy . Uptake rate = (Number of InternalizedParticles)/(Number of NucleatedCells).
</figcaption>
</figure>
</div>
</div>
</section>
<section id="sec-Visualization-materials" class="level2" data-number="9.4">
<h2 data-number="9.4" class="anchored" data-anchor-id="sec-Visualization-materials"><span class="header-section-number">9.4</span> Materials</h2>
<ol type="1">
<li><p>HL-60 peripheral blood promyeloblasts (ATCC) (<a href="#sec-FootNote1-HTML" class="quarto-xref">Footnote&nbsp;<span>9.6.1</span></a>).</p></li>
<li><p>Growth medium: RPMI 1640 supplemented with 10% fetal calf serum, nonessential amino acids, and optional antibiotics/antimycotics.</p></li>
<li><p>Phosphate-buffered saline (PBS).</p></li>
<li><p>Differentiation medium: Growth medium supplemented with 1.2% DMSO (v/v)(<a href="#sec-FootNote1-HTML" class="quarto-xref">Footnote&nbsp;<span>9.6.1</span></a>).</p></li>
<li><p>Imaging surface:</p>
<ol type="a">
<li><em>Option 1:</em> Senso Microplate glass-bottom 96-well plate (Greiner Bio-One).</li>
<li><em>Option 2:</em> Microscope coverslips(1.2 cm diameter) and standard 24-well plate.</li>
</ol></li>
<li><p>20× poly-L-lysine coating solution: 50 mg poly-L-lysine in 50 mL diH2O, filter through a 0.22 μm filter. Dilute in H2O for a 1× working stock.</p></li>
<li><p>Model phagocytic particle (<a href="#sec-FootNote2-HTML" class="quarto-xref">Footnote&nbsp;<span>9.6.2</span></a>):</p>
<ol type="a">
<li><em>Option 1:</em> Zymosan A (S. cerevisiae) Bioparticles™ Alexa Fluor™ conjugate (Thermo Fisher Scientific; multiple fluorescence choices).</li>
<li><em>Option 2:</em> Fluoro-Max™ Fluorescent Carboxylate-Modified Particles (Thermo Fisher Scientific).</li>
<li><em>Option 3:</em> Mycobacterium tuberculosis H37Rv (ATCC 25618) expressing mCherry reporter (or another fluorescent reporter microbe).</li>
</ol></li>
<li><p>4% paraformaldehyde solution</p></li>
<li><p>Permeabilization/blocking buffer: 1% bovine serum albumin (w/v) and 0.1% Triton-X100 (v/v) in PBS.</p></li>
<li><p>Cell surface stain (<a href="#sec-FootNote3-HTML" class="quarto-xref">Footnote&nbsp;<span>9.6.3</span></a>): Phalloidin Alexa Fluor™ (e.g., Thermo Fisher Scientific; multiple fluorescence options) in permeabilization/blocking buffer at approximately 1:50 dilution (titration may be necessary).</p></li>
<li><p>Nuclear stain (<a href="#sec-FootNote3-HTML" class="quarto-xref">Footnote&nbsp;<span>9.6.3</span></a>): DAPI (4′,6-diamidino-2-phenylindole) in PBS at approximately 1μg/mL (titration may be necessary).</p></li>
</ol>
</section>
<section id="sec-Visualization-methods" class="level2" data-number="9.5">
<h2 data-number="9.5" class="anchored" data-anchor-id="sec-Visualization-methods"><span class="header-section-number">9.5</span> Methods</h2>
<section id="differentiation-of-hl-60-cells-into-granulocytic-neutrophils" class="level3" data-number="9.5.1">
<h3 data-number="9.5.1" class="anchored" data-anchor-id="differentiation-of-hl-60-cells-into-granulocytic-neutrophils"><span class="header-section-number">9.5.1</span> Differentiation of HL-60 Cells into Granulocytic Neutrophils</h3>
<ol type="1">
<li><p>Grow HL-60 cells in growth media using standard tissue culture techniques. When cultured cells reach a density of approximately 1 × 10<sup>6</sup> cells/mL, replace growth media with an equal volume of differentiation media (<a href="#sec-FootNote1-HTML" class="quarto-xref">Footnote&nbsp;<span>9.6.1</span></a>).</p></li>
<li><p>Incubate cells at 37 °C and 5% CO2 for at least 5 days to obtain fully differentiated HL-60 (dHL-60) cells (<a href="#sec-FootNote4-HTML" class="quarto-xref">Footnote&nbsp;<span>9.6.4</span></a>).</p></li>
</ol>
</section>
<section id="immobilization-of-dhl-60-cells-to-glass-bottom-plate-or-coverslip" class="level3" data-number="9.5.2">
<h3 data-number="9.5.2" class="anchored" data-anchor-id="immobilization-of-dhl-60-cells-to-glass-bottom-plate-or-coverslip"><span class="header-section-number">9.5.2</span> Immobilization of dHL-60 Cells to Glass-Bottom Plate or Coverslip</h3>
<ol type="1">
<li><p>Coat desired imaging surface (96-well plate, coverslips, etc.) according to steps below (<a href="#sec-FootNote5-HTML" class="quarto-xref">Footnote&nbsp;<span>9.6.5</span></a>)<span class="citation" data-cites="PolylysinecoatedTissueCulture2012"><sup><a href="../references.html#ref-PolylysinecoatedTissueCulture2012" role="doc-biblioref">12</a></sup></span>.</p></li>
<li><p>Add 1× polylysine solution to sterile multi-well plates or coverslips and incubate at 37 °C for 1 h.</p></li>
<li><p>Aspirate polylysine solution and leave to dry. Plates and coverslips may be kept at 4 °C.</p></li>
<li><p>From cultured dHL-60 cells, isolate an appropriate number of cells to your preferred imaging surfaces (<a href="#sec-FootNote6-HTML" class="quarto-xref">Footnote&nbsp;<span>9.6.6</span></a>, <a href="#sec-FootNote7-HTML" class="quarto-xref">Footnote&nbsp;<span>9.6.7</span></a>).</p></li>
<li><p>Resuspend cells in the minimum volume to cover surface of the well (approximately 50 μL for 96-well and 250 μL for 24-well plate), and add cells to the appropriate wells.</p></li>
<li><p>Centrifuge cells in plate for 5 min at 250 × g, and allow the cells to recover in differentiation media at 37 °C and 5% CO2 for at least 1 h (or overnight) to maximize immobilization.</p></li>
</ol>
</section>
<section id="phagocytosis-of-phagocytic-particles-by-dhl-60" class="level3" data-number="9.5.3">
<h3 data-number="9.5.3" class="anchored" data-anchor-id="phagocytosis-of-phagocytic-particles-by-dhl-60"><span class="header-section-number">9.5.3</span> Phagocytosis of Phagocytic Particles by dHL-60</h3>
<ol type="1">
<li><p>Isolate an appropriate number of phagocytic particles for a multiplicity of infection (MOI) of approximately ten particles per cell.</p></li>
<li><p>Resuspend in growth medium or PBS at the minimum volume to cover the surface of the well (approximately 50 μL or 250 μL for 96- and 24-well respectively).</p></li>
<li><p>Particle solution may be sonicated for 5–10 min in a water bath or passed through a narrow-gauge syringe needle 10–20 times to reach a single-particle suspension.</p></li>
<li><p>Aspirate growth media from cells affixed to well-plate, wash gently with PBS, and add resuspended particle solution to the cells. Centrifuge plate for 1 min at 250 × g to sediment the particles to the bottom of the well.</p></li>
<li><p>Incubate cells and particles at 37 °C and 5% CO2 for 10 min (<a href="#sec-FootNote8-HTML" class="quarto-xref">Footnote&nbsp;<span>9.6.8</span></a>).</p></li>
<li><p>Gently wash cells twice with PBS to remove unbound particles.</p></li>
<li><p>Fix cells using ice-cold 4% paraformaldehyde solution for 30 min on ice. After fixation, remove formaldehyde solution and hydrate wells with PBS. Plate may be stored at 4 °C for up to a week after fixation, although it is recommended to proceed with staining as early as possible.</p></li>
</ol>
</section>
<section id="fluorescent-cell-staining" class="level3" data-number="9.5.4">
<h3 data-number="9.5.4" class="anchored" data-anchor-id="fluorescent-cell-staining"><span class="header-section-number">9.5.4</span> Fluorescent Cell Staining</h3>
<ol type="1">
<li><p>If using phalloidin (<a href="#sec-FootNote3-HTML" class="quarto-xref">Footnote&nbsp;<span>9.6.3</span></a>): aspirate PBS, and treat cells for 15 min with permeabilization/blocking buffer.</p></li>
<li><p>Remove permeabilization/blocking buffer, add cell surface stain, and incubate for approximately 1 h at room temperature under cover from light (<a href="#sec-FootNote9-HTML" class="quarto-xref">Footnote&nbsp;<span>9.6.9</span></a>).</p></li>
<li><p>Remove cell surface stain solution and wash twice with PBS.</p></li>
<li><p>Stain nuclear DNA using nuclear staining solution for 10 min at room temperature under cover from light (<a href="#sec-FootNote3-HTML" class="quarto-xref">Footnote&nbsp;<span>9.6.3</span></a>).</p></li>
<li><p>Remove nuclear staining solution and wash twice with PBS. Leave cells in PBS under cover from light until imaging is completed.</p></li>
<li><p>If using a 96-well plate, imaging may be performed with no further preparation. If using glass coverslips, follow the steps below to mount the slips to microscopy slides.</p></li>
<li><p>On clean microscopy slide, pipette a single droplet (~15 μL) of anti-fade mounting medium and carefully place coverslip cell-side down on top of this droplet, taking care to minimize air bubbles.</p></li>
<li><p>It is recommended to use a lacquer to glue down the coverslips and prevent drying. Standard clear nail polish is a sufficient affixation agent.</p></li>
<li><p>Allow lacquer to fully dry before imaging .</p></li>
</ol>
</section>
<section id="image-acquisition" class="level3" data-number="9.5.5">
<h3 data-number="9.5.5" class="anchored" data-anchor-id="image-acquisition"><span class="header-section-number">9.5.5</span> Image Acquisition</h3>
<ol type="1">
<li><p>Regardless of acquisition setup (<a href="#sec-FootNote10-HTML" class="quarto-xref">Footnote&nbsp;<span>9.6.10</span></a>), optimize parameters for excitation and exposure times for the respective fluorophores used in the experimental workflow.</p></li>
<li><p>We recommend image collection at 10× or 20× magnification to maximize the total number of cells captured per image. It is best to capture at least 500 cells across the majority of the well or coverslip per technical replicate per experimental condition.</p></li>
<li><p>For optimal quantification, images should be collected in a plane such that the nucleus, cell boundary, and any internalized beads are all clearly distinguishable across the relevant channels with minimal background and inter-channel fluorescence.</p></li>
</ol>
</section>
<section id="image-analysis-quantification" class="level3" data-number="9.5.6">
<h3 data-number="9.5.6" class="anchored" data-anchor-id="image-analysis-quantification"><span class="header-section-number">9.5.6</span> Image Analysis (Quantification)</h3>
<ol type="1">
<li><p>To analyze images, identify the total number of cells captured per image and the number of fluorescent particles fully internalized (<a href="#sec-FootNote11-HTML" class="quarto-xref">Footnote&nbsp;<span>9.6.11</span></a>).</p></li>
<li><p>If using a manual counting method, count the number of nuclei cells and internalized beads per image.</p></li>
<li><p>If using an automated or semiautomated counting method, define threshold parameters for each fluorescent channel, and apply the same parameters for every experimental condition within a biological replicate.</p></li>
<li><p>Calculate rate of phagocytic uptake by aggregating the total number of internalized particles and dividing by the total number of isolated cells for every technical replicate across every experimental condition. Technical replicates may be aggregated to report a single uptake rate for every biological replicate.</p></li>
<li><p>Normalization between biological replicates may be necessary to compensate for inter-experimental variability.</p></li>
<li><p>Uptake rates may be reported as box- or dot-plots and differences between experimental condition may be assessed using a two-tailed, unpaired Student’s T-test.</p></li>
<li><p>Alternatively, one may depict uptake rate using a histogram or violin-plot to represent the number of beads internalized per cell in order to visualize the distribution of uptake events across experimental conditions.</p></li>
<li><p>Differences between experimental conditions may be assessed using a discrete nonparametric test, such as the Chi-squared test or the Kolmogorov–Smirnov Test.</p></li>
</ol>
</section>
</section>
<section id="footnotes" class="level2" data-number="9.6">
<h2 data-number="9.6" class="anchored" data-anchor-id="footnotes"><span class="header-section-number">9.6</span> Footnotes</h2>
<section id="sec-FootNote1-HTML" class="level3" data-number="9.6.1">
<h3 data-number="9.6.1" class="anchored" data-anchor-id="sec-FootNote1-HTML"><span class="header-section-number">9.6.1</span> Footnote 1</h3>
<p>HL-60 cells are derived from peripheral blood leukocytes isolated from a patient with acute promyelocytic leukemia, and can be differentiated into granulocytic neutrophil-like cells using 5-day treatments with DMSO, all trans-retinoic acid (ATRA), or phorbol 12-myristate 13-acetate (PMA)<span class="citation" data-cites="hauertDifferentiatedHL60Cells2002 fleckUseHL60Cell2005 martinHL60CellsInduced1990 milliusManipulationNeutrophilLikeHL602010"><sup><a href="../references.html#ref-hauertDifferentiatedHL60Cells2002" role="doc-biblioref">2</a>,<a href="../references.html#ref-fleckUseHL60Cell2005" role="doc-biblioref">8</a>,<a href="../references.html#ref-martinHL60CellsInduced1990" role="doc-biblioref">13</a>,<a href="../references.html#ref-milliusManipulationNeutrophilLikeHL602010" role="doc-biblioref">14</a></sup></span>. It is advisable to titrate concentrations of activating agent to identify the optimal differentiation conditions in one’s own laboratory. Spontaneous, non-granulocytic differentiation can occur in the absence of activating reagents, and it is recommended to limit passage number to avoid spurious differentiation<span class="citation" data-cites="fleckUseHL60Cell2005"><sup><a href="../references.html#ref-fleckUseHL60Cell2005" role="doc-biblioref">8</a></sup></span>.</p>
</section>
<section id="sec-FootNote2-HTML" class="level3" data-number="9.6.2">
<h3 data-number="9.6.2" class="anchored" data-anchor-id="sec-FootNote2-HTML"><span class="header-section-number">9.6.2</span> Footnote 2</h3>
<p>One may select from a range of model phagocytic particles. This methodology requires only that the selected model particle is fluorophore-conjugated/labeled. Common models include: Zymosan A (which consists of a fungal glycan that acts as ligand to the pathogen recognition receptor Dectin-1), opsonized and non-opsonized silica beads (which act as ligands for FCγR and CR3, respectively), and live or heat-inactivated fluorescent reporter strains of bacteria such as <em>Salmonella enterica</em> Serovar Typhimurium or <em>Mycobacterium tuberculosis</em> (which are both decorated with ligands for a host of pathogen recognition receptors)<span class="citation" data-cites="futosiReprintNeutrophilCell2013 agramonte-heviaGramnegativeBacteriaPhagocytic2002"><sup><a href="../references.html#ref-futosiReprintNeutrophilCell2013" role="doc-biblioref">15</a>,<a href="../references.html#ref-agramonte-heviaGramnegativeBacteriaPhagocytic2002" role="doc-biblioref">16</a></sup></span>.</p>
</section>
<section id="sec-FootNote3-HTML" class="level3" data-number="9.6.3">
<h3 data-number="9.6.3" class="anchored" data-anchor-id="sec-FootNote3-HTML"><span class="header-section-number">9.6.3</span> Footnote 3</h3>
<p>It is necessary to select cellular and nuclear stains with compatibility with the fluorescently labeled model phagocytic particle. We recommend phalloidin, a fungus-derived toxin that effectively stains filamentous actin. Phalloidin may be purchased as a fluorophore-conjugated cell boundary marker in multiple ranges of excitation/emission.</p>
</section>
<section id="sec-FootNote4-HTML" class="level3" data-number="9.6.4">
<h3 data-number="9.6.4" class="anchored" data-anchor-id="sec-FootNote4-HTML"><span class="header-section-number">9.6.4</span> Footnote 4</h3>
<p>Differentiated HL-60 cells will remain in suspension, and previous reports suggest that dHL-60 cells will die through apoptosis beginning on day 8 of treatment<span class="citation" data-cites="martinHL60CellsInduced1990 milliusManipulationNeutrophilLikeHL602010"><sup><a href="../references.html#ref-martinHL60CellsInduced1990" role="doc-biblioref">13</a>,<a href="../references.html#ref-milliusManipulationNeutrophilLikeHL602010" role="doc-biblioref">14</a></sup></span>. If performing genetic or chemical screens, it is recommended that transfection/transduction or treatment begin during the 5-day differentiation period such that cells are prepared for phagocytosis analysis by day 5.</p>
</section>
<section id="sec-FootNote5-HTML" class="level3" data-number="9.6.5">
<h3 data-number="9.6.5" class="anchored" data-anchor-id="sec-FootNote5-HTML"><span class="header-section-number">9.6.5</span> Footnote 5</h3>
<p>Poly-L-lysine coating may be used to affix dHL-60 cells to imaging glassware. Such coated products are available for purchase (<a href="#sec-Visualization-materials" class="quarto-xref">See Materials&nbsp;<span>9.4</span></a>), or may be prepared in-house as described previously<span class="citation" data-cites="PolylysinecoatedTissueCulture2012"><sup><a href="../references.html#ref-PolylysinecoatedTissueCulture2012" role="doc-biblioref">12</a></sup></span>.</p>
</section>
<section id="sec-FootNote6-HTML" class="level3" data-number="9.6.6">
<h3 data-number="9.6.6" class="anchored" data-anchor-id="sec-FootNote6-HTML"><span class="header-section-number">9.6.6</span> Footnote 6</h3>
<p>We recommend seeding approximately 2 × 10<sup>4</sup> cells when using a 96-well glass-bottom imaging plate, and approximately 5 × 10<sup>4</sup> cells when using 24 mm diameter coverslips in a standard 24-well plate.</p>
</section>
<section id="sec-FootNote7-HTML" class="level3" data-number="9.6.7">
<h3 data-number="9.6.7" class="anchored" data-anchor-id="sec-FootNote7-HTML"><span class="header-section-number">9.6.7</span> Footnote 7</h3>
<p>It is highly recommended to perform this uptake assay with multiple technical replicates, with differentiation and uptake assays performed on independent days for each biological replicate. Experimental conditions should be assayed in parallel for every biological replicate, as significant variation may occur between replicates.</p>
</section>
<section id="sec-FootNote8-HTML" class="level3" data-number="9.6.8">
<h3 data-number="9.6.8" class="anchored" data-anchor-id="sec-FootNote8-HTML"><span class="header-section-number">9.6.8</span> Footnote 8</h3>
<p>Exact uptake time may need to be optimized: 10 min may be too long if cells are highly activated, and one is characterizing a weak phenotype, or too short if the model phagocytic particle is taken up poorly. It is recommended to optimize this timing using a three- or four-point time course on wild-type cells.</p>
</section>
<section id="sec-FootNote9-HTML" class="level3" data-number="9.6.9">
<h3 data-number="9.6.9" class="anchored" data-anchor-id="sec-FootNote9-HTML"><span class="header-section-number">9.6.9</span> Footnote 9</h3>
<p>As with many staining protocols, it may be necessary to modulate both the concentration of staining reagent and the time to maximize signal and minimize background.</p>
</section>
<section id="sec-FootNote10-HTML" class="level3" data-number="9.6.10">
<h3 data-number="9.6.10" class="anchored" data-anchor-id="sec-FootNote10-HTML"><span class="header-section-number">9.6.10</span> Footnote 10</h3>
<p>Images for phagocytic analysis may be collected on a range of acquisition platforms—so long as the microscope has the excitation/detection capacities to capture the fluorophores used, it ought to be sufficient for this methodology. We recommend the KEYENCE BZ-X700 all-in-one fluorescence microscope (KEYENCE) due to its capacity for semiautomated image acquisition and multi-field image stitching.</p>
</section>
<section id="sec-FootNote11-HTML" class="level3" data-number="9.6.11">
<h3 data-number="9.6.11" class="anchored" data-anchor-id="sec-FootNote11-HTML"><span class="header-section-number">9.6.11</span> Footnote 11</h3>
<p>A number of image analysis software tools with compatible cell counting protocols are available, including CellProfiler (open access), FIJI (open access), Imaris (Oxford Instruments), and KEYENCE BZ-X Analyzer (KEYENCE). It is notable that a semiautomated cell counting tool is preferable to manual counting for enhanced throughput and reduced user bias.</p>
</section>
</section>
<section id="contributions" class="level2" data-number="9.7">
<h2 data-number="9.7" class="anchored" data-anchor-id="contributions"><span class="header-section-number">9.7</span> Contributions</h2>
<p>GG compiled the concepts and references reviewed and wrote this methods chapter. FGT provided text edits and oversaw the design of this methods chapter.</p>


<div class="hidden" aria-hidden="true">
<span class="glightbox-desc lightbox-desc-1">Figure&nbsp;9.1: <strong>An overview of the experimental workflow described in this methodology.</strong> (a) Induce polymorphonuclear differentiation of cultured HL-60 cells using 5-day treatment with one of the following: DMSO, ATRA, or PMA . Plate on polylysine-coated imaging surface. (b) Add fluorescent phagocytic particles to cells at a multiplicity of infection of approximately 1:10. (c) Wash unbound particles, fix using 4% paraformaldehyde, and stain using fluorescent cell boundary marker and nuclear stain. (d) Collect high-content images of “infected” cells using fluorescence microscopy . Uptake rate = (Number of InternalizedParticles)/(Number of NucleatedCells).</span>
</div>
<div id="refs" class="references csl-bib-body" role="list" style="display: none">
<div id="ref-aulakhNeutrophilsLungFirst2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">1. </div><div class="csl-right-inline">Aulakh, G.K. (2018). Neutrophils in the lung: <span>“The first responders.”</span> Cell and Tissue Research <em>371</em>, 577–588. <a href="https://doi.org/10.1007/s00441-017-2748-z">10.1007/s00441-017-2748-z</a>.</div>
</div>
<div id="ref-hauertDifferentiatedHL60Cells2002" class="csl-entry" role="listitem">
<div class="csl-left-margin">2. </div><div class="csl-right-inline">Hauert, A.B., Martinelli, S., Marone, C., and Niggli, V. (2002). Differentiated <span>HL-60</span> cells are a valid model system for the analysis of human neutrophil migration and chemotaxis. The International Journal of Biochemistry &amp; Cell Biology <em>34</em>, 838–854. <a href="https://doi.org/10.1016/S1357-2725(02)00010-9">10.1016/S1357-2725(02)00010-9</a>.</div>
</div>
<div id="ref-cassatellaHumanMatureNeutrophils2017a" class="csl-entry" role="listitem">
<div class="csl-left-margin">3. </div><div class="csl-right-inline">Cassatella, M.A. (2017). Human mature neutrophils as atypical <span>APC</span>. Blood <em>129</em>, 1895–1896. <a href="https://doi.org/10.1182/blood-2017-02-767574">10.1182/blood-2017-02-767574</a>.</div>
</div>
<div id="ref-leePhagocytosisNeutrophils2003" class="csl-entry" role="listitem">
<div class="csl-left-margin">4. </div><div class="csl-right-inline">Lee, W.L., Harrison, R.E., and Grinstein, S. (2003). Phagocytosis by neutrophils. Microbes and Infection <em>5</em>, 1299–1306. <a href="https://doi.org/10.1016/j.micinf.2003.09.014">10.1016/j.micinf.2003.09.014</a>.</div>
</div>
<div id="ref-koupNeutrophilsAcquireCapacity2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">5. </div><div class="csl-right-inline">Koup, R.A., Liang, F., Loré, K., Vono, M., Lin, A., and Norrby-Teglund, A. (2017). Neutrophils acquire the capacity for antigen presentation to memory <span>CD4</span> + <span>T</span> cells in vitro and ex vivo. Blood <em>129</em>, 1991–2001. <a href="https://doi.org/10.1182/blood-2016-10-744441">10.1182/blood-2016-10-744441</a>.</div>
</div>
<div id="ref-herantMechanicsNeutrophilPhagocytosis2006" class="csl-entry" role="listitem">
<div class="csl-left-margin">6. </div><div class="csl-right-inline">Herant, M. (2006). Mechanics of neutrophil phagocytosis: Experiments and quantitative models. Journal of Cell Science <em>119</em>, 1903–1913. <a href="https://doi.org/10.1242/jcs.02876">10.1242/jcs.02876</a>.</div>
</div>
<div id="ref-stuartPhagocytosisElegantComplexity2005" class="csl-entry" role="listitem">
<div class="csl-left-margin">7. </div><div class="csl-right-inline">Stuart, L.M., and Ezekowitz, R.A.B. (2005). Phagocytosis: Elegant complexity. Immunity <em>22</em>, 539–550. <a href="https://doi.org/10.1016/j.immuni.2005.05.002">10.1016/j.immuni.2005.05.002</a>.</div>
</div>
<div id="ref-fleckUseHL60Cell2005" class="csl-entry" role="listitem">
<div class="csl-left-margin">8. </div><div class="csl-right-inline">Fleck, R.A., Romero-Steiner, S., and Nahm, M.H. (2005). Use of <span>HL-60 Cell Line To Measure Opsonic Capacity</span> of <span>Pneumococcal Antibodies</span>. Clinical and Vaccine Immunology <em>12</em>, 19–27. <a href="https://doi.org/10.1128/CDLI.12.1.19-27.2005">10.1128/CDLI.12.1.19-27.2005</a>.</div>
</div>
<div id="ref-collinsHL60PromyelocyticLeukemia1987" class="csl-entry" role="listitem">
<div class="csl-left-margin">9. </div><div class="csl-right-inline">Collins, S.J. (1987). <a href="http://www.bloodjournal.org/content/70/5/1233.abstract">The <span>HL-60</span> promyelocytic leukemia cell line: Proliferation, differentiation, and cellular oncogene expression.</a> Blood <em>70</em>, 1233–1244.</div>
</div>
<div id="ref-tafesseDisruptionSphingolipidBiosynthesis2015a" class="csl-entry" role="listitem">
<div class="csl-left-margin">10. </div><div class="csl-right-inline">Tafesse, F.G., Rashidfarrokhi, A., Schmidt, F.I., Freinkman, E., Dougan, S., Dougan, M., Esteban, A., Maruyama, T., Strijbis, K., and Ploegh, H.L. (2015). Disruption of <span>Sphingolipid Biosynthesis Blocks Phagocytosis</span> of <span>Candida</span> albicans. PLoS pathogens <em>11</em>, e1005188. <a href="https://doi.org/10.1371/journal.ppat.1005188">10.1371/journal.ppat.1005188</a>.</div>
</div>
<div id="ref-rashidfarrokhiVisualizingEarlyStages2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">11. </div><div class="csl-right-inline">Rashidfarrokhi, A., Richina, V., and Tafesse, F.G. (2017). Visualizing the <span>Early Stages</span> of <span>Phagocytosis</span>. Journal of Visualized Experiments. <a href="https://doi.org/10.3791/54646">10.3791/54646</a>.</div>
</div>
<div id="ref-PolylysinecoatedTissueCulture2012" class="csl-entry" role="listitem">
<div class="csl-left-margin">12. </div><div class="csl-right-inline">Polylysine-coated tissue culture surfaces (2012). <a href="https://www.protocolsonline.com/recipes/stock-solutions/polylysine-coated-tissue-culture-surfaces/">https://www.protocolsonline.com/recipes/stock-solutions/polylysine-coated-tissue-culture-surfaces/</a>.</div>
</div>
<div id="ref-martinHL60CellsInduced1990" class="csl-entry" role="listitem">
<div class="csl-left-margin">13. </div><div class="csl-right-inline">Martin, S.J., Bradley, J.G., and Cotter, T.G. (1990). <span>HL-60</span> cells induced to differentiate towards neutrophils subsequently die via apoptosis. Clinical and experimental immunology <em>79</em>, 448–453. <a href="https://doi.org/10.1111/j.1365-2249.1990.tb08110.x">10.1111/j.1365-2249.1990.tb08110.x</a>.</div>
</div>
<div id="ref-milliusManipulationNeutrophilLikeHL602010" class="csl-entry" role="listitem">
<div class="csl-left-margin">14. </div><div class="csl-right-inline">Millius, A., and Weiner, O.D. (2010). Manipulation of <span>Neutrophil-Like HL-60 Cells</span> for the <span>Study</span> of <span>Directed Cell Migration</span>. In Histochemical <span>Journal</span>, pp. 147–158. <a href="https://doi.org/10.1007/978-1-60761-404-3_9">10.1007/978-1-60761-404-3_9</a>.</div>
</div>
<div id="ref-futosiReprintNeutrophilCell2013" class="csl-entry" role="listitem">
<div class="csl-left-margin">15. </div><div class="csl-right-inline">Futosi, K., Fodor, S., and Mócsai, A. (2013). Reprint of <span>Neutrophil</span> cell surface receptors and their intracellular signal transduction pathways. International Immunopharmacology <em>17</em>, 1185–1197. <a href="https://doi.org/10.1016/j.intimp.2013.11.010">10.1016/j.intimp.2013.11.010</a>.</div>
</div>
<div id="ref-agramonte-heviaGramnegativeBacteriaPhagocytic2002" class="csl-entry" role="listitem">
<div class="csl-left-margin">16. </div><div class="csl-right-inline">Agramonte-Hevia, J. (2002). Gram-negative bacteria and phagocytic cell interaction mediated by complement receptor 3. FEMS Immunology and Medical Microbiology <em>34</em>, 255–266. <a href="https://doi.org/10.1016/S0928-8244(02)00408-X">10.1016/S0928-8244(02)00408-X</a>.</div>
</div>
</div>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleGiscusIfUsed = (isAlternate, darkModeDefault) => {
    const baseTheme = document.querySelector('#giscus-base-theme')?.value ?? 'light';
    const alternateTheme = document.querySelector('#giscus-alt-theme')?.value ?? 'dark';
    let newTheme = '';
    if(darkModeDefault) {
      newTheme = isAlternate ? baseTheme : alternateTheme;
    } else {
      newTheme = isAlternate ? alternateTheme : baseTheme;
    }
    const changeGiscusTheme = () => {
      // From: https://github.com/giscus/giscus/issues/336
      const sendMessage = (message) => {
        const iframe = document.querySelector('iframe.giscus-frame');
        if (!iframe) return;
        iframe.contentWindow.postMessage({ giscus: message }, 'https://giscus.app');
      }
      sendMessage({
        setConfig: {
          theme: newTheme
        }
      });
    }
    const isGiscussLoaded = window.document.querySelector('iframe.giscus-frame') !== null;
    if (isGiscussLoaded) {
      changeGiscusTheme();
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  const darkModeDefault = false;
  let localAlternateSentinel = darkModeDefault ? 'alternate' : 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
    toggleGiscusIfUsed(toAlternate, darkModeDefault);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } else {
    toggleColorMode(false);
  }
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../../parts/Addenda/addenda_SLs_Fungus.html" class="pagination-link  aria-label=" &lt;span="" squeaky="" yeast="" gets="" greased&lt;="" span&gt;"="">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">The Squeaky Yeast Gets Greased</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="../../parts/Addenda/addenda_Celegans_SMS_family.html" class="pagination-link" aria-label="<span class='chapter-number'>10</span>&nbsp; <span class='chapter-title'>Systematic analysis of the sphingomyelin synthase family in _C. elegans_</span>">
        <span class="nav-page-text"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Systematic analysis of the sphingomyelin synthase family in <em>C. elegans</em></span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
<p>Copyright 2024, Gaelen Guzman</p>
</div>   
    <div class="nav-footer-center">
      &nbsp;
    </div>
    <div class="nav-footer-right">
      &nbsp;
    </div>
  </div>
</footer>
<script>var lightboxQuarto = GLightbox({"descPosition":"bottom","closeEffect":"fade","selector":".lightbox","openEffect":"fade","loop":true});
window.onload = () => {
  lightboxQuarto.on('slide_before_load', (data) => {
    const { slideIndex, slideNode, slideConfig, player, trigger } = data;
    const href = trigger.getAttribute('href');
    if (href !== null) {
      const imgEl = window.document.querySelector(`a[href="${href}"] img`);
      if (imgEl !== null) {
        const srcAttr = imgEl.getAttribute("src");
        if (srcAttr && srcAttr.startsWith("data:")) {
          slideConfig.href = srcAttr;
        }
      }
    } 
  });

  lightboxQuarto.on('slide_after_load', (data) => {
    const { slideIndex, slideNode, slideConfig, player, trigger } = data;
    if (window.Quarto?.typesetMath) {
      window.Quarto.typesetMath(slideNode);
    }
  });

};
          </script>




</body></html>